{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "UUdjG4-XsE0I"
      },
      "source": [
        "# **SaprotHub: Making Protein Modeling Accessible to All Biologists**\n",
        "\n",
        "<a href=\"https://www.biorxiv.org/content/10.1101/2024.05.24.595648v2\"><img src=\"https://img.shields.io/badge/Paper-bioRxiv-green\" style=\"max-width: 100%;\"></a>\n",
        "<a href=\"https://huggingface.co/SaProtHub\"><img src=\"https://img.shields.io/badge/%F0%9F%A4%97%20Hugging%20Face-red?label=SaprotHub\" style=\"max-width: 100%;\"></a>\n",
        "<a href=\"https://github.com/westlake-repl/SaprotHub\"><img src=\"https://img.shields.io/badge/Github-black?logo=github\" style=\"max-width: 100%;\"></a>\n",
        "<a href=\"https://theopmc.github.io/\"><img src=\"https://img.shields.io/badge/Website-OPMC-yellow\" style=\"max-width: 100%;\"></a>\n",
        "<a href=\"https://cbirt.net/no-coding-required-saprothub-brings-protein-modeling-to-every-biologist/\" alt=\"blog\"><img src=\"https://img.shields.io/badge/Blog-Medium-purple\" /></a>\n",
        "<a href=\"https://x.com/sokrypton/status/1795525127653986415\"><img src=\"https://img.shields.io/badge/Twitter-blue?logo=twitter\" style=\"max-width: 100%;\"></a>\n",
        "\n",
        "\n",
        "This is the Colab version of [SaProt](https://github.com/westlake-repl/SaProt), a pre-trained protein language model designed for various downstream protein tasks.\n",
        "\n",
        "**ColabSaprot** is a platform where **Protein Language Models(PLMs)** are more accessible and user-friendly for biologists, enabling effortless model training and sharing within the scientific community.\n",
        "\n",
        "We've established the [SaprotHub](https://huggingface.co/SaProtHub) for storing and sharing models and datasets, where you can explore extensive collections for specific protein prediction tasks.\n",
        "\n",
        "We hope ColabSaprot and SaprotHub can contribute to advancing biological research, fostering collaboration, and accelerating discoveries in the field. You can access [our paper](https://www.biorxiv.org/content/10.1101/2024.05.24.595648v2) for further details.\n",
        "\n",
        "Check these videos ([training](https://www.youtube.com/watch?v=r42z1hvYKfw), [predicting](https://www.youtube.com/watch?v=N5VMBwM_ukQ)) to see how to use  ColabSaprot.\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1nLb_im9sJWw"
      },
      "source": [
        "## ColabSaprot\n",
        "\n",
        "| Function                             | Tutorial                                                     | Video                                                        |\n",
        "| ------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |\n",
        "| <a href=\"#train\">Train your model</a>                     | [How to train your model](https://github.com/westlake-repl/SaprotHub/wiki/2.1:-Train-your-model) | - [YouTube](https://www.youtube.com/watch?v=r42z1hvYKfw)<br />- [Bilibili](https://www.bilibili.com/video/BV1HDhHeTEmH/?spm_id_from=333.337.search-card.all.click&vd_source=a418185fadee73ac65d8fab69eee0b52) |\n",
        "| <a href=\"#prediction\">Classification/Regression Prediction</a> | [How to use model for classification/regression prediction](https://github.com/westlake-repl/SaprotHub/wiki/3.1:-Classification-Regression-Prediction) | - [YouTube](https://www.youtube.com/watch?v=N5VMBwM_ukQ)      |\n",
        "| <a href=\"#mutational_effect_prediction\">Mutational Effect Prediction</a>         | [How to use model for mutational effect prediction](https://github.com/westlake-repl/SaprotHub/wiki/3.2:-Mutational-Effect-Prediction) | -                                                            |\n",
        "| <a href=\"#inverse_folding_prediction\">Inverse Folding Prediction</a>           | [How to use model for inverse folding prediction](https://github.com/westlake-repl/SaprotHub/wiki/3.3:-Inverse-Folding-Prediction) | -                                                            |\n",
        "| <a href=\"#upload_model\">Contribute to SaprotHub</a>              | [How to contribute to SaprotHub](https://github.com/westlake-repl/SaprotHub/wiki/0.4:-Contribute-to-SaprotHub) | -                                                            |\n",
        "\n",
        "<br>\n",
        "\n",
        "<font color=red>**To view the content, please click on the first option in the left sidebar.**</font>\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "zVQ6vaQTjYO3"
      },
      "source": [
        "## SaprotHub\n",
        "\n",
        "Find awesome models and datasets for specific protein task on SaprotHub!\n",
        "\n",
        "<img src=\"https://github.com/westlake-repl/SaProtHub/blob/main/Figure/SaProtHub.png?raw=true\" height=\"350\" width=\"600px\" align=\"center\">"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "3-dw1U1uBI7d"
      },
      "source": [
        "# **1: Installation**\n",
        "\n",
        "‚ö†Ô∏è**IMPORTANT**‚ö†Ô∏è\n",
        "\n",
        "Before installing SaProt, please **<font color=red>SWITCH YOUR RUNTIME TYPE TO GPU!!!</font>**\n",
        "\n",
        "> üìçPlease check this [page](https://github.com/westlake-repl/SaprotHub/wiki/1.1:-Switch-your-runtime-type-to-GPU) to learn **how to switch your runtime type to GPU**.\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "Tgvb8ibwBI7d"
      },
      "outputs": [],
      "source": [
        "#@title **1.1: ‚ñ∂Ô∏è Click the run button to install SaProt**\n",
        "\n",
        "#@markdown (Please waiting for 2-8 minutes to install...)\n",
        "################################################################################\n",
        "########################### install saprot #####################################\n",
        "################################################################################\n",
        "%load_ext autoreload\n",
        "%autoreload 2\n",
        "\n",
        "import os\n",
        "# Check whether the server is local or from google cloud\n",
        "root_dir = os.getcwd()\n",
        "\n",
        "from google.colab import output\n",
        "output.enable_custom_widget_manager()\n",
        "\n",
        "try:\n",
        "  import sys\n",
        "  sys.path.append(f\"{root_dir}/SaprotHub\")\n",
        "  import saprot\n",
        "  print(\"SaProt is installed successfully!\")\n",
        "  os.system(f\"chmod +x {root_dir}/SaprotHub/bin/*\")\n",
        "\n",
        "except ImportError:\n",
        "  print(\"Installing SaProt...\")\n",
        "  os.system(f\"rm -rf {root_dir}/SaprotHub\")\n",
        "  # !rm -rf /content/SaprotHub/\n",
        "\n",
        "  !git clone https://github.com/westlake-repl/SaprotHub.git\n",
        "\n",
        "  # !pip install /content/SaprotHub/saprot-0.4.7-py3-none-any.whl\n",
        "  os.system(f\"pip install -r {root_dir}/SaprotHub/requirements.txt\")\n",
        "  # !pip install -r /content/SaprotHub/requirements.txt\n",
        "\n",
        "  os.system(f\"pip install {root_dir}/SaprotHub\")\n",
        "\n",
        "\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/LMDB\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/bin\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/output\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/datasets\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/classification/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/regression/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/token_classification/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/pair_classification/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/pair_regression/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/structures\")\n",
        "  # !mkdir -p /content/SaprotHub/LMDB\n",
        "  # !mkdir -p /content/SaprotHub/bin\n",
        "  # !mkdir -p /content/SaprotHub/output\n",
        "  # !mkdir -p /content/SaprotHub/datasets\n",
        "  # !mkdir -p /content/SaprotHub/adapters/classification/Local\n",
        "  # !mkdir -p /content/SaprotHub/adapters/regression/Local\n",
        "  # !mkdir -p /content/SaprotHub/adapters/token_classification/Local\n",
        "  # !mkdir -p /content/SaprotHub/adapters/pair_classification/Local\n",
        "  # !mkdir -p /content/SaprotHub/adapters/pair_regression/Local\n",
        "  # !mkdir -p /content/SaprotHub/structures\n",
        "\n",
        "  # !pip install gdown==v4.6.3 --force-reinstall --quiet\n",
        "  # os.system(\n",
        "  #   f\"wget 'https://drive.usercontent.google.com/download?id=1B_9t3n_nlj8Y3Kpc_mMjtMdY0OPYa7Re&export=download&authuser=0' -O {root_dir}/SaprotHub/bin/foldseek\"\n",
        "  # )\n",
        "\n",
        "  os.system(f\"chmod +x {root_dir}/SaprotHub/bin/*\")\n",
        "  # !chmod +x /content/SaprotHub/bin/foldseek\n",
        "  import sys\n",
        "  sys.path.append(f\"{root_dir}/SaprotHub\")\n",
        "\n",
        "  # !mv /content/SaprotHub/ColabSaprotSetup/foldseek /content/SaprotHub/bin/\n",
        "\n",
        "################################################################################\n",
        "################################################################################\n",
        "################################## global ######################################\n",
        "################################################################################\n",
        "################################################################################\n",
        "\n",
        "import ipywidgets\n",
        "import pandas as pd\n",
        "import torch\n",
        "import numpy as np\n",
        "import lmdb\n",
        "import base64\n",
        "import copy\n",
        "import os\n",
        "import json\n",
        "import zipfile\n",
        "import yaml\n",
        "import argparse\n",
        "import pprint\n",
        "import subprocess\n",
        "import py3Dmol\n",
        "import matplotlib.pyplot as plt\n",
        "\n",
        "\n",
        "from loguru import logger\n",
        "from easydict import EasyDict\n",
        "from colorama import init, Fore, Back, Style\n",
        "from IPython.display import clear_output\n",
        "from saprot.utils.mpr import MultipleProcessRunnerSimplifier\n",
        "from huggingface_hub import snapshot_download\n",
        "from ipywidgets import HTML\n",
        "from IPython.display import display\n",
        "from google.colab import widgets\n",
        "from pathlib import Path\n",
        "from tqdm import tqdm\n",
        "from datetime import datetime\n",
        "from google.colab import files\n",
        "from transformers import AutoTokenizer, EsmForProteinFolding\n",
        "from transformers.models.esm.openfold_utils.protein import to_pdb, Protein as OFProtein\n",
        "from transformers.models.esm.openfold_utils.feats import atom14_to_atom37\n",
        "from string import ascii_uppercase,ascii_lowercase\n",
        "from saprot.data.parse import get_chain_ids\n",
        "import torch.nn.functional as F\n",
        "\n",
        "DATASET_HOME = Path(f'{root_dir}/SaprotHub/datasets')\n",
        "ADAPTER_HOME = Path(f'{root_dir}/SaprotHub/adapters')\n",
        "STRUCTURE_HOME = Path(f\"{root_dir}/SaprotHub/structures\")\n",
        "LMDB_HOME = Path(f'{root_dir}/SaprotHub/LMDB')\n",
        "OUTPUT_HOME = Path(f'{root_dir}/SaprotHub/output')\n",
        "UPLOAD_FILE_HOME = Path(f'{root_dir}/SaprotHub/upload_files')\n",
        "FOLDSEEK_PATH = Path(f\"{root_dir}/SaprotHub/bin/foldseek\")\n",
        "aa_set = {\"A\", \"C\", \"D\", \"E\", \"F\", \"G\", \"H\", \"I\", \"K\", \"L\", \"M\", \"N\", \"P\", \"Q\", \"R\", \"S\", \"T\", \"V\", \"W\", \"Y\"}\n",
        "foldseek_struc_vocab = \"pynwrqhgdlvtmfsaeikc#\"\n",
        "\n",
        "data_type_list = [\"Single AA Sequence\",\n",
        "                  \"Single SA Sequence\",\n",
        "                  \"Single UniProt ID\",\n",
        "                  \"Single PDB/CIF Structure\",\n",
        "                  \"Multiple AA Sequences\",\n",
        "                  \"Multiple SA Sequences\",\n",
        "                  \"Multiple UniProt IDs\",\n",
        "                  \"Multiple PDB/CIF Structures\",\n",
        "                  \"SaprotHub Dataset\",\n",
        "                  \"A pair of AA Sequences\",\n",
        "                  \"A pair of SA Sequences\",\n",
        "                  \"A pair of UniProt IDs\",\n",
        "                  \"A pair of PDB/CIF Structures\",\n",
        "                  \"Multiple pairs of AA Sequences\",\n",
        "                  \"Multiple pairs of SA Sequences\",\n",
        "                  \"Multiple pairs of UniProt IDs\",\n",
        "                  \"Multiple pairs of PDB/CIF Structures\",]\n",
        "\n",
        "task_type_dict = {\n",
        "  \"Protein-level Classification\": \"classification\",\n",
        "  \"Residue-level Classification\" : \"token_classification\",\n",
        "  \"Protein-level Regression\" : \"regression\",\n",
        "  \"Protein-protein Classification\": \"pair_classification\",\n",
        "  \"Protein-protein Regression\": \"pair_regression\",\n",
        "}\n",
        "model_type_dict = {\n",
        "  \"classification\" : \"saprot/saprot_classification_model\",\n",
        "  \"token_classification\" : \"saprot/saprot_token_classification_model\",\n",
        "  \"regression\" : \"saprot/saprot_regression_model\",\n",
        "  \"pair_classification\" : \"saprot/saprot_pair_classification_model\",\n",
        "  \"pair_regression\" : \"saprot/saprot_pair_regression_model\",\n",
        "}\n",
        "dataset_type_dict = {\n",
        "  \"classification\": \"saprot/saprot_classification_dataset\",\n",
        "  \"token_classification\" : \"saprot/saprot_token_classification_dataset\",\n",
        "  \"regression\": \"saprot/saprot_regression_dataset\",\n",
        "  \"pair_classification\" : \"saprot/saprot_pair_classification_dataset\",\n",
        "  \"pair_regression\" : \"saprot/saprot_pair_regression_dataset\",\n",
        "}\n",
        "training_data_type_dict = {\n",
        "  \"Single AA Sequence\": \"AA\",\n",
        "  \"Single SA Sequence\": \"SA\",\n",
        "  \"Single UniProt ID\": \"SA\",\n",
        "  \"Single PDB/CIF Structure\": \"SA\",\n",
        "  \"Multiple AA Sequences\": \"AA\",\n",
        "  \"Multiple SA Sequences\": \"SA\",\n",
        "  \"Multiple UniProt IDs\": \"SA\",\n",
        "  \"Multiple PDB/CIF Structures\": \"SA\",\n",
        "  \"SaprotHub Dataset\": \"SA\",\n",
        "  \"A pair of AA Sequences\": \"AA\",\n",
        "  \"A pair of SA Sequences\": \"SA\",\n",
        "  \"A pair of UniProt IDs\": \"SA\",\n",
        "  \"A pair of PDB/CIF Structures\": \"SA\",\n",
        "  \"Multiple pairs of AA Sequences\": \"AA\",\n",
        "  \"Multiple pairs of SA Sequences\": \"SA\",\n",
        "  \"Multiple pairs of UniProt IDs\": \"SA\",\n",
        "  \"Multiple pairs of PDB/CIF Structures\": \"SA\",\n",
        "}\n",
        "\n",
        "\n",
        "class font:\n",
        "    RED = '\\033[91m'\n",
        "    GREEN = '\\033[92m'\n",
        "    YELLOW = '\\033[93m'\n",
        "    BLUE = '\\033[94m'\n",
        "\n",
        "    BOLD = '\\033[1m'\n",
        "    UNDERLINE = '\\033[4m'\n",
        "\n",
        "    RESET = '\\033[0m'\n",
        "\n",
        "\n",
        "################################################################################\n",
        "############################### adapters #######################################\n",
        "################################################################################\n",
        "def get_adapters_list(task_type=None):\n",
        "\n",
        "    adapters_list = []\n",
        "\n",
        "    if task_type:\n",
        "      for file_path in (ADAPTER_HOME / task_type).glob('**/adapter_config.json'):\n",
        "        adapters_list.append(file_path.parent)\n",
        "    else:\n",
        "      for file_path in ADAPTER_HOME.glob('**/adapter_config.json'):\n",
        "        adapters_list.append(file_path.parent)\n",
        "\n",
        "    return adapters_list\n",
        "\n",
        "\n",
        "def show_adapters_info(adapters_list):\n",
        "  grid = widgets.Grid(len(adapters_list)+1, 2, header_row=True, header_column=True)\n",
        "\n",
        "  with grid.output_to(0, 0):\n",
        "    print(\"ID\")\n",
        "\n",
        "  with grid.output_to(0, 1):\n",
        "    print(\"Local Model\")\n",
        "\n",
        "  # with grid.output_to(0, 2):\n",
        "  #   print(\"Adapter Path\")\n",
        "\n",
        "  for i in range(len(adapters_list)):\n",
        "    with grid.output_to(i+1, 0):\n",
        "      print(i)\n",
        "    with grid.output_to(i+1, 1):\n",
        "      print(adapters_list[i].stem)\n",
        "    # with grid.output_to(i+1, 2):\n",
        "    #   print(adapters_list[i])\n",
        "\n",
        "def adapters_text(adapters_list):\n",
        "  input = ipywidgets.Text(\n",
        "    value=None,\n",
        "    placeholder='Enter SaprotHub Model ID',\n",
        "    # description='Selected:',\n",
        "    disabled=False)\n",
        "  input.layout.width = '500px'\n",
        "  display(input)\n",
        "\n",
        "  return input\n",
        "\n",
        "def adapters_dropdown(adapters_list):\n",
        "  dropdown = ipywidgets.Dropdown(\n",
        "    options=[f\"{adapter_path.parent.stem}/{adapter_path.stem}\" for index, adapter_path in enumerate(adapters_list)],\n",
        "    value=None,\n",
        "    placeholder='Select a Local Model here',\n",
        "    # description='Selected:',\n",
        "    disabled=False)\n",
        "  dropdown.layout.width = '500px'\n",
        "  display(dropdown)\n",
        "\n",
        "  return dropdown\n",
        "\n",
        "def adapters_combobox(adapters_list):\n",
        "  combobox = ipywidgets.Combobox(\n",
        "    options=[f\"{adapter_path.parent.stem}/{adapter_path.stem}\" for index, adapter_path in enumerate(adapters_list)],\n",
        "    value=None,\n",
        "    placeholder='Enter SaprotHub Model repository id or select a Local Model here',\n",
        "    # description='Selected:',\n",
        "    disabled=False)\n",
        "  combobox.layout.width = '500px'\n",
        "  display(combobox)\n",
        "\n",
        "  return combobox\n",
        "\n",
        "def select_adapter():\n",
        "  adapters_list = get_adapters_list()\n",
        "  print(Fore.BLUE+\"Existing Models:\"+Style.RESET_ALL)\n",
        "  # print(\"=\"*100)\n",
        "  # show_adapters_info(adapters_list)\n",
        "  # print(\"=\"*100)\n",
        "  return adapters_combobox(adapters_list)\n",
        "\n",
        "def adapters_selectmultiple(adapters_list):\n",
        "  selectmulitiple = ipywidgets.SelectMultiple(\n",
        "  options=[f\"{adapter_path.parent.stem}/{adapter_path.stem}\" for index, adapter_path in enumerate(adapters_list)],\n",
        "  value=[],\n",
        "  #rows=10,\n",
        "  placeholder='Select multiple models',\n",
        "  # description='Fruits',\n",
        "  disabled=False,\n",
        "  layout={'width': '500px'})\n",
        "  display(selectmulitiple)\n",
        "\n",
        "  return selectmulitiple\n",
        "\n",
        "def adapters_textmultiple(adapters_list):\n",
        "  textmultiple = ipywidgets.Text(\n",
        "  value=None,\n",
        "  placeholder='Enter multiple SaprotHub Model IDs, separated by commas.',\n",
        "  # description='Fruits',\n",
        "  disabled=False,\n",
        "  layout={'width': '500px'})\n",
        "  display(textmultiple)\n",
        "\n",
        "  return textmultiple\n",
        "\n",
        "\n",
        "def select_adapter_from(task_type, use_model_from):\n",
        "  adapters_list = get_adapters_list(task_type)\n",
        "\n",
        "  if use_model_from == 'Trained by yourself on ColabSaprot':\n",
        "    print(Fore.BLUE+f\"Local Model ({task_type}):\"+Style.RESET_ALL)\n",
        "    return adapters_dropdown(adapters_list)\n",
        "\n",
        "  elif use_model_from == 'Shared by peers on SaprotHub':\n",
        "    print(Fore.BLUE+\"SaprotHub Model:\"+Style.RESET_ALL)\n",
        "    return adapters_text(adapters_list)\n",
        "\n",
        "  elif use_model_from == \"Saved in your local computer\":\n",
        "    print(Fore.BLUE+\"Click the button to upload the \\\"Model-<task_name>-<model_size>.zip\\\" file of your Model:\"+Style.RESET_ALL)\n",
        "    # 1. upload model.zip\n",
        "    adapter_upload_path = ADAPTER_HOME / task_type / \"Local\"\n",
        "    adapter_zip_path = upload_file(adapter_upload_path)\n",
        "    adapter_path = adapter_upload_path / adapter_zip_path.stem\n",
        "    # 2. unzip model.zip\n",
        "    with zipfile.ZipFile(adapter_zip_path, 'r') as zip_ref:\n",
        "        zip_ref.extractall(adapter_path)\n",
        "    os.remove(adapter_zip_path)\n",
        "    # 3. check adapter_config.json\n",
        "    adapter_config_path = adapter_path / \"adapter_config.json\"\n",
        "    assert adapter_config_path.exists(), f\"Can't find {adapter_config_path}\"\n",
        "\n",
        "    return EasyDict({\"value\":  f\"Local/{adapter_zip_path.stem}\"})\n",
        "\n",
        "  elif use_model_from == \"Multi-models on ColabSaprot\":\n",
        "    # 1. select the list of adapters\n",
        "    print(Fore.BLUE+f\"Local Model ({task_type}):\"+Style.RESET_ALL)\n",
        "    print(Fore.BLUE+f\"Multiple values can be selected with \\\"shift\\\" and/or \\\"ctrl\\\" (or \\\"command\\\") pressed and mouse clicks or arrow keys.\"+Style.RESET_ALL)\n",
        "    return adapters_selectmultiple(adapters_list)\n",
        "\n",
        "  elif use_model_from == \"Multi-models on SaprotHub\":\n",
        "    # 1. enter the list of adapters\n",
        "    print(Fore.BLUE+f\"SaprotHub Model IDs, separated by commas ({task_type}):\"+Style.RESET_ALL)\n",
        "    return adapters_textmultiple(adapters_list)\n",
        "\n",
        "\n",
        "\n",
        "################################################################################\n",
        "########################### download dataset ###################################\n",
        "################################################################################\n",
        "def download_dataset(task_name):\n",
        "  import gdown\n",
        "  import tarfile\n",
        "\n",
        "  filepath = LMDB_HOME / f\"{task_name}.tar.gz\"\n",
        "  download_links = {\n",
        "    \"ClinVar\" : \"https://drive.google.com/uc?id=1Le6-v8ddXa1eLJZFo7HPij7NhaBmNUbo\",\n",
        "    \"DeepLoc_cls2\" : \"https://drive.google.com/uc?id=1dGlojkCt1DwUXWiUk4kXRGRNu5sz2uxf\",\n",
        "    \"DeepLoc_cls10\" : \"https://drive.google.com/uc?id=1dGlojkCt1DwUXWiUk4kXRGRNu5sz2uxf\",\n",
        "    \"EC\" : \"https://drive.google.com/uc?id=1VFLFA-jK1tkTZBVbMw8YSsjZqAqlVQVQ\",\n",
        "    \"GO_BP\" : \"https://drive.google.com/uc?id=1DGiGErWbRnEK8jmE2Jpb996By8KVDBfF\",\n",
        "    \"GO_CC\" : \"https://drive.google.com/uc?id=1DGiGErWbRnEK8jmE2Jpb996By8KVDBfF\",\n",
        "    \"GO_MF\" : \"https://drive.google.com/uc?id=1DGiGErWbRnEK8jmE2Jpb996By8KVDBfF\",\n",
        "    \"HumanPPI\" : \"https://drive.google.com/uc?id=1ahgj-IQTtv3Ib5iaiXO_ASh2hskEsvoX\",\n",
        "    \"MetalIonBinding\" : \"https://drive.google.com/uc?id=1rwknPWIHrXKQoiYvgQy4Jd-efspY16x3\",\n",
        "    \"ProteinGym\" : \"https://drive.google.com/uc?id=1L-ODrhfeSjDom-kQ2JNDa2nDEpS8EGfD\",\n",
        "    \"Thermostability\" : \"https://drive.google.com/uc?id=1I9GR1stFDHc8W3FCsiykyrkNprDyUzSz\",\n",
        "  }\n",
        "\n",
        "  try:\n",
        "    gdown.download(download_links[task_name], str(filepath), quiet=False)\n",
        "    with tarfile.open(filepath, 'r:gz') as tar:\n",
        "      tar.extractall(path=str(LMDB_HOME))\n",
        "      print(f\"Extracted: {filepath}\")\n",
        "  except Exception as e:\n",
        "    raise RuntimeError(\"The dataset has not prepared.\")\n",
        "\n",
        "################################################################################\n",
        "############################# upload file ######################################\n",
        "################################################################################\n",
        "def upload_file(upload_path):\n",
        "  import shutil\n",
        "  import os\n",
        "  from pathlib import Path\n",
        "  import sys\n",
        "\n",
        "  upload_path = Path(upload_path)\n",
        "  upload_path.mkdir(parents=True, exist_ok=True)\n",
        "  basepath = Path().resolve()\n",
        "  try:\n",
        "    uploaded = files.upload()\n",
        "    filenames = []\n",
        "    for filename in uploaded.keys():\n",
        "      filenames.append(filename)\n",
        "      shutil.move(basepath / filename, upload_path / filename)\n",
        "    if len(filenames) == 0:\n",
        "      logger.info(\"The uploading process has been interrupted by the user.\")\n",
        "      raise RuntimeError(\"The uploading process has been interrupted by the user.\")\n",
        "  except Exception as e:\n",
        "    logger.error(\"Upload file fail! Please click the button to run again.\")\n",
        "    raise(e)\n",
        "\n",
        "  return upload_path / filenames[0]\n",
        "\n",
        "################################################################################\n",
        "############################ upload dataset ####################################\n",
        "################################################################################\n",
        "\n",
        "\n",
        "def input_raw_data_by_data_type(data_type):\n",
        "  print(Fore.BLUE+\"Dataset: \"+Style.RESET_ALL, end='')\n",
        "\n",
        "  # 0-2. 0. Single AA Sequence, 1. Single SA Sequence, 2. Single UniProt ID\n",
        "  if data_type in data_type_list[:3]:\n",
        "    input_seq = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder=f'Enter {data_type} here',\n",
        "      disabled=False)\n",
        "    input_seq.layout.width = '500px'\n",
        "    print(Fore.BLUE+f\"{data_type}\"+Style.RESET_ALL)\n",
        "    display(input_seq)\n",
        "    return input_seq\n",
        "\n",
        "  # 3. Single PDB/CIF Structure\n",
        "  elif data_type == data_type_list[3]:\n",
        "    print(\"Please provide the structure type, chain and your structure file.\")\n",
        "\n",
        "    dropdown_type = ipywidgets.Dropdown(\n",
        "      value=\"AF2\",\n",
        "      options=[\"PDB\", \"AF2\"],\n",
        "      disabled=False)\n",
        "    dropdown_type.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"Structure type:\"+Style.RESET_ALL)\n",
        "    display(dropdown_type)\n",
        "\n",
        "    input_chain = ipywidgets.Text(\n",
        "      value=\"A\",\n",
        "      placeholder=f'Enter the name of chain here',\n",
        "      disabled=False)\n",
        "    input_chain.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"Chain:\"+Style.RESET_ALL)\n",
        "    display(input_chain)\n",
        "\n",
        "    print(Fore.BLUE+\"Please upload a .pdb/.cif file\"+Style.RESET_ALL)\n",
        "    pdb_file_path = upload_file(STRUCTURE_HOME)\n",
        "    return pdb_file_path.stem, dropdown_type, input_chain\n",
        "\n",
        "  # 4-7 & 13-16. Multiple Sequences\n",
        "  elif data_type in data_type_list[4:8] or data_type in data_type_list[13:17]:\n",
        "    print(Fore.BLUE+f\"Please upload the .csv file which contains {data_type}\"+Style.RESET_ALL)\n",
        "    uploaded_csv_path = upload_file(UPLOAD_FILE_HOME)\n",
        "    print(Fore.BLUE+\"Successfully upload your .csv file!\"+Style.RESET_ALL)\n",
        "    print(\"=\"*100)\n",
        "\n",
        "    if data_type == data_type_list[7] or data_type == data_type_list[16]:\n",
        "      # upload and unzip PDB files\n",
        "      print(Fore.BLUE+f\"Please upload your .zip file which contains {data_type} files\"+Style.RESET_ALL)\n",
        "      pdb_zip_path = upload_file(UPLOAD_FILE_HOME)\n",
        "      if pdb_zip_path.suffix != \".zip\":\n",
        "        logger.error(\"The data type does not match. Please click the run button again to upload a .zip file!\")\n",
        "        raise RuntimeError(\"The data type does not match.\")\n",
        "      print(Fore.BLUE+\"Successfully upload your .zip file!\"+Style.RESET_ALL)\n",
        "      print(\"=\"*100)\n",
        "\n",
        "      import zipfile\n",
        "      with zipfile.ZipFile(pdb_zip_path, 'r') as zip_ref:\n",
        "        zip_ref.extractall(STRUCTURE_HOME)\n",
        "\n",
        "    return uploaded_csv_path\n",
        "\n",
        "  # 8. SaprotHub Dataset\n",
        "  elif data_type == data_type_list[8]:\n",
        "    input_repo_id = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder=f'Copy and paste the SaprotHub Dataset ID here',\n",
        "      disabled=False)\n",
        "    input_repo_id.layout.width = '500px'\n",
        "    print(Fore.BLUE+f\"{data_type}\"+Style.RESET_ALL)\n",
        "    display(input_repo_id)\n",
        "    return input_repo_id\n",
        "\n",
        "  # 9-11. A pair of seq\n",
        "  elif data_type in [\"A pair of AA Sequences\", \"A pair of SA Sequences\", \"A pair of UniProt IDs\"]:\n",
        "    print()\n",
        "\n",
        "    seq_type = data_type[len(\"A pair of \"):-1]\n",
        "\n",
        "    input_seq1 = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder=f'Enter the {seq_type} of Sequence 1 here',\n",
        "      disabled=False)\n",
        "    input_seq1.layout.width = '500px'\n",
        "    print(Fore.BLUE+f\"Sequence 1:\"+Style.RESET_ALL)\n",
        "    display(input_seq1)\n",
        "\n",
        "    input_seq2 = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder=f'Enter the {seq_type} of Sequence 2 here',\n",
        "      disabled=False)\n",
        "    input_seq2.layout.width = '500px'\n",
        "    print(Fore.BLUE+f\"Sequence 2:\"+Style.RESET_ALL)\n",
        "    display(input_seq2)\n",
        "\n",
        "    return (input_seq1, input_seq2)\n",
        "\n",
        "  # 12. Pair Single PDB/CIF Structure\n",
        "  elif data_type == data_type_list[12]:\n",
        "    print(\"Please provide the structure type, chain and your structure file.\")\n",
        "\n",
        "    dropdown_type1 = ipywidgets.Dropdown(\n",
        "      value=\"PDB\",\n",
        "      options=[\"PDB\", \"AF2\"],\n",
        "      disabled=False)\n",
        "    dropdown_type1.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"The first structure type:\"+Style.RESET_ALL)\n",
        "    display(dropdown_type1)\n",
        "\n",
        "    input_chain1 = ipywidgets.Text(\n",
        "      value=\"A\",\n",
        "      placeholder=f'Enter the name of chain of the first structure here',\n",
        "      disabled=False)\n",
        "    input_chain1.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"Chain of the first structure:\"+Style.RESET_ALL)\n",
        "    display(input_chain1)\n",
        "\n",
        "    print(Fore.BLUE+\"Please upload a .pdb/.cif file\"+Style.RESET_ALL)\n",
        "    pdb_file_path1 = upload_file(STRUCTURE_HOME)\n",
        "\n",
        "\n",
        "    dropdown_type2 = ipywidgets.Dropdown(\n",
        "      value=\"PDB\",\n",
        "      options=[\"PDB\", \"AF2\"],\n",
        "      disabled=False)\n",
        "    dropdown_type2.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"The second structure type:\"+Style.RESET_ALL)\n",
        "    display(dropdown_type2)\n",
        "\n",
        "    input_chain2 = ipywidgets.Text(\n",
        "      value=\"A\",\n",
        "      placeholder=f'Enter the name of chain of the second structure here',\n",
        "      disabled=False)\n",
        "    input_chain2.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"Chain of the second structure:\"+Style.RESET_ALL)\n",
        "    display(input_chain2)\n",
        "\n",
        "    print(Fore.BLUE+\"Please upload a .pdb/.cif file\"+Style.RESET_ALL)\n",
        "    pdb_file_path2 = upload_file(STRUCTURE_HOME)\n",
        "    return (pdb_file_path1.stem, dropdown_type1, input_chain1, pdb_file_path2.stem, dropdown_type2, input_chain2)\n",
        "\n",
        "\n",
        "  # elif data_type == \"Multiple pairs of PDB/CIF Structures\":\n",
        "  #   print(Fore.BLUE+f\"Please upload the .csv file which contains {data_type}\"+Style.RESET_ALL)\n",
        "  #   uploaded_csv_path = upload_file(UPLOAD_FILE_HOME)\n",
        "  #   print(Fore.BLUE+\"Successfully upload your .csv file!\"+Style.RESET_ALL)\n",
        "  #   print(\"=\"*100)\n",
        "\n",
        "  #   if data_type == data_type_list[7]:\n",
        "  #     # upload and unzip PDB files\n",
        "  #     print(Fore.BLUE+f\"Please upload your .zip file which contains {data_type} files\"+Style.RESET_ALL)\n",
        "  #     pdb_zip_path = upload_file(UPLOAD_FILE_HOME)\n",
        "  #     if pdb_zip_path.suffix != \".zip\":\n",
        "  #       logger.error(\"The data type does not match. Please click the run button again to upload a .zip file!\")\n",
        "  #       raise RuntimeError(\"The data type does not match.\")\n",
        "  #     print(Fore.BLUE+\"Successfully upload your .zip file!\"+Style.RESET_ALL)\n",
        "  #     print(\"=\"*100)\n",
        "\n",
        "  #     import zipfile\n",
        "  #     with zipfile.ZipFile(pdb_zip_path, 'r') as zip_ref:\n",
        "  #       zip_ref.extractall(STRUCTURE_HOME)\n",
        "\n",
        "  #   return uploaded_csv_path\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "def get_SA_sequence_by_data_type(data_type, raw_data):\n",
        "\n",
        "  # 0. Single AA Sequence\n",
        "  if data_type == data_type_list[0]:\n",
        "    input_seq = raw_data\n",
        "    aa_seq = input_seq.value\n",
        "\n",
        "    sa_seq = ''\n",
        "    for aa in aa_seq:\n",
        "        sa_seq += aa + '#'\n",
        "    return sa_seq\n",
        "\n",
        "  # 1. Single SA Sequence\n",
        "  if data_type == data_type_list[1]:\n",
        "    input_seq = raw_data\n",
        "    sa_seq = input_seq.value\n",
        "\n",
        "    return sa_seq\n",
        "\n",
        "  # 2. Single UniProt ID\n",
        "  if data_type == data_type_list[2]:\n",
        "    input_seq = raw_data\n",
        "    uniprot_id = input_seq.value\n",
        "\n",
        "\n",
        "    protein_list = [(uniprot_id, \"AF2\", \"A\")]\n",
        "    uniprot2pdb([protein_list[0][0]])\n",
        "    mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "    seqs = mprs.run()\n",
        "    sa_seq = seqs[0].split('\\t')[1]\n",
        "    return sa_seq\n",
        "\n",
        "  # 3. Single PDB/CIF Structure\n",
        "  if data_type == data_type_list[3]:\n",
        "    uniprot_id = raw_data[0]\n",
        "    struc_type = raw_data[1].value\n",
        "    chain = raw_data[2].value\n",
        "\n",
        "    protein_list = [(uniprot_id, struc_type, chain)]\n",
        "    mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "    seqs = mprs.run()\n",
        "    sa_seq = seqs[0].split('\\t')[1]\n",
        "    return sa_seq\n",
        "\n",
        "  # Multiple sequences\n",
        "  # raw_data = upload_files/xxx.csv\n",
        "  if data_type in data_type_list[4:8] or data_type in data_type_list[13:17]:\n",
        "    uploaded_csv_path = raw_data\n",
        "    csv_dataset_path = DATASET_HOME / uploaded_csv_path.name\n",
        "\n",
        "  # 4. Multiple AA Sequences\n",
        "  if data_type == data_type_list[4]:\n",
        "    protein_df = pd.read_csv(uploaded_csv_path)\n",
        "    for index, value in protein_df['sequence'].items():\n",
        "      sa_seq = ''\n",
        "      for aa in value:\n",
        "        sa_seq += aa + '#'\n",
        "      protein_df.at[index, 'sequence'] = sa_seq\n",
        "\n",
        "    protein_df.to_csv(csv_dataset_path, index=None)\n",
        "    return csv_dataset_path\n",
        "\n",
        "  # 5. Multiple SA Sequences\n",
        "  if data_type == data_type_list[5]:\n",
        "    protein_df = pd.read_csv(uploaded_csv_path)\n",
        "\n",
        "    protein_df.to_csv(csv_dataset_path, index=None)\n",
        "    return csv_dataset_path\n",
        "\n",
        "  # 6. Multiple UniProt IDs\n",
        "  if data_type == data_type_list[6]:\n",
        "    protein_df = pd.read_csv(uploaded_csv_path)\n",
        "    protein_list = protein_df.iloc[:, 0].tolist()\n",
        "    uniprot2pdb(protein_list)\n",
        "    protein_list = [(uniprot_id, \"AF2\", \"A\") for uniprot_id in protein_list]\n",
        "    mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "    outputs = mprs.run()\n",
        "\n",
        "    protein_df['sequence'] = [output.split(\"\\t\")[1] for output in outputs]\n",
        "    protein_df.to_csv(csv_dataset_path, index=None)\n",
        "    return csv_dataset_path\n",
        "\n",
        "  # 7. Multiple PDB/CIF Structures\n",
        "  if data_type == data_type_list[7]:\n",
        "    protein_df = pd.read_csv(uploaded_csv_path)\n",
        "    # protein_list = [(uniprot_id, type, chain), ...]\n",
        "    # protein_list = [item.split('.')[0] for item in protein_df.iloc[:, 0].tolist()]\n",
        "    # uniprot2pdb(protein_list)\n",
        "    protein_list = []\n",
        "    for row_tuple in protein_df.itertuples(index=False):\n",
        "      assert row_tuple.type in ['PDB', 'AF2'],  \"The type of structure must be either \\\"PDB\\\" or \\\"AF2\\\"!\"\n",
        "      protein_list.append(row_tuple)\n",
        "    mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "    outputs = mprs.run()\n",
        "\n",
        "    protein_df['sequence'] = [output.split(\"\\t\")[1] for output in outputs]\n",
        "    protein_df.to_csv(csv_dataset_path, index=None)\n",
        "    return csv_dataset_path\n",
        "\n",
        "  # 8. SaprotHub Dataset\n",
        "  elif data_type == data_type_list[8]:\n",
        "    input_repo_id = raw_data\n",
        "    REPO_ID = input_repo_id.value\n",
        "\n",
        "    if REPO_ID.startswith('/'):\n",
        "      return Path(REPO_ID)\n",
        "\n",
        "    snapshot_download(repo_id=REPO_ID, repo_type=\"dataset\", local_dir=LMDB_HOME/REPO_ID)\n",
        "\n",
        "    return LMDB_HOME/REPO_ID\n",
        "\n",
        "  # 9. Pair Single AA Sequences\n",
        "  elif data_type == \"A pair of AA Sequences\":\n",
        "    input_seq_1, input_seq_2 = raw_data\n",
        "    sa_seq1 = get_SA_sequence_by_data_type(data_type_list[0], input_seq_1)\n",
        "    sa_seq2 = get_SA_sequence_by_data_type(data_type_list[0], input_seq_2)\n",
        "\n",
        "    return (sa_seq1, sa_seq2)\n",
        "\n",
        "  # 10. Pair Single SA Sequences\n",
        "  elif data_type ==  \"A pair of SA Sequences\":\n",
        "    input_seq_1, input_seq_2 = raw_data\n",
        "    sa_seq1 = get_SA_sequence_by_data_type(data_type_list[1], input_seq_1)\n",
        "    sa_seq2 = get_SA_sequence_by_data_type(data_type_list[1], input_seq_2)\n",
        "\n",
        "    return (sa_seq1, sa_seq2)\n",
        "\n",
        "  # 11. Pair Single UniProt IDs\n",
        "  elif data_type ==  \"A pair of UniProt IDs\":\n",
        "    input_seq_1, input_seq_2 = raw_data\n",
        "    sa_seq1 = get_SA_sequence_by_data_type(data_type_list[2], input_seq_1)\n",
        "    sa_seq2 = get_SA_sequence_by_data_type(data_type_list[2], input_seq_2)\n",
        "\n",
        "    return (sa_seq1, sa_seq2)\n",
        "\n",
        "  # 12. Pair Single PDB/CIF Structure\n",
        "  if data_type == \"A pair of PDB/CIF Structures\":\n",
        "    uniprot_id1 = raw_data[0]\n",
        "    struc_type1 = raw_data[1].value\n",
        "    chain1 = raw_data[2].value\n",
        "\n",
        "    protein_list1 = [(uniprot_id1, struc_type1, chain1)]\n",
        "    mprs1 = MultipleProcessRunnerSimplifier(protein_list1, pdb2sequence, n_process=2, return_results=True)\n",
        "    seqs1 = mprs1.run()\n",
        "    sa_seq1 = seqs1[0].split('\\t')[1]\n",
        "\n",
        "    uniprot_id2 = raw_data[3]\n",
        "    struc_type2 = raw_data[4].value\n",
        "    chain2 = raw_data[5].value\n",
        "\n",
        "    protein_list2 = [(uniprot_id2, struc_type2, chain2)]\n",
        "    mprs2 = MultipleProcessRunnerSimplifier(protein_list2, pdb2sequence, n_process=2, return_results=True)\n",
        "    seqs2 = mprs2.run()\n",
        "    sa_seq2 = seqs2[0].split('\\t')[1]\n",
        "    return sa_seq1, sa_seq2\n",
        "\n",
        "  # # Pair raw_data = upload_files/xxx.csv\n",
        "  # if data_type in data_type_list[12:16]:\n",
        "  #   uploaded_csv_path = raw_data\n",
        "  #   csv_dataset_path = DATASET_HOME / uploaded_csv_path.name\n",
        "\n",
        "  # 13. Pair Multiple AA Sequences\n",
        "  if data_type == \"Multiple pairs of AA Sequences\":\n",
        "    protein_df = pd.read_csv(uploaded_csv_path)\n",
        "    for index, value in protein_df['seq_1'].items():\n",
        "      sa_seq1 = ''\n",
        "      for aa in value:\n",
        "        sa_seq1 += aa + '#'\n",
        "      protein_df.at[index, 'seq_1'] = sa_seq1\n",
        "\n",
        "    protein_df['name_1'] = 'name_1'\n",
        "    protein_df['chain_1'] = 'A'\n",
        "\n",
        "    for index, value in protein_df['seq_2'].items():\n",
        "      sa_seq2 = ''\n",
        "      for aa in value:\n",
        "        sa_seq2 += aa + '#'\n",
        "      protein_df.at[index, 'seq_2'] = sa_seq2\n",
        "\n",
        "    protein_df['name_2'] = 'name_2'\n",
        "    protein_df['chain_2'] = 'A'\n",
        "\n",
        "    protein_df.to_csv(csv_dataset_path, index=None)\n",
        "    return csv_dataset_path\n",
        "\n",
        "  # 14. Pair Multiple SA Sequences\n",
        "  if data_type == \"Multiple pairs of SA Sequences\":\n",
        "    protein_df = pd.read_csv(uploaded_csv_path)\n",
        "\n",
        "    protein_df['name_1'] = 'name_1'\n",
        "    protein_df['chain_1'] = 'A'\n",
        "\n",
        "\n",
        "    protein_df['name_2'] = 'name_2'\n",
        "    protein_df['chain_2'] = 'A'\n",
        "\n",
        "\n",
        "    protein_df.to_csv(csv_dataset_path, index=None)\n",
        "    return csv_dataset_path\n",
        "\n",
        "  # 15. Pair Multiple UniProt IDs\n",
        "  if data_type == \"Multiple pairs of UniProt IDs\":\n",
        "    protein_df = pd.read_csv(uploaded_csv_path)\n",
        "    protein_list1 = protein_df.loc[:, \"seq_1\"].tolist()\n",
        "    uniprot2pdb(protein_list1)\n",
        "    protein_df['name_1'] = protein_list1\n",
        "    protein_list1 = [(uniprot_id, \"AF2\", \"A\") for uniprot_id in protein_list1]\n",
        "    mprs1 = MultipleProcessRunnerSimplifier(protein_list1, pdb2sequence, n_process=2, return_results=True)\n",
        "    outputs1 = mprs1.run()\n",
        "\n",
        "    protein_df['seq_1'] = [output.split(\"\\t\")[1] for output in outputs1]\n",
        "    protein_df['chain_1'] = 'A'\n",
        "\n",
        "    protein_list2 = protein_df.loc[:, \"seq_2\"].tolist()\n",
        "    uniprot2pdb(protein_list2)\n",
        "    protein_df['name_2'] = protein_list2\n",
        "    protein_list2 = [(uniprot_id, \"AF2\", \"A\") for uniprot_id in protein_list2]\n",
        "    mprs2 = MultipleProcessRunnerSimplifier(protein_list2, pdb2sequence, n_process=2, return_results=True)\n",
        "    outputs2 = mprs2.run()\n",
        "\n",
        "    protein_df['seq_2'] = [output.split(\"\\t\")[1] for output in outputs2]\n",
        "    protein_df['chain_2'] = 'A'\n",
        "\n",
        "    protein_df.to_csv(csv_dataset_path, index=None)\n",
        "    return csv_dataset_path\n",
        "\n",
        "\n",
        "  # # 13-16. Pair Multiple Sequences\n",
        "  # elif data_type in data_type_list[12:16]:\n",
        "  #   print(Fore.BLUE+f\"Please upload the .csv file which contains {data_type}\"+Style.RESET_ALL)\n",
        "  #   uploaded_csv_path = upload_file(UPLOAD_FILE_HOME)\n",
        "  #   print(Fore.BLUE+\"Successfully upload your .csv file!\"+Style.RESET_ALL)\n",
        "  #   print(\"=\"*100)\n",
        "\n",
        "  elif data_type ==  \"Multiple pairs of PDB/CIF Structures\":\n",
        "    protein_df = pd.read_csv(uploaded_csv_path)\n",
        "    # columns: seq_1, seq_2, type_1, type_2, chain_1, chain_2, label, stage\n",
        "\n",
        "    # protein_list = [(uniprot_id, type, chain), ...]\n",
        "    # protein_list = [item.split('.')[0] for item in protein_df.iloc[:, 0].tolist()]\n",
        "    # uniprot2pdb(protein_list)\n",
        "\n",
        "    for i in range(1, 3):\n",
        "      protein_list = []\n",
        "      for index, row in protein_df.iterrows():\n",
        "        assert row[f\"type_{i}\"] in ['PDB', 'AF2'],  \"The type of structure must be either \\\"PDB\\\" or \\\"AF2\\\"!\"\n",
        "        row_tuple = (row[f\"seq_{i}\"], row[f\"type_{i}\"], row[f\"chain_{i}\"])\n",
        "        protein_list.append(row_tuple)\n",
        "      mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "      outputs = mprs.run()\n",
        "\n",
        "      # add name column, del type column\n",
        "      protein_df[f'name_{i}'] = protein_df[f'seq_{i}'].apply(lambda x: x.split('.')[0])\n",
        "      protein_df.drop(f\"type_{i}\", axis=1, inplace=True)\n",
        "      print(outputs)\n",
        "      protein_df[f'seq_{i}'] = [output.split(\"\\t\")[1] for output in outputs]\n",
        "\n",
        "    # columns: name_1, name_2, chain_1, chain_2, seq_1, seq_2, label, stage\n",
        "    protein_df.to_csv(csv_dataset_path, index=None)\n",
        "    return csv_dataset_path\n",
        "\n",
        "\n",
        "'''\n",
        "  elif data_type == \"A pair of AA Sequences\",\n",
        "  elif data_type == \"A pair of SA Sequences\",\n",
        "  elif data_type == \"A pair of UniProt IDs\",\n",
        "  elif data_type == \"A pair of PDB/CIF Structures\",\n",
        "  elif data_type == \"Multiple pairs of AA Sequences\",\n",
        "  elif data_type == \"Multiple pairs of SA Sequences\",\n",
        "  elif data_type == \"Multiple pairs of UniProt IDs\",\n",
        "  elif data_type == \"Multiple pairs of PDB/CIF Structures\",\n",
        "'''\n",
        "\n",
        "\n",
        "# # return a SA Sequence or a csv dataset path\n",
        "# def get_raw_dataset(data_type, raw_data):\n",
        "#   if data_type in data_type_list[:3]:\n",
        "#     raw_dataset = get_SA_sequence_by_data_type(data_type, raw_data.value)\n",
        "#   elif data_type == data_type_list[3]:\n",
        "#     raw_dataset = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "#   elif data_type in data_type_list[4:8]:\n",
        "#     raw_dataset = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "#   elif data_type in data_type_list[8]:\n",
        "#     raw_dataset = get_SA_sequence_by_data_type(data_type, raw_data.value)\n",
        "\n",
        "#   return raw_dataset\n",
        "\n",
        "# def upload_dataset(data_type):\n",
        "#   print(Fore.BLUE+f\"Please upload the .csv file which contains {data_type}\"+Style.RESET_ALL)\n",
        "#   uploaded_csv_path = upload_file(UPLOAD_FILE_HOME)\n",
        "#   print(Fore.BLUE+\"Successfully upload your .csv file!\"+Style.RESET_ALL)\n",
        "#   print(\"=\"*100)\n",
        "\n",
        "#   # selected_csv_dataset = DATASET_HOME / f\"[DATASET]{Path(uploaded_csv_path).stem}.csv\"\n",
        "#   # get_SASequence_by_data_type(data_type, uploaded_csv_path, selected_csv_dataset)\n",
        "#   # get_SA_sequence_by_data_type(data_type, uploaded_csv_path)\n",
        "#   # print()\n",
        "#   # print(\"=\"*100)\n",
        "#   # print(Fore.BLUE+\"Successfully upload your dataset!\"+Style.RESET_ALL)\n",
        "\n",
        "#   return uploaded_csv_path\n",
        "\n",
        "\n",
        "\n",
        "################################################################################\n",
        "########################## Download predicted structures #######################\n",
        "################################################################################\n",
        "def uniprot2pdb(uniprot_ids, nprocess=20):\n",
        "  from saprot.utils.downloader import AlphaDBDownloader\n",
        "\n",
        "  os.makedirs(STRUCTURE_HOME, exist_ok=True)\n",
        "  af2_downloader = AlphaDBDownloader(uniprot_ids, \"pdb\", save_dir=STRUCTURE_HOME, n_process=20)\n",
        "  af2_downloader.run()\n",
        "\n",
        "\n",
        "\n",
        "################################################################################\n",
        "############### Form foldseek sequences by multiple processes ##################\n",
        "################################################################################\n",
        "# def pdb2sequence(process_id, idx, uniprot_id, writer):\n",
        "#   from saprot.utils.foldseek_util import get_struc_seq\n",
        "\n",
        "#   try:\n",
        "#     pdb_path = f\"{STRUCTURE_HOME}/{uniprot_id}.pdb\"\n",
        "#     cif_path = f\"{STRUCTURE_HOME}/{uniprot_id}.cif\"\n",
        "#     if Path(pdb_path).exists():\n",
        "#       seq = get_struc_seq(FOLDSEEK_PATH, pdb_path, [\"A\"], process_id=process_id)[\"A\"][-1]\n",
        "#     if Path(cif_path).exists():\n",
        "#       seq = get_struc_seq(FOLDSEEK_PATH, cif_path, [\"A\"], process_id=process_id)[\"A\"][-1]\n",
        "\n",
        "#     writer.write(f\"{uniprot_id}\\t{seq}\\n\")\n",
        "#   except Exception as e:\n",
        "#     print(f\"Error: {uniprot_id}, {e}\")\n",
        "\n",
        "# clear_output(wait=True)\n",
        "# print(\"Installation finished!\")\n",
        "\n",
        "def pdb2sequence(process_id, idx, row_tuple, writer):\n",
        "\n",
        "  # print(\"=\"*100)\n",
        "  # print(row_tuple)\n",
        "  # print(\"=\"*100)\n",
        "  uniprot_id = row_tuple[0].split('.')[0]     #\n",
        "  struc_type = row_tuple[1]                   # PDB or AF2\n",
        "  chain = row_tuple[2]\n",
        "\n",
        "  if struc_type==\"AF2\":\n",
        "    plddt_mask= True\n",
        "    chain = 'A'\n",
        "  else:\n",
        "    plddt_mask= False\n",
        "\n",
        "  from saprot.utils.foldseek_util import get_struc_seq\n",
        "\n",
        "  try:\n",
        "    pdb_path = f\"{STRUCTURE_HOME}/{uniprot_id}.pdb\"\n",
        "    cif_path = f\"{STRUCTURE_HOME}/{uniprot_id}.cif\"\n",
        "    if Path(pdb_path).exists():\n",
        "      seq = get_struc_seq(FOLDSEEK_PATH, pdb_path, [chain], process_id=process_id, plddt_mask=plddt_mask)[chain][-1]\n",
        "    elif Path(cif_path).exists():\n",
        "      seq = get_struc_seq(FOLDSEEK_PATH, cif_path, [chain], process_id=process_id, plddt_mask=plddt_mask)[chain][-1]\n",
        "    else:\n",
        "      raise BaseException(f\"The {uniprot_id}.pdb/{uniprot_id}.cif file doesn't exists!\")\n",
        "    writer.write(f\"{uniprot_id}\\t{seq}\\n\")\n",
        "\n",
        "  except Exception as e:\n",
        "    print(f\"Error: {uniprot_id}, {e}\")\n",
        "\n",
        "\n",
        "pymol_color_list = [\"#33ff33\",\"#00ffff\",\"#ff33cc\",\"#ffff00\",\"#ff9999\",\"#e5e5e5\",\"#7f7fff\",\"#ff7f00\",\n",
        "          \"#7fff7f\",\"#199999\",\"#ff007f\",\"#ffdd5e\",\"#8c3f99\",\"#b2b2b2\",\"#007fff\",\"#c4b200\",\n",
        "          \"#8cb266\",\"#00bfbf\",\"#b27f7f\",\"#fcd1a5\",\"#ff7f7f\",\"#ffbfdd\",\"#7fffff\",\"#ffff7f\",\n",
        "          \"#00ff7f\",\"#337fcc\",\"#d8337f\",\"#bfff3f\",\"#ff7fff\",\"#d8d8ff\",\"#3fffbf\",\"#b78c4c\",\n",
        "          \"#339933\",\"#66b2b2\",\"#ba8c84\",\"#84bf00\",\"#b24c66\",\"#7f7f7f\",\"#3f3fa5\",\"#a5512b\"]\n",
        "\n",
        "alphabet_list = list(ascii_uppercase+ascii_lowercase)\n",
        "\n",
        "\n",
        "def convert_outputs_to_pdb(outputs):\n",
        "\tfinal_atom_positions = atom14_to_atom37(outputs[\"positions\"][-1], outputs)\n",
        "\toutputs = {k: v.to(\"cpu\").numpy() for k, v in outputs.items()}\n",
        "\tfinal_atom_positions = final_atom_positions.cpu().numpy()\n",
        "\tfinal_atom_mask = outputs[\"atom37_atom_exists\"]\n",
        "\tpdbs = []\n",
        "\toutputs[\"plddt\"] *= 100\n",
        "\n",
        "\tfor i in range(outputs[\"aatype\"].shape[0]):\n",
        "\t\taa = outputs[\"aatype\"][i]\n",
        "\t\tpred_pos = final_atom_positions[i]\n",
        "\t\tmask = final_atom_mask[i]\n",
        "\t\tresid = outputs[\"residue_index\"][i] + 1\n",
        "\t\tpred = OFProtein(\n",
        "\t\t    aatype=aa,\n",
        "\t\t    atom_positions=pred_pos,\n",
        "\t\t    atom_mask=mask,\n",
        "\t\t    residue_index=resid,\n",
        "\t\t    b_factors=outputs[\"plddt\"][i],\n",
        "\t\t    chain_index=outputs[\"chain_index\"][i] if \"chain_index\" in outputs else None,\n",
        "\t\t)\n",
        "\t\tpdbs.append(to_pdb(pred))\n",
        "\treturn pdbs\n",
        "\n",
        "\n",
        "# This function is copied from ColabFold!\n",
        "def show_pdb(path, show_sidechains=False, show_mainchains=False, color=\"lddt\"):\n",
        "  file_type = str(path).split(\".\")[-1]\n",
        "  if file_type == \"cif\":\n",
        "    file_type == \"mmcif\"\n",
        "\n",
        "  view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js',)\n",
        "  view.addModel(open(path,'r').read(),file_type)\n",
        "\n",
        "  if color == \"lDDT\":\n",
        "    view.setStyle({'cartoon': {'colorscheme': {'prop':'b','gradient': 'roygb','min':50,'max':90}}})\n",
        "  elif color == \"rainbow\":\n",
        "    view.setStyle({'cartoon': {'color':'spectrum'}})\n",
        "  elif color == \"chain\":\n",
        "    chains = len(get_chain_ids(path))\n",
        "    for n,chain,color in zip(range(chains),alphabet_list,pymol_color_list):\n",
        "       view.setStyle({'chain':chain},{'cartoon': {'color':color}})\n",
        "\n",
        "  if show_sidechains:\n",
        "    BB = ['C','O','N']\n",
        "    view.addStyle({'and':[{'resn':[\"GLY\",\"PRO\"],'invert':True},{'atom':BB,'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"GLY\"},{'atom':'CA'}]},\n",
        "                        {'sphere':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"PRO\"},{'atom':['C','O'],'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "  if show_mainchains:\n",
        "    BB = ['C','O','N','CA']\n",
        "    view.addStyle({'atom':BB},{'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "\n",
        "  view.zoomTo()\n",
        "  return view\n",
        "\n",
        "\n",
        "def plot_plddt_legend(dpi=100):\n",
        "  thresh = ['plDDT:','Very low (<50)','Low (60)','OK (70)','Confident (80)','Very high (>90)']\n",
        "  plt.figure(figsize=(1,0.1),dpi=dpi)\n",
        "  ########################################\n",
        "  for c in [\"#FFFFFF\",\"#FF0000\",\"#FFFF00\",\"#00FF00\",\"#00FFFF\",\"#0000FF\"]:\n",
        "    plt.bar(0, 0, color=c)\n",
        "  plt.legend(thresh, frameon=False,\n",
        "             loc='center', ncol=6,\n",
        "             handletextpad=1,\n",
        "             columnspacing=1,\n",
        "             markerscale=0.5,)\n",
        "  plt.axis(False)\n",
        "  return plt\n",
        "\n",
        "\n",
        "################################################################################\n",
        "###############   Download file to local computer   ##################\n",
        "################################################################################\n",
        "def file_download(path: str):\n",
        "  with open(path, \"rb\") as r:\n",
        "    res = r.read()\n",
        "\n",
        "  #FILE\n",
        "  filename = os.path.basename(path)\n",
        "  b64 = base64.b64encode(res)\n",
        "  payload = b64.decode()\n",
        "\n",
        "  #BUTTONS\n",
        "  html_buttons = '''<html>\n",
        "  <head>\n",
        "  <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">\n",
        "  </head>\n",
        "  <body>\n",
        "  <a download=\"{filename}\" href=\"data:text/csv;base64,{payload}\" download>\n",
        "  <button class=\"p-Widget jupyter-widgets jupyter-button widget-button mod-warning\">Download File</button>\n",
        "  </a>\n",
        "  </body>\n",
        "  </html>\n",
        "  '''\n",
        "\n",
        "  html_button = html_buttons.format(payload=payload,filename=filename)\n",
        "  display(HTML(html_button))\n",
        "\n",
        "  # Automatically download file if the server is from google cloud.\n",
        "  if root_dir == \"/content\":\n",
        "    files.download(path)\n",
        "\n",
        "\n",
        "clear_output(wait=True)\n",
        "print(\"Installation finished!\")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "3Uxag_RSBI7e"
      },
      "source": [
        "# **2: Train and Share your Protein Model** <a name=\"train\"></a>\n",
        "\n",
        "You can **train** a model based on pre-trained SaProt, or **continually train** a fine-tuned model in SaprotHub.\n",
        "\n",
        "\n",
        "\n",
        "<!-- ## Training Dataset\n",
        "\n",
        "For the training dataset, **two additional columns** are required in the CSV file: `label` and `stage`.\n",
        "\n",
        "<img src=\"https://github.com/westlake-repl/SaProtHub/blob/main/Figure/Multiple_AA_Sequences_data_format_training.png\n",
        "?raw=true\" height=\"200\" width=\"400px\" align=\"center\">\n",
        "\n",
        "### Column `label`\n",
        "\n",
        "The content of column `label` depends on your **task type**:\n",
        "\n",
        "| Task Type                         | Content in the Column                          |\n",
        "|-----------------------------------|------------------------------------------------|\n",
        "| Classification tasks              | Category index starting from zero              |\n",
        "| Amino Acid Classification tasks   | A list of category indices for each amino acid |\n",
        "| Regression tasks                  | Numerical values                               |\n",
        "\n",
        "<img src=\"https://github.com/westlake-repl/SaProtHub/blob/main/Figure/label_format.png?raw=true\" height=\"300\" width=\"800px\" align=\"center\">\n",
        "<br>\n",
        "\n",
        "\n",
        "### Column `stage`\n",
        "\n",
        "The column `stage` indicate whether the sample is used for training, validation, or testing. Ensure your dataset includes samples for all three stages. The values are: `train`, `valid`, `test`.\n",
        "\n",
        "<br>\n",
        "\n",
        "### **Note:**\n",
        "\n",
        "1. **Examples are available** at /content/SaprotHub/upload_files (if you connect to your local server, then the path is /SaprotHub/upload_files). Download to review their format, and then upload them for a trial.\n",
        "\n",
        "2.  <a href=\"#get_sa\">Here</a> you can **convert your data into SA Sequence** format.\n",
        "\n",
        "3. <a href=\"#fa2csv\">Here</a> you can **convert your .fa/.fasta file to a .csv file**, which corresponds to the data format for Multiple AA Sequences.\n",
        "\n",
        "4. <a href=\"#split_dataset\">Here</a> you can **randomly split your .csv dataset**, which means to add a `stage` column, where the ratio of `train`:`valid`:`test` is 8:1:1.\n",
        "\n",
        "4. The maximum input length of the model is 1024, and protein sequences exceeding this length will only retain the first 1024 amino acids. -->\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "vqdmLslQBI7e"
      },
      "outputs": [],
      "source": [
        "#@title **2.1: Train your Model** <a name=\"train\"></a>\n",
        "\n",
        "################################################################################\n",
        "############################# ADVANCED CONFIG ##################################\n",
        "################################################################################\n",
        "\n",
        "GPU_batch_size = 0\n",
        "accumulate_grad_batches = 0\n",
        "num_workers = 2\n",
        "seed = 20000812\n",
        "\n",
        "limit_train_batches=1.0\n",
        "limit_val_batches=1.0\n",
        "limit_test_batches=1.0\n",
        "\n",
        "val_check_interval=0.5\n",
        "\n",
        "mask_struc_ratio=None\n",
        "\n",
        "################################################################################\n",
        "################################## MARKDOWN #################################\n",
        "################################################################################\n",
        "\n",
        "#@markdown > üìçPlease see the [tutorial](https://github.com/westlake-repl/SaprotHub/wiki/2.1:-Train-your-model)\n",
        "\n",
        "if torch.cuda.is_available() is False:\n",
        "  raise BaseException(\"Please refer to Section 1.1 to switch your Runtime to a GPU!\")\n",
        "\n",
        "################################################################################\n",
        "################################## TASK CONFIG #################################\n",
        "################################################################################\n",
        "#@markdown # 1. Task\n",
        "task_name = \"demo\" # @param {type:\"string\"}\n",
        "task_type = \"Protein-level Classification\" # @param [\"Protein-level Classification\", \"Protein-level Regression\", \"Residue-level Classification\", \"Protein-protein Classification\", \"Protein-protein Regression\"]\n",
        "task_type = task_type_dict[task_type]\n",
        "\n",
        "if task_type in [\"classification\", 'token_classification', 'pair_classification']:\n",
        "\n",
        "  print(Fore.BLUE+'Enter the number of category in your training dataset here:'+Style.RESET_ALL)\n",
        "  num_of_categories = ipywidgets.BoundedIntText(\n",
        "                                              # value=7,\n",
        "                                              min=2,\n",
        "                                              max=1000000,\n",
        "                                              step=1,\n",
        "                                              # description='num_of_category: \\n',\n",
        "                                              disabled=False)\n",
        "  num_of_categories.layout.width = \"100px\"\n",
        "  display(num_of_categories)\n",
        "\n",
        "#@markdown <br>\n",
        "\n",
        "################################################################################\n",
        "#################################### MODEL CONFIG #####################################\n",
        "################################################################################\n",
        "#@markdown # 2. Model\n",
        "\n",
        "base_model = \"Shared by peers on SaprotHub\" # @param [\"Official pretrained SaProt (35M)\", \"Official pretrained SaProt (650M)\", \"Trained by yourself on ColabSaprot\", \"Shared by peers on SaprotHub\", \"Saved in your local computer\"]\n",
        "\n",
        "# continue learning\n",
        "if base_model in [\"Trained by yourself on ColabSaprot\", \"Shared by peers on SaprotHub\", \"Saved in your local computer\"]:\n",
        "  continue_learning = True\n",
        "  adapter_combobox = select_adapter_from(task_type, use_model_from=base_model)\n",
        "else:\n",
        "  continue_learning = False\n",
        "\n",
        "#@markdown <br>\n",
        "\n",
        "################################################################################\n",
        "################################### DATASET CONFIG ####################################\n",
        "################################################################################\n",
        "#@markdown # 3. Dataset\n",
        "\n",
        "data_type = \"Multiple SA Sequences\" # @param [\"Multiple AA Sequences\", \"Multiple SA Sequences\", \"Multiple UniProt IDs\", \"Multiple PDB/CIF Structures\", \"SaprotHub Dataset\", \"Multiple pairs of AA Sequences\", \"Multiple pairs of SA Sequences\", \"Multiple pairs of UniProt IDs\", \"Multiple pairs of PDB/CIF Structures\"]\n",
        "raw_data = input_raw_data_by_data_type(data_type)\n",
        "\n",
        "#@markdown <br>\n",
        "\n",
        "################################################################################\n",
        "################################### TRAIN CONFIG ####################################\n",
        "################################################################################\n",
        "#@markdown # 4. Training\n",
        "\n",
        "batch_size = \"Adaptive\" # @param [\"Adaptive\", \"1\", \"2\", \"4\", \"8\", \"16\", \"32\", \"64\", \"128\", \"256\"]\n",
        "max_epochs = 1 # @param [\"10\", \"20\", \"50\"] {type:\"raw\", allow-input: true}\n",
        "learning_rate = 1.0e-3 # @param [\"1.0e-3\", \"5.0e-4\", \"1.0e-4\"] {type:\"raw\", allow-input: true}\n",
        "\n",
        "#@markdown <br>\n",
        "\n",
        "################################################################################\n",
        "################################### TRAIN ####################################\n",
        "################################################################################\n",
        "\n",
        "def train(button):\n",
        "  global base_model\n",
        "  global GPU_batch_size\n",
        "  global accumulate_grad_batches\n",
        "\n",
        "  button.disabled = True\n",
        "  button.description = 'Training...'\n",
        "  button.button_style = ''\n",
        "\n",
        "################################################################################\n",
        "################################### DATASET CONFIRM ####################################\n",
        "################################################################################\n",
        "\n",
        "  if data_type == data_type_list[8]:\n",
        "    lmdb_dataset_path = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "  else:\n",
        "    csv_dataset_path = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "    from saprot.utils.construct_lmdb import construct_lmdb\n",
        "    construct_lmdb(csv_dataset_path, LMDB_HOME, task_name, task_type)\n",
        "\n",
        "    lmdb_dataset_path = LMDB_HOME / task_name\n",
        "\n",
        "################################################################################\n",
        "################################### MODEL CONFIRM ####################################\n",
        "################################################################################\n",
        "\n",
        "  # base_model\n",
        "  if continue_learning:\n",
        "    adapter_path = ADAPTER_HOME / task_type / adapter_combobox.value\n",
        "    print(f\"Training on an existing model: {adapter_path}\")\n",
        "\n",
        "    if base_model == \"Shared by peers on SaprotHub\":\n",
        "      if not adapter_path.exists():\n",
        "        snapshot_download(repo_id=adapter_combobox.value, repo_type=\"model\", local_dir=adapter_path)\n",
        "\n",
        "    adapter_config_path = Path(adapter_path) / \"adapter_config.json\"\n",
        "    assert adapter_config_path.exists(), f\"Can't find {adapter_config_path}\"\n",
        "    with open(adapter_config_path, 'r') as f:\n",
        "      adapter_config = json.load(f)\n",
        "      base_model = adapter_config['base_model_name_or_path']\n",
        "\n",
        "  elif base_model == \"Official pretrained SaProt (35M)\":\n",
        "    base_model = \"westlake-repl/SaProt_35M_AF2\"\n",
        "\n",
        "  elif base_model == \"Official pretrained SaProt (650M)\":\n",
        "    base_model = \"westlake-repl/SaProt_650M_AF2\"\n",
        "\n",
        "  # model size and model name\n",
        "  if base_model == \"westlake-repl/SaProt_650M_AF2\":\n",
        "    model_size = \"650M\"\n",
        "    model_name = f\"Model-{task_name}-{model_size}\"\n",
        "  elif base_model == \"westlake-repl/SaProt_35M_AF2\":\n",
        "    model_size = \"35M\"\n",
        "    model_name = f\"Model-{task_name}-{model_size}\"\n",
        "\n",
        "################################################################################\n",
        "################################# CONFIG #######################################\n",
        "################################################################################\n",
        "\n",
        "  from saprot.config.config_dict import Default_config\n",
        "  config = copy.deepcopy(Default_config)\n",
        "\n",
        "  config.setting.run_mode = \"train\"\n",
        "  config.setting.seed = seed\n",
        "\n",
        "################################################################################\n",
        "################################# MODEL ########################################\n",
        "################################################################################\n",
        "\n",
        "  if task_type in [\"classification\", \"token_classification\", \"pair_classification\"]:\n",
        "    config.model.kwargs.num_labels = num_of_categories.value\n",
        "\n",
        "  config.model.model_py_path = model_type_dict[task_type]\n",
        "  config.model.kwargs.config_path = base_model\n",
        "  config.dataset.kwargs.tokenizer = base_model\n",
        "\n",
        "  config.model.save_path = str(ADAPTER_HOME / f\"{task_type}\" / \"Local\" / model_name)\n",
        "\n",
        "  if task_type in [\"regression\", \"pair_regression\", \"pair_classification\"]:\n",
        "    config.model.kwargs.extra_config = {}\n",
        "    config.model.kwargs.extra_config.attention_probs_dropout_prob=0\n",
        "    config.model.kwargs.extra_config.hidden_dropout_prob=0\n",
        "\n",
        "  config.model.kwargs.lora_kwargs = EasyDict({\n",
        "    \"is_trainable\": True,\n",
        "    \"num_lora\": 1,\n",
        "    \"config_list\": []})\n",
        "  if continue_learning:\n",
        "    config.model.kwargs.lora_kwargs.config_list.append({\"lora_config_path\": adapter_path})\n",
        "\n",
        "################################################################################\n",
        "################################# DATASET ######################################\n",
        "################################################################################\n",
        "\n",
        "  config.dataset.dataset_py_path = dataset_type_dict[task_type]\n",
        "\n",
        "  config.dataset.train_lmdb = str(lmdb_dataset_path / \"train\")\n",
        "  config.dataset.valid_lmdb = str(lmdb_dataset_path / \"valid\")\n",
        "  config.dataset.test_lmdb = str(lmdb_dataset_path / \"test\")\n",
        "\n",
        "  # num_workers\n",
        "  config.dataset.dataloader_kwargs.num_workers = num_workers\n",
        "\n",
        "  # mask_struc\n",
        "  # config.dataset.kwargs.mask_struc_ratio= mask_struc_ratio\n",
        "\n",
        "  ################################################################################\n",
        "  ######################## batch size ############################################\n",
        "  ################################################################################\n",
        "  def get_accumulate_grad_samples(num_samples):\n",
        "      if num_samples > 3200:\n",
        "          return 64\n",
        "      elif 1600 < num_samples <= 3200:\n",
        "          return 32\n",
        "      elif 800 < num_samples <= 1600:\n",
        "          return 16\n",
        "      elif 400 < num_samples <= 800:\n",
        "          return 8\n",
        "      elif 200 < num_samples <= 400:\n",
        "          return 4\n",
        "      elif 100 < num_samples <= 200:\n",
        "          return 2\n",
        "      else:\n",
        "          return 1\n",
        "\n",
        "  # advanced config\n",
        "  if (GPU_batch_size > 0) and (accumulate_grad_batches > 0):\n",
        "    config.dataset.dataloader_kwargs.batch_size = GPU_batch_size\n",
        "    config.Trainer.accumulate_grad_batches= accumulate_grad_batches\n",
        "\n",
        "  elif (GPU_batch_size == 0) and (accumulate_grad_batches == 0):\n",
        "\n",
        "    # batch_size\n",
        "    if base_model == \"westlake-repl/SaProt_650M_AF2\" and root_dir == \"/content\":\n",
        "      GPU_batch_size = 1\n",
        "    else:\n",
        "      GPU_batch_size_dict = {\n",
        "        \"Tesla T4\": 2,\n",
        "        \"NVIDIA L4\": 2,\n",
        "        \"NVIDIA A100-SXM4-40GB\": 4,\n",
        "        }\n",
        "      GPU_name = torch.cuda.get_device_name(0)\n",
        "      GPU_batch_size = GPU_batch_size_dict[GPU_name] if GPU_name in GPU_batch_size_dict else 2\n",
        "\n",
        "      if task_type in [\"pair_classification\", \"pair_regression\"]:\n",
        "        GPU_batch_size = int(max(GPU_batch_size / 2, 1))\n",
        "\n",
        "    config.dataset.dataloader_kwargs.batch_size = GPU_batch_size\n",
        "\n",
        "    # accumulate_grad_batches\n",
        "    if batch_size == \"Adaptive\":\n",
        "\n",
        "      env = lmdb.open(config.dataset.train_lmdb, readonly=True)\n",
        "\n",
        "      with env.begin() as txn:\n",
        "        stat = txn.stat()\n",
        "        num_samples = stat['entries']\n",
        "\n",
        "      accumulate_grad_samples = get_accumulate_grad_samples(num_samples)\n",
        "\n",
        "    else:\n",
        "      accumulate_grad_samples = int(batch_size)\n",
        "\n",
        "    accumulate_grad_batches = max(int(accumulate_grad_samples / GPU_batch_size), 1)\n",
        "\n",
        "    config.Trainer.accumulate_grad_batches= accumulate_grad_batches\n",
        "\n",
        "  else:\n",
        "    raise BaseException(f\"Please make sure `GPU_batch_size`({GPU_batch_size}) and `accumulate_grad_batches`({accumulate_grad_batches}) are both greater than zero!\")\n",
        "\n",
        "  ################################################################################\n",
        "  ############################## TRAINER #########################################\n",
        "  ################################################################################\n",
        "\n",
        "  config.Trainer.accelerator = \"gpu\" if torch.cuda.is_available() else \"cpu\"\n",
        "\n",
        "  # epoch\n",
        "  config.Trainer.max_epochs = max_epochs\n",
        "  # test only: load the existing model\n",
        "  if config.Trainer.max_epochs == 0:\n",
        "    config.model.save_path = config.model.kwargs.lora_kwargs.config_list[0].lora_config_path\n",
        "\n",
        "  # learning rate\n",
        "  config.model.lr_scheduler_kwargs.init_lr = learning_rate\n",
        "\n",
        "  # trainer\n",
        "  config.Trainer.limit_train_batches=limit_train_batches\n",
        "  config.Trainer.limit_val_batches=limit_val_batches\n",
        "  config.Trainer.limit_test_batches=limit_test_batches\n",
        "  config.Trainer.val_check_interval=val_check_interval\n",
        "\n",
        "  # strategy\n",
        "  strategy = {\n",
        "      # - deepspeed\n",
        "      # 'class': 'DeepSpeedStrategy',\n",
        "      # 'stage': 2\n",
        "\n",
        "      # - None\n",
        "      # 'class': None,\n",
        "\n",
        "      # - DP\n",
        "      # 'class': 'DataParallelStrategy',\n",
        "\n",
        "      # - DDP\n",
        "      # 'class': 'DDPStrategy',\n",
        "      # 'find_unused_parameter': True\n",
        "  }\n",
        "  config.Trainer.strategy = strategy\n",
        "\n",
        "  ################################################################################\n",
        "  ############################## Run the task ####################################\n",
        "  ################################################################################\n",
        "\n",
        "  print('='*100)\n",
        "  print(Fore.BLUE+f\"Training task type: {task_type}\"+Style.RESET_ALL)\n",
        "  print(Fore.BLUE+f\"Dataset: {lmdb_dataset_path}\"+Style.RESET_ALL)\n",
        "  print(Fore.BLUE+f\"Base Model: {config.model.kwargs.config_path}\"+Style.RESET_ALL)\n",
        "  if continue_learning:\n",
        "    print(Fore.BLUE+f\"Existing model: {config.model.kwargs.lora_kwargs.config_list[0]['lora_config_path']}\"+Style.RESET_ALL)\n",
        "  print('='*100)\n",
        "  pprint.pprint(config)\n",
        "  print('='*100)\n",
        "\n",
        "  from saprot.scripts.training import finetune\n",
        "  finetune(config)\n",
        "\n",
        "\n",
        "  ################################################################################\n",
        "  ############################## Save the adapter ################################\n",
        "  ################################################################################\n",
        "\n",
        "  def add_training_data_type_to_config(metadata_path, training_data_type):\n",
        "    if metadata_path.exists() is False:\n",
        "      config_data = {\n",
        "          'training_data_type': training_data_type\n",
        "          }\n",
        "      with open(metadata_path, 'w') as file:\n",
        "          json.dump(config_data, file, indent=4)\n",
        "\n",
        "    else:\n",
        "      with open(metadata_path, 'r') as file:\n",
        "          config_data = json.load(file)\n",
        "\n",
        "      config_data['training_data_type'] = training_data_type\n",
        "\n",
        "      with open(metadata_path, 'w') as file:\n",
        "          json.dump(config_data, file, indent=4)\n",
        "\n",
        "  metadata_path = Path(config.model.save_path) / \"metadata.json\"\n",
        "  training_data_type = training_data_type_dict[data_type]\n",
        "  add_training_data_type_to_config(metadata_path, training_data_type)\n",
        "\n",
        "  print(Fore.BLUE)\n",
        "  print(f\"Model is saved to \\\"{config.model.save_path}\\\" on Colab Server\")\n",
        "  print(Style.RESET_ALL)\n",
        "\n",
        "\n",
        "  adapter_zip = Path(config.model.save_path) / f\"{model_name}.zip\"\n",
        "  !cd $config.model.save_path && zip -r $adapter_zip \"adapter_config.json\" \"adapter_model.safetensors\" \"README.md\" \"metadata.json\"\n",
        "  # !cd $config.model.save_path && zip -r $adapter_zip \"adapter_config.json\" \"adapter_model.safetensors\" \"adapter_model.bin\" \"README.md\" \"metadata.json\"\n",
        "  print(\"Click to download the model to your local computer\")\n",
        "  if adapter_zip.exists():\n",
        "    # files.download(adapter_zip)\n",
        "    file_download(adapter_zip)\n",
        "\n",
        "  ################################################################################\n",
        "  ############################### Modify README ##################################\n",
        "  ################################################################################\n",
        "\n",
        "button_train = ipywidgets.Button(\n",
        "    description='Start Training',\n",
        "    disabled=False,\n",
        "    button_style='success', # 'success', 'info', 'warning', 'danger' or ''\n",
        "    tooltip='Apply',\n",
        "    # icon='check' # (FontAwesome names without the `fa-` prefix)\n",
        "    )\n",
        "button_train.on_click(train)\n",
        "button_train.layout.width = '300px'\n",
        "display(button_train)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "UnKX1BTZBI7f"
      },
      "outputs": [],
      "source": [
        "#@title **2.2: Login HuggingFace to Upload your model (Optional)** <a name=\"upload_model\"></a>\n",
        "################################################################################\n",
        "###################### Login HuggingFace #######################################\n",
        "################################################################################\n",
        "\n",
        "from huggingface_hub import notebook_login\n",
        "notebook_login()\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "6XlluTsPBI7m"
      },
      "outputs": [],
      "source": [
        "#@title **2.3: Upload your Model (Optional)**\n",
        "\n",
        "#@markdown > üìçPlease see the [tutorial](https://github.com/westlake-repl/SaprotHub/wiki/2.2:-Upload-your-model)\n",
        "\n",
        "# #@markdown Your Huggingface adapter repository names follow the format `<username>/<task_name>`.\n",
        "\n",
        "################################################################################\n",
        "########################## Metadata  ###########################################\n",
        "################################################################################\n",
        "name = \"demo_cls\" # @param {type:\"string\"}\n",
        "description = \"This model is used for a demo classification task\" # @param {type:\"string\"}\n",
        "\n",
        "\n",
        "\n",
        "# #@markdown > 0: Nucleus <br>\n",
        "# #@markdown > 1: Cytoplasm <br>\n",
        "# #@markdown > 2: Extracellular <br>\n",
        "# #@markdown > ... <br>\n",
        "# #@markdown > 9: Peroxisome <br>\n",
        "\n",
        "label_meanings = \"A, B\" #@param {type:\"string\"}\n",
        "\n",
        "\n",
        "\n",
        "################################################################################\n",
        "########################### Move Files  ########################################\n",
        "################################################################################\n",
        "\n",
        "from huggingface_hub import HfApi, Repository, ModelFilter\n",
        "\n",
        "api = HfApi()\n",
        "\n",
        "user = api.whoami()\n",
        "\n",
        "if name == \"\":\n",
        "  name = model_name\n",
        "repo_name = user['name'] + '/' + name\n",
        "local_dir = Path(\"/content/SaprotHub/model_to_push\") / repo_name\n",
        "local_dir.mkdir(parents=True, exist_ok=True)\n",
        "\n",
        "repo_list = [repo.id for repo in api.list_models(filter=ModelFilter(author=user['name']))]\n",
        "if repo_name not in repo_list:\n",
        "  api.create_repo(repo_name, private=False)\n",
        "\n",
        "repo = Repository(local_dir=local_dir, clone_from=repo_name)\n",
        "\n",
        "command = f\"cp {config.model.save_path}/* {local_dir}/\"\n",
        "subprocess.run(command, shell=True)\n",
        "\n",
        "################################################################################\n",
        "########################## Modify README  ######################################\n",
        "################################################################################\n",
        "import json\n",
        "\n",
        "md_path = local_dir / \"README.md\"\n",
        "\n",
        "if task_type in [\"classification\", \"token_classification\", \"pair_classification\"]:\n",
        "  label_meanings_md = ''\n",
        "  for index, label in enumerate(label_meanings.split(', ')):\n",
        "    label_meanings_md += f\"{index}: {label} <br> \"\n",
        "\n",
        "  # print(label_meanings_md)\n",
        "  description = description + \"<br><br> The digital label means: <br>\" + label_meanings_md\n",
        "\n",
        "replace_data = {\n",
        "    \"<!-- Provide a quick summary of what the model is/does. -->\": description\n",
        "}\n",
        "\n",
        "with open(md_path, \"r\") as file:\n",
        "    content = file.read()\n",
        "\n",
        "for key, value in replace_data.items():\n",
        "    if value != \"\":\n",
        "        content = content.replace(key, value)\n",
        "\n",
        "# new_md_path = \"README.md\"\n",
        "with open(md_path, \"w\") as file:\n",
        "    file.write(content)\n",
        "\n",
        "################################################################################\n",
        "########################## Upload Model  #######################################\n",
        "################################################################################\n",
        "\n",
        "\n",
        "repo.push_to_hub(commit_message=\"Upload adapter model\")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "bQ1JgmrsBI7m"
      },
      "source": [
        "# **3: Use SaProt to Predict**"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "h8qHRJtIQxU4"
      },
      "outputs": [],
      "source": [
        "#@title **3.1: Classification&Regression Prediction** <a name=\"prediction\"></a>\n",
        "\n",
        "#@markdown > üìçPlease see the [tutorial](https://github.com/westlake-repl/SaprotHub/wiki/3.1:-Classification-Regression-Prediction)\n",
        "\n",
        "from transformers import EsmTokenizer\n",
        "import torch\n",
        "import copy\n",
        "\n",
        "################################################################################\n",
        "################################# TASK #########################################\n",
        "################################################################################\n",
        "#@markdown # 1. Task\n",
        "\n",
        "task_type = \"Protein-level Regression\" # @param [\"Protein-level Classification\", \"Protein-level Regression\", \"Residue-level Classification\", \"Protein-protein Classification\", \"Protein-protein Regression\"]\n",
        "task_type = task_type_dict[task_type]\n",
        "\n",
        "if task_type in [\"classification\", 'token_classification', 'pair_classification']:\n",
        "\n",
        "  print(Fore.BLUE+'The number of categories in your classification task:'+Style.RESET_ALL)\n",
        "  num_of_categories = ipywidgets.BoundedIntText(\n",
        "                                              # value=7,\n",
        "                                              min=2,\n",
        "                                              # max=10,\n",
        "                                              step=1,\n",
        "                                              # description='num_of_category: \\n',\n",
        "                                              disabled=False)\n",
        "  num_of_categories.layout.width = \"100px\"\n",
        "  display(num_of_categories)\n",
        "\n",
        "#@markdown <br>\n",
        "\n",
        "\n",
        "################################################################################\n",
        "################################## MODEL #######################################\n",
        "################################################################################\n",
        "#@markdown # 2. Model\n",
        "\n",
        "##@markdown As we use Parameter-Efficient Fine-Tuning Technique, which allows us to store model weights into an small adapter without adjusting the original model weights during training, it's necessary to specify both the original model and adapter for prediction.\n",
        "##@markdown\n",
        "##@markdown 1. Select a **base model**\n",
        "##@markdown\n",
        "##@markdown 2. By running this cell, you will see an **model combobox**. We provide two ways to select your adapter:\n",
        "##@markdown  - Select a **local model** from the combobox.\n",
        "##@markdown   - Enter a **huggingface repository name** to the combobox. (e.g. \"SaProtHub/DeepLoc_cls10_35M\")\n",
        "##@markdown\n",
        "##@markdown You can also find some officical adapters in [here](https://huggingface.co/SaProtHub)\n",
        "# base_model = \"westlake-repl/SaProt_35M_AF2\" #@param ['westlake-repl/SaProt_35M_AF2', 'westlake-repl/SaProt_650M_AF2'] {allow-input:true}\n",
        "use_model_from = \"Trained by yourself on ColabSaprot\" # @param [\"Trained by yourself on ColabSaprot\", \"Shared by peers on SaprotHub\", \"Saved in your local computer\", \"Multi-models on SaprotHub\"]\n",
        "if use_model_from == \"Multi-models on SaprotHub\":\n",
        "  multi_lora = True\n",
        "else:\n",
        "  multi_lora = False\n",
        "\n",
        "# use_existing_model = True # @param {type:\"boolean\"}\n",
        "# use_existing_model = True\n",
        "# if use_existing_model:\n",
        "#   adapter_combobox = select_adapter()\n",
        "\n",
        "adapter_input = select_adapter_from(task_type, use_model_from)\n",
        "#@markdown <br>\n",
        "\n",
        "################################################################################\n",
        "################################################################################\n",
        "################################################################################\n",
        "\n",
        "# # @markdown Please ensure that the selected task type aligns with the training task type of the model you intend to utilize.\n",
        "\n",
        "## @markdown If you are conducting inference on a classification task, please ensure that the `num_of_category` matches the number of categories in the training dataset. Otherwise, you do not need to assign `num_of_category`.\n",
        "\n",
        "\n",
        "##@markdown You have two options to provide your protein sequences:\n",
        "##@markdown - **Single Sequence: Enter a single SA sequence** into the input box, you can get a SA Sequence by clicking <a href=\"#get_SA_seq\">here</a>\n",
        "##@markdown - **Multiple Sequences: Select a dataset**, you can upload a dataset from <a href=\"#upload_dataset\">here</a>\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "# print(Fore.BLUE+f\"Data type: {data_type}\"+Style.RESET_ALL)\n",
        "\n",
        "\n",
        "################################################################################\n",
        "################################ DATASET #######################################\n",
        "################################################################################\n",
        "#@markdown # 3. Dataset\n",
        "data_type = \"Multiple PDB/CIF Structures\" # @param [\"Single AA Sequence\", \"Single SA Sequence\", \"Single UniProt ID\", \"Single PDB/CIF Structure\", \"Multiple AA Sequences\", \"Multiple SA Sequences\", \"Multiple UniProt IDs\", \"Multiple PDB/CIF Structures\", \"A pair of AA Sequences\", \"A pair of SA Sequences\", \"A pair of UniProt IDs\", \"A pair of PDB/CIF Structures\", \"Multiple pairs of AA Sequences\", \"Multiple pairs of SA Sequences\", \"Multiple pairs of UniProt IDs\", \"Multiple pairs of PDB/CIF Structures\"]\n",
        "\n",
        "mode = \"Multiple Sequences\" if (data_type in data_type_list[4:8] or data_type in data_type_list[13:17]) else \"Single Sequence\"\n",
        "\n",
        "raw_data = input_raw_data_by_data_type(data_type)\n",
        "\n",
        "# [\"Single AA Sequence\",\"Single SA Sequence\",\"Single UniProt ID\",\"Single PDB/CIF Structure\",\"Multiple AA Sequences\",\"Multiple SA Sequences\",\"Multiple UniProt IDs\",\"Multiple PDB/CIF Structures\",\"SaprotHub Dataset\",\"A pair of AA Sequences\",\"A pair of SA Sequences\",\"A pair of UniProt IDs\",\"A pair of PDB/CIF Structures\",\"Multiple pairs of AA Sequences\",\"Multiple pairs of SA Sequences\",\"Multiple pairs of UniProt IDs\",\"Multiple pairs of PDB/CIF Structures\"]\n",
        "if mode == \"Multiple Sequences\":\n",
        "  csv_dataset_path = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "else:\n",
        "  single_sa_seq = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "\n",
        "def apply(button):\n",
        "  global single_sa_seq\n",
        "  button.disabled = True\n",
        "  button.description = 'Predicting...'\n",
        "  button.button_style = ''\n",
        "\n",
        "  print()\n",
        "  print('='*100)\n",
        "  print(Fore.BLUE+f'Current Model ({use_model_from}): {adapter_input.value}'+Style.RESET_ALL)\n",
        "  if data_type in [\"Single AA Sequence\", \"Single SA Sequence\", \"Single UniProt ID\"]:\n",
        "    print(Fore.BLUE+f'Current Dataset ({data_type}): {raw_data.value}'+Style.RESET_ALL)\n",
        "  elif data_type == \"A pair of PDB/CIF Structures\":\n",
        "    print(Fore.BLUE+f'Current Dataset ({data_type}): Sequence 1: {raw_data[0]}, Sequence 2: {raw_data[1]}'+Style.RESET_ALL)\n",
        "  elif data_type in [\"A pair of AA Sequences\",\"A pair of SA Sequences\",\"A pair of UniProt IDs\"]:\n",
        "    print(Fore.BLUE+f'Current Dataset ({data_type}): Sequence 1: {raw_data[0].value}, Sequence 2: {raw_data[1].value}'+Style.RESET_ALL)\n",
        "  elif data_type == \"Single PDB/CIF Structure\":\n",
        "    print(Fore.BLUE+f'Current Dataset ({data_type}): {raw_data[0]}' +Style.RESET_ALL)\n",
        "  elif data_type in [\"Multiple AA Sequences\", \"Multiple SA Sequences\", \"Multiple UniProt IDs\", \"Multiple PDB/CIF Structures\"]:\n",
        "    print(Fore.BLUE+f'Current Dataset ({data_type}): {raw_data}'+Style.RESET_ALL)\n",
        "\n",
        "  predict()\n",
        "\n",
        "  # button_predict.on_click(predict)\n",
        "  # button_apply.layout.width = '500px'\n",
        "  # display(button_predict)\n",
        "\n",
        "button_apply = ipywidgets.Button(\n",
        "    description='Make Prediction',\n",
        "    disabled=False,\n",
        "    button_style='success', # 'success', 'info', 'warning', 'danger' or ''\n",
        "    tooltip='Apply',\n",
        "    # icon='check' # (FontAwesome names without the `fa-` prefix)\n",
        "    )\n",
        "button_apply.on_click(apply)\n",
        "# button_apply.layout.width = '500px'\n",
        "display(button_apply)\n",
        "\n",
        "\n",
        "# def apply(button):\n",
        "#     global single_sa_seq\n",
        "#     # button.disabled = True\n",
        "#     # button.description = 'Clicked'\n",
        "#     # button.button_style = ''\n",
        "\n",
        "#     # print(Fore.BLUE+'Construct dataset...'+Style.RESET_ALL)\n",
        "#     if mode == \"Multiple Sequences\":\n",
        "#       raw_data = input_raw_data_by_data_type(data_type)\n",
        "#       csv_dataset_path = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "\n",
        "#     elif mode == \"Single Sequence\":\n",
        "#       single_sa_seq = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "\n",
        "\n",
        "#     print()\n",
        "#     print('='*100)\n",
        "#     print(Fore.BLUE+f'Current Model ({use_model_from}): {adapter_input.value}'+Style.RESET_ALL)\n",
        "#     print(Fore.BLUE+f'Current Dataset ({data_type}): {raw_data.value}'+Style.RESET_ALL)\n",
        "\n",
        "# button_apply = ipywidgets.Button(\n",
        "#     description='Apply',\n",
        "#     disabled=False,\n",
        "#     button_style='success', # 'success', 'info', 'warning', 'danger' or ''\n",
        "#     tooltip='Apply',\n",
        "#     icon='check' # (FontAwesome names without the `fa-` prefix)\n",
        "#     )\n",
        "# button_apply.on_click(apply)\n",
        "# # button_apply.layout.width = '500px'\n",
        "# if\n",
        "# display(button_apply)\n",
        "\n",
        "\n",
        "##@markdown <br>\n",
        "\n",
        "##@markdown  <font color=\"red\"> **Note that:** </font> If `use_model_from` is set to `Multi-models on SaprotHub`, each sample will be predicted using multiple models. For classification tasks, voting will be used to determine the final predicted category; for regression tasks, the predicted values from each model will be averaged.\n",
        "\n",
        "################################################################################\n",
        "################################################################################\n",
        "################################################################################\n",
        "from transformers import EsmTokenizer\n",
        "import torch\n",
        "import copy\n",
        "import sys\n",
        "from saprot.scripts.training import my_load_model\n",
        "\n",
        "def get_base_model(adapter_path):\n",
        "  adapter_config = Path(adapter_path) / \"adapter_config.json\"\n",
        "  with open(adapter_config, 'r') as f:\n",
        "    adapter_config_dict = json.load(f)\n",
        "    base_model = adapter_config_dict['base_model_name_or_path']\n",
        "    if 'SaProt_650M_AF2' in base_model:\n",
        "      base_model = \"westlake-repl/SaProt_650M_AF2\"\n",
        "    elif 'SaProt_35M_AF2' in base_model:\n",
        "      base_model = \"westlake-repl/SaProt_35M_AF2\"\n",
        "    else:\n",
        "      raise RuntimeError(\"Please ensure the base model is \\\"SaProt_650M_AF2\\\" or \\\"SaProt_35M_AF2\\\"\")\n",
        "  return base_model\n",
        "\n",
        "def check_training_data_type(adapter_path, data_type):\n",
        "  metadata_path = Path(adapter_path) / \"metadata.json\"\n",
        "  if metadata_path.exists():\n",
        "    with open(metadata_path, 'r') as f:\n",
        "      metadata = json.load(f)\n",
        "      required_training_data_type = metadata['training_data_type']\n",
        "  else:\n",
        "    required_training_data_type = \"SA\"\n",
        "  assert required_training_data_type == training_data_type_dict[data_type], f\"This model ({adapter_path}) is trained on {required_training_data_type} sequences. Please ensure your data type is also {required_training_data_type} sequences for accurate predictions.\"\n",
        "\n",
        "# def predict(button):\n",
        "#   button.disabled = True\n",
        "#   button.description = 'Predicting'\n",
        "#   button.button_style = ''\n",
        "def predict():\n",
        "  # base_model = \"westlake-repl/SaProt_35M_AF2\"\n",
        "  if multi_lora:\n",
        "    if use_model_from == \"Multi-models on ColabSaprot\":\n",
        "      config_list = [EasyDict({'lora_config_path': ADAPTER_HOME / task_type / lora_config_path}) for lora_config_path in list(adapter_input.value)]\n",
        "    elif use_model_from == \"Multi-models on SaprotHub\":\n",
        "      #1. get adapter_list\n",
        "      repo_id_list = adapter_input.value.replace(\" \", \"\").split(',')\n",
        "      #2. download adapters\n",
        "      for repo_id in repo_id_list:\n",
        "        snapshot_download(repo_id=repo_id, repo_type=\"model\", local_dir=ADAPTER_HOME / task_type / repo_id)\n",
        "      config_list = [EasyDict({'lora_config_path': ADAPTER_HOME / task_type / repo_id}) for repo_id in repo_id_list]\n",
        "\n",
        "    assert len(config_list) > 0, \"Please select your models from the dropdown menu on the output of 3.1.1!\"\n",
        "    base_model = get_base_model(ADAPTER_HOME / task_type / config_list[0].lora_config_path)\n",
        "\n",
        "    for lora_config in config_list:\n",
        "      check_training_data_type(lora_config.lora_config_path, data_type)\n",
        "\n",
        "    lora_kwargs = EasyDict({\n",
        "      \"num_lora\": len(config_list),\n",
        "      \"config_list\": config_list\n",
        "    })\n",
        "\n",
        "\n",
        "  else:\n",
        "    if use_model_from == \"Shared by peers on SaprotHub\":\n",
        "      snapshot_download(repo_id=adapter_input.value, repo_type=\"model\", local_dir=ADAPTER_HOME / task_type / adapter_input.value)\n",
        "\n",
        "    adapter_path = ADAPTER_HOME / task_type / adapter_input.value\n",
        "    base_model = get_base_model(adapter_path)\n",
        "    check_training_data_type(adapter_path, data_type)\n",
        "    lora_kwargs = {\n",
        "        \"num_lora\": 1,\n",
        "        \"config_list\": [{\"lora_config_path\": adapter_path}]\n",
        "    }\n",
        "\n",
        "\n",
        "\n",
        "  ################################################################################\n",
        "  ##################################### config ###################################\n",
        "  ################################################################################\n",
        "  from saprot.config.config_dict import Default_config\n",
        "  config = copy.deepcopy(Default_config)\n",
        "\n",
        "  # task\n",
        "  if task_type in [ \"classification\", \"token_classification\"]:\n",
        "    # config.model.kwargs.num_labels = num_of_categories.value\n",
        "    config.model.kwargs.num_labels = num_of_categories.value\n",
        "\n",
        "  # base model\n",
        "  config.model.model_py_path = model_type_dict[task_type]\n",
        "  # config.model.save_path = model_save_path\n",
        "  config.model.kwargs.config_path = base_model\n",
        "\n",
        "  # lora\n",
        "  config.model.kwargs.lora_kwargs = lora_kwargs\n",
        "\n",
        "  ################################################################################\n",
        "  ################################### inference ##################################\n",
        "  ################################################################################\n",
        "  from peft import PeftModelForSequenceClassification\n",
        "\n",
        "  model = my_load_model(config.model)\n",
        "  tokenizer = EsmTokenizer.from_pretrained(config.model.kwargs.config_path)\n",
        "  device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "  model.to(device)\n",
        "\n",
        "  clear_output(wait=True)\n",
        "\n",
        "  print(Fore.BLUE+f\"Inference task type: {task_type}\"+Style.RESET_ALL)\n",
        "  if mode == \"Multiple Sequences\":\n",
        "    print(Fore.BLUE+f\"Dataset: {csv_dataset_path}\"+Style.RESET_ALL)\n",
        "  else:\n",
        "    if data_type == \"Single PDB/CIF Structure\":\n",
        "      print(Fore.BLUE+f'Dataset ({data_type}): {raw_data[0]}' +Style.RESET_ALL)\n",
        "    else:\n",
        "      print(Fore.BLUE+f\"Dataset: {raw_data.value}\"+Style.RESET_ALL)\n",
        "\n",
        "  if multi_lora:\n",
        "    print(Fore.BLUE+f\"Model: {base_model} - {[str(lora_config.lora_config_path) for lora_config in lora_kwargs.config_list]}\"+Style.RESET_ALL)\n",
        "  else:\n",
        "    print(Fore.BLUE+f\"Model: {base_model} - {adapter_path}\"+Style.RESET_ALL)\n",
        "\n",
        "  outputs_list=[]\n",
        "\n",
        "  timestamp = str(datetime.now().strftime(\"%Y%m%d%H%M%S\"))\n",
        "  output_file = OUTPUT_HOME / f'output_{timestamp}.csv'\n",
        "\n",
        "  if mode == \"Single Sequence\":\n",
        "    if task_type in [\"pair_classification\", \"pair_regression\"]:\n",
        "      df = pd.DataFrame({\n",
        "          'seq_1': [single_sa_seq[0]],\n",
        "          'seq_2': [single_sa_seq[1]]\n",
        "      })\n",
        "    else:\n",
        "      df = pd.DataFrame({\n",
        "          'sequence': [single_sa_seq]\n",
        "      })\n",
        "  elif mode == \"Multiple Sequences\":\n",
        "    df = pd.read_csv(csv_dataset_path)\n",
        "\n",
        "  print()\n",
        "  print('='*100)\n",
        "  print(Fore.BLUE+\"Prediction Result:\"+Style.RESET_ALL)\n",
        "\n",
        "  if task_type in [\"pair_classification\", \"pair_regression\"]:\n",
        "    for index, row in tqdm(df.iterrows(), total=df.shape[0]):\n",
        "      input_1 = tokenizer(row[\"seq_1\"], return_tensors=\"pt\")\n",
        "      input_1 = {k: v.to(device) for k, v in input_1.items()}\n",
        "      input_2 = tokenizer(row[\"seq_2\"], return_tensors=\"pt\")\n",
        "      input_2 = {k: v.to(device) for k, v in input_2.items()}\n",
        "\n",
        "      with torch.no_grad(): outputs = model(input_1, input_2)\n",
        "      outputs_list.append(outputs)\n",
        "  else:\n",
        "    for index in tqdm(range(len(df))):\n",
        "      seq = df['sequence'].iloc[index]\n",
        "      inputs = tokenizer(seq, return_tensors=\"pt\")\n",
        "      inputs = {k: v.to(device) for k, v in inputs.items()}\n",
        "      with torch.no_grad(): outputs = model(inputs)\n",
        "      outputs_list.append(outputs)\n",
        "\n",
        "\n",
        "    if task_type == \"pair_classification\":\n",
        "      softmax_output_list = [F.softmax(output, dim=1).squeeze().tolist() for output in outputs_list]\n",
        "      print()\n",
        "      for index, output in enumerate(softmax_output_list):\n",
        "        print(f\"For Sequence {index}, Category {output.index(max(output))}, Probability: {output}\")\n",
        "        df.loc[index, 'result'] = output.index(max(output))\n",
        "        df.loc[index, 'probability'] = ', '.join(map(str, output))\n",
        "      df.to_csv(output_file, index=False)\n",
        "\n",
        "    elif task_type == \"pair_regression\":\n",
        "      print()\n",
        "      for index, output in enumerate(outputs_list):\n",
        "        print(f\"For Sequence {index}, Value {output.cpu().item()}\")\n",
        "      df['score'] = [output.cpu().item() for output in outputs_list]\n",
        "      df.to_csv(output_file, index=False)\n",
        "\n",
        "    elif task_type == \"classification\":\n",
        "      print()\n",
        "      softmax_output_list = [F.softmax(output, dim=1).squeeze().tolist() for output in outputs_list]\n",
        "      for index, output in enumerate(softmax_output_list):\n",
        "        print(f\"For Sequence {index}, Category {output.index(max(output))}, Probability: {output}\")\n",
        "        df.loc[index, 'result'] = output.index(max(output))\n",
        "        df.loc[index, 'probability'] = ', '.join(map(str, output))\n",
        "      df.to_csv(output_file, index=False)\n",
        "\n",
        "    elif task_type == \"regression\":\n",
        "      print()\n",
        "      for index, output in enumerate(outputs_list):\n",
        "        print(f\"For Sequence {index}, Value {output.item()}\")\n",
        "      df['score'] = [output.cpu().item() for output in outputs_list]\n",
        "      df.to_csv(output_file, index=False)\n",
        "\n",
        "    elif task_type == \"token_classification\":\n",
        "      seq_prob_df_list = []\n",
        "      softmax_output_list = [F.softmax(output, dim=-1).squeeze().tolist() for output in outputs_list]\n",
        "      print(\"The probability of each category:\")\n",
        "      for seq_index, seq in enumerate(softmax_output_list):\n",
        "        seq_prob_df = pd.DataFrame(seq)[1:-1]\n",
        "        print('='*100)\n",
        "        print(f'Sequence {seq_index + 1}:')\n",
        "        print(seq_prob_df.to_string())\n",
        "        seq_prob_df['seq_index'] = seq_index\n",
        "        seq_prob_df['aa_index'] = seq_prob_df.index\n",
        "        seq_prob_df['sequence'] = df.loc[seq_index, 'sequence']\n",
        "        seq_prob_df_list.append(seq_prob_df)\n",
        "      combined_df = pd.concat(seq_prob_df_list, ignore_index=False)\n",
        "      combined_df.to_csv(output_file, index=True)\n",
        "\n",
        "    print()\n",
        "    print('='*100)\n",
        "    print(Fore.BLUE+f\"The prediction result is saved to {output_file} and your local computer.\"+Style.RESET_ALL)\n",
        "    file_download(output_file)\n",
        "\n",
        "# button_predict = ipywidgets.Button(\n",
        "#     description='Make Prediction',\n",
        "#     disabled=False,\n",
        "#     button_style='success', # 'success', 'info', 'warning', 'danger' or ''\n",
        "#     tooltip='Apply',\n",
        "#     icon='check' # (FontAwesome names without the `fa-` prefix)\n",
        "#     )\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "uxD_KOF1BI7n"
      },
      "outputs": [],
      "source": [
        "#@title **3.2: Mutational Effect Prediction** <a name=\"mutational_effect_prediction\"></a>\n",
        "#@markdown <font color=\"red\">Our model is pre-trained based on protein structures and performs best when provided with structural data.\n",
        "#@markdown **Therefore, if you only have the amino acid sequence, we strongly recommend using AF2 to predict the structure and generate a PDB file before proceeding with mutation prediction.**\n",
        "\n",
        "#@markdown <a href=\"#get_sa\">Here</a> you can **convert your data into SA Sequence** format.\n",
        "#@markdown > üìçPlease see the [tutorial](https://github.com/westlake-repl/SaprotHub/wiki/3.2:-Mutational-Effect-Prediction)\n",
        "\n",
        "\n",
        "mutation_task = \"Saturation mutagenesis\" #@param [\"Single-site or Multi-site mutagenesis\", \"Saturation mutagenesis\"]\n",
        "\n",
        "# data_type = \"Single AA Sequence\" # @param [\"Single AA Sequence\", \"Single SA Sequence\", \"Single UniProt ID\", \"Single PDB/CIF Structure\", \"Multiple AA Sequences\", \"Multiple SA Sequences\", \"Multiple UniProt IDs\", \"Multiple PDB/CIF Structures\"]\n",
        "data_type = \"Single SA Sequence\" # @param [\"Single SA Sequence\", \"Single UniProt ID\", \"Single PDB/CIF Structure\", \"Multiple SA Sequences\", \"Multiple UniProt IDs\", \"Multiple PDB/CIF Structures\"]\n",
        "raw_data = input_raw_data_by_data_type(data_type)\n",
        "\n",
        "mode = \"Multiple Sequences\" if data_type in data_type_list[4:8] else \"Single Sequence\"\n",
        "\n",
        "if mutation_task == \"Single-site or Multi-site mutagenesis\":\n",
        "  if mode == \"Single Sequence\":\n",
        "    input_mut = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder='Enter Single Mutation Information here',\n",
        "      # description='SA Sequence:',\n",
        "      disabled=False)\n",
        "    print(Fore.BLUE+\"Mutation:\"+Style.RESET_ALL)\n",
        "    input_mut.layout.width = '500px'\n",
        "    display(input_mut)\n",
        "\n",
        "def mutational_effect_predict(button):\n",
        "  button.disabled = True\n",
        "  button.description = 'Clicked'\n",
        "  button.button_style = ''\n",
        "\n",
        "\n",
        "  #@title 3.2.2: Get your Result\n",
        "\n",
        "  ################################################################################\n",
        "  ################################# DATASET ###################################\n",
        "  ################################################################################\n",
        "  if mode == \"Single Sequence\":\n",
        "    seq = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "  else:\n",
        "    dataset_csv_path = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "\n",
        "  ################################################################################\n",
        "  ################################# Task Info ####################################\n",
        "  ################################################################################\n",
        "  base_model = \"westlake-repl/SaProt_650M_AF2\"\n",
        "\n",
        "  # clear_output(wait=True)\n",
        "\n",
        "  print(Fore.BLUE)\n",
        "  print(f\"Mutation task: {mutation_task}\")\n",
        "  print(f\"Mode: {mode}\")\n",
        "  print(f\"Model: {base_model}\")\n",
        "  if mode == \"Multiple Sequences\":\n",
        "    print(Fore.BLUE+f\"Dataset: {dataset_csv_path}\"+Style.RESET_ALL)\n",
        "  else:\n",
        "    print(Fore.BLUE+f\"Dataset: {seq}\"+Style.RESET_ALL)\n",
        "\n",
        "  print(Style.RESET_ALL)\n",
        "\n",
        "  print(f\"Predicting...\")\n",
        "  timestamp = datetime.now().strftime(\"%y%m%d%H%M%S\")\n",
        "\n",
        "  ################################################################################\n",
        "  ################################# load model ###################################\n",
        "  ################################################################################\n",
        "\n",
        "  from saprot.model.saprot.saprot_foldseek_mutation_model import SaprotFoldseekMutationModel\n",
        "\n",
        "  config = {\n",
        "      \"foldseek_path\": None,\n",
        "      \"config_path\": base_model,\n",
        "      \"load_pretrained\": True,\n",
        "  }\n",
        "\n",
        "  try:\n",
        "    zero_shot_model\n",
        "  except Exception:\n",
        "    zero_shot_model = SaprotFoldseekMutationModel(**config)\n",
        "    device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "    zero_shot_model.to(device)\n",
        "\n",
        "  ################################################################################\n",
        "  ########################### Single Sequence ####################################\n",
        "  ################################################################################\n",
        "  if mode == \"Single Sequence\":\n",
        "\n",
        "    if mutation_task == \"Single-site or Multi-site mutagenesis\":\n",
        "      mut = input_mut.value\n",
        "      # validate mut\n",
        "      aa_seq = seq[0::2]\n",
        "      for m in mut.split(':'):\n",
        "        ori_aa = m[0]\n",
        "        pos = int(m[1:-1])\n",
        "        mut_aa = m[-1]\n",
        "        assert aa_seq[pos-1] == ori_aa, f\"The provided mutation information contains an error ({m}): the original amino acid at position {pos} ({ori_aa}) does not match your sequence ({aa_seq[pos-1]}).\"\n",
        "\n",
        "      score = zero_shot_model.predict_mut(seq, mut)\n",
        "\n",
        "      print()\n",
        "      print(\"=\"*100)\n",
        "      print(Fore.BLUE+\"Output:\"+Style.RESET_ALL)\n",
        "      print(f\"The score of mutation {mut} is {Fore.BLUE}{score}{Style.RESET_ALL}\")\n",
        "\n",
        "    if mutation_task==\"Saturation mutagenesis\":\n",
        "      timestamp = datetime.now().strftime(\"%y%m%d%H%M%S\")\n",
        "      output_path = OUTPUT_HOME / f'{timestamp}_prediction_output.csv'\n",
        "\n",
        "      mut_dicts = []\n",
        "      for pos in tqdm(range(1, int(len(seq) / 2)+1), total=int(len(seq) / 2)+1, leave=False, desc=f\"Predicting\"):\n",
        "        mut_dict = zero_shot_model.predict_pos_mut(seq, pos)\n",
        "        mut_dicts.append(mut_dict)\n",
        "\n",
        "      mut_list = [{'mutation': key, 'score': value} for d in mut_dicts for key, value in d.items()]\n",
        "      df = pd.DataFrame(mut_list)\n",
        "      df.to_csv(output_path, index=None)\n",
        "\n",
        "      print()\n",
        "      print(\"=\"*100)\n",
        "      print(Fore.BLUE+\"Output:\"+Style.RESET_ALL)\n",
        "      # files.download(output_path)\n",
        "      file_download(output_path)\n",
        "      print(f\"\\n{Fore.BLUE}The result has been saved to {output_path} and your local computer.{Style.RESET_ALL}\")\n",
        "\n",
        "  ################################################################################\n",
        "  ########################### Multiple Sequences #################################\n",
        "  ################################################################################\n",
        "  if mode == \"Multiple Sequences\":\n",
        "\n",
        "    dataset_df = pd.read_csv(dataset_csv_path)\n",
        "    results = []\n",
        "\n",
        "    if mutation_task==\"Single-site or Multi-site mutagenesis\":\n",
        "      for index, row in tqdm(dataset_df.iterrows(), total=len(dataset_df), leave=False, desc=f\"Predicting\"):\n",
        "       seq = row['sequence']\n",
        "       mut_info = row['mutation']\n",
        "       results.append(zero_shot_model.predict_mut(seq, mut_info).cpu().item())\n",
        "\n",
        "      print()\n",
        "      print(\"=\"*100)\n",
        "      print(Fore.BLUE+\"Output:\"+Style.RESET_ALL)\n",
        "\n",
        "      # result_df = pd.DataFrame()\n",
        "      # result_df['sequence'] = dataset_df['sequence']\n",
        "      # result_df['mutation'] = dataset_df['mutation']\n",
        "      dataset_df['score'] = results\n",
        "\n",
        "      output_path = OUTPUT_HOME / f\"{timestamp}_prediction_output_{Path(dataset_csv_path).stem}.csv\"\n",
        "      dataset_df.to_csv(output_path, index=None)\n",
        "      file_download(output_path)\n",
        "      print(f\"{Fore.BLUE}The result has been saved to {output_path} and your local computer {Style.RESET_ALL}\")\n",
        "\n",
        "    else:\n",
        "      for index, row in tqdm(dataset_df.iterrows(), total=len(dataset_df), leave=False, desc=f\"Predicting\"):\n",
        "        seq = row['sequence']\n",
        "        mut_dicts = []\n",
        "        for pos in range(1, int(len(seq) / 2)+1):\n",
        "          mut_dict = zero_shot_model.predict_pos_mut(seq, pos)\n",
        "          mut_dicts.append(mut_dict)\n",
        "        mut_list = [{'mutation': key, 'score': value} for d in mut_dicts for key, value in d.items()]\n",
        "        result_df = pd.DataFrame(mut_list)\n",
        "        results.append(result_df)\n",
        "\n",
        "      print()\n",
        "      print(\"=\"*100)\n",
        "      print(Fore.BLUE+\"Output:\"+Style.RESET_ALL)\n",
        "\n",
        "      zip_files = []\n",
        "      for i in range(len(results)):\n",
        "        output_path = OUTPUT_HOME / f\"{timestamp}_prediction_output_{Path(dataset_csv_path).stem}_Sequence{i+1}.csv\"\n",
        "        results[i].to_csv(output_path, index=None)\n",
        "        zip_files.append(output_path)\n",
        "\n",
        "      # zip and download zip to local computer\n",
        "      zip_path = OUTPUT_HOME / f\"{timestamp}_{Path(dataset_csv_path).stem}.zip\"\n",
        "      with zipfile.ZipFile(zip_path, 'w') as zipf:\n",
        "          for file in zip_files:\n",
        "              zipf.write(file, os.path.basename(file))\n",
        "      # files.download(zip_path)\n",
        "      print(f\"{Fore.BLUE}The result has been saved to {zip_path} and your local computer{Style.RESET_ALL}\")\n",
        "      file_download(zip_path)\n",
        "\n",
        "button_mutational_effect_predict = ipywidgets.Button(\n",
        "  description='Mutational Effect Predict',\n",
        "  disabled=False,\n",
        "  button_style='success', # 'success', 'info', 'warning', 'danger' or ''\n",
        "  tooltip='Apply',\n",
        "  # icon='check' # (FontAwesome names without the `fa-` prefix)\n",
        "  )\n",
        "button_mutational_effect_predict.on_click(mutational_effect_predict)\n",
        "button_mutational_effect_predict.layout.width = '300px'\n",
        "display(button_mutational_effect_predict)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "uAlQdqTcBI7n"
      },
      "source": [
        "## **3.3: Inverse Folding Prediction** <a name=\"inverse_folding_prediction\"></a>\n",
        "\n",
        "<!-- Predict the amino acid sequence from protein backbone structure.\n",
        "\n",
        "<br>\n",
        "\n",
        "### Dataset\n",
        "\n",
        "The protein backbone structure should be provided in .pdb/.cif file format. -->\n",
        "\n",
        "> üìçPlease see the [tutorial](https://github.com/westlake-repl/SaprotHub/wiki/3.3:-Inverse-Folding-Prediction)\n",
        "\n",
        "\n",
        "<!-- Predict the residue sequence of a structure-aware sequence with masked amino acids (which could be all masked or partially masked).\n",
        "\n",
        "<br>\n",
        "\n",
        "### Dataset\n",
        "\n",
        "Enter a **SA sequence with masked amino acids** into the `sa_seq` input box.\n",
        "\n",
        "<br>\n",
        "\n",
        "For example,\n",
        "**input** is a SA Sequence with masked amino acids:\n",
        "\n",
        "`#d#v#v#v#p#p#p#p#a#p#a#q#k#k#k#k#w`\n",
        "\n",
        "and the **output** predicted by model is an AA Sequence:\n",
        "\n",
        "`MEELGLPDLPPGGVVVV`.\n",
        "\n",
        "<br> -->\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "DT7M_DU2BI7n",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 615,
          "referenced_widgets": [
            "43bcba7abdfe423abdef9ec4925216a2",
            "f469cbc8326c4d74882de4a1d860854b",
            "682d12c78c9f4b79a104d3cf9edaf7ec",
            "14ba4e672efd48b2812f1adaef897c4f",
            "7bbeedb8def74786b81633f1d6428f97",
            "a98ba678e25b4fb186a589d416c238b3",
            "703318884d354d5f8fb8fec1de6eccb8",
            "a23903fc8c2d4ed88e77c1f3a41943f7",
            "558c855445594b8485b9fba5c1af3a84",
            "8952766795894d0a98b19f7cc1097701",
            "2fc9c110abd24ed1991bcf1d6bfd0ba5",
            "a81da4401d1243ae9d342b19416439a4"
          ]
        },
        "outputId": "6c38957b-8566-4a3f-fe0f-0d47d7bd8e10"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[34mAmino Acid Sequence:\u001b[0m\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Text(value='ARIRHVQGDITEFQGDAIVNAANNYLKLGAGVAGAILRKGGPSIQEECDRIGKIRVGEAAVTGAGNLPVRYVIHAAVLGDEPASLETVRKATKSALEK‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "558c855445594b8485b9fba5c1af3a84"
            }
          },
          "metadata": {
            "application/vnd.jupyter.widget-view+json": {
              "colab": {
                "custom_widget_manager": {
                  "url": "https://ssl.gstatic.com/colaboratory-static/widgets/colab-cdn-widget-manager/2b70e893a8ba7c0f/manager.min.js"
                }
              }
            }
          }
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[34mStructure Sequence:\u001b[0m\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Text(value='daeaedadqqlqdqfaeeeaqaaqqrdqdddvsvvqcvqqhdvsvvvsvvvggddqlawdwdqshngnhgiyiyqhqhhphagdlvsllssllnslvv‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "a81da4401d1243ae9d342b19416439a4"
            }
          },
          "metadata": {
            "application/vnd.jupyter.widget-view+json": {
              "colab": {
                "custom_widget_manager": {
                  "url": "https://ssl.gstatic.com/colaboratory-static/widgets/colab-cdn-widget-manager/2b70e893a8ba7c0f/manager.min.js"
                }
              }
            }
          }
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_17228382996107566\"  style=\"position: relative; width: 640px; height: 480px;\">\n        <p id=\"3dmolwarning_17228382996107566\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_17228382996107566 = null;\nvar warn = document.getElementById(\"3dmolwarning_17228382996107566\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_17228382996107566 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17228382996107566\"),{backgroundColor:\"white\"});\nviewer_17228382996107566.zoomTo();\n\tviewer_17228382996107566.addModel(\"HEADER    DE NOVO PROTEIN                         24-APR-12   4ETJ              \\nTITLE     CRYSTAL STRUCTURE OF E6H VARIANT OF DE NOVO DESIGNED SERINE HYDROLASE \\nTITLE    2 OSH55, NORTHEAST STRUCTURAL GENOMICS CONSORTIUM (NESG) TARGET OR185  \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: SERINE HYDROLASE;                                          \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES;                                                     \\nCOMPND   5 MUTATION: YES                                                        \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: ARTIFICIAL GENE;                                \\nSOURCE   3 ORGANISM_TAXID: 32630                                                \\nKEYWDS    STRUCTURAL GENOMICS, PSI-BIOLOGY, PROTEIN STRUCTURE INITIATIVE,       \\nKEYWDS   2 NORTHEAST STRUCTURAL GENOMICS CONSORTIUM, NESG, DE NOVO PROTEIN      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    A.KUZIN,M.SU,J.SEETHARAMAN,K.KORNHABER,G.KORNHABER,S.RAJAGOPALAN,     \\nAUTHOR   2 D.BAKER,J.K.EVERETT,T.B.ACTON,G.T.MONTELIONE,L.TONG,J.F.HUNT,        \\nAUTHOR   3 NORTHEAST STRUCTURAL GENOMICS CONSORTIUM (NESG)                      \\nREVDAT   4   13-SEP-23 4ETJ    1       REMARK HETSYN                            \\nREVDAT   3   14-MAY-14 4ETJ    1       JRNL                                     \\nREVDAT   2   16-APR-14 4ETJ    1       JRNL                                     \\nREVDAT   1   13-JUN-12 4ETJ    0                                                \\nJRNL        AUTH   S.RAJAGOPALAN,C.WANG,K.YU,A.P.KUZIN,F.RICHTER,S.LEW,         \\nJRNL        AUTH 2 A.E.MIKLOS,M.L.MATTHEWS,J.SEETHARAMAN,M.SU,J.F.HUNT,         \\nJRNL        AUTH 3 B.F.CRAVATT,D.BAKER                                          \\nJRNL        TITL   DESIGN OF ACTIVATED SERINE-CONTAINING CATALYTIC TRIADS WITH  \\nJRNL        TITL 2 ATOMIC-LEVEL ACCURACY.                                       \\nJRNL        REF    NAT.CHEM.BIOL.                V.  10   386 2014              \\nJRNL        REFN                   ISSN 1552-4450                               \\nJRNL        PMID   24705591                                                     \\nJRNL        DOI    10.1038/NCHEMBIO.1498                                        \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.20 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PHENIX DEV_988                                       \\nREMARK   3   AUTHORS     : PAUL ADAMS,PAVEL AFONINE,VINCENT CHEN,IAN            \\nREMARK   3               : DAVIS,KRESHNA GOPAL,RALF GROSSE-KUNSTLEVE,           \\nREMARK   3               : LI-WEI HUNG,ROBERT IMMORMINO,TOM IOERGER,            \\nREMARK   3               : AIRLIE MCCOY,ERIK MCKEE,NIGEL MORIARTY,              \\nREMARK   3               : REETAL PAI,RANDY READ,JANE RICHARDSON,               \\nREMARK   3               : DAVID RICHARDSON,TOD ROMO,JIM SACCHETTINI,           \\nREMARK   3               : NICHOLAS SAUTER,JACOB SMITH,LAURENT                  \\nREMARK   3               : STORONI,TOM TERWILLIGER,PETER ZWART                  \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : ML                                            \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.20                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 36.96                          \\nREMARK   3   MIN(FOBS/SIGMA_FOBS)              : 1.340                          \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 99.9                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 10470                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.185                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.183                           \\nREMARK   3   FREE R VALUE                     : 0.226                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 4.790                           \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 502                             \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT (IN BINS).                           \\nREMARK   3   BIN  RESOLUTION RANGE  COMPL.    NWORK NFREE   RWORK  RFREE        \\nREMARK   3     1 36.9636 -  3.4958    1.00     2633   122  0.1813 0.2065        \\nREMARK   3     2  3.4958 -  2.7750    1.00     2472   126  0.1753 0.2493        \\nREMARK   3     3  2.7750 -  2.4243    1.00     2437   134  0.1943 0.2220        \\nREMARK   3     4  2.4243 -  2.2027    1.00     2426   120  0.1952 0.2595        \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED        : FLAT BULK SOLVENT MODEL                       \\nREMARK   3   SOLVENT RADIUS     : 1.00                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.73                                          \\nREMARK   3   K_SOL              : 0.38                                          \\nREMARK   3   B_SOL              : 49.03                                         \\nREMARK   3                                                                      \\nREMARK   3  ERROR ESTIMATES.                                                    \\nREMARK   3   COORDINATE ERROR (MAXIMUM-LIKELIHOOD BASED)     : 0.210            \\nREMARK   3   PHASE ERROR (DEGREES, MAXIMUM-LIKELIHOOD BASED) : 18.790           \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : 36.45                          \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : 1.42180                                              \\nREMARK   3    B22 (A**2) : 1.42180                                              \\nREMARK   3    B33 (A**2) : -2.84360                                             \\nREMARK   3    B12 (A**2) : 0.00000                                              \\nREMARK   3    B13 (A**2) : 0.00000                                              \\nREMARK   3    B23 (A**2) : 0.00000                                              \\nREMARK   3                                                                      \\nREMARK   3  TWINNING INFORMATION.                                               \\nREMARK   3   FRACTION: NULL                                                     \\nREMARK   3   OPERATOR: NULL                                                     \\nREMARK   3                                                                      \\nREMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       \\nREMARK   3                 RMSD          COUNT                                  \\nREMARK   3   BOND      :  0.008           1273                                  \\nREMARK   3   ANGLE     :  1.085           1704                                  \\nREMARK   3   CHIRALITY :  0.070            196                                  \\nREMARK   3   PLANARITY :  0.005            217                                  \\nREMARK   3   DIHEDRAL  : 17.508            485                                  \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : 1                                          \\nREMARK   3   TLS GROUP : 1                                                      \\nREMARK   3    SELECTION: ALL                                                    \\nREMARK   3    ORIGIN FOR THE GROUP (A):   5.6904  13.5459  14.1463              \\nREMARK   3    T TENSOR                                                          \\nREMARK   3      T11:   0.1942 T22:   0.2940                                     \\nREMARK   3      T33:   0.2653 T12:   0.0050                                     \\nREMARK   3      T13:   0.0100 T23:   0.0623                                     \\nREMARK   3    L TENSOR                                                          \\nREMARK   3      L11:   2.4074 L22:   3.1653                                     \\nREMARK   3      L33:   4.5167 L12:   1.0594                                     \\nREMARK   3      L13:   0.0247 L23:   0.4689                                     \\nREMARK   3    S TENSOR                                                          \\nREMARK   3      S11:  -0.0585 S12:   0.0363 S13:   0.1890                       \\nREMARK   3      S21:  -0.1093 S22:   0.0609 S23:   0.0638                       \\nREMARK   3      S31:  -0.0745 S32:   0.3546 S33:  -0.0012                       \\nREMARK   3                                                                      \\nREMARK   3  NCS DETAILS                                                         \\nREMARK   3   NUMBER OF NCS GROUPS : NULL                                        \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 4ETJ COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 27-APR-12.                  \\nREMARK 100 THE DEPOSITION ID IS D_1000072050.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 21-MAR-12                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : 7                                  \\nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \\nREMARK 200  RADIATION SOURCE               : NSLS                               \\nREMARK 200  BEAMLINE                       : X4C                                \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.97925                            \\nREMARK 200  MONOCHROMATOR                  : SI 111 CHANNEL                     \\nREMARK 200  OPTICS                         : MIRROR                             \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : MAR CCD 165 MM                     \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 19058                              \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.200                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 50.000                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : -3.000                             \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 100.0                              \\nREMARK 200  DATA REDUNDANCY                : 14.90                              \\nREMARK 200  R MERGE                    (I) : 0.09000                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 57.6000                            \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MIR                          \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: 3V45                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 55.15                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.74                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: PROTEIN SOLUTION: 100MM NACL, 5MM DTT,   \\nREMARK 280  0.02% NAN3, 10MM TRIS-HCL (PH 7.5), RESERVOIR SOLUTION: 0.1M        \\nREMARK 280  MGCL2, 0.1M BIS-TRIS, 80% PEG400, MICROBATCH UNDER OIL,             \\nREMARK 280  TEMPERATURE 293KK                                                   \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 43 21 2                        \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X,-Y,Z+1/2                                             \\nREMARK 290       3555   -Y+1/2,X+1/2,Z+3/4                                      \\nREMARK 290       4555   Y+1/2,-X+1/2,Z+1/4                                      \\nREMARK 290       5555   -X+1/2,Y+1/2,-Z+3/4                                     \\nREMARK 290       6555   X+1/2,-Y+1/2,-Z+1/4                                     \\nREMARK 290       7555   Y,X,-Z                                                  \\nREMARK 290       8555   -Y,-X,-Z+1/2                                            \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       44.63000            \\nREMARK 290   SMTRY1   3  0.000000 -1.000000  0.000000       32.96500            \\nREMARK 290   SMTRY2   3  1.000000  0.000000  0.000000       32.96500            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       66.94500            \\nREMARK 290   SMTRY1   4  0.000000  1.000000  0.000000       32.96500            \\nREMARK 290   SMTRY2   4 -1.000000  0.000000  0.000000       32.96500            \\nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       22.31500            \\nREMARK 290   SMTRY1   5 -1.000000  0.000000  0.000000       32.96500            \\nREMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       32.96500            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       66.94500            \\nREMARK 290   SMTRY1   6  1.000000  0.000000  0.000000       32.96500            \\nREMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       32.96500            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000       22.31500            \\nREMARK 290   SMTRY1   7  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   8  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000       44.63000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 300 REMARK: MONOMER,17.05 KD,95.7%, DIMER,34.77 KD,4.68%                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 4310 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 15060 ANGSTROM**2                       \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -22.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       44.63000            \\nREMARK 375                                                                      \\nREMARK 375 SPECIAL POSITION                                                     \\nREMARK 375 THE FOLLOWING ATOMS ARE FOUND TO BE WITHIN 0.15 ANGSTROMS            \\nREMARK 375 OF A SYMMETRY RELATED ATOM AND ARE ASSUMED TO BE ON SPECIAL          \\nREMARK 375 POSITIONS.                                                           \\nREMARK 375                                                                      \\nREMARK 375 ATOM RES CSSEQI                                                      \\nREMARK 375 CL    CL A 201  LIES ON A SPECIAL POSITION.                          \\nREMARK 465                                                                      \\nREMARK 465 MISSING RESIDUES                                                     \\nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \\nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \\nREMARK 465                                                                      \\nREMARK 465   M RES C SSSEQI                                                     \\nREMARK 465     MET A     1                                                      \\nREMARK 465     HIS A   166                                                      \\nREMARK 465     HIS A   167                                                      \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \\nREMARK 500                                                                      \\nREMARK 500 THE FOLLOWING ATOMS THAT ARE RELATED BY CRYSTALLOGRAPHIC             \\nREMARK 500 SYMMETRY ARE IN CLOSE CONTACT.  AN ATOM LOCATED WITHIN 0.15          \\nREMARK 500 ANGSTROMS OF A SYMMETRY RELATED ATOM IS ASSUMED TO BE ON A           \\nREMARK 500 SPECIAL POSITION AND IS, THEREFORE, LISTED IN REMARK 375             \\nREMARK 500 INSTEAD OF REMARK 500.  ATOMS WITH NON-BLANK ALTERNATE               \\nREMARK 500 LOCATION INDICATORS ARE NOT INCLUDED IN THE CALCULATIONS.            \\nREMARK 500                                                                      \\nREMARK 500 DISTANCE CUTOFF:                                                     \\nREMARK 500 2.2 ANGSTROMS FOR CONTACTS NOT INVOLVING HYDROGEN ATOMS              \\nREMARK 500 1.6 ANGSTROMS FOR CONTACTS INVOLVING HYDROGEN ATOMS                  \\nREMARK 500                                                                      \\nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI  SSYMOP   DISTANCE          \\nREMARK 500   O8   PE4 A   202     O8   PE4 A   202     7555     2.00            \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ILE A  55     -162.58   -110.53                                   \\nREMARK 500    TRP A 116      -75.21    -96.81                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 610                                                                      \\nREMARK 610 MISSING HETEROATOM                                                   \\nREMARK 610 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 610 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 610 I=INSERTION CODE):                                                   \\nREMARK 610   M RES C SSEQI                                                      \\nREMARK 610     PE4 A  202                                                       \\nREMARK 610     PE5 A  203                                                       \\nREMARK 610     PE6 A  204                                                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CL A 201                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE PE4 A 202                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC3                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE PE5 A 203                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC4                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE PE6 A 204                 \\nREMARK 900                                                                      \\nREMARK 900 RELATED ENTRIES                                                      \\nREMARK 900 RELATED ID: 3V45   RELATED DB: PDB                                   \\nREMARK 900 99.4% IDENTITY                                                       \\nREMARK 900 RELATED ID: NESG-OR185   RELATED DB: TARGETTRACK                     \\nDBREF  4ETJ A    1   167  PDB    4ETJ     4ETJ             1    167             \\nSEQRES   1 A  167  MET ALA ARG ILE ARG HIS VAL GLN GLY ASP ILE THR GLU          \\nSEQRES   2 A  167  PHE GLN GLY ASP ALA ILE VAL ASN ALA ALA ASN ASN TYR          \\nSEQRES   3 A  167  LEU LYS LEU GLY ALA GLY VAL ALA GLY ALA ILE LEU ARG          \\nSEQRES   4 A  167  LYS GLY GLY PRO SER ILE GLN GLU GLU CYS ASP ARG ILE          \\nSEQRES   5 A  167  GLY LYS ILE ARG VAL GLY GLU ALA ALA VAL THR GLY ALA          \\nSEQRES   6 A  167  GLY ASN LEU PRO VAL ARG TYR VAL ILE HIS ALA ALA VAL          \\nSEQRES   7 A  167  LEU GLY ASP GLU PRO ALA SER LEU GLU THR VAL ARG LYS          \\nSEQRES   8 A  167  ALA THR LYS SER ALA LEU GLU LYS ALA VAL GLU LEU GLY          \\nSEQRES   9 A  167  LEU LYS THR VAL ALA PHE THR ALA LEU GLY ALA TRP VAL          \\nSEQRES  10 A  167  GLY GLY LEU PRO ALA GLU ALA VAL LEU ARG VAL MET LEU          \\nSEQRES  11 A  167  GLU GLU ILE LYS LYS ALA PRO ASP THR LEU GLU VAL THR          \\nSEQRES  12 A  167  GLY VAL HIS GLY THR GLU LYS SER ALA GLU ALA ALA ARG          \\nSEQRES  13 A  167  ARG ALA LEU LEU GLU HIS HIS HIS HIS HIS HIS                  \\nHET     CL  A 201       1                                                       \\nHET    PE4  A 202      13                                                       \\nHET    PE5  A 203      16                                                       \\nHET    PE6  A 204      16                                                       \\nHETNAM      CL CHLORIDE ION                                                     \\nHETNAM     PE4 2-{2-[2-(2-{2-[2-(2-ETHOXY-ETHOXY)-ETHOXY]-ETHOXY}-              \\nHETNAM   2 PE4  ETHOXY)-ETHOXY]-ETHOXY}-ETHANOL                                 \\nHETNAM     PE5 3,6,9,12,15,18,21,24-OCTAOXAHEXACOSAN-1-OL                       \\nHETNAM     PE6 PHOSPHORYL-HEXAETHYLENE GLYCOL                                   \\nHETSYN     PE4 POLYETHYLENE GLYCOL PEG4000                                      \\nHETSYN     PE5 2-(2-{2-[2-(2-{2-[2-(2-ETHOXY-ETHOXY)-ETHOXY]-ETHOXY}-           \\nHETSYN   2 PE5  ETHOXY)-ETHOXY]-ETHOXY}-ETHOXY)-ETHANOL; POLYETHYLENE           \\nHETSYN   3 PE5  GLYCOL PEG400                                                   \\nFORMUL   2   CL    CL 1-                                                        \\nFORMUL   3  PE4    C16 H34 O8                                                   \\nFORMUL   4  PE5    C18 H38 O9                                                   \\nFORMUL   5  PE6    C12 H27 O10 P                                                \\nFORMUL   6  HOH   *37(H2 O)                                                     \\nHELIX    1   1 ASP A   10  PHE A   14  5                                   5    \\nHELIX    2   2 GLY A   32  GLY A   42  1                                  11    \\nHELIX    3   3 PRO A   43  GLY A   53  1                                  11    \\nHELIX    4   4 SER A   85  LEU A  103  1                                  19    \\nHELIX    5   5 PRO A  121  LYS A  134  1                                  14    \\nHELIX    6   6 THR A  148  HIS A  162  1                                  15    \\nSHEET    1   A 6 ARG A   3  GLN A   8  0                                        \\nSHEET    2   A 6 GLU A 141  HIS A 146  1  O  GLY A 144   N  ARG A   5           \\nSHEET    3   A 6 THR A 107  PHE A 110  1  N  PHE A 110   O  THR A 143           \\nSHEET    4   A 6 ALA A  18  ASN A  24  1  N  VAL A  20   O  ALA A 109           \\nSHEET    5   A 6 TYR A  72  LEU A  79  1  O  ILE A  74   N  ASN A  21           \\nSHEET    6   A 6 ALA A  60  GLY A  64 -1  N  ALA A  61   O  HIS A  75           \\nCISPEP   1 HIS A  164    HIS A  165          0        -0.83                     \\nSITE     1 AC1  1 ARG A  90                                                     \\nSITE     1 AC2  7 ARG A   5  VAL A   7  GLY A   9  ASP A  10                    \\nSITE     2 AC2  7 GLU A  13  PHE A  14  THR A 107                               \\nSITE     1 AC3  6 GLY A 119  LEU A 120  LEU A 126  ARG A 127                    \\nSITE     2 AC3  6 LEU A 130  ALA A 158                                          \\nSITE     1 AC4  7 HIS A   6  THR A 111  ALA A 112  LEU A 126                    \\nSITE     2 AC4  7 GLY A 144  VAL A 145  HIS A 146                               \\nCRYST1   65.930   65.930   89.260  90.00  90.00  90.00 P 43 21 2     8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.015168  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.015168  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.011203        0.00000                         \\nATOM      1  N   ALA A   2      -4.297   9.078   4.794  1.00 42.80           N  \\nANISOU    1  N   ALA A   2     5133   5774   5353   -498   -855    869       N  \\nATOM      2  CA  ALA A   2      -2.998   9.524   4.298  1.00 45.02           C  \\nANISOU    2  CA  ALA A   2     5564   6033   5509   -462   -784    899       C  \\nATOM      3  C   ALA A   2      -2.069   8.349   4.013  1.00 50.16           C  \\nANISOU    3  C   ALA A   2     6397   6670   5993   -509   -643    794       C  \\nATOM      4  O   ALA A   2      -2.135   7.320   4.690  1.00 49.71           O  \\nANISOU    4  O   ALA A   2     6357   6567   5965   -509   -550    705       O  \\nATOM      5  CB  ALA A   2      -2.352  10.475   5.292  1.00 29.16           C  \\nANISOU    5  CB  ALA A   2     3474   3977   3629   -329   -686    924       C  \\nATOM      6  N   ARG A   3      -1.200   8.513   3.016  1.00 39.44           N  \\nANISOU    6  N   ARG A   3     5184   5344   4459   -544   -617    806       N  \\nATOM      7  CA  ARG A   3      -0.190   7.510   2.703  1.00 37.63           C  \\nANISOU    7  CA  ARG A   3     5111   5106   4080   -548   -458    691       C  \\nATOM      8  C   ARG A   3       0.922   7.478   3.746  1.00 35.42           C  \\nANISOU    8  C   ARG A   3     4772   4802   3883   -401   -283    647       C  \\nATOM      9  O   ARG A   3       1.173   8.474   4.427  1.00 27.61           O  \\nANISOU    9  O   ARG A   3     3658   3824   3009   -333   -278    714       O  \\nATOM     10  CB  ARG A   3       0.427   7.786   1.341  1.00 42.67           C  \\nANISOU   10  CB  ARG A   3     5900   5817   4497   -636   -458    706       C  \\nATOM     11  CG  ARG A   3      -0.558   7.879   0.223  1.00 37.51           C  \\nANISOU   11  CG  ARG A   3     5329   5206   3719   -791   -655    764       C  \\nATOM     12  CD  ARG A   3       0.073   8.660  -0.895  1.00 50.80           C  \\nANISOU   12  CD  ARG A   3     7136   6963   5205   -870   -667    841       C  \\nATOM     13  NE  ARG A   3      -0.298  10.069  -0.871  1.00 50.27           N  \\nANISOU   13  NE  ARG A   3     6985   6891   5225   -852   -817   1020       N  \\nATOM     14  CZ  ARG A   3       0.500  11.057  -1.258  1.00 42.75           C  \\nANISOU   14  CZ  ARG A   3     6095   5957   4192   -876   -775   1112       C  \\nATOM     15  NH1 ARG A   3       1.734  10.800  -1.677  1.00 42.14           N  \\nANISOU   15  NH1 ARG A   3     6119   5944   3947   -923   -573   1033       N  \\nATOM     16  NH2 ARG A   3       0.064  12.307  -1.219  1.00 50.69           N  \\nANISOU   16  NH2 ARG A   3     7059   6912   5288   -853   -929   1279       N  \\nATOM     17  N   ILE A   4       1.573   6.323   3.873  1.00 34.63           N  \\nANISOU   17  N   ILE A   4     4771   4663   3723   -347   -152    532       N  \\nATOM     18  CA  ILE A   4       2.642   6.131   4.849  1.00 31.66           C  \\nANISOU   18  CA  ILE A   4     4335   4276   3417   -189    -12    490       C  \\nATOM     19  C   ILE A   4       3.853   5.432   4.230  1.00 32.50           C  \\nANISOU   19  C   ILE A   4     4545   4415   3389   -121    134    384       C  \\nATOM     20  O   ILE A   4       3.716   4.460   3.492  1.00 36.11           O  \\nANISOU   20  O   ILE A   4     5172   4819   3730   -160    162    292       O  \\nATOM     21  CB  ILE A   4       2.173   5.293   6.045  1.00 34.39           C  \\nANISOU   21  CB  ILE A   4     4675   4514   3878   -128      1    463       C  \\nATOM     22  CG1 ILE A   4       1.005   5.975   6.743  1.00 40.16           C  \\nANISOU   22  CG1 ILE A   4     5272   5240   4748   -186   -105    543       C  \\nATOM     23  CG2 ILE A   4       3.294   5.109   7.062  1.00 28.10           C  \\nANISOU   23  CG2 ILE A   4     3824   3717   3135     44    111    439       C  \\nATOM     24  CD1 ILE A   4       0.560   5.239   7.977  1.00 40.30           C  \\nANISOU   24  CD1 ILE A   4     5283   5171   4856   -158    -69    522       C  \\nATOM     25  N   ARG A   5       5.041   5.932   4.537  1.00 27.75           N  \\nANISOU   25  N   ARG A   5     3832   3905   2804    -22    234    382       N  \\nATOM     26  CA  ARG A   5       6.262   5.239   4.159  1.00 29.32           C  \\nANISOU   26  CA  ARG A   5     4066   4158   2916     90    392    266       C  \\nATOM     27  C   ARG A   5       7.198   5.262   5.348  1.00 30.26           C  \\nANISOU   27  C   ARG A   5     4023   4317   3156    268    452    262       C  \\nATOM     28  O   ARG A   5       7.180   6.203   6.157  1.00 25.25           O  \\nANISOU   28  O   ARG A   5     3246   3726   2623    251    396    351       O  \\nATOM     29  CB  ARG A   5       6.931   5.912   2.948  1.00 31.36           C  \\nANISOU   29  CB  ARG A   5     4329   4569   3016    -15    466    255       C  \\nATOM     30  CG  ARG A   5       7.344   7.357   3.233  1.00 29.80           C  \\nANISOU   30  CG  ARG A   5     3968   4485   2869    -80    445    367       C  \\nATOM     31  CD  ARG A   5       7.739   8.108   1.992  1.00 28.89           C  \\nANISOU   31  CD  ARG A   5     3907   4492   2578   -247    492    395       C  \\nATOM     32  NE  ARG A   5       9.058   7.740   1.496  1.00 30.53           N  \\nANISOU   32  NE  ARG A   5     4071   4853   2677   -201    698    272       N  \\nATOM     33  CZ  ARG A   5      10.134   8.508   1.598  1.00 31.06           C  \\nANISOU   33  CZ  ARG A   5     3977   5084   2742   -232    802    282       C  \\nATOM     34  NH1 ARG A   5      10.051   9.686   2.204  1.00 31.62           N  \\nANISOU   34  NH1 ARG A   5     3953   5153   2909   -316    714    409       N  \\nATOM     35  NH2 ARG A   5      11.285   8.106   1.098  1.00 32.94           N  \\nANISOU   35  NH2 ARG A   5     4144   5488   2884   -188   1002    154       N  \\nATOM     36  N   HIS A   6       8.016   4.222   5.449  1.00 31.14           N  \\nANISOU   36  N   HIS A   6     4165   4410   3256    447    558    156       N  \\nATOM     37  CA  HIS A   6       9.012   4.125   6.501  1.00 29.49           C  \\nANISOU   37  CA  HIS A   6     3797   4261   3146    642    596    150       C  \\nATOM     38  C   HIS A   6      10.409   4.014   5.922  1.00 33.46           C  \\nANISOU   38  C   HIS A   6     4187   4937   3591    758    748     46       C  \\nATOM     39  O   HIS A   6      10.712   3.100   5.155  1.00 32.55           O  \\nANISOU   39  O   HIS A   6     4188   4784   3396    847    853    -77       O  \\nATOM     40  CB  HIS A   6       8.722   2.926   7.404  1.00 27.85           C  \\nANISOU   40  CB  HIS A   6     3709   3863   3008    800    560    138       C  \\nATOM     41  CG  HIS A   6       7.572   3.147   8.328  1.00 30.27           C  \\nANISOU   41  CG  HIS A   6     4050   4060   3393    697    438    242       C  \\nATOM     42  ND1 HIS A   6       7.671   3.929   9.457  1.00 27.37           N  \\nANISOU   42  ND1 HIS A   6     3527   3761   3113    703    381    327       N  \\nATOM     43  CD2 HIS A   6       6.286   2.728   8.269  1.00 29.80           C  \\nANISOU   43  CD2 HIS A   6     4146   3847   3329    566    373    263       C  \\nATOM     44  CE1 HIS A   6       6.502   3.958  10.072  1.00 27.00           C  \\nANISOU   44  CE1 HIS A   6     3542   3604   3114    600    304    389       C  \\nATOM     45  NE2 HIS A   6       5.643   3.247   9.366  1.00 25.26           N  \\nANISOU   45  NE2 HIS A   6     3492   3259   2846    512    296    355       N  \\nATOM     46  N   VAL A   7      11.271   4.940   6.310  1.00 31.89           N  \\nANISOU   46  N   VAL A   7     3753   4933   3432    752    769     81       N  \\nATOM     47  CA  VAL A   7      12.626   4.922   5.794  1.00 35.73           C  \\nANISOU   47  CA  VAL A   7     4073   5633   3871    840    924    -21       C  \\nATOM     48  C   VAL A   7      13.640   5.216   6.895  1.00 31.69           C  \\nANISOU   48  C   VAL A   7     3291   5279   3472    975    903     -3       C  \\nATOM     49  O   VAL A   7      13.310   5.764   7.936  1.00 34.46           O  \\nANISOU   49  O   VAL A   7     3591   5598   3903    931    775    100       O  \\nATOM     50  CB  VAL A   7      12.790   5.936   4.635  1.00 39.39           C  \\nANISOU   50  CB  VAL A   7     4516   6253   4198    591   1007    -15       C  \\nATOM     51  CG1 VAL A   7      12.113   5.428   3.345  1.00 40.49           C  \\nANISOU   51  CG1 VAL A   7     4912   6293   4179    492   1054    -73       C  \\nATOM     52  CG2 VAL A   7      12.219   7.269   5.037  1.00 33.34           C  \\nANISOU   52  CG2 VAL A   7     3717   5479   3470    379    881    136       C  \\nATOM     53  N   GLN A   8      14.880   4.834   6.652  1.00 35.53           N  \\nANISOU   53  N   GLN A   8     3595   5951   3955   1140   1032   -115       N  \\nATOM     54  CA  GLN A   8      15.973   5.195   7.536  1.00 43.70           C  \\nANISOU   54  CA  GLN A   8     4322   7204   5080   1243   1010   -108       C  \\nATOM     55  C   GLN A   8      16.491   6.568   7.130  1.00 49.93           C  \\nANISOU   55  C   GLN A   8     4924   8234   5811    969   1074    -85       C  \\nATOM     56  O   GLN A   8      16.609   6.873   5.936  1.00 42.97           O  \\nANISOU   56  O   GLN A   8     4081   7437   4809    811   1216   -136       O  \\nATOM     57  CB  GLN A   8      17.102   4.163   7.430  1.00 45.62           C  \\nANISOU   57  CB  GLN A   8     4411   7562   5362   1565   1118   -245       C  \\nATOM     58  CG  GLN A   8      18.152   4.282   8.523  1.00 62.26           C  \\nANISOU   58  CG  GLN A   8     6198   9878   7581   1734   1038   -227       C  \\nATOM     59  CD  GLN A   8      19.270   5.238   8.162  1.00 72.38           C  \\nANISOU   59  CD  GLN A   8     7131  11532   8838   1587   1152   -281       C  \\nATOM     60  OE1 GLN A   8      19.688   5.308   7.006  1.00 81.18           O  \\nANISOU   60  OE1 GLN A   8     8195  12785   9866   1513   1352   -391       O  \\nATOM     61  NE2 GLN A   8      19.758   5.985   9.149  1.00 69.60           N  \\nANISOU   61  NE2 GLN A   8     6547  11352   8546   1516   1030   -211       N  \\nATOM     62  N   GLY A   9      16.803   7.394   8.122  1.00 50.85           N  \\nANISOU   62  N   GLY A   9     4866   8456   5998    891    970    -10       N  \\nATOM     63  CA  GLY A   9      17.429   8.674   7.857  1.00 47.46           C  \\nANISOU   63  CA  GLY A   9     4258   8252   5524    624   1029      5       C  \\nATOM     64  C   GLY A   9      17.530   9.576   9.068  1.00 45.69           C  \\nANISOU   64  C   GLY A   9     3916   8072   5373    515    885     88       C  \\nATOM     65  O   GLY A   9      17.388   9.152  10.221  1.00 44.74           O  \\nANISOU   65  O   GLY A   9     3783   7881   5336    681    746    121       O  \\nATOM     66  N   ASP A  10      17.780  10.842   8.787  1.00 45.52           N  \\nANISOU   66  N   ASP A  10     3836   8159   5302    214    922    122       N  \\nATOM     67  CA  ASP A  10      17.893  11.860   9.811  1.00 45.61           C  \\nANISOU   67  CA  ASP A  10     3765   8202   5361     53    805    181       C  \\nATOM     68  C   ASP A  10      16.771  12.837   9.528  1.00 38.29           C  \\nANISOU   68  C   ASP A  10     3111   7032   4406   -180    760    283       C  \\nATOM     69  O   ASP A  10      16.756  13.488   8.481  1.00 42.39           O  \\nANISOU   69  O   ASP A  10     3712   7558   4835   -393    853    307       O  \\nATOM     70  CB  ASP A  10      19.260  12.542   9.699  1.00 48.13           C  \\nANISOU   70  CB  ASP A  10     3774   8860   5652   -119    896    121       C  \\nATOM     71  CG  ASP A  10      19.416  13.724  10.643  1.00 54.24           C  \\nANISOU   71  CG  ASP A  10     4492   9664   6451   -354    786    168       C  \\nATOM     72  OD1 ASP A  10      18.526  13.951  11.490  1.00 54.81           O  \\nANISOU   72  OD1 ASP A  10     4748   9506   6570   -340    644    234       O  \\nATOM     73  OD2 ASP A  10      20.450  14.422  10.542  1.00 58.67           O  \\nANISOU   73  OD2 ASP A  10     4891  10411   6988   -553    830    122       O  \\nATOM     74  N   ILE A  11      15.818  12.931  10.446  1.00 37.84           N  \\nANISOU   74  N   ILE A  11     3199   6759   4420   -133    618    344       N  \\nATOM     75  CA  ILE A  11      14.607  13.703  10.165  1.00 28.38           C  \\nANISOU   75  CA  ILE A  11     2253   5309   3222   -280    567    433       C  \\nATOM     76  C   ILE A  11      14.906  15.182   9.903  1.00 34.07           C  \\nANISOU   76  C   ILE A  11     2987   6051   3907   -581    589    476       C  \\nATOM     77  O   ILE A  11      14.165  15.850   9.184  1.00 36.46           O  \\nANISOU   77  O   ILE A  11     3492   6184   4179   -715    580    554       O  \\nATOM     78  CB  ILE A  11      13.521  13.507  11.262  1.00 28.33           C  \\nANISOU   78  CB  ILE A  11     2368   5091   3306   -164    436    470       C  \\nATOM     79  CG1 ILE A  11      12.214  14.188  10.849  1.00 28.19           C  \\nANISOU   79  CG1 ILE A  11     2573   4830   3308   -267    388    549       C  \\nATOM     80  CG2 ILE A  11      14.024  13.972  12.648  1.00 28.63           C  \\nANISOU   80  CG2 ILE A  11     2280   5209   3388   -190    362    448       C  \\nATOM     81  CD1 ILE A  11      11.068  13.937  11.796  1.00 25.86           C  \\nANISOU   81  CD1 ILE A  11     2374   4350   3100   -158    295    568       C  \\nATOM     82  N   THR A  12      16.019  15.682  10.446  1.00 35.00           N  \\nANISOU   82  N   THR A  12     2895   6378   4023   -696    609    428       N  \\nATOM     83  CA  THR A  12      16.407  17.079  10.244  1.00 40.07           C  \\nANISOU   83  CA  THR A  12     3564   7037   4625  -1020    639    460       C  \\nATOM     84  C   THR A  12      16.910  17.367   8.826  1.00 40.68           C  \\nANISOU   84  C   THR A  12     3655   7224   4578  -1211    788    478       C  \\nATOM     85  O   THR A  12      17.111  18.528   8.449  1.00 46.83           O  \\nANISOU   85  O   THR A  12     4525   7969   5301  -1508    821    532       O  \\nATOM     86  CB  THR A  12      17.475  17.539  11.265  1.00 43.90           C  \\nANISOU   86  CB  THR A  12     3813   7736   5131  -1134    610    393       C  \\nATOM     87  OG1 THR A  12      18.667  16.766  11.096  1.00 46.75           O  \\nANISOU   87  OG1 THR A  12     3862   8435   5466  -1044    691    307       O  \\nATOM     88  CG2 THR A  12      16.960  17.376  12.688  1.00 39.56           C  \\nANISOU   88  CG2 THR A  12     3291   7075   4664   -989    463    380       C  \\nATOM     89  N   GLU A  13      17.119  16.315   8.043  1.00 36.37           N  \\nANISOU   89  N   GLU A  13     3044   6797   3977  -1054    886    430       N  \\nATOM     90  CA  GLU A  13      17.549  16.477   6.659  1.00 39.43           C  \\nANISOU   90  CA  GLU A  13     3465   7301   4216  -1229   1049    433       C  \\nATOM     91  C   GLU A  13      16.413  16.140   5.687  1.00 42.12           C  \\nANISOU   91  C   GLU A  13     4101   7417   4487  -1171   1028    506       C  \\nATOM     92  O   GLU A  13      16.602  16.151   4.476  1.00 36.31           O  \\nANISOU   92  O   GLU A  13     3446   6753   3596  -1300   1151    514       O  \\nATOM     93  CB  GLU A  13      18.794  15.632   6.367  1.00 40.85           C  \\nANISOU   93  CB  GLU A  13     3332   7831   4358  -1130   1212    294       C  \\nATOM     94  CG  GLU A  13      20.041  16.067   7.126  1.00 53.55           C  \\nANISOU   94  CG  GLU A  13     4680   9629   6037  -1198   1191    216       C  \\nATOM     95  CD  GLU A  13      20.461  17.499   6.814  1.00 66.41           C  \\nANISOU   95  CD  GLU A  13     6400  11231   7601  -1575   1213    263       C  \\nATOM     96  OE1 GLU A  13      20.308  17.935   5.653  1.00 69.17           O  \\nANISOU   96  OE1 GLU A  13     6941  11515   7827  -1760   1309    316       O  \\nATOM     97  OE2 GLU A  13      20.939  18.194   7.737  1.00 72.13           O  \\nANISOU   97  OE2 GLU A  13     7029  11991   8387  -1691   1126    249       O  \\nATOM     98  N   PHE A  14      15.234  15.845   6.237  1.00 41.85           N  \\nANISOU   98  N   PHE A  14     4217   7130   4553   -994    872    554       N  \\nATOM     99  CA  PHE A  14      14.032  15.597   5.444  1.00 37.50           C  \\nANISOU   99  CA  PHE A  14     3927   6367   3955   -951    809    629       C  \\nATOM    100  C   PHE A  14      13.780  16.761   4.507  1.00 40.01           C  \\nANISOU  100  C   PHE A  14     4454   6590   4157  -1226    804    756       C  \\nATOM    101  O   PHE A  14      13.855  17.915   4.922  1.00 37.63           O  \\nANISOU  101  O   PHE A  14     4193   6207   3899  -1395    756    824       O  \\nATOM    102  CB  PHE A  14      12.823  15.421   6.359  1.00 35.54           C  \\nANISOU  102  CB  PHE A  14     3769   5880   3854   -781    638    669       C  \\nATOM    103  CG  PHE A  14      11.566  15.009   5.642  1.00 38.00           C  \\nANISOU  103  CG  PHE A  14     4291   6012   4136   -718    557    729       C  \\nATOM    104  CD1 PHE A  14      11.471  13.759   5.043  1.00 28.42           C  \\nANISOU  104  CD1 PHE A  14     3100   4846   2853   -588    610    657       C  \\nATOM    105  CD2 PHE A  14      10.466  15.858   5.602  1.00 33.27           C  \\nANISOU  105  CD2 PHE A  14     3861   5193   3586   -781    419    848       C  \\nATOM    106  CE1 PHE A  14      10.308  13.377   4.394  1.00 27.91           C  \\nANISOU  106  CE1 PHE A  14     3222   4632   2750   -563    520    704       C  \\nATOM    107  CE2 PHE A  14       9.296  15.479   4.974  1.00 27.28           C  \\nANISOU  107  CE2 PHE A  14     3255   4303   2808   -724    319    903       C  \\nATOM    108  CZ  PHE A  14       9.214  14.240   4.367  1.00 33.17           C  \\nANISOU  108  CZ  PHE A  14     4020   5116   3467   -635    366    832       C  \\nATOM    109  N   GLN A  15      13.456  16.461   3.253  1.00 33.44           N  \\nANISOU  109  N   GLN A  15     3785   5751   3170  -1276    843    790       N  \\nATOM    110  CA  GLN A  15      13.376  17.494   2.242  1.00 38.91           C  \\nANISOU  110  CA  GLN A  15     4691   6385   3707  -1551    848    923       C  \\nATOM    111  C   GLN A  15      11.952  17.688   1.714  1.00 47.06           C  \\nANISOU  111  C   GLN A  15     5999   7158   4724  -1517    661   1062       C  \\nATOM    112  O   GLN A  15      11.740  17.746   0.511  1.00 66.16           O  \\nANISOU  112  O   GLN A  15     8609   9579   6951  -1644    668   1136       O  \\nATOM    113  CB  GLN A  15      14.327  17.140   1.089  1.00 45.47           C  \\nANISOU  113  CB  GLN A  15     5492   7470   4314  -1699   1064    857       C  \\nATOM    114  CG  GLN A  15      14.878  18.327   0.334  1.00 70.07           C  \\nANISOU  114  CG  GLN A  15     8734  10630   7260  -2059   1146    964       C  \\nATOM    115  CD  GLN A  15      16.301  18.681   0.736  1.00 74.60           C  \\nANISOU  115  CD  GLN A  15     9049  11409   7886  -2158   1287    842       C  \\nATOM    116  OE1 GLN A  15      16.859  18.109   1.676  1.00 72.04           O  \\nANISOU  116  OE1 GLN A  15     8431  11247   7693  -2003   1332    718       O  \\nATOM    117  NE2 GLN A  15      16.894  19.632   0.020  1.00 72.94           N  \\nANISOU  117  NE2 GLN A  15     8949  11191   7573  -2411   1337    881       N  \\nATOM    118  N   GLY A  16      10.979  17.798   2.610  1.00 42.64           N  \\nANISOU  118  N   GLY A  16     5449   6395   4357  -1352    492   1096       N  \\nATOM    119  CA  GLY A  16       9.598  18.006   2.204  1.00 39.94           C  \\nANISOU  119  CA  GLY A  16     5309   5832   4035  -1296    300   1221       C  \\nATOM    120  C   GLY A  16       8.975  19.264   2.783  1.00 38.32           C  \\nANISOU  120  C   GLY A  16     5200   5387   3975  -1317    155   1339       C  \\nATOM    121  O   GLY A  16       9.661  20.242   3.058  1.00 41.27           O  \\nANISOU  121  O   GLY A  16     5592   5735   4354  -1478    206   1369       O  \\nATOM    122  N   ASP A  17       7.664  19.229   2.993  1.00 38.86           N  \\nANISOU  122  N   ASP A  17     5322   5276   4167  -1152    -21   1394       N  \\nATOM    123  CA  ASP A  17       6.940  20.374   3.554  1.00 38.38           C  \\nANISOU  123  CA  ASP A  17     5346   4967   4269  -1113   -158   1488       C  \\nATOM    124  C   ASP A  17       7.150  20.654   5.051  1.00 37.56           C  \\nANISOU  124  C   ASP A  17     5095   4818   4357  -1032   -112   1381       C  \\nATOM    125  O   ASP A  17       7.258  21.806   5.452  1.00 40.79           O  \\nANISOU  125  O   ASP A  17     5595   5068   4837  -1110   -135   1425       O  \\nATOM    126  CB  ASP A  17       5.445  20.235   3.257  1.00 41.40           C  \\nANISOU  126  CB  ASP A  17     5790   5209   4731   -949   -358   1570       C  \\nATOM    127  CG  ASP A  17       5.154  20.269   1.774  1.00 42.91           C  \\nANISOU  127  CG  ASP A  17     6177   5409   4719  -1058   -455   1710       C  \\nATOM    128  OD1 ASP A  17       5.790  21.088   1.070  1.00 55.57           O  \\nANISOU  128  OD1 ASP A  17     7961   6983   6171  -1262   -426   1820       O  \\nATOM    129  OD2 ASP A  17       4.309  19.477   1.311  1.00 35.78           O  \\nANISOU  129  OD2 ASP A  17     5259   4546   3787   -966   -558   1711       O  \\nATOM    130  N   ALA A  18       7.179  19.615   5.878  1.00 29.25           N  \\nANISOU  130  N   ALA A  18     3849   3887   3377   -883    -56   1243       N  \\nATOM    131  CA  ALA A  18       7.267  19.816   7.316  1.00 27.26           C  \\nANISOU  131  CA  ALA A  18     3479   3599   3280   -806    -30   1147       C  \\nATOM    132  C   ALA A  18       7.883  18.616   7.996  1.00 29.70           C  \\nANISOU  132  C   ALA A  18     3594   4113   3580   -717     67   1013       C  \\nATOM    133  O   ALA A  18       7.847  17.511   7.462  1.00 32.56           O  \\nANISOU  133  O   ALA A  18     3920   4582   3868   -647     93    986       O  \\nATOM    134  CB  ALA A  18       5.871  20.069   7.903  1.00 26.68           C  \\nANISOU  134  CB  ALA A  18     3427   3328   3384   -633   -152   1163       C  \\nATOM    135  N   ILE A  19       8.439  18.835   9.182  1.00 28.26           N  \\nANISOU  135  N   ILE A  19     3305   3968   3463   -717    111    928       N  \\nATOM    136  CA  ILE A  19       8.765  17.729  10.087  1.00 28.40           C  \\nANISOU  136  CA  ILE A  19     3159   4130   3502   -583    155    821       C  \\nATOM    137  C   ILE A  19       8.018  17.960  11.395  1.00 28.77           C  \\nANISOU  137  C   ILE A  19     3197   4054   3681   -486    107    777       C  \\nATOM    138  O   ILE A  19       7.703  19.097  11.753  1.00 28.52           O  \\nANISOU  138  O   ILE A  19     3247   3870   3719   -550     75    791       O  \\nATOM    139  CB  ILE A  19      10.292  17.579  10.347  1.00 26.34           C  \\nANISOU  139  CB  ILE A  19     2742   4101   3164   -667    251    750       C  \\nATOM    140  CG1 ILE A  19      10.839  18.742  11.172  1.00 28.19           C  \\nANISOU  140  CG1 ILE A  19     2961   4317   3432   -820    248    725       C  \\nATOM    141  CG2 ILE A  19      11.063  17.461   9.024  1.00 27.29           C  \\nANISOU  141  CG2 ILE A  19     2860   4363   3146   -784    337    776       C  \\nATOM    142  CD1 ILE A  19      12.378  18.738  11.279  1.00 30.31           C  \\nANISOU  142  CD1 ILE A  19     3047   4850   3619   -951    329    662       C  \\nATOM    143  N   VAL A  20       7.693  16.887  12.093  1.00 27.48           N  \\nANISOU  143  N   VAL A  20     2959   3938   3544   -338    109    721       N  \\nATOM    144  CA  VAL A  20       6.964  17.032  13.337  1.00 28.29           C  \\nANISOU  144  CA  VAL A  20     3057   3947   3743   -264     88    673       C  \\nATOM    145  C   VAL A  20       7.939  16.828  14.490  1.00 37.76           C  \\nANISOU  145  C   VAL A  20     4162   5286   4901   -276    121    595       C  \\nATOM    146  O   VAL A  20       8.679  15.844  14.524  1.00 31.21           O  \\nANISOU  146  O   VAL A  20     3242   4612   4002   -216    140    577       O  \\nATOM    147  CB  VAL A  20       5.778  16.051  13.421  1.00 26.49           C  \\nANISOU  147  CB  VAL A  20     2834   3667   3564   -126     64    675       C  \\nATOM    148  CG1 VAL A  20       5.075  16.161  14.770  1.00 23.98           C  \\nANISOU  148  CG1 VAL A  20     2500   3285   3324    -70     76    612       C  \\nATOM    149  CG2 VAL A  20       4.803  16.328  12.292  1.00 21.73           C  \\nANISOU  149  CG2 VAL A  20     2303   2953   3000   -124     -1    753       C  \\nATOM    150  N   ASN A  21       7.961  17.797  15.401  1.00 28.05           N  \\nANISOU  150  N   ASN A  21     2958   3992   3707   -350    118    546       N  \\nATOM    151  CA  ASN A  21       8.798  17.737  16.588  1.00 32.04           C  \\nANISOU  151  CA  ASN A  21     3387   4628   4157   -386    123    471       C  \\nATOM    152  C   ASN A  21       8.002  17.214  17.777  1.00 29.03           C  \\nANISOU  152  C   ASN A  21     3030   4206   3796   -277    122    423       C  \\nATOM    153  O   ASN A  21       6.908  17.698  18.047  1.00 32.04           O  \\nANISOU  153  O   ASN A  21     3490   4424   4260   -252    139    402       O  \\nATOM    154  CB  ASN A  21       9.304  19.145  16.900  1.00 36.32           C  \\nANISOU  154  CB  ASN A  21     3978   5121   4702   -573    125    429       C  \\nATOM    155  CG  ASN A  21      10.131  19.200  18.155  1.00 40.47           C  \\nANISOU  155  CG  ASN A  21     4432   5791   5155   -644    109    342       C  \\nATOM    156  OD1 ASN A  21      10.873  18.265  18.456  1.00 41.43           O  \\nANISOU  156  OD1 ASN A  21     4417   6119   5203   -582     84    339       O  \\nATOM    157  ND2 ASN A  21      10.019  20.308  18.898  1.00 36.78           N  \\nANISOU  157  ND2 ASN A  21     4064   5209   4703   -769    113    267       N  \\nATOM    158  N   ALA A  22       8.530  16.214  18.470  1.00 30.31           N  \\nANISOU  158  N   ALA A  22     3124   4515   3879   -208    106    410       N  \\nATOM    159  CA  ALA A  22       7.971  15.799  19.747  1.00 27.08           C  \\nANISOU  159  CA  ALA A  22     2760   4088   3442   -155    109    369       C  \\nATOM    160  C   ALA A  22       8.268  16.875  20.796  1.00 33.07           C  \\nANISOU  160  C   ALA A  22     3551   4848   4165   -285    108    280       C  \\nATOM    161  O   ALA A  22       9.370  16.944  21.336  1.00 26.15           O  \\nANISOU  161  O   ALA A  22     2609   4132   3195   -357     58    254       O  \\nATOM    162  CB  ALA A  22       8.564  14.465  20.173  1.00 32.12           C  \\nANISOU  162  CB  ALA A  22     3351   4866   3987    -47     68    402       C  \\nATOM    163  N   ALA A  23       7.284  17.718  21.083  1.00 27.63           N  \\nANISOU  163  N   ALA A  23     2958   3988   3554   -313    162    223       N  \\nATOM    164  CA  ALA A  23       7.519  18.876  21.946  1.00 27.66           C  \\nANISOU  164  CA  ALA A  23     3031   3947   3531   -448    178    116       C  \\nATOM    165  C   ALA A  23       6.942  18.724  23.352  1.00 27.59           C  \\nANISOU  165  C   ALA A  23     3092   3937   3454   -437    226     24       C  \\nATOM    166  O   ALA A  23       6.069  17.880  23.598  1.00 29.02           O  \\nANISOU  166  O   ALA A  23     3279   4106   3641   -329    269     45       O  \\nATOM    167  CB  ALA A  23       6.949  20.136  21.275  1.00 28.66           C  \\nANISOU  167  CB  ALA A  23     3246   3850   3791   -487    213     98       C  \\nATOM    168  N   ASN A  24       7.434  19.538  24.280  1.00 33.09           N  \\nANISOU  168  N   ASN A  24     3854   4653   4067   -577    225    -85       N  \\nATOM    169  CA  ASN A  24       6.721  19.760  25.531  1.00 33.67           C  \\nANISOU  169  CA  ASN A  24     4037   4676   4081   -595    307   -207       C  \\nATOM    170  C   ASN A  24       5.722  20.905  25.337  1.00 32.36           C  \\nANISOU  170  C   ASN A  24     3963   4260   4073   -573    412   -303       C  \\nATOM    171  O   ASN A  24       5.823  21.645  24.358  1.00 28.12           O  \\nANISOU  171  O   ASN A  24     3434   3592   3659   -583    388   -267       O  \\nATOM    172  CB  ASN A  24       7.682  20.040  26.682  1.00 30.93           C  \\nANISOU  172  CB  ASN A  24     3737   4473   3543   -760    252   -293       C  \\nATOM    173  CG  ASN A  24       8.619  21.189  26.398  1.00 39.97           C  \\nANISOU  173  CG  ASN A  24     4893   5597   4695   -940    202   -347       C  \\nATOM    174  OD1 ASN A  24       8.200  22.347  26.336  1.00 38.10           O  \\nANISOU  174  OD1 ASN A  24     4780   5151   4547  -1005    275   -447       O  \\nATOM    175  ND2 ASN A  24       9.907  20.879  26.250  1.00 39.44           N  \\nANISOU  175  ND2 ASN A  24     4698   5749   4537  -1025     78   -286       N  \\nATOM    176  N   ASN A  25       4.755  21.059  26.239  1.00 32.91           N  \\nANISOU  176  N   ASN A  25     4105   4260   4140   -535    531   -423       N  \\nATOM    177  CA  ASN A  25       3.712  22.062  26.015  1.00 29.87           C  \\nANISOU  177  CA  ASN A  25     3778   3635   3937   -452    634   -517       C  \\nATOM    178  C   ASN A  25       4.169  23.512  26.251  1.00 34.77           C  \\nANISOU  178  C   ASN A  25     4552   4086   4571   -576    646   -645       C  \\nATOM    179  O   ASN A  25       3.437  24.454  25.971  1.00 32.91           O  \\nANISOU  179  O   ASN A  25     4391   3611   4503   -491    711   -714       O  \\nATOM    180  CB  ASN A  25       2.418  21.732  26.780  1.00 30.45           C  \\nANISOU  180  CB  ASN A  25     3838   3699   4033   -348    787   -615       C  \\nATOM    181  CG  ASN A  25       2.585  21.822  28.281  1.00 41.87           C  \\nANISOU  181  CG  ASN A  25     5400   5229   5277   -472    876   -772       C  \\nATOM    182  OD1 ASN A  25       3.524  21.261  28.839  1.00 43.26           O  \\nANISOU  182  OD1 ASN A  25     5604   5584   5246   -597    791   -731       O  \\nATOM    183  ND2 ASN A  25       1.682  22.539  28.942  1.00 33.85           N  \\nANISOU  183  ND2 ASN A  25     4455   4092   4316   -431   1044   -957       N  \\nATOM    184  N   TYR A  26       5.389  23.680  26.750  1.00 34.58           N  \\nANISOU  184  N   TYR A  26     4579   4185   4377   -777    572   -675       N  \\nATOM    185  CA  TYR A  26       6.009  25.004  26.861  1.00 36.43           C  \\nANISOU  185  CA  TYR A  26     4968   4269   4606   -953    562   -783       C  \\nATOM    186  C   TYR A  26       6.767  25.390  25.593  1.00 37.31           C  \\nANISOU  186  C   TYR A  26     5047   4331   4796  -1029    459   -647       C  \\nATOM    187  O   TYR A  26       7.260  26.511  25.478  1.00 37.46           O  \\nANISOU  187  O   TYR A  26     5208   4194   4829  -1194    449   -708       O  \\nATOM    188  CB  TYR A  26       6.929  25.069  28.085  1.00 35.78           C  \\nANISOU  188  CB  TYR A  26     4951   4360   4282  -1175    529   -901       C  \\nATOM    189  CG  TYR A  26       6.156  24.938  29.373  1.00 36.85           C  \\nANISOU  189  CG  TYR A  26     5182   4507   4314  -1142    656  -1062       C  \\nATOM    190  CD1 TYR A  26       5.908  26.044  30.173  1.00 45.94           C  \\nANISOU  190  CD1 TYR A  26     6541   5481   5434  -1238    766  -1287       C  \\nATOM    191  CD2 TYR A  26       5.632  23.711  29.767  1.00 46.14           C  \\nANISOU  191  CD2 TYR A  26     6261   5855   5417  -1021    686   -995       C  \\nATOM    192  CE1 TYR A  26       5.170  25.929  31.354  1.00 46.14           C  \\nANISOU  192  CE1 TYR A  26     6657   5528   5344  -1215    914  -1453       C  \\nATOM    193  CE2 TYR A  26       4.899  23.583  30.939  1.00 42.11           C  \\nANISOU  193  CE2 TYR A  26     5845   5367   4788  -1018    827  -1140       C  \\nATOM    194  CZ  TYR A  26       4.670  24.693  31.727  1.00 39.40           C  \\nANISOU  194  CZ  TYR A  26     5691   4875   4406  -1112    947  -1373       C  \\nATOM    195  OH  TYR A  26       3.943  24.558  32.888  1.00 40.92           O  \\nANISOU  195  OH  TYR A  26     5978   5109   4461  -1118   1113  -1532       O  \\nATOM    196  N   LEU A  27       6.825  24.464  24.639  1.00 32.87           N  \\nANISOU  196  N   LEU A  27     4321   3893   4276   -922    395   -470       N  \\nATOM    197  CA  LEU A  27       7.533  24.667  23.376  1.00 33.50           C  \\nANISOU  197  CA  LEU A  27     4357   3968   4401   -995    317   -333       C  \\nATOM    198  C   LEU A  27       8.978  25.092  23.596  1.00 33.35           C  \\nANISOU  198  C   LEU A  27     4333   4094   4246  -1268    252   -365       C  \\nATOM    199  O   LEU A  27       9.514  25.921  22.854  1.00 38.35           O  \\nANISOU  199  O   LEU A  27     5033   4626   4914  -1424    232   -329       O  \\nATOM    200  CB  LEU A  27       6.797  25.667  22.473  1.00 34.18           C  \\nANISOU  200  CB  LEU A  27     4581   3736   4671   -928    340   -299       C  \\nATOM    201  CG  LEU A  27       5.356  25.307  22.088  1.00 36.53           C  \\nANISOU  201  CG  LEU A  27     4839   3913   5127   -654    378   -255       C  \\nATOM    202  CD1 LEU A  27       4.732  26.375  21.214  1.00 32.69           C  \\nANISOU  202  CD1 LEU A  27     4493   3112   4816   -579    363   -208       C  \\nATOM    203  CD2 LEU A  27       5.274  23.939  21.403  1.00 29.13           C  \\nANISOU  203  CD2 LEU A  27     3714   3186   4170   -546    330   -110       C  \\nATOM    204  N   LYS A  28       9.596  24.534  24.636  1.00 33.08           N  \\nANISOU  204  N   LYS A  28     4220   4301   4046  -1340    213   -429       N  \\nATOM    205  CA  LYS A  28      11.012  24.758  24.901  1.00 38.01           C  \\nANISOU  205  CA  LYS A  28     4771   5140   4532  -1592    125   -456       C  \\nATOM    206  C   LYS A  28      11.804  23.619  24.264  1.00 41.87           C  \\nANISOU  206  C   LYS A  28     5005   5919   4983  -1524     44   -312       C  \\nATOM    207  O   LYS A  28      11.629  22.462  24.645  1.00 37.52           O  \\nANISOU  207  O   LYS A  28     4354   5518   4382  -1348     14   -265       O  \\nATOM    208  CB  LYS A  28      11.287  24.820  26.411  1.00 35.98           C  \\nANISOU  208  CB  LYS A  28     4571   5001   4099  -1710     98   -605       C  \\nATOM    209  CG  LYS A  28      10.612  25.991  27.149  1.00 37.78           C  \\nANISOU  209  CG  LYS A  28     5068   4948   4338  -1798    198   -792       C  \\nATOM    210  CD  LYS A  28      10.990  27.344  26.570  1.00 52.62           C  \\nANISOU  210  CD  LYS A  28     7099   6602   6294  -2011    213   -838       C  \\nATOM    211  CE  LYS A  28      10.088  28.449  27.111  1.00 64.68           C  \\nANISOU  211  CE  LYS A  28     8917   7769   7889  -2007    333  -1015       C  \\nATOM    212  NZ  LYS A  28      10.280  29.738  26.382  1.00 66.39           N  \\nANISOU  212  NZ  LYS A  28     9330   7681   8214  -2167    350  -1025       N  \\nATOM    213  N   LEU A  29      12.651  23.941  23.283  1.00 39.05           N  \\nANISOU  213  N   LEU A  29     4558   5629   4651  -1660     22   -245       N  \\nATOM    214  CA  LEU A  29      13.387  22.912  22.542  1.00 36.08           C  \\nANISOU  214  CA  LEU A  29     3935   5516   4256  -1576    -24   -127       C  \\nATOM    215  C   LEU A  29      14.623  22.451  23.300  1.00 37.18           C  \\nANISOU  215  C   LEU A  29     3866   6005   4254  -1668   -132   -164       C  \\nATOM    216  O   LEU A  29      15.514  23.243  23.598  1.00 42.04           O  \\nANISOU  216  O   LEU A  29     4448   6726   4800  -1940   -172   -240       O  \\nATOM    217  CB  LEU A  29      13.767  23.387  21.134  1.00 42.19           C  \\nANISOU  217  CB  LEU A  29     4686   6246   5096  -1684     20    -44       C  \\nATOM    218  CG  LEU A  29      12.694  23.401  20.034  1.00 45.68           C  \\nANISOU  218  CG  LEU A  29     5257   6434   5666  -1526     84     56       C  \\nATOM    219  CD1 LEU A  29      11.804  24.621  20.132  1.00 49.81           C  \\nANISOU  219  CD1 LEU A  29     6051   6599   6277  -1580    124     11       C  \\nATOM    220  CD2 LEU A  29      13.361  23.365  18.682  1.00 54.43           C  \\nANISOU  220  CD2 LEU A  29     6273   7639   6769  -1612    110    158       C  \\nATOM    221  N   GLY A  30      14.672  21.162  23.622  1.00 47.01           N  \\nANISOU  221  N   GLY A  30     4977   7428   5458  -1444   -192   -106       N  \\nATOM    222  CA  GLY A  30      15.769  20.639  24.418  1.00 50.89           C  \\nANISOU  222  CA  GLY A  30     5271   8248   5818  -1477   -327   -123       C  \\nATOM    223  C   GLY A  30      16.109  19.185  24.150  1.00 53.65           C  \\nANISOU  223  C   GLY A  30     5424   8790   6169  -1204   -386    -15       C  \\nATOM    224  O   GLY A  30      16.982  18.877  23.340  1.00 50.75           O  \\nANISOU  224  O   GLY A  30     4830   8614   5837  -1184   -393     26       O  \\nATOM    225  N   ALA A  31      15.418  18.287  24.841  1.00 61.31           N  \\nANISOU  225  N   ALA A  31     6494   9703   7098   -998   -417     24       N  \\nATOM    226  CA  ALA A  31      15.739  16.865  24.779  1.00 64.96           C  \\nANISOU  226  CA  ALA A  31     6823  10312   7547   -733   -492    126       C  \\nATOM    227  C   ALA A  31      15.191  16.180  23.526  1.00 55.16           C  \\nANISOU  227  C   ALA A  31     5588   8933   6437   -540   -384    204       C  \\nATOM    228  O   ALA A  31      14.068  16.455  23.085  1.00 54.99           O  \\nANISOU  228  O   ALA A  31     5743   8658   6493   -537   -273    206       O  \\nATOM    229  CB  ALA A  31      15.252  16.145  26.053  1.00 60.83           C  \\nANISOU  229  CB  ALA A  31     6443   9770   6899   -621   -573    152       C  \\nATOM    230  N   GLY A  32      16.001  15.289  22.960  1.00 44.48           N  \\nANISOU  230  N   GLY A  32     4034   7760   5106   -375   -423    259       N  \\nATOM    231  CA  GLY A  32      15.567  14.442  21.864  1.00 47.32           C  \\nANISOU  231  CA  GLY A  32     4412   8009   5557   -178   -334    321       C  \\nATOM    232  C   GLY A  32      15.819  15.063  20.507  1.00 46.61           C  \\nANISOU  232  C   GLY A  32     4243   7922   5545   -292   -219    302       C  \\nATOM    233  O   GLY A  32      16.821  15.744  20.304  1.00 53.00           O  \\nANISOU  233  O   GLY A  32     4872   8927   6337   -462   -223    257       O  \\nATOM    234  N   VAL A  33      14.902  14.833  19.574  1.00 41.95           N  \\nANISOU  234  N   VAL A  33     3792   7125   5023   -222   -119    340       N  \\nATOM    235  CA  VAL A  33      15.026  15.410  18.235  1.00 45.34           C  \\nANISOU  235  CA  VAL A  33     4196   7534   5496   -340    -13    340       C  \\nATOM    236  C   VAL A  33      14.988  16.940  18.271  1.00 36.79           C  \\nANISOU  236  C   VAL A  33     3186   6378   4415   -628     10    305       C  \\nATOM    237  O   VAL A  33      15.628  17.596  17.459  1.00 43.05           O  \\nANISOU  237  O   VAL A  33     3903   7252   5201   -802     69    299       O  \\nATOM    238  CB  VAL A  33      13.925  14.889  17.288  1.00 46.07           C  \\nANISOU  238  CB  VAL A  33     4453   7410   5642   -221     59    393       C  \\nATOM    239  CG1 VAL A  33      14.190  13.438  16.925  1.00 49.42           C  \\nANISOU  239  CG1 VAL A  33     4811   7906   6061     26     64    411       C  \\nATOM    240  CG2 VAL A  33      12.545  15.044  17.934  1.00 54.38           C  \\nANISOU  240  CG2 VAL A  33     5716   8215   6729   -203     41    407       C  \\nATOM    241  N   ALA A  34      14.232  17.493  19.219  1.00 35.40           N  \\nANISOU  241  N   ALA A  34     3173   6036   4240   -683    -25    277       N  \\nATOM    242  CA  ALA A  34      14.087  18.937  19.371  1.00 34.71           C  \\nANISOU  242  CA  ALA A  34     3206   5818   4163   -930     -4    229       C  \\nATOM    243  C   ALA A  34      15.434  19.619  19.453  1.00 32.95           C  \\nANISOU  243  C   ALA A  34     2821   5821   3878  -1170    -27    178       C  \\nATOM    244  O   ALA A  34      15.634  20.676  18.860  1.00 47.63           O  \\nANISOU  244  O   ALA A  34     4743   7605   5748  -1400     27    171       O  \\nATOM    245  CB  ALA A  34      13.242  19.271  20.596  1.00 38.41           C  \\nANISOU  245  CB  ALA A  34     3842   6126   4624   -925    -31    171       C  \\nATOM    246  N   GLY A  35      16.366  19.006  20.177  1.00 36.76           N  \\nANISOU  246  N   GLY A  35     3094   6583   4290  -1126   -116    148       N  \\nATOM    247  CA  GLY A  35      17.717  19.531  20.272  1.00 34.72           C  \\nANISOU  247  CA  GLY A  35     2614   6605   3972  -1353   -153     93       C  \\nATOM    248  C   GLY A  35      18.414  19.593  18.926  1.00 46.44           C  \\nANISOU  248  C   GLY A  35     3940   8223   5484  -1433    -49    121       C  \\nATOM    249  O   GLY A  35      19.074  20.583  18.584  1.00 37.32           O  \\nANISOU  249  O   GLY A  35     2737   7140   4301  -1739     -5     87       O  \\nATOM    250  N   ALA A  36      18.276  18.520  18.155  1.00 40.45           N  \\nANISOU  250  N   ALA A  36     3113   7494   4762  -1178      2    178       N  \\nATOM    251  CA  ALA A  36      18.828  18.491  16.808  1.00 42.87           C  \\nANISOU  251  CA  ALA A  36     3299   7917   5073  -1237    128    195       C  \\nATOM    252  C   ALA A  36      18.148  19.551  15.931  1.00 40.70           C  \\nANISOU  252  C   ALA A  36     3285   7373   4808  -1453    223    241       C  \\nATOM    253  O   ALA A  36      18.799  20.221  15.131  1.00 36.37           O  \\nANISOU  253  O   ALA A  36     2681   6919   4220  -1701    314    242       O  \\nATOM    254  CB  ALA A  36      18.659  17.106  16.207  1.00 45.04           C  \\nANISOU  254  CB  ALA A  36     3513   8223   5378   -906    167    229       C  \\nATOM    255  N   ILE A  37      16.833  19.694  16.094  1.00 38.42           N  \\nANISOU  255  N   ILE A  37     3275   6753   4568  -1358    199    284       N  \\nATOM    256  CA  ILE A  37      16.061  20.675  15.334  1.00 41.55           C  \\nANISOU  256  CA  ILE A  37     3936   6862   4990  -1505    252    343       C  \\nATOM    257  C   ILE A  37      16.565  22.093  15.592  1.00 41.32           C  \\nANISOU  257  C   ILE A  37     3980   6789   4932  -1852    256    307       C  \\nATOM    258  O   ILE A  37      16.807  22.865  14.658  1.00 39.74           O  \\nANISOU  258  O   ILE A  37     3869   6531   4700  -2078    329    356       O  \\nATOM    259  CB  ILE A  37      14.543  20.538  15.627  1.00 37.23           C  \\nANISOU  259  CB  ILE A  37     3623   6000   4523  -1306    209    379       C  \\nATOM    260  CG1 ILE A  37      13.996  19.323  14.868  1.00 30.02           C  \\nANISOU  260  CG1 ILE A  37     2694   5088   3626  -1053    232    437       C  \\nATOM    261  CG2 ILE A  37      13.803  21.822  15.261  1.00 32.46           C  \\nANISOU  261  CG2 ILE A  37     3285   5086   3964  -1458    220    421       C  \\nATOM    262  CD1 ILE A  37      12.587  18.949  15.178  1.00 28.25           C  \\nANISOU  262  CD1 ILE A  37     2627   4630   3476   -858    191    463       C  \\nATOM    263  N   LEU A  38      16.758  22.417  16.865  1.00 38.38           N  \\nANISOU  263  N   LEU A  38     3588   6445   4550  -1916    177    219       N  \\nATOM    264  CA  LEU A  38      17.285  23.723  17.245  1.00 41.56           C  \\nANISOU  264  CA  LEU A  38     4070   6805   4915  -2267    172    158       C  \\nATOM    265  C   LEU A  38      18.745  23.982  16.801  1.00 44.09           C  \\nANISOU  265  C   LEU A  38     4141   7455   5155  -2556    218    132       C  \\nATOM    266  O   LEU A  38      19.096  25.112  16.455  1.00 46.01           O  \\nANISOU  266  O   LEU A  38     4505   7612   5365  -2896    268    130       O  \\nATOM    267  CB  LEU A  38      17.134  23.920  18.755  1.00 47.77           C  \\nANISOU  267  CB  LEU A  38     4902   7565   5685  -2268     76     51       C  \\nATOM    268  CG  LEU A  38      17.534  25.286  19.311  1.00 54.54           C  \\nANISOU  268  CG  LEU A  38     5900   8326   6499  -2630     63    -42       C  \\nATOM    269  CD1 LEU A  38      16.687  26.386  18.680  1.00 54.18           C  \\nANISOU  269  CD1 LEU A  38     6208   7855   6523  -2726    131      5       C  \\nATOM    270  CD2 LEU A  38      17.413  25.290  20.826  1.00 52.20           C  \\nANISOU  270  CD2 LEU A  38     5633   8050   6152  -2610    -32   -162       C  \\nATOM    271  N   ARG A  39      19.604  22.963  16.819  1.00 50.25           N  \\nANISOU  271  N   ARG A  39     4574   8611   5908  -2429    207    109       N  \\nATOM    272  CA  ARG A  39      20.997  23.148  16.369  1.00 51.24           C  \\nANISOU  272  CA  ARG A  39     4506   8993   5972  -2564    252     66       C  \\nATOM    273  C   ARG A  39      21.075  23.400  14.859  1.00 50.12           C  \\nANISOU  273  C   ARG A  39     4452   8782   5808  -2649    400    137       C  \\nATOM    274  O   ARG A  39      21.784  24.291  14.399  1.00 56.26           O  \\nANISOU  274  O   ARG A  39     5279   9571   6527  -2916    459    122       O  \\nATOM    275  CB  ARG A  39      21.852  21.907  16.634  1.00 58.92           C  \\nANISOU  275  CB  ARG A  39     5135  10309   6943  -2294    205     25       C  \\nATOM    276  CG  ARG A  39      22.070  21.459  18.057  1.00 63.00           C  \\nANISOU  276  CG  ARG A  39     5516  10972   7448  -2175     39    -30       C  \\nATOM    277  CD  ARG A  39      22.489  19.995  17.988  1.00 61.50           C  \\nANISOU  277  CD  ARG A  39     5077  11000   7290  -1803      9    -17       C  \\nATOM    278  NE  ARG A  39      23.185  19.739  16.725  1.00 69.30           N  \\nANISOU  278  NE  ARG A  39     5940  12102   8289  -1777    152    -17       N  \\nATOM    279  CZ  ARG A  39      23.172  18.585  16.058  1.00 66.22           C  \\nANISOU  279  CZ  ARG A  39     5452  11768   7941  -1476    213      3       C  \\nATOM    280  NH1 ARG A  39      22.499  17.541  16.523  1.00 73.56           N  \\nANISOU  280  NH1 ARG A  39     6400  12638   8912  -1162    136     41       N  \\nATOM    281  NH2 ARG A  39      23.844  18.470  14.921  1.00 52.22           N  \\nANISOU  281  NH2 ARG A  39     3578  10103   6160  -1498    356    -20       N  \\nATOM    282  N   LYS A  40      20.366  22.573  14.093  1.00 45.55           N  \\nANISOU  282  N   LYS A  40     3899   8148   5261  -2427    458    212       N  \\nATOM    283  CA  LYS A  40      20.464  22.596  12.642  1.00 51.13           C  \\nANISOU  283  CA  LYS A  40     4676   8835   5919  -2476    592    274       C  \\nATOM    284  C   LYS A  40      19.550  23.640  12.013  1.00 48.72           C  \\nANISOU  284  C   LYS A  40     4741   8171   5600  -2673    619    384       C  \\nATOM    285  O   LYS A  40      19.806  24.102  10.905  1.00 48.77           O  \\nANISOU  285  O   LYS A  40     4864   8136   5530  -2825    710    437       O  \\nATOM    286  CB  LYS A  40      20.132  21.220  12.059  1.00 49.30           C  \\nANISOU  286  CB  LYS A  40     4329   8699   5704  -2155    640    297       C  \\nATOM    287  CG  LYS A  40      21.182  20.143  12.286  1.00 49.00           C  \\nANISOU  287  CG  LYS A  40     3954   8989   5676  -1933    642    200       C  \\nATOM    288  CD  LYS A  40      21.095  19.124  11.165  1.00 60.33           C  \\nANISOU  288  CD  LYS A  40     5350  10481   7091  -1725    759    211       C  \\nATOM    289  CE  LYS A  40      21.798  17.832  11.496  1.00 64.36           C  \\nANISOU  289  CE  LYS A  40     5581  11225   7649  -1404    741    127       C  \\nATOM    290  NZ  LYS A  40      21.552  16.840  10.415  1.00 65.36           N  \\nANISOU  290  NZ  LYS A  40     5725  11353   7754  -1201    864    124       N  \\nATOM    291  N   GLY A  41      18.471  23.986  12.712  1.00 43.32           N  \\nANISOU  291  N   GLY A  41     4251   7220   4988  -2656    535    421       N  \\nATOM    292  CA  GLY A  41      17.512  24.946  12.201  1.00 42.32           C  \\nANISOU  292  CA  GLY A  41     4509   6690   4881  -2751    526    527       C  \\nATOM    293  C   GLY A  41      17.847  26.336  12.683  1.00 45.01           C  \\nANISOU  293  C   GLY A  41     5025   6864   5210  -3068    505    491       C  \\nATOM    294  O   GLY A  41      17.571  27.333  12.007  1.00 43.53           O  \\nANISOU  294  O   GLY A  41     5139   6398   5002  -3226    522    577       O  \\nATOM    295  N   GLY A  42      18.443  26.391  13.871  1.00 42.81           N  \\nANISOU  295  N   GLY A  42     4585   6740   4940  -3119    449    359       N  \\nATOM    296  CA  GLY A  42      18.941  27.632  14.435  1.00 45.37           C  \\nANISOU  296  CA  GLY A  42     5056   6947   5235  -3399    424    286       C  \\nATOM    297  C   GLY A  42      17.963  28.308  15.367  1.00 44.74           C  \\nANISOU  297  C   GLY A  42     5250   6514   5236  -3420    353    250       C  \\nATOM    298  O   GLY A  42      16.841  27.844  15.531  1.00 42.31           O  \\nANISOU  298  O   GLY A  42     5035   6025   5017  -3127    316    283       O  \\nATOM    299  N   PRO A  43      18.385  29.434  15.958  1.00 47.27           N  \\nANISOU  299  N   PRO A  43     5729   6707   5525  -3663    332    162       N  \\nATOM    300  CA  PRO A  43      17.715  30.149  17.053  1.00 51.32           C  \\nANISOU  300  CA  PRO A  43     6489   6918   6093  -3708    276     64       C  \\nATOM    301  C   PRO A  43      16.321  30.666  16.707  1.00 53.24           C  \\nANISOU  301  C   PRO A  43     7105   6667   6455  -3577    281    148       C  \\nATOM    302  O   PRO A  43      15.496  30.813  17.609  1.00 45.84           O  \\nANISOU  302  O   PRO A  43     6306   5516   5595  -3389    242     56       O  \\nATOM    303  CB  PRO A  43      18.646  31.336  17.317  1.00 58.07           C  \\nANISOU  303  CB  PRO A  43     7456   7743   6863  -4023    281    -18       C  \\nATOM    304  CG  PRO A  43      19.958  30.961  16.704  1.00 52.61           C  \\nANISOU  304  CG  PRO A  43     6456   7465   6069  -4130    320      2       C  \\nATOM    305  CD  PRO A  43      19.636  30.096  15.545  1.00 50.54           C  \\nANISOU  305  CD  PRO A  43     6091   7280   5833  -3930    377    140       C  \\nATOM    306  N   SER A  44      16.068  30.948  15.429  1.00 49.10           N  \\nANISOU  306  N   SER A  44     6738   5979   5941  -3580    319    312       N  \\nATOM    307  CA  SER A  44      14.796  31.547  15.020  1.00 48.91           C  \\nANISOU  307  CA  SER A  44     7073   5477   6031  -3409    291    411       C  \\nATOM    308  C   SER A  44      13.615  30.639  15.341  1.00 43.62           C  \\nANISOU  308  C   SER A  44     6316   4779   5478  -2939    244    405       C  \\nATOM    309  O   SER A  44      12.518  31.112  15.625  1.00 51.16           O  \\nANISOU  309  O   SER A  44     7496   5384   6559  -2736    210    394       O  \\nATOM    310  CB  SER A  44      14.806  31.929  13.530  1.00 51.34           C  \\nANISOU  310  CB  SER A  44     7544   5670   6295  -3474    313    607       C  \\nATOM    311  OG  SER A  44      14.619  30.793  12.696  1.00 48.63           O  \\nANISOU  311  OG  SER A  44     7004   5551   5922  -3309    327    720       O  \\nATOM    312  N   ILE A  45      13.847  29.331  15.314  1.00 46.48           N  \\nANISOU  312  N   ILE A  45     6348   5511   5802  -2769    247    404       N  \\nATOM    313  CA  ILE A  45      12.802  28.361  15.651  1.00 38.55           C  \\nANISOU  313  CA  ILE A  45     5248   4512   4889  -2368    210    395       C  \\nATOM    314  C   ILE A  45      12.299  28.508  17.099  1.00 39.87           C  \\nANISOU  314  C   ILE A  45     5454   4580   5115  -2258    192    231       C  \\nATOM    315  O   ILE A  45      11.090  28.620  17.334  1.00 37.61           O  \\nANISOU  315  O   ILE A  45     5299   4041   4949  -2011    180    221       O  \\nATOM    316  CB  ILE A  45      13.273  26.913  15.383  1.00 41.19           C  \\nANISOU  316  CB  ILE A  45     5249   5244   5157  -2235    222    417       C  \\nATOM    317  CG1 ILE A  45      13.634  26.744  13.899  1.00 43.60           C  \\nANISOU  317  CG1 ILE A  45     5539   5634   5394  -2321    265    562       C  \\nATOM    318  CG2 ILE A  45      12.206  25.917  15.809  1.00 35.59           C  \\nANISOU  318  CG2 ILE A  45     4469   4524   4531  -1867    188    404       C  \\nATOM    319  CD1 ILE A  45      13.924  25.320  13.480  1.00 35.27           C  \\nANISOU  319  CD1 ILE A  45     4207   4904   4291  -2143    290    578       C  \\nATOM    320  N   GLN A  46      13.228  28.515  18.056  1.00 40.44           N  \\nANISOU  320  N   GLN A  46     5402   4870   5092  -2448    190     98       N  \\nATOM    321  CA  GLN A  46      12.897  28.776  19.458  1.00 39.58           C  \\nANISOU  321  CA  GLN A  46     5367   4680   4991  -2416    179    -72       C  \\nATOM    322  C   GLN A  46      12.301  30.165  19.669  1.00 41.70           C  \\nANISOU  322  C   GLN A  46     5995   4508   5340  -2500    206   -137       C  \\nATOM    323  O   GLN A  46      11.381  30.338  20.462  1.00 41.57           O  \\nANISOU  323  O   GLN A  46     6099   4297   5397  -2316    226   -243       O  \\nATOM    324  CB  GLN A  46      14.124  28.646  20.356  1.00 40.78           C  \\nANISOU  324  CB  GLN A  46     5339   5159   4997  -2659    145   -191       C  \\nATOM    325  CG  GLN A  46      13.778  28.810  21.827  1.00 41.25           C  \\nANISOU  325  CG  GLN A  46     5485   5167   5022  -2631    129   -367       C  \\nATOM    326  CD  GLN A  46      12.845  27.701  22.311  1.00 38.74           C  \\nANISOU  326  CD  GLN A  46     5070   4909   4739  -2265    128   -361       C  \\nATOM    327  OE1 GLN A  46      13.262  26.552  22.449  1.00 38.05           O  \\nANISOU  327  OE1 GLN A  46     4731   5141   4584  -2160     80   -316       O  \\nATOM    328  NE2 GLN A  46      11.574  28.035  22.533  1.00 38.33           N  \\nANISOU  328  NE2 GLN A  46     5218   4547   4801  -2068    187   -402       N  \\nATOM    329  N   GLU A  47      12.834  31.157  18.965  1.00 43.97           N  \\nANISOU  329  N   GLU A  47     6465   4630   5610  -2782    216    -79       N  \\nATOM    330  CA  GLU A  47      12.320  32.519  19.097  1.00 51.53           C  \\nANISOU  330  CA  GLU A  47     7808   5119   6650  -2863    236   -130       C  \\nATOM    331  C   GLU A  47      10.848  32.574  18.711  1.00 51.23           C  \\nANISOU  331  C   GLU A  47     7917   4756   6792  -2477    230    -56       C  \\nATOM    332  O   GLU A  47      10.036  33.180  19.409  1.00 49.29           O  \\nANISOU  332  O   GLU A  47     7867   4210   6650  -2335    255   -179       O  \\nATOM    333  CB  GLU A  47      13.126  33.495  18.246  1.00 49.20           C  \\nANISOU  333  CB  GLU A  47     7708   4688   6297  -3241    245    -38       C  \\nATOM    334  CG  GLU A  47      14.516  33.761  18.777  1.00 68.48           C  \\nANISOU  334  CG  GLU A  47    10047   7391   8581  -3678    255   -152       C  \\nATOM    335  CD  GLU A  47      15.304  34.710  17.893  1.00 73.98           C  \\nANISOU  335  CD  GLU A  47    10867   8033   9211  -3907    271    -53       C  \\nATOM    336  OE1 GLU A  47      16.277  35.314  18.397  1.00 81.43           O  \\nANISOU  336  OE1 GLU A  47    11796   9098  10046  -4175    275   -162       O  \\nATOM    337  OE2 GLU A  47      14.955  34.844  16.696  1.00 69.41           O  \\nANISOU  337  OE2 GLU A  47    10402   7299   8671  -3830    277    133       O  \\nATOM    338  N   GLU A  48      10.511  31.921  17.606  1.00 43.82           N  \\nANISOU  338  N   GLU A  48     6867   3898   5886  -2306    196    133       N  \\nATOM    339  CA  GLU A  48       9.131  31.871  17.147  1.00 47.31           C  \\nANISOU  339  CA  GLU A  48     7391   4093   6493  -1944    163    219       C  \\nATOM    340  C   GLU A  48       8.253  31.097  18.136  1.00 43.00           C  \\nANISOU  340  C   GLU A  48     6672   3647   6020  -1637    190     87       C  \\nATOM    341  O   GLU A  48       7.130  31.502  18.422  1.00 42.15           O  \\nANISOU  341  O   GLU A  48     6681   3268   6065  -1391    201     35       O  \\nATOM    342  CB  GLU A  48       9.056  31.261  15.743  1.00 46.56           C  \\nANISOU  342  CB  GLU A  48     7204   4111   6374  -1875    110    443       C  \\nATOM    343  CG  GLU A  48       7.690  31.366  15.087  1.00 47.94           C  \\nANISOU  343  CG  GLU A  48     7486   4023   6708  -1548     37    562       C  \\nATOM    344  CD  GLU A  48       7.725  31.076  13.591  1.00 49.55           C  \\nANISOU  344  CD  GLU A  48     7700   4280   6849  -1567    -31    794       C  \\nATOM    345  OE1 GLU A  48       8.826  31.079  12.995  1.00 51.62           O  \\nANISOU  345  OE1 GLU A  48     7958   4705   6950  -1867      1    863       O  \\nATOM    346  OE2 GLU A  48       6.640  30.851  13.013  1.00 44.98           O  \\nANISOU  346  OE2 GLU A  48     7124   3594   6373  -1290   -116    899       O  \\nATOM    347  N   CYS A  49       8.776  29.994  18.665  1.00 45.54           N  \\nANISOU  347  N   CYS A  49     6717   4357   6231  -1652    206     34       N  \\nATOM    348  CA  CYS A  49       8.096  29.251  19.731  1.00 41.42           C  \\nANISOU  348  CA  CYS A  49     6057   3949   5732  -1431    244    -94       C  \\nATOM    349  C   CYS A  49       7.775  30.123  20.942  1.00 45.18           C  \\nANISOU  349  C   CYS A  49     6720   4210   6236  -1457    309   -304       C  \\nATOM    350  O   CYS A  49       6.733  29.955  21.578  1.00 44.88           O  \\nANISOU  350  O   CYS A  49     6673   4094   6287  -1217    365   -400       O  \\nATOM    351  CB  CYS A  49       8.940  28.059  20.183  1.00 35.91           C  \\nANISOU  351  CB  CYS A  49     5088   3675   4880  -1496    233   -111       C  \\nATOM    352  SG  CYS A  49       8.793  26.592  19.138  1.00 33.27           S  \\nANISOU  352  SG  CYS A  49     4506   3593   4543  -1304    193     68       S  \\nATOM    353  N   ASP A  50       8.691  31.033  21.266  1.00 47.28           N  \\nANISOU  353  N   ASP A  50     7150   4398   6415  -1769    313   -388       N  \\nATOM    354  CA  ASP A  50       8.512  31.947  22.389  1.00 51.60           C  \\nANISOU  354  CA  ASP A  50     7920   4722   6962  -1843    380   -609       C  \\nATOM    355  C   ASP A  50       7.377  32.922  22.101  1.00 53.58           C  \\nANISOU  355  C   ASP A  50     8433   4505   7419  -1628    418   -624       C  \\nATOM    356  O   ASP A  50       6.638  33.298  23.008  1.00 54.89           O  \\nANISOU  356  O   ASP A  50     8708   4498   7650  -1486    505   -812       O  \\nATOM    357  CB  ASP A  50       9.804  32.718  22.693  1.00 52.79           C  \\nANISOU  357  CB  ASP A  50     8198   4894   6966  -2270    361   -690       C  \\nATOM    358  CG  ASP A  50      10.939  31.814  23.160  1.00 53.93           C  \\nANISOU  358  CG  ASP A  50     8056   5522   6914  -2462    309   -704       C  \\nATOM    359  OD1 ASP A  50      10.671  30.666  23.578  1.00 57.14           O  \\nANISOU  359  OD1 ASP A  50     8229   6197   7283  -2259    301   -697       O  \\nATOM    360  OD2 ASP A  50      12.103  32.260  23.113  1.00 53.53           O  \\nANISOU  360  OD2 ASP A  50     8012   5579   6747  -2820    272   -720       O  \\nATOM    361  N   ARG A  51       7.241  33.325  20.838  1.00 49.94           N  \\nANISOU  361  N   ARG A  51     8075   3845   7056  -1596    352   -428       N  \\nATOM    362  CA  ARG A  51       6.148  34.208  20.433  1.00 57.18           C  \\nANISOU  362  CA  ARG A  51     9228   4314   8182  -1347    350   -402       C  \\nATOM    363  C   ARG A  51       4.802  33.504  20.642  1.00 55.84           C  \\nANISOU  363  C   ARG A  51     8859   4198   8161   -929    379   -427       C  \\nATOM    364  O   ARG A  51       3.843  34.108  21.124  1.00 55.19           O  \\nANISOU  364  O   ARG A  51     8895   3836   8237   -695    443   -558       O  \\nATOM    365  CB  ARG A  51       6.317  34.699  18.982  1.00 49.38           C  \\nANISOU  365  CB  ARG A  51     8397   3133   7234  -1412    245   -150       C  \\nATOM    366  CG  ARG A  51       7.612  35.499  18.738  1.00 60.48           C  \\nANISOU  366  CG  ARG A  51    10022   4463   8495  -1866    237   -124       C  \\nATOM    367  CD  ARG A  51       7.581  36.360  17.462  1.00 65.05           C  \\nANISOU  367  CD  ARG A  51    10895   4697   9125  -1932    156    100       C  \\nATOM    368  NE  ARG A  51       7.471  35.597  16.214  1.00 70.83           N  \\nANISOU  368  NE  ARG A  51    11456   5627   9830  -1840     73    350       N  \\nATOM    369  CZ  ARG A  51       8.485  35.328  15.388  1.00 72.65           C  \\nANISOU  369  CZ  ARG A  51    11626   6088   9889  -2144     59    495       C  \\nATOM    370  NH1 ARG A  51       9.718  35.741  15.671  1.00 73.52           N  \\nANISOU  370  NH1 ARG A  51    11795   6289   9850  -2570    118    423       N  \\nATOM    371  NH2 ARG A  51       8.266  34.639  14.273  1.00 64.16           N  \\nANISOU  371  NH2 ARG A  51    10421   5170   8785  -2033     -6    700       N  \\nATOM    372  N   ILE A  52       4.747  32.221  20.304  1.00 49.52           N  \\nANISOU  372  N   ILE A  52     7750   3758   7308   -845    344   -316       N  \\nATOM    373  CA  ILE A  52       3.553  31.408  20.543  1.00 49.58           C  \\nANISOU  373  CA  ILE A  52     7537   3876   7426   -512    378   -343       C  \\nATOM    374  C   ILE A  52       3.255  31.252  22.038  1.00 47.18           C  \\nANISOU  374  C   ILE A  52     7189   3653   7083   -474    519   -597       C  \\nATOM    375  O   ILE A  52       2.124  31.440  22.475  1.00 51.34           O  \\nANISOU  375  O   ILE A  52     7702   4045   7759   -214    602   -711       O  \\nATOM    376  CB  ILE A  52       3.687  30.003  19.906  1.00 43.73           C  \\nANISOU  376  CB  ILE A  52     6509   3503   6604   -485    315   -182       C  \\nATOM    377  CG1 ILE A  52       3.881  30.111  18.395  1.00 44.61           C  \\nANISOU  377  CG1 ILE A  52     6668   3550   6732   -515    189     59       C  \\nATOM    378  CG2 ILE A  52       2.464  29.148  20.208  1.00 42.53           C  \\nANISOU  378  CG2 ILE A  52     6138   3468   6553   -194    357   -218       C  \\nATOM    379  CD1 ILE A  52       3.995  28.764  17.715  1.00 48.85           C  \\nANISOU  379  CD1 ILE A  52     6955   4419   7187   -486    138    196       C  \\nATOM    380  N   GLY A  53       4.274  30.909  22.820  1.00 48.34           N  \\nANISOU  380  N   GLY A  53     7307   4038   7021   -736    547   -687       N  \\nATOM    381  CA  GLY A  53       4.095  30.717  24.250  1.00 43.61           C  \\nANISOU  381  CA  GLY A  53     6694   3546   6331   -744    668   -914       C  \\nATOM    382  C   GLY A  53       3.538  29.346  24.590  1.00 48.09           C  \\nANISOU  382  C   GLY A  53     6987   4427   6857   -586    703   -890       C  \\nATOM    383  O   GLY A  53       3.363  28.502  23.720  1.00 48.39           O  \\nANISOU  383  O   GLY A  53     6844   4607   6935   -484    626   -705       O  \\nATOM    384  N   LYS A  54       3.247  29.138  25.868  1.00 47.40           N  \\nANISOU  384  N   LYS A  54     6900   4436   6675   -585    825  -1082       N  \\nATOM    385  CA  LYS A  54       2.770  27.863  26.374  1.00 48.16           C  \\nANISOU  385  CA  LYS A  54     6786   4819   6694   -488    875  -1072       C  \\nATOM    386  C   LYS A  54       1.491  27.377  25.697  1.00 51.58           C  \\nANISOU  386  C   LYS A  54     7045   5225   7329   -199    899   -982       C  \\nATOM    387  O   LYS A  54       0.503  28.106  25.615  1.00 54.30           O  \\nANISOU  387  O   LYS A  54     7429   5335   7869     -2    975  -1067       O  \\nATOM    388  CB  LYS A  54       2.534  27.963  27.887  1.00 47.76           C  \\nANISOU  388  CB  LYS A  54     6823   4820   6505   -545   1029  -1315       C  \\nATOM    389  CG  LYS A  54       1.956  26.700  28.514  1.00 46.23           C  \\nANISOU  389  CG  LYS A  54     6455   4898   6212   -469   1104  -1310       C  \\nATOM    390  CD  LYS A  54       1.833  26.847  30.030  1.00 51.25           C  \\nANISOU  390  CD  LYS A  54     7216   5597   6660   -572   1259  -1547       C  \\nATOM    391  CE  LYS A  54       1.205  25.604  30.662  1.00 51.20           C  \\nANISOU  391  CE  LYS A  54     7068   5846   6539   -521   1349  -1529       C  \\nATOM    392  NZ  LYS A  54       1.075  25.694  32.154  1.00 56.00           N  \\nANISOU  392  NZ  LYS A  54     7818   6539   6922   -644   1510  -1750       N  \\nATOM    393  N   ILE A  55       1.509  26.137  25.218  1.00 43.06           N  \\nANISOU  393  N   ILE A  55     5769   4386   6205   -171    830   -817       N  \\nATOM    394  CA  ILE A  55       0.276  25.508  24.778  1.00 41.20           C  \\nANISOU  394  CA  ILE A  55     5350   4184   6120     55    860   -758       C  \\nATOM    395  C   ILE A  55      -0.166  24.369  25.700  1.00 41.50           C  \\nANISOU  395  C   ILE A  55     5263   4469   6035     51    968   -814       C  \\nATOM    396  O   ILE A  55       0.263  24.250  26.848  1.00 41.71           O  \\nANISOU  396  O   ILE A  55     5374   4601   5874    -87   1045   -935       O  \\nATOM    397  CB  ILE A  55       0.339  25.021  23.317  1.00 41.47           C  \\nANISOU  397  CB  ILE A  55     5278   4246   6233    112    703   -524       C  \\nATOM    398  CG1 ILE A  55       1.358  23.883  23.167  1.00 38.47           C  \\nANISOU  398  CG1 ILE A  55     4830   4126   5660    -40    623   -402       C  \\nATOM    399  CG2 ILE A  55       0.616  26.206  22.375  1.00 41.36           C  \\nANISOU  399  CG2 ILE A  55     5415   3962   6337    118    606   -453       C  \\nATOM    400  CD1 ILE A  55       1.440  23.306  21.779  1.00 33.93           C  \\nANISOU  400  CD1 ILE A  55     4163   3597   5131      2    495   -200       C  \\nATOM    401  N   ARG A  56      -1.050  23.540  25.180  1.00 37.24           N  \\nANISOU  401  N   ARG A  56     4538   4020   5592    185    966   -722       N  \\nATOM    402  CA  ARG A  56      -1.658  22.497  25.966  1.00 38.87           C  \\nANISOU  402  CA  ARG A  56     4636   4427   5705    176   1083   -768       C  \\nATOM    403  C   ARG A  56      -1.095  21.142  25.521  1.00 33.32           C  \\nANISOU  403  C   ARG A  56     3869   3919   4874     93    968   -582       C  \\nATOM    404  O   ARG A  56      -0.866  20.930  24.336  1.00 31.26           O  \\nANISOU  404  O   ARG A  56     3555   3636   4687    132    830   -427       O  \\nATOM    405  CB  ARG A  56      -3.151  22.591  25.706  1.00 50.72           C  \\nANISOU  405  CB  ARG A  56     5962   5881   7428    377   1174   -818       C  \\nATOM    406  CG  ARG A  56      -3.988  21.721  26.521  1.00 57.43           C  \\nANISOU  406  CG  ARG A  56     6691   6917   8214    359   1337   -895       C  \\nATOM    407  CD  ARG A  56      -4.408  22.357  27.837  1.00 59.88           C  \\nANISOU  407  CD  ARG A  56     7070   7205   8476    352   1563  -1145       C  \\nATOM    408  NE  ARG A  56      -5.508  21.532  28.298  1.00 52.45           N  \\nANISOU  408  NE  ARG A  56     5948   6446   7536    363   1726  -1196       N  \\nATOM    409  CZ  ARG A  56      -5.349  20.320  28.807  1.00 38.82           C  \\nANISOU  409  CZ  ARG A  56     4234   4921   5596    192   1757  -1126       C  \\nATOM    410  NH1 ARG A  56      -4.129  19.818  28.998  1.00 35.12           N  \\nANISOU  410  NH1 ARG A  56     3940   4503   4901     33   1633  -1015       N  \\nATOM    411  NH2 ARG A  56      -6.416  19.627  29.155  1.00 40.95           N  \\nANISOU  411  NH2 ARG A  56     4341   5341   5877    177   1915  -1171       N  \\nATOM    412  N   VAL A  57      -0.821  20.240  26.457  1.00 42.67           N  \\nANISOU  412  N   VAL A  57     5082   5278   5851    -21   1021   -596       N  \\nATOM    413  CA  VAL A  57      -0.412  18.886  26.070  1.00 36.22           C  \\nANISOU  413  CA  VAL A  57     4219   4612   4932    -62    923   -427       C  \\nATOM    414  C   VAL A  57      -1.515  18.305  25.192  1.00 38.37           C  \\nANISOU  414  C   VAL A  57     4333   4880   5366     48    925   -351       C  \\nATOM    415  O   VAL A  57      -2.688  18.377  25.563  1.00 34.99           O  \\nANISOU  415  O   VAL A  57     3812   4455   5028    102   1061   -447       O  \\nATOM    416  CB  VAL A  57      -0.194  17.955  27.290  1.00 35.42           C  \\nANISOU  416  CB  VAL A  57     4195   4673   4591   -179    985   -444       C  \\nATOM    417  CG1 VAL A  57      -0.117  16.502  26.828  1.00 30.76           C  \\nANISOU  417  CG1 VAL A  57     3564   4181   3943   -178    908   -275       C  \\nATOM    418  CG2 VAL A  57       1.075  18.349  28.081  1.00 29.06           C  \\nANISOU  418  CG2 VAL A  57     3533   3919   3590   -305    923   -486       C  \\nATOM    419  N   GLY A  58      -1.145  17.755  24.030  1.00 33.16           N  \\nANISOU  419  N   GLY A  58     3632   4229   4740     72    782   -193       N  \\nATOM    420  CA  GLY A  58      -2.112  17.248  23.067  1.00 25.78           C  \\nANISOU  420  CA  GLY A  58     2559   3293   3945    152    750   -118       C  \\nATOM    421  C   GLY A  58      -2.357  18.198  21.891  1.00 37.99           C  \\nANISOU  421  C   GLY A  58     4055   4701   5678    263    649    -75       C  \\nATOM    422  O   GLY A  58      -2.958  17.823  20.882  1.00 37.19           O  \\nANISOU  422  O   GLY A  58     3854   4606   5673    320    569     14       O  \\nATOM    423  N   GLU A  59      -1.901  19.441  22.017  1.00 34.12           N  \\nANISOU  423  N   GLU A  59     3657   4077   5230    282    645   -134       N  \\nATOM    424  CA  GLU A  59      -2.021  20.414  20.933  1.00 30.53           C  \\nANISOU  424  CA  GLU A  59     3214   3456   4932    376    537    -72       C  \\nATOM    425  C   GLU A  59      -0.754  20.417  20.074  1.00 29.62           C  \\nANISOU  425  C   GLU A  59     3201   3337   4717    277    409     57       C  \\nATOM    426  O   GLU A  59       0.234  19.766  20.420  1.00 28.43           O  \\nANISOU  426  O   GLU A  59     3089   3313   4399    161    410     74       O  \\nATOM    427  CB  GLU A  59      -2.304  21.824  21.495  1.00 29.56           C  \\nANISOU  427  CB  GLU A  59     3166   3142   4923    452    612   -211       C  \\nATOM    428  CG  GLU A  59      -3.789  22.105  21.671  1.00 38.06           C  \\nANISOU  428  CG  GLU A  59     4088   4175   6197    637    694   -304       C  \\nATOM    429  CD  GLU A  59      -4.089  23.266  22.612  1.00 56.76           C  \\nANISOU  429  CD  GLU A  59     6534   6383   8647    717    834   -501       C  \\nATOM    430  OE1 GLU A  59      -3.182  24.082  22.906  1.00 49.79           O  \\nANISOU  430  OE1 GLU A  59     5859   5362   7696    630    831   -549       O  \\nATOM    431  OE2 GLU A  59      -5.247  23.348  23.080  1.00 66.75           O  \\nANISOU  431  OE2 GLU A  59     7648   7670  10044    856    959   -625       O  \\nATOM    432  N   ALA A  60      -0.800  21.147  18.960  1.00 32.03           N  \\nANISOU  432  N   ALA A  60     3545   3505   5122    327    300    151       N  \\nATOM    433  CA  ALA A  60       0.342  21.296  18.063  1.00 29.83           C  \\nANISOU  433  CA  ALA A  60     3364   3220   4750    215    202    267       C  \\nATOM    434  C   ALA A  60       0.463  22.758  17.669  1.00 36.60           C  \\nANISOU  434  C   ALA A  60     4366   3842   5698    220    157    282       C  \\nATOM    435  O   ALA A  60      -0.528  23.478  17.662  1.00 35.04           O  \\nANISOU  435  O   ALA A  60     4168   3477   5667    371    151    252       O  \\nATOM    436  CB  ALA A  60       0.167  20.446  16.829  1.00 32.14           C  \\nANISOU  436  CB  ALA A  60     3592   3599   5022    232    102    409       C  \\nATOM    437  N   ALA A  61       1.675  23.198  17.349  1.00 32.60           N  \\nANISOU  437  N   ALA A  61     3981   3319   5086     56    126    328       N  \\nATOM    438  CA  ALA A  61       1.886  24.566  16.884  1.00 38.93           C  \\nANISOU  438  CA  ALA A  61     4966   3875   5950     15     79    365       C  \\nATOM    439  C   ALA A  61       2.905  24.595  15.759  1.00 35.48           C  \\nANISOU  439  C   ALA A  61     4603   3483   5394   -154      5    512       C  \\nATOM    440  O   ALA A  61       3.850  23.809  15.738  1.00 39.24           O  \\nANISOU  440  O   ALA A  61     5002   4184   5725   -279     28    520       O  \\nATOM    441  CB  ALA A  61       2.339  25.459  18.022  1.00 38.92           C  \\nANISOU  441  CB  ALA A  61     5088   3761   5939    -77    169    208       C  \\nATOM    442  N   VAL A  62       2.715  25.517  14.829  1.00 41.66           N  \\nANISOU  442  N   VAL A  62     5542   4050   6237   -151    -82    627       N  \\nATOM    443  CA  VAL A  62       3.568  25.576  13.659  1.00 40.86           C  \\nANISOU  443  CA  VAL A  62     5527   3991   6007   -326   -140    776       C  \\nATOM    444  C   VAL A  62       4.492  26.794  13.704  1.00 33.29           C  \\nANISOU  444  C   VAL A  62     4782   2864   5002   -545   -117    776       C  \\nATOM    445  O   VAL A  62       4.062  27.903  14.024  1.00 35.39           O  \\nANISOU  445  O   VAL A  62     5223   2838   5386   -498   -129    744       O  \\nATOM    446  CB  VAL A  62       2.721  25.584  12.368  1.00 46.08           C  \\nANISOU  446  CB  VAL A  62     6231   4561   6715   -206   -276    949       C  \\nATOM    447  CG1 VAL A  62       3.605  25.371  11.142  1.00 41.48           C  \\nANISOU  447  CG1 VAL A  62     5720   4089   5950   -401   -310   1092       C  \\nATOM    448  CG2 VAL A  62       1.643  24.506  12.448  1.00 42.23           C  \\nANISOU  448  CG2 VAL A  62     5533   4214   6299     -1   -302    925       C  \\nATOM    449  N   THR A  63       5.771  26.563  13.414  1.00 33.38           N  \\nANISOU  449  N   THR A  63     4773   3064   4846   -789    -75    798       N  \\nATOM    450  CA  THR A  63       6.727  27.642  13.195  1.00 34.64           C  \\nANISOU  450  CA  THR A  63     5126   3102   4933  -1062    -56    827       C  \\nATOM    451  C   THR A  63       7.437  27.432  11.860  1.00 44.64           C  \\nANISOU  451  C   THR A  63     6416   4497   6048  -1238    -72    984       C  \\nATOM    452  O   THR A  63       7.304  26.372  11.239  1.00 45.21           O  \\nANISOU  452  O   THR A  63     6339   4776   6062  -1147    -85   1038       O  \\nATOM    453  CB  THR A  63       7.799  27.702  14.313  1.00 43.86           C  \\nANISOU  453  CB  THR A  63     6218   4424   6023  -1260     36    663       C  \\nATOM    454  OG1 THR A  63       8.663  26.556  14.234  1.00 33.04           O  \\nANISOU  454  OG1 THR A  63     4603   3428   4522  -1324     74    652       O  \\nATOM    455  CG2 THR A  63       7.147  27.778  15.674  1.00 34.42           C  \\nANISOU  455  CG2 THR A  63     5001   3147   4931  -1106     69    493       C  \\nATOM    456  N   GLY A  64       8.198  28.438  11.431  1.00 40.92           N  \\nANISOU  456  N   GLY A  64     6146   3902   5501  -1511    -56   1047       N  \\nATOM    457  CA  GLY A  64       9.088  28.298  10.292  1.00 41.01           C  \\nANISOU  457  CA  GLY A  64     6171   4078   5333  -1748    -24   1167       C  \\nATOM    458  C   GLY A  64      10.221  27.369  10.683  1.00 46.30           C  \\nANISOU  458  C   GLY A  64     6549   5151   5893  -1863     85   1050       C  \\nATOM    459  O   GLY A  64      10.458  27.148  11.867  1.00 41.65           O  \\nANISOU  459  O   GLY A  64     5816   4660   5350  -1828    116    896       O  \\nATOM    460  N   ALA A  65      10.928  26.819   9.702  1.00 36.66           N  \\nANISOU  460  N   ALA A  65     5240   4170   4520  -1990    141   1119       N  \\nATOM    461  CA  ALA A  65      11.937  25.816  10.018  1.00 36.30           C  \\nANISOU  461  CA  ALA A  65     4884   4516   4394  -2027    237   1006       C  \\nATOM    462  C   ALA A  65      13.369  26.319   9.980  1.00 37.74           C  \\nANISOU  462  C   ALA A  65     4997   4886   4456  -2382    340    966       C  \\nATOM    463  O   ALA A  65      14.294  25.516   9.914  1.00 43.66           O  \\nANISOU  463  O   ALA A  65     5477   5987   5124  -2421    424    899       O  \\nATOM    464  CB  ALA A  65      11.782  24.605   9.124  1.00 36.69           C  \\nANISOU  464  CB  ALA A  65     4810   4762   4371  -1874    255   1052       C  \\nATOM    465  N   GLY A  66      13.555  27.636  10.041  1.00 40.04           N  \\nANISOU  465  N   GLY A  66     5524   4945   4744  -2639    335   1000       N  \\nATOM    466  CA  GLY A  66      14.893  28.205  10.052  1.00 42.45           C  \\nANISOU  466  CA  GLY A  66     5769   5424   4937  -3031    433    956       C  \\nATOM    467  C   GLY A  66      15.742  27.733   8.883  1.00 62.93           C  \\nANISOU  467  C   GLY A  66     8224   8313   7373  -3123    537    992       C  \\nATOM    468  O   GLY A  66      15.320  27.809   7.723  1.00 43.94           O  \\nANISOU  468  O   GLY A  66     6003   5795   4898  -3103    533   1127       O  \\nATOM    469  N   ASN A  67      16.931  27.224   9.195  1.00 43.99           N  \\nANISOU  469  N   ASN A  67     5497   6294   4922  -3188    618    856       N  \\nATOM    470  CA  ASN A  67      17.875  26.772   8.172  1.00 45.42           C  \\nANISOU  470  CA  ASN A  67     5517   6773   4969  -3251    734    842       C  \\nATOM    471  C   ASN A  67      17.674  25.312   7.749  1.00 54.16           C  \\nANISOU  471  C   ASN A  67     6411   8106   6060  -2993    779    835       C  \\nATOM    472  O   ASN A  67      18.470  24.770   6.981  1.00 51.71           O  \\nANISOU  472  O   ASN A  67     5938   8058   5651  -3002    890    790       O  \\nATOM    473  CB  ASN A  67      19.320  26.988   8.633  1.00 47.77           C  \\nANISOU  473  CB  ASN A  67     5563   7363   5224  -3434    795    698       C  \\nATOM    474  CG  ASN A  67      19.676  28.449   8.750  1.00 58.96           C  \\nANISOU  474  CG  ASN A  67     7212   8582   6607  -3740    780    707       C  \\nATOM    475  OD1 ASN A  67      19.367  29.252   7.861  1.00 52.23           O  \\nANISOU  475  OD1 ASN A  67     6667   7489   5691  -3872    790    827       O  \\nATOM    476  ND2 ASN A  67      20.322  28.813   9.856  1.00 54.91           N  \\nANISOU  476  ND2 ASN A  67     6570   8164   6128  -3854    746    583       N  \\nATOM    477  N   LEU A  68      16.617  24.677   8.253  1.00 46.62           N  \\nANISOU  477  N   LEU A  68     5467   7046   5202  -2765    702    871       N  \\nATOM    478  CA  LEU A  68      16.264  23.330   7.803  1.00 44.93           C  \\nANISOU  478  CA  LEU A  68     5121   6973   4976  -2479    723    864       C  \\nATOM    479  C   LEU A  68      15.641  23.366   6.399  1.00 54.09           C  \\nANISOU  479  C   LEU A  68     6529   7997   6024  -2503    738   1005       C  \\nATOM    480  O   LEU A  68      14.997  24.347   6.012  1.00 39.77           O  \\nANISOU  480  O   LEU A  68     5027   5883   4200  -2620    663   1140       O  \\nATOM    481  CB  LEU A  68      15.333  22.642   8.810  1.00 43.86           C  \\nANISOU  481  CB  LEU A  68     4953   6716   4995  -2125    594    813       C  \\nATOM    482  CG  LEU A  68      15.918  22.376  10.209  1.00 37.62           C  \\nANISOU  482  CG  LEU A  68     3920   6091   4282  -2064    564    678       C  \\nATOM    483  CD1 LEU A  68      14.833  22.018  11.181  1.00 38.37           C  \\nANISOU  483  CD1 LEU A  68     4083   5991   4504  -1789    444    656       C  \\nATOM    484  CD2 LEU A  68      16.972  21.282  10.174  1.00 38.87           C  \\nANISOU  484  CD2 LEU A  68     3736   6638   4393  -1976    649    583       C  \\nATOM    485  N   PRO A  69      15.836  22.295   5.622  1.00 53.09           N  \\nANISOU  485  N   PRO A  69     6281   8085   5804  -2386    827    975       N  \\nATOM    486  CA  PRO A  69      15.320  22.297   4.251  1.00 52.93           C  \\nANISOU  486  CA  PRO A  69     6501   7974   5636  -2441    842   1099       C  \\nATOM    487  C   PRO A  69      13.816  22.015   4.167  1.00 50.79           C  \\nANISOU  487  C   PRO A  69     6429   7421   5447  -2178    663   1185       C  \\nATOM    488  O   PRO A  69      13.325  21.670   3.100  1.00 63.81           O  \\nANISOU  488  O   PRO A  69     8226   9045   6975  -2157    650   1261       O  \\nATOM    489  CB  PRO A  69      16.116  21.170   3.592  1.00 52.32           C  \\nANISOU  489  CB  PRO A  69     6204   8223   5451  -2361   1002    975       C  \\nATOM    490  CG  PRO A  69      16.356  20.204   4.713  1.00 49.30           C  \\nANISOU  490  CG  PRO A  69     5516   8006   5211  -2110    999    837       C  \\nATOM    491  CD  PRO A  69      16.574  21.058   5.937  1.00 49.13           C  \\nANISOU  491  CD  PRO A  69     5436   7912   5318  -2202    918    822       C  \\nATOM    492  N   VAL A  70      13.096  22.151   5.273  1.00 47.31           N  \\nANISOU  492  N   VAL A  70     5983   6793   5199  -1994    531   1162       N  \\nATOM    493  CA  VAL A  70      11.641  22.054   5.245  1.00 40.29           C  \\nANISOU  493  CA  VAL A  70     5258   5644   4406  -1773    365   1242       C  \\nATOM    494  C   VAL A  70      11.007  23.426   5.355  1.00 37.00           C  \\nANISOU  494  C   VAL A  70     5097   4899   4062  -1859    245   1366       C  \\nATOM    495  O   VAL A  70      11.645  24.393   5.751  1.00 41.88           O  \\nANISOU  495  O   VAL A  70     5766   5458   4687  -2068    286   1363       O  \\nATOM    496  CB  VAL A  70      11.073  21.159   6.385  1.00 38.71           C  \\nANISOU  496  CB  VAL A  70     4883   5451   4375  -1475    308   1128       C  \\nATOM    497  CG1 VAL A  70      11.098  19.705   5.980  1.00 38.59           C  \\nANISOU  497  CG1 VAL A  70     4737   5625   4299  -1312    360   1058       C  \\nATOM    498  CG2 VAL A  70      11.838  21.371   7.667  1.00 34.55           C  \\nANISOU  498  CG2 VAL A  70     4182   5014   3932  -1507    353   1011       C  \\nATOM    499  N   ARG A  71       9.733  23.494   5.012  1.00 42.02           N  \\nANISOU  499  N   ARG A  71     5892   5315   4760  -1691     91   1469       N  \\nATOM    500  CA  ARG A  71       8.972  24.731   5.097  1.00 51.92           C  \\nANISOU  500  CA  ARG A  71     7389   6225   6113  -1693    -45   1589       C  \\nATOM    501  C   ARG A  71       8.577  25.040   6.551  1.00 47.74           C  \\nANISOU  501  C   ARG A  71     6776   5557   5804  -1538    -75   1474       C  \\nATOM    502  O   ARG A  71       8.675  26.193   7.015  1.00 39.56           O  \\nANISOU  502  O   ARG A  71     5893   4299   4837  -1641    -90   1488       O  \\nATOM    503  CB  ARG A  71       7.727  24.589   4.218  1.00 59.09           C  \\nANISOU  503  CB  ARG A  71     8441   6993   7017  -1531   -215   1732       C  \\nATOM    504  CG  ARG A  71       7.040  25.865   3.812  1.00 66.82           C  \\nANISOU  504  CG  ARG A  71     9720   7630   8040  -1548   -374   1914       C  \\nATOM    505  CD  ARG A  71       6.170  25.595   2.587  1.00 71.41           C  \\nANISOU  505  CD  ARG A  71    10432   8182   8519  -1472   -536   2079       C  \\nATOM    506  NE  ARG A  71       5.299  26.720   2.285  1.00 77.35           N  \\nANISOU  506  NE  ARG A  71    11444   8592   9355  -1395   -738   2262       N  \\nATOM    507  CZ  ARG A  71       5.706  27.818   1.663  1.00 88.61           C  \\nANISOU  507  CZ  ARG A  71    13103   9876  10688  -1546   -732   2334       C  \\nATOM    508  NH1 ARG A  71       6.969  27.925   1.285  1.00 88.67           N  \\nANISOU  508  NH1 ARG A  71    13141  10043  10506  -1841   -552   2295       N  \\nATOM    509  NH2 ARG A  71       4.853  28.807   1.420  1.00 97.17           N  \\nANISOU  509  NH2 ARG A  71    14372  10671  11876  -1393   -901   2442       N  \\nATOM    510  N   TYR A  72       8.137  24.005   7.264  1.00 32.61           N  \\nANISOU  510  N   TYR A  72     4644   3762   3983  -1307    -74   1356       N  \\nATOM    511  CA  TYR A  72       7.570  24.190   8.590  1.00 31.65           C  \\nANISOU  511  CA  TYR A  72     4456   3519   4049  -1143   -100   1249       C  \\nATOM    512  C   TYR A  72       8.056  23.164   9.609  1.00 31.17           C  \\nANISOU  512  C   TYR A  72     4142   3700   4001  -1068    -11   1086       C  \\nATOM    513  O   TYR A  72       8.298  22.002   9.272  1.00 37.33           O  \\nANISOU  513  O   TYR A  72     4783   4698   4705  -1010     29   1061       O  \\nATOM    514  CB  TYR A  72       6.039  24.095   8.522  1.00 35.83           C  \\nANISOU  514  CB  TYR A  72     5021   3873   4719   -884   -235   1300       C  \\nATOM    515  CG  TYR A  72       5.356  25.210   7.766  1.00 46.52           C  \\nANISOU  515  CG  TYR A  72     6625   4938   6113   -881   -368   1463       C  \\nATOM    516  CD1 TYR A  72       5.645  26.546   8.037  1.00 57.49           C  \\nANISOU  516  CD1 TYR A  72     8217   6077   7547   -999   -368   1490       C  \\nATOM    517  CD2 TYR A  72       4.419  24.928   6.782  1.00 49.96           C  \\nANISOU  517  CD2 TYR A  72     7108   5338   6535   -760   -509   1593       C  \\nATOM    518  CE1 TYR A  72       5.013  27.573   7.347  1.00 60.28           C  \\nANISOU  518  CE1 TYR A  72     8833   6128   7944   -970   -506   1656       C  \\nATOM    519  CE2 TYR A  72       3.782  25.947   6.082  1.00 59.71           C  \\nANISOU  519  CE2 TYR A  72     8576   6306   7804   -731   -664   1764       C  \\nATOM    520  CZ  TYR A  72       4.082  27.266   6.370  1.00 62.44           C  \\nANISOU  520  CZ  TYR A  72     9138   6381   8206   -822   -662   1801       C  \\nATOM    521  OH  TYR A  72       3.452  28.276   5.678  1.00 71.87           O  \\nANISOU  521  OH  TYR A  72    10594   7275   9440   -769   -830   1986       O  \\nATOM    522  N   VAL A  73       8.166  23.587  10.864  1.00 34.99           N  \\nANISOU  522  N   VAL A  73     4591   4130   4575  -1063     13    976       N  \\nATOM    523  CA  VAL A  73       8.248  22.630  11.968  1.00 36.94           C  \\nANISOU  523  CA  VAL A  73     4637   4548   4851   -935     53    843       C  \\nATOM    524  C   VAL A  73       6.924  22.614  12.737  1.00 30.71           C  \\nANISOU  524  C   VAL A  73     3863   3591   4214   -720      5    800       C  \\nATOM    525  O   VAL A  73       6.421  23.656  13.167  1.00 31.49           O  \\nANISOU  525  O   VAL A  73     4090   3461   4415   -712    -18    781       O  \\nATOM    526  CB  VAL A  73       9.409  22.911  12.947  1.00 33.63           C  \\nANISOU  526  CB  VAL A  73     4130   4263   4385  -1094    116    731       C  \\nATOM    527  CG1 VAL A  73       9.323  21.940  14.130  1.00 27.04           C  \\nANISOU  527  CG1 VAL A  73     3131   3571   3573   -937    126    620       C  \\nATOM    528  CG2 VAL A  73      10.775  22.782  12.240  1.00 29.51           C  \\nANISOU  528  CG2 VAL A  73     3514   3977   3721  -1300    183    752       C  \\nATOM    529  N   ILE A  74       6.351  21.431  12.882  1.00 36.64           N  \\nANISOU  529  N   ILE A  74     4491   4452   4981   -552      0    777       N  \\nATOM    530  CA  ILE A  74       5.089  21.278  13.602  1.00 28.20           C  \\nANISOU  530  CA  ILE A  74     3396   3274   4043   -369    -21    727       C  \\nATOM    531  C   ILE A  74       5.423  20.701  14.958  1.00 25.29           C  \\nANISOU  531  C   ILE A  74     2923   3034   3651   -348     48    601       C  \\nATOM    532  O   ILE A  74       5.942  19.588  15.065  1.00 24.07           O  \\nANISOU  532  O   ILE A  74     2664   3072   3409   -331     71    588       O  \\nATOM    533  CB  ILE A  74       4.098  20.370  12.847  1.00 30.28           C  \\nANISOU  533  CB  ILE A  74     3609   3565   4333   -232    -79    794       C  \\nATOM    534  CG1 ILE A  74       3.804  20.946  11.457  1.00 28.33           C  \\nANISOU  534  CG1 ILE A  74     3482   3207   4074   -265   -175    935       C  \\nATOM    535  CG2 ILE A  74       2.789  20.210  13.641  1.00 27.64           C  \\nANISOU  535  CG2 ILE A  74     3208   3155   4139    -68    -82    730       C  \\nATOM    536  CD1 ILE A  74       3.027  19.977  10.534  1.00 29.93           C  \\nANISOU  536  CD1 ILE A  74     3641   3480   4251   -183   -249   1002       C  \\nATOM    537  N   HIS A  75       5.169  21.491  15.994  1.00 26.51           N  \\nANISOU  537  N   HIS A  75     3128   3072   3872   -351     79    506       N  \\nATOM    538  CA  HIS A  75       5.541  21.106  17.344  1.00 25.84           C  \\nANISOU  538  CA  HIS A  75     2982   3106   3731   -365    136    388       C  \\nATOM    539  C   HIS A  75       4.355  20.436  18.035  1.00 30.00           C  \\nANISOU  539  C   HIS A  75     3455   3621   4322   -206    170    336       C  \\nATOM    540  O   HIS A  75       3.395  21.086  18.450  1.00 33.10           O  \\nANISOU  540  O   HIS A  75     3889   3858   4828   -130    202    275       O  \\nATOM    541  CB  HIS A  75       6.031  22.320  18.133  1.00 26.19           C  \\nANISOU  541  CB  HIS A  75     3130   3050   3770   -502    164    291       C  \\nATOM    542  CG  HIS A  75       7.189  23.022  17.503  1.00 30.82           C  \\nANISOU  542  CG  HIS A  75     3769   3652   4289   -708    141    336       C  \\nATOM    543  ND1 HIS A  75       8.498  22.746  17.835  1.00 29.74           N  \\nANISOU  543  ND1 HIS A  75     3535   3743   4023   -861    145    303       N  \\nATOM    544  CD2 HIS A  75       7.237  24.003  16.566  1.00 33.47           C  \\nANISOU  544  CD2 HIS A  75     4244   3809   4662   -801    115    416       C  \\nATOM    545  CE1 HIS A  75       9.301  23.538  17.145  1.00 33.20           C  \\nANISOU  545  CE1 HIS A  75     4029   4160   4424  -1060    141    346       C  \\nATOM    546  NE2 HIS A  75       8.562  24.307  16.363  1.00 34.77           N  \\nANISOU  546  NE2 HIS A  75     4395   4101   4713  -1037    125    422       N  \\nATOM    547  N   ALA A  76       4.439  19.123  18.150  1.00 26.99           N  \\nANISOU  547  N   ALA A  76     2983   3405   3867   -156    174    357       N  \\nATOM    548  CA  ALA A  76       3.351  18.332  18.676  1.00 27.48           C  \\nANISOU  548  CA  ALA A  76     2999   3475   3968    -47    214    328       C  \\nATOM    549  C   ALA A  76       3.630  18.053  20.133  1.00 28.22           C  \\nANISOU  549  C   ALA A  76     3101   3655   3968    -81    275    232       C  \\nATOM    550  O   ALA A  76       4.517  17.265  20.450  1.00 26.08           O  \\nANISOU  550  O   ALA A  76     2808   3530   3571   -110    251    253       O  \\nATOM    551  CB  ALA A  76       3.235  17.029  17.898  1.00 28.62           C  \\nANISOU  551  CB  ALA A  76     3090   3711   4075      5    179    413       C  \\nATOM    552  N   ALA A  77       2.868  18.690  21.018  1.00 27.67           N  \\nANISOU  552  N   ALA A  77     3065   3496   3953    -66    351    125       N  \\nATOM    553  CA  ALA A  77       3.139  18.623  22.457  1.00 27.94           C  \\nANISOU  553  CA  ALA A  77     3144   3606   3867   -129    415     21       C  \\nATOM    554  C   ALA A  77       2.627  17.311  23.057  1.00 29.96           C  \\nANISOU  554  C   ALA A  77     3371   3970   4045    -91    458     38       C  \\nATOM    555  O   ALA A  77       1.422  17.109  23.164  1.00 25.00           O  \\nANISOU  555  O   ALA A  77     2704   3297   3497    -33    536      8       O  \\nATOM    556  CB  ALA A  77       2.501  19.813  23.157  1.00 28.05           C  \\nANISOU  556  CB  ALA A  77     3226   3474   3956   -132    506   -123       C  \\nATOM    557  N   VAL A  78       3.547  16.433  23.456  1.00 31.98           N  \\nANISOU  557  N   VAL A  78     3642   4366   4144   -128    404     89       N  \\nATOM    558  CA  VAL A  78       3.186  15.098  23.940  1.00 28.67           C  \\nANISOU  558  CA  VAL A  78     3242   4018   3633    -99    426    137       C  \\nATOM    559  C   VAL A  78       3.356  14.977  25.437  1.00 29.98           C  \\nANISOU  559  C   VAL A  78     3500   4263   3629   -175    469     73       C  \\nATOM    560  O   VAL A  78       2.931  13.998  26.040  1.00 32.98           O  \\nANISOU  560  O   VAL A  78     3938   4682   3914   -180    508    107       O  \\nATOM    561  CB  VAL A  78       4.003  13.957  23.243  1.00 33.61           C  \\nANISOU  561  CB  VAL A  78     3849   4718   4205    -43    324    263       C  \\nATOM    562  CG1 VAL A  78       3.608  13.837  21.796  1.00 40.61           C  \\nANISOU  562  CG1 VAL A  78     4674   5534   5222     16    303    321       C  \\nATOM    563  CG2 VAL A  78       5.521  14.183  23.370  1.00 25.50           C  \\nANISOU  563  CG2 VAL A  78     2797   3808   3084    -70    227    276       C  \\nATOM    564  N   LEU A  79       3.973  15.977  26.043  1.00 29.90           N  \\nANISOU  564  N   LEU A  79     3527   4271   3563   -256    460    -18       N  \\nATOM    565  CA  LEU A  79       4.218  15.925  27.473  1.00 38.16           C  \\nANISOU  565  CA  LEU A  79     4679   5408   4411   -351    483    -85       C  \\nATOM    566  C   LEU A  79       4.513  17.313  28.015  1.00 38.08           C  \\nANISOU  566  C   LEU A  79     4722   5362   4385   -458    514   -236       C  \\nATOM    567  O   LEU A  79       4.557  18.290  27.264  1.00 27.57           O  \\nANISOU  567  O   LEU A  79     3356   3915   3203   -453    513   -274       O  \\nATOM    568  CB  LEU A  79       5.364  14.957  27.792  1.00 42.03           C  \\nANISOU  568  CB  LEU A  79     5185   6054   4731   -348    336     29       C  \\nATOM    569  CG  LEU A  79       6.832  15.365  27.582  1.00 49.10           C  \\nANISOU  569  CG  LEU A  79     6007   7065   5585   -385    188     47       C  \\nATOM    570  CD1 LEU A  79       7.719  14.142  27.744  1.00 51.84           C  \\nANISOU  570  CD1 LEU A  79     6332   7556   5810   -303     46    179       C  \\nATOM    571  CD2 LEU A  79       7.086  16.005  26.237  1.00 49.76           C  \\nANISOU  571  CD2 LEU A  79     5976   7080   5849   -361    175     58       C  \\nATOM    572  N   GLY A  80       4.708  17.393  29.324  1.00 40.31           N  \\nANISOU  572  N   GLY A  80     5119   5730   4469   -567    539   -321       N  \\nATOM    573  CA  GLY A  80       5.023  18.656  29.966  1.00 38.12           C  \\nANISOU  573  CA  GLY A  80     4928   5416   4139   -697    569   -487       C  \\nATOM    574  C   GLY A  80       4.388  18.715  31.333  1.00 40.73           C  \\nANISOU  574  C   GLY A  80     5403   5775   4299   -780    707   -624       C  \\nATOM    575  O   GLY A  80       4.950  18.184  32.289  1.00 38.99           O  \\nANISOU  575  O   GLY A  80     5270   5713   3831   -877    637   -600       O  \\nATOM    576  N   ASP A  81       3.220  19.348  31.441  1.00 33.38           N  \\nANISOU  576  N   ASP A  81     4493   4699   3489   -738    902   -767       N  \\nATOM    577  CA  ASP A  81       2.543  19.406  32.738  1.00 35.40           C  \\nANISOU  577  CA  ASP A  81     4879   4995   3576   -821   1077   -922       C  \\nATOM    578  C   ASP A  81       1.985  18.042  33.073  1.00 35.04           C  \\nANISOU  578  C   ASP A  81     4825   5068   3420   -806   1122   -803       C  \\nATOM    579  O   ASP A  81       1.829  17.696  34.228  1.00 36.69           O  \\nANISOU  579  O   ASP A  81     5171   5382   3386   -923   1202   -855       O  \\nATOM    580  CB  ASP A  81       1.443  20.465  32.768  1.00 39.88           C  \\nANISOU  580  CB  ASP A  81     5448   5385   4322   -753   1292  -1129       C  \\nATOM    581  CG  ASP A  81       1.995  21.866  32.672  1.00 44.01           C  \\nANISOU  581  CG  ASP A  81     6061   5751   4910   -801   1263  -1268       C  \\nATOM    582  OD1 ASP A  81       2.990  22.169  33.364  1.00 49.58           O  \\nANISOU  582  OD1 ASP A  81     6900   6536   5404   -981   1175  -1322       O  \\nATOM    583  OD2 ASP A  81       1.441  22.663  31.895  1.00 47.33           O  \\nANISOU  583  OD2 ASP A  81     6428   5967   5590   -668   1315  -1317       O  \\nATOM    584  N   GLU A  82       1.706  17.259  32.038  1.00 33.51           N  \\nANISOU  584  N   GLU A  82     4495   4850   3389   -681   1067   -641       N  \\nATOM    585  CA  GLU A  82       1.302  15.877  32.226  1.00 38.93           C  \\nANISOU  585  CA  GLU A  82     5197   5621   3974   -685   1081   -504       C  \\nATOM    586  C   GLU A  82       2.342  14.988  31.570  1.00 32.21           C  \\nANISOU  586  C   GLU A  82     4321   4817   3100   -627    853   -295       C  \\nATOM    587  O   GLU A  82       2.965  15.378  30.582  1.00 32.47           O  \\nANISOU  587  O   GLU A  82     4248   4805   3284   -550    735   -254       O  \\nATOM    588  CB  GLU A  82      -0.077  15.614  31.604  1.00 42.35           C  \\nANISOU  588  CB  GLU A  82     5493   5982   4616   -599   1234   -514       C  \\nATOM    589  CG  GLU A  82      -1.223  16.386  32.238  1.00 46.38           C  \\nANISOU  589  CG  GLU A  82     5980   6467   5176   -620   1485   -730       C  \\nATOM    590  CD  GLU A  82      -1.272  17.844  31.787  1.00 41.56           C  \\nANISOU  590  CD  GLU A  82     5305   5709   4774   -520   1507   -879       C  \\nATOM    591  OE1 GLU A  82      -1.128  18.107  30.582  1.00 45.79           O  \\nANISOU  591  OE1 GLU A  82     5729   6144   5526   -399   1385   -795       O  \\nATOM    592  OE2 GLU A  82      -1.436  18.731  32.643  1.00 47.09           O  \\nANISOU  592  OE2 GLU A  82     6098   6384   5411   -569   1645  -1080       O  \\nATOM    593  N   PRO A  83       2.539  13.789  32.118  1.00 31.69           N  \\nANISOU  593  N   PRO A  83     4365   4836   2840   -662    797   -163       N  \\nATOM    594  CA  PRO A  83       3.440  12.818  31.479  1.00 32.87           C  \\nANISOU  594  CA  PRO A  83     4491   5010   2986   -563    595     31       C  \\nATOM    595  C   PRO A  83       2.875  12.279  30.146  1.00 39.03           C  \\nANISOU  595  C   PRO A  83     5148   5684   3999   -442    611    110       C  \\nATOM    596  O   PRO A  83       1.665  12.320  29.895  1.00 33.66           O  \\nANISOU  596  O   PRO A  83     4417   4934   3438   -454    767     54       O  \\nATOM    597  CB  PRO A  83       3.572  11.715  32.534  1.00 36.05           C  \\nANISOU  597  CB  PRO A  83     5091   5488   3118   -628    555    143       C  \\nATOM    598  CG  PRO A  83       2.346  11.860  33.410  1.00 37.10           C  \\nANISOU  598  CG  PRO A  83     5322   5616   3158   -770    792     28       C  \\nATOM    599  CD  PRO A  83       1.950  13.297  33.379  1.00 33.64           C  \\nANISOU  599  CD  PRO A  83     4782   5149   2849   -795    925   -189       C  \\nATOM    600  N   ALA A  84       3.758  11.815  29.276  1.00 27.65           N  \\nANISOU  600  N   ALA A  84     3643   4244   2618   -331    451    227       N  \\nATOM    601  CA  ALA A  84       3.337  11.275  27.995  1.00 29.75           C  \\nANISOU  601  CA  ALA A  84     3820   4417   3068   -231    452    295       C  \\nATOM    602  C   ALA A  84       2.549   9.992  28.185  1.00 28.51           C  \\nANISOU  602  C   ALA A  84     3776   4207   2850   -253    511    382       C  \\nATOM    603  O   ALA A  84       2.862   9.187  29.059  1.00 32.27           O  \\nANISOU  603  O   ALA A  84     4416   4714   3130   -285    468    464       O  \\nATOM    604  CB  ALA A  84       4.542  11.028  27.084  1.00 25.30           C  \\nANISOU  604  CB  ALA A  84     3177   3882   2553   -113    289    382       C  \\nATOM    605  N   SER A  85       1.531   9.806  27.353  1.00 29.18           N  \\nANISOU  605  N   SER A  85     3782   4210   3094   -248    596    371       N  \\nATOM    606  CA  SER A  85       0.687   8.619  27.427  1.00 34.03           C  \\nANISOU  606  CA  SER A  85     4496   4768   3667   -311    664    440       C  \\nATOM    607  C   SER A  85       0.211   8.320  26.007  1.00 25.17           C  \\nANISOU  607  C   SER A  85     3262   3566   2735   -253    644    463       C  \\nATOM    608  O   SER A  85       0.420   9.142  25.121  1.00 28.45           O  \\nANISOU  608  O   SER A  85     3533   3981   3298   -174    598    421       O  \\nATOM    609  CB  SER A  85      -0.508   8.911  28.318  1.00 31.98           C  \\nANISOU  609  CB  SER A  85     4239   4546   3366   -461    860    340       C  \\nATOM    610  OG  SER A  85      -1.373   9.821  27.662  1.00 33.92           O  \\nANISOU  610  OG  SER A  85     4275   4785   3827   -437    947    223       O  \\nATOM    611  N   LEU A  86      -0.422   7.173  25.773  1.00 25.62           N  \\nANISOU  611  N   LEU A  86     3406   3552   2775   -313    674    530       N  \\nATOM    612  CA  LEU A  86      -0.949   6.885  24.426  1.00 26.05           C  \\nANISOU  612  CA  LEU A  86     3364   3543   2989   -289    650    537       C  \\nATOM    613  C   LEU A  86      -2.033   7.887  24.042  1.00 24.61           C  \\nANISOU  613  C   LEU A  86     2962   3409   2979   -324    735    428       C  \\nATOM    614  O   LEU A  86      -2.263   8.154  22.871  1.00 27.04           O  \\nANISOU  614  O   LEU A  86     3148   3695   3432   -269    675    423       O  \\nATOM    615  CB  LEU A  86      -1.499   5.460  24.319  1.00 29.03           C  \\nANISOU  615  CB  LEU A  86     3899   3826   3306   -388    673    613       C  \\nATOM    616  CG  LEU A  86      -0.505   4.325  24.581  1.00 34.07           C  \\nANISOU  616  CG  LEU A  86     4786   4365   3795   -318    576    735       C  \\nATOM    617  CD1 LEU A  86      -1.182   2.983  24.427  1.00 27.50           C  \\nANISOU  617  CD1 LEU A  86     4139   3395   2914   -441    613    799       C  \\nATOM    618  CD2 LEU A  86       0.723   4.465  23.655  1.00 25.07           C  \\nANISOU  618  CD2 LEU A  86     3596   3214   2716   -111    435    755       C  \\nATOM    619  N   GLU A  87      -2.673   8.460  25.044  1.00 25.69           N  \\nANISOU  619  N   GLU A  87     3053   3614   3093   -402    870    339       N  \\nATOM    620  CA  GLU A  87      -3.732   9.430  24.807  1.00 26.81           C  \\nANISOU  620  CA  GLU A  87     2974   3800   3412   -398    960    223       C  \\nATOM    621  C   GLU A  87      -3.157  10.780  24.343  1.00 25.16           C  \\nANISOU  621  C   GLU A  87     2669   3569   3320   -254    885    172       C  \\nATOM    622  O   GLU A  87      -3.708  11.423  23.452  1.00 31.56           O  \\nANISOU  622  O   GLU A  87     3321   4358   4314   -183    853    144       O  \\nATOM    623  CB  GLU A  87      -4.602   9.604  26.072  1.00 27.99           C  \\nANISOU  623  CB  GLU A  87     3103   4034   3496   -520   1164    118       C  \\nATOM    624  CG  GLU A  87      -5.946  10.246  25.773  1.00 37.91           C  \\nANISOU  624  CG  GLU A  87     4099   5351   4956   -513   1275      1       C  \\nATOM    625  CD  GLU A  87      -6.921  10.246  26.956  1.00 46.62           C  \\nANISOU  625  CD  GLU A  87     5153   6566   5994   -651   1514   -117       C  \\nATOM    626  OE1 GLU A  87      -6.482  10.150  28.126  1.00 39.01           O  \\nANISOU  626  OE1 GLU A  87     4374   5629   4821   -736   1603   -135       O  \\nATOM    627  OE2 GLU A  87      -8.141  10.353  26.699  1.00 61.20           O  \\nANISOU  627  OE2 GLU A  87     6764   8493   7995   -678   1614   -195       O  \\nATOM    628  N   THR A  88      -2.072  11.228  24.966  1.00 31.22           N  \\nANISOU  628  N   THR A  88     3543   4342   3977   -227    849    165       N  \\nATOM    629  CA  THR A  88      -1.480  12.502  24.554  1.00 27.62           C  \\nANISOU  629  CA  THR A  88     3025   3851   3617   -135    785    118       C  \\nATOM    630  C   THR A  88      -0.834  12.334  23.165  1.00 27.88           C  \\nANISOU  630  C   THR A  88     3033   3841   3720    -58    631    220       C  \\nATOM    631  O   THR A  88      -0.848  13.243  22.338  1.00 29.26           O  \\nANISOU  631  O   THR A  88     3126   3964   4028      4    581    207       O  \\nATOM    632  CB  THR A  88      -0.479  13.049  25.606  1.00 24.77           C  \\nANISOU  632  CB  THR A  88     2777   3529   3106   -167    783     69       C  \\nATOM    633  OG1 THR A  88       0.729  12.283  25.583  1.00 30.39           O  \\nANISOU  633  OG1 THR A  88     3588   4279   3680   -159    655    180       O  \\nATOM    634  CG2 THR A  88      -1.075  12.999  27.002  1.00 29.73           C  \\nANISOU  634  CG2 THR A  88     3475   4215   3607   -269    941    -24       C  \\nATOM    635  N   VAL A  89      -0.297  11.147  22.911  1.00 24.60           N  \\nANISOU  635  N   VAL A  89     2707   3435   3205    -63    565    319       N  \\nATOM    636  CA  VAL A  89       0.295  10.835  21.616  1.00 27.49           C  \\nANISOU  636  CA  VAL A  89     3061   3771   3612      3    451    397       C  \\nATOM    637  C   VAL A  89      -0.794  10.839  20.546  1.00 29.86           C  \\nANISOU  637  C   VAL A  89     3262   4031   4054      1    443    401       C  \\nATOM    638  O   VAL A  89      -0.596  11.350  19.433  1.00 28.17           O  \\nANISOU  638  O   VAL A  89     2995   3792   3918     49    363    426       O  \\nATOM    639  CB  VAL A  89       1.026   9.465  21.646  1.00 26.76           C  \\nANISOU  639  CB  VAL A  89     3101   3678   3390     24    399    483       C  \\nATOM    640  CG1 VAL A  89       1.443   9.036  20.258  1.00 24.90           C  \\nANISOU  640  CG1 VAL A  89     2856   3409   3195     89    319    533       C  \\nATOM    641  CG2 VAL A  89       2.226   9.517  22.586  1.00 22.38           C  \\nANISOU  641  CG2 VAL A  89     2614   3188   2701     54    358    496       C  \\nATOM    642  N   ARG A  90      -1.954  10.283  20.875  1.00 28.56           N  \\nANISOU  642  N   ARG A  90     3067   3874   3910    -73    521    378       N  \\nATOM    643  CA  ARG A  90      -3.049  10.280  19.907  1.00 28.14           C  \\nANISOU  643  CA  ARG A  90     2885   3815   3990    -87    494    377       C  \\nATOM    644  C   ARG A  90      -3.481  11.716  19.567  1.00 27.95           C  \\nANISOU  644  C   ARG A  90     2710   3781   4128     -2    474    327       C  \\nATOM    645  O   ARG A  90      -3.575  12.082  18.404  1.00 26.70           O  \\nANISOU  645  O   ARG A  90     2497   3593   4053     48    365    372       O  \\nATOM    646  CB  ARG A  90      -4.238   9.470  20.418  1.00 25.81           C  \\nANISOU  646  CB  ARG A  90     2553   3561   3692   -212    595    349       C  \\nATOM    647  CG  ARG A  90      -5.372   9.399  19.401  1.00 25.15           C  \\nANISOU  647  CG  ARG A  90     2306   3506   3743   -245    543    346       C  \\nATOM    648  CD  ARG A  90      -6.531   8.501  19.850  1.00 25.84           C  \\nANISOU  648  CD  ARG A  90     2339   3659   3821   -415    645    315       C  \\nATOM    649  NE  ARG A  90      -6.143   7.098  19.947  1.00 27.22           N  \\nANISOU  649  NE  ARG A  90     2739   3768   3837   -535    649    380       N  \\nATOM    650  CZ  ARG A  90      -6.126   6.245  18.927  1.00 26.30           C  \\nANISOU  650  CZ  ARG A  90     2704   3595   3695   -589    551    430       C  \\nATOM    651  NH1 ARG A  90      -6.477   6.644  17.715  1.00 27.65           N  \\nANISOU  651  NH1 ARG A  90     2742   3792   3970   -551    431    431       N  \\nATOM    652  NH2 ARG A  90      -5.756   4.986  19.117  1.00 25.95           N  \\nANISOU  652  NH2 ARG A  90     2897   3453   3511   -680    567    480       N  \\nATOM    653  N   LYS A  91      -3.721  12.523  20.597  1.00 27.24           N  \\nANISOU  653  N   LYS A  91     2578   3701   4069     16    580    234       N  \\nATOM    654  CA  LYS A  91      -4.169  13.903  20.428  1.00 32.06           C  \\nANISOU  654  CA  LYS A  91     3074   4263   4844    120    578    171       C  \\nATOM    655  C   LYS A  91      -3.144  14.763  19.677  1.00 23.43           C  \\nANISOU  655  C   LYS A  91     2058   3082   3763    185    459    226       C  \\nATOM    656  O   LYS A  91      -3.492  15.516  18.775  1.00 31.44           O  \\nANISOU  656  O   LYS A  91     3010   4029   4906    264    370    258       O  \\nATOM    657  CB  LYS A  91      -4.498  14.525  21.791  1.00 36.37           C  \\nANISOU  657  CB  LYS A  91     3604   4824   5390    119    742     34       C  \\nATOM    658  CG  LYS A  91      -5.768  13.971  22.454  1.00 37.96           C  \\nANISOU  658  CG  LYS A  91     3677   5127   5618     54    892    -43       C  \\nATOM    659  CD  LYS A  91      -5.868  14.402  23.925  1.00 43.64           C  \\nANISOU  659  CD  LYS A  91     4438   5881   6263     18   1083   -184       C  \\nATOM    660  CE  LYS A  91      -6.990  13.663  24.683  1.00 45.59           C  \\nANISOU  660  CE  LYS A  91     4585   6258   6478   -101   1264   -254       C  \\nATOM    661  NZ  LYS A  91      -7.069  14.039  26.145  1.00 40.63           N  \\nANISOU  661  NZ  LYS A  91     4024   5680   5734   -161   1472   -399       N  \\nATOM    662  N   ALA A  92      -1.879  14.633  20.041  1.00 23.56           N  \\nANISOU  662  N   ALA A  92     2208   3108   3637    142    450    245       N  \\nATOM    663  CA  ALA A  92      -0.832  15.418  19.411  1.00 22.22           C  \\nANISOU  663  CA  ALA A  92     2101   2884   3460    159    361    288       C  \\nATOM    664  C   ALA A  92      -0.630  15.030  17.959  1.00 25.36           C  \\nANISOU  664  C   ALA A  92     2497   3277   3861    168    245    399       C  \\nATOM    665  O   ALA A  92      -0.398  15.901  17.126  1.00 31.83           O  \\nANISOU  665  O   ALA A  92     3332   4027   4735    189    171    443       O  \\nATOM    666  CB  ALA A  92       0.478  15.292  20.180  1.00 21.61           C  \\nANISOU  666  CB  ALA A  92     2120   2864   3227     99    375    274       C  \\nATOM    667  N   THR A  93      -0.703  13.732  17.656  1.00 26.27           N  \\nANISOU  667  N   THR A  93     2624   3454   3903    139    232    443       N  \\nATOM    668  CA  THR A  93      -0.572  13.264  16.275  1.00 22.58           C  \\nANISOU  668  CA  THR A  93     2175   2988   3416    135    136    525       C  \\nATOM    669  C   THR A  93      -1.734  13.819  15.462  1.00 28.82           C  \\nANISOU  669  C   THR A  93     2874   3736   4340    166     63    550       C  \\nATOM    670  O   THR A  93      -1.555  14.362  14.374  1.00 30.00           O  \\nANISOU  670  O   THR A  93     3048   3848   4502    177    -36    618       O  \\nATOM    671  CB  THR A  93      -0.577  11.717  16.180  1.00 27.03           C  \\nANISOU  671  CB  THR A  93     2795   3593   3883     98    148    544       C  \\nATOM    672  OG1 THR A  93       0.487  11.184  16.971  1.00 25.96           O  \\nANISOU  672  OG1 THR A  93     2742   3490   3632    106    193    536       O  \\nATOM    673  CG2 THR A  93      -0.382  11.255  14.734  1.00 23.70           C  \\nANISOU  673  CG2 THR A  93     2417   3169   3419     85     63    603       C  \\nATOM    674  N   LYS A  94      -2.933  13.704  16.016  1.00 22.08           N  \\nANISOU  674  N   LYS A  94     1908   2900   3581    176    109    497       N  \\nATOM    675  CA  LYS A  94      -4.110  14.271  15.381  1.00 24.55           C  \\nANISOU  675  CA  LYS A  94     2084   3197   4046    234     29    512       C  \\nATOM    676  C   LYS A  94      -3.991  15.799  15.216  1.00 31.34           C  \\nANISOU  676  C   LYS A  94     2949   3943   5014    341    -19    522       C  \\nATOM    677  O   LYS A  94      -4.378  16.338  14.177  1.00 28.09           O  \\nANISOU  677  O   LYS A  94     2516   3484   4673    395   -156    602       O  \\nATOM    678  CB  LYS A  94      -5.357  13.912  16.187  1.00 26.19           C  \\nANISOU  678  CB  LYS A  94     2130   3476   4343    224    122    427       C  \\nATOM    679  CG  LYS A  94      -6.653  14.387  15.581  1.00 34.44           C  \\nANISOU  679  CG  LYS A  94     2974   4547   5563    299     32    433       C  \\nATOM    680  CD  LYS A  94      -7.695  14.543  16.676  1.00 47.54           C  \\nANISOU  680  CD  LYS A  94     4445   6274   7344    330    179    309       C  \\nATOM    681  CE  LYS A  94      -9.097  14.351  16.158  1.00 53.95           C  \\nANISOU  681  CE  LYS A  94     5001   7199   8297    343    107    304       C  \\nATOM    682  NZ  LYS A  94      -9.376  12.898  15.990  1.00 65.85           N  \\nANISOU  682  NZ  LYS A  94     6512   8819   9690    140    117    320       N  \\nATOM    683  N   SER A  95      -3.471  16.501  16.229  1.00 27.42           N  \\nANISOU  683  N   SER A  95     2505   3391   4521    361     84    445       N  \\nATOM    684  CA  SER A  95      -3.316  17.961  16.118  1.00 29.96           C  \\nANISOU  684  CA  SER A  95     2878   3563   4942    445     48    445       C  \\nATOM    685  C   SER A  95      -2.291  18.325  15.044  1.00 35.21           C  \\nANISOU  685  C   SER A  95     3688   4170   5520    389    -62    563       C  \\nATOM    686  O   SER A  95      -2.490  19.282  14.282  1.00 36.31           O  \\nANISOU  686  O   SER A  95     3873   4183   5739    448   -167    637       O  \\nATOM    687  CB  SER A  95      -2.915  18.614  17.450  1.00 29.38           C  \\nANISOU  687  CB  SER A  95     2860   3438   4864    443    187    317       C  \\nATOM    688  OG  SER A  95      -3.952  18.554  18.412  1.00 33.81           O  \\nANISOU  688  OG  SER A  95     3295   4037   5516    503    309    194       O  \\nATOM    689  N   ALA A  96      -1.206  17.552  14.984  1.00 30.23           N  \\nANISOU  689  N   ALA A  96     3130   3634   4723    279    -35    583       N  \\nATOM    690  CA  ALA A  96      -0.158  17.748  13.979  1.00 30.54           C  \\nANISOU  690  CA  ALA A  96     3281   3667   4656    202   -103    677       C  \\nATOM    691  C   ALA A  96      -0.686  17.543  12.565  1.00 31.63           C  \\nANISOU  691  C   ALA A  96     3425   3804   4789    209   -233    790       C  \\nATOM    692  O   ALA A  96      -0.382  18.318  11.664  1.00 34.24           O  \\nANISOU  692  O   ALA A  96     3853   4056   5099    181   -317    883       O  \\nATOM    693  CB  ALA A  96       1.014  16.801  14.242  1.00 21.55           C  \\nANISOU  693  CB  ALA A  96     2168   2661   3359    120    -37    656       C  \\nATOM    694  N   LEU A  97      -1.461  16.481  12.376  1.00 31.02           N  \\nANISOU  694  N   LEU A  97     3261   3814   4710    221   -254    783       N  \\nATOM    695  CA  LEU A  97      -2.084  16.204  11.093  1.00 37.50           C  \\nANISOU  695  CA  LEU A  97     4081   4656   5512    211   -393    874       C  \\nATOM    696  C   LEU A  97      -3.048  17.319  10.690  1.00 34.67           C  \\nANISOU  696  C   LEU A  97     3677   4192   5306    315   -523    939       C  \\nATOM    697  O   LEU A  97      -3.084  17.716   9.534  1.00 36.62           O  \\nANISOU  697  O   LEU A  97     4004   4400   5508    301   -665   1057       O  \\nATOM    698  CB  LEU A  97      -2.803  14.843  11.117  1.00 35.00           C  \\nANISOU  698  CB  LEU A  97     3678   4447   5171    176   -385    833       C  \\nATOM    699  CG  LEU A  97      -1.884  13.623  11.251  1.00 32.70           C  \\nANISOU  699  CG  LEU A  97     3475   4226   4723     96   -290    793       C  \\nATOM    700  CD1 LEU A  97      -2.700  12.362  11.474  1.00 28.50           C  \\nANISOU  700  CD1 LEU A  97     2891   3752   4185     49   -270    745       C  \\nATOM    701  CD2 LEU A  97      -0.963  13.475  10.033  1.00 24.27           C  \\nANISOU  701  CD2 LEU A  97     2541   3179   3502     33   -334    854       C  \\nATOM    702  N   GLU A  98      -3.823  17.826  11.648  1.00 39.19           N  \\nANISOU  702  N   GLU A  98     4126   4716   6051    430   -474    860       N  \\nATOM    703  CA  GLU A  98      -4.780  18.892  11.361  1.00 37.46           C  \\nANISOU  703  CA  GLU A  98     3840   4382   6009    583   -594    907       C  \\nATOM    704  C   GLU A  98      -4.085  20.165  10.892  1.00 34.44           C  \\nANISOU  704  C   GLU A  98     3657   3811   5620    600   -659    999       C  \\nATOM    705  O   GLU A  98      -4.595  20.856  10.032  1.00 34.74           O  \\nANISOU  705  O   GLU A  98     3730   3749   5718    684   -830   1119       O  \\nATOM    706  CB  GLU A  98      -5.689  19.182  12.565  1.00 40.51           C  \\nANISOU  706  CB  GLU A  98     4047   4760   6585    716   -486    770       C  \\nATOM    707  CG  GLU A  98      -6.794  18.143  12.761  1.00 49.27           C  \\nANISOU  707  CG  GLU A  98     4925   6051   7745    707   -469    711       C  \\nATOM    708  CD  GLU A  98      -7.491  18.243  14.113  1.00 58.83           C  \\nANISOU  708  CD  GLU A  98     5966   7296   9090    783   -292    549       C  \\nATOM    709  OE1 GLU A  98      -8.577  17.640  14.264  1.00 59.23           O  \\nANISOU  709  OE1 GLU A  98     5791   7491   9221    786   -278    498       O  \\nATOM    710  OE2 GLU A  98      -6.953  18.912  15.027  1.00 63.08           O  \\nANISOU  710  OE2 GLU A  98     6598   7731   9639    816   -160    463       O  \\nATOM    711  N   LYS A  99      -2.925  20.475  11.458  1.00 34.13           N  \\nANISOU  711  N   LYS A  99     3750   3720   5497    507   -534    951       N  \\nATOM    712  CA  LYS A  99      -2.173  21.644  11.012  1.00 36.00           C  \\nANISOU  712  CA  LYS A  99     4196   3780   5705    463   -580   1037       C  \\nATOM    713  C   LYS A  99      -1.651  21.425   9.590  1.00 35.65           C  \\nANISOU  713  C   LYS A  99     4280   3780   5487    337   -693   1194       C  \\nATOM    714  O   LYS A  99      -1.700  22.329   8.766  1.00 39.59           O  \\nANISOU  714  O   LYS A  99     4929   4129   5986    343   -821   1329       O  \\nATOM    715  CB  LYS A  99      -1.017  21.956  11.963  1.00 33.22           C  \\nANISOU  715  CB  LYS A  99     3931   3404   5289    352   -421    935       C  \\nATOM    716  CG  LYS A  99      -1.456  22.372  13.363  1.00 33.73           C  \\nANISOU  716  CG  LYS A  99     3925   3398   5491    454   -304    774       C  \\nATOM    717  CD  LYS A  99      -2.133  23.744  13.369  1.00 35.21           C  \\nANISOU  717  CD  LYS A  99     4192   3323   5863    608   -364    784       C  \\nATOM    718  CE  LYS A  99      -2.436  24.192  14.798  1.00 47.52           C  \\nANISOU  718  CE  LYS A  99     5710   4811   7535    692   -210    590       C  \\nATOM    719  NZ  LYS A  99      -2.916  25.602  14.852  1.00 58.93           N  \\nANISOU  719  NZ  LYS A  99     7277   5957   9158    849   -247    575       N  \\nATOM    720  N   ALA A 100      -1.174  20.216   9.310  1.00 30.88           N  \\nANISOU  720  N   ALA A 100     3635   3372   4727    225   -643   1173       N  \\nATOM    721  CA  ALA A 100      -0.625  19.889   8.000  1.00 33.32           C  \\nANISOU  721  CA  ALA A 100     4065   3751   4845     93   -711   1287       C  \\nATOM    722  C   ALA A 100      -1.687  20.069   6.924  1.00 37.19           C  \\nANISOU  722  C   ALA A 100     4571   4202   5356    158   -923   1420       C  \\nATOM    723  O   ALA A 100      -1.432  20.654   5.884  1.00 38.42           O  \\nANISOU  723  O   ALA A 100     4902   4291   5405     86  -1030   1563       O  \\nATOM    724  CB  ALA A 100      -0.097  18.461   7.988  1.00 29.22           C  \\nANISOU  724  CB  ALA A 100     3485   3431   4186     10   -611   1208       C  \\nATOM    725  N   VAL A 101      -2.883  19.561   7.192  1.00 42.17           N  \\nANISOU  725  N   VAL A 101     5016   4892   6116    280   -986   1375       N  \\nATOM    726  CA  VAL A 101      -3.982  19.659   6.248  1.00 38.12           C  \\nANISOU  726  CA  VAL A 101     4463   4383   5636    351  -1212   1491       C  \\nATOM    727  C   VAL A 101      -4.421  21.107   6.120  1.00 41.49           C  \\nANISOU  727  C   VAL A 101     4962   4593   6211    502  -1348   1602       C  \\nATOM    728  O   VAL A 101      -4.737  21.573   5.020  1.00 53.32           O  \\nANISOU  728  O   VAL A 101     6574   6034   7652    512  -1557   1773       O  \\nATOM    729  CB  VAL A 101      -5.160  18.762   6.690  1.00 38.72           C  \\nANISOU  729  CB  VAL A 101     4278   4601   5833    425  -1229   1396       C  \\nATOM    730  CG1 VAL A 101      -6.377  18.941   5.781  1.00 38.16           C  \\nANISOU  730  CG1 VAL A 101     4113   4562   5823    509  -1490   1510       C  \\nATOM    731  CG2 VAL A 101      -4.706  17.312   6.740  1.00 36.10           C  \\nANISOU  731  CG2 VAL A 101     3937   4435   5342    267  -1107   1302       C  \\nATOM    732  N   GLU A 102      -4.411  21.835   7.235  1.00 36.55           N  \\nANISOU  732  N   GLU A 102     4298   3828   5760    618  -1233   1508       N  \\nATOM    733  CA  GLU A 102      -4.831  23.241   7.214  1.00 45.37           C  \\nANISOU  733  CA  GLU A 102     5507   4689   7041    790  -1345   1591       C  \\nATOM    734  C   GLU A 102      -3.902  24.099   6.348  1.00 48.47           C  \\nANISOU  734  C   GLU A 102     6226   4911   7278    656  -1413   1759       C  \\nATOM    735  O   GLU A 102      -4.308  25.131   5.810  1.00 54.55           O  \\nANISOU  735  O   GLU A 102     7139   5465   8121    771  -1588   1907       O  \\nATOM    736  CB  GLU A 102      -4.902  23.812   8.632  1.00 52.62           C  \\nANISOU  736  CB  GLU A 102     6354   5486   8152    915  -1173   1422       C  \\nATOM    737  CG  GLU A 102      -6.233  23.571   9.335  1.00 68.38           C  \\nANISOU  737  CG  GLU A 102     8046   7558  10375   1133  -1166   1303       C  \\nATOM    738  CD  GLU A 102      -6.159  23.821  10.836  1.00 77.05           C  \\nANISOU  738  CD  GLU A 102     9076   8606  11593   1193   -933   1094       C  \\nATOM    739  OE1 GLU A 102      -5.237  24.543  11.279  1.00 76.30           O  \\nANISOU  739  OE1 GLU A 102     9191   8343  11457   1121   -832   1060       O  \\nATOM    740  OE2 GLU A 102      -7.020  23.285  11.570  1.00 78.89           O  \\nANISOU  740  OE2 GLU A 102     9049   8981  11946   1286   -847    960       O  \\nATOM    741  N   LEU A 103      -2.656  23.662   6.203  1.00 42.86           N  \\nANISOU  741  N   LEU A 103     5634   4299   6353    414  -1273   1738       N  \\nATOM    742  CA  LEU A 103      -1.681  24.409   5.416  1.00 46.28           C  \\nANISOU  742  CA  LEU A 103     6361   4609   6614    232  -1294   1880       C  \\nATOM    743  C   LEU A 103      -1.540  23.832   4.014  1.00 46.47           C  \\nANISOU  743  C   LEU A 103     6483   4778   6397     85  -1408   2022       C  \\nATOM    744  O   LEU A 103      -0.736  24.303   3.226  1.00 49.18           O  \\nANISOU  744  O   LEU A 103     7068   5067   6551   -102  -1412   2146       O  \\nATOM    745  CB  LEU A 103      -0.335  24.410   6.136  1.00 43.50           C  \\nANISOU  745  CB  LEU A 103     6057   4294   6178     45  -1059   1759       C  \\nATOM    746  CG  LEU A 103      -0.393  25.153   7.474  1.00 48.75           C  \\nANISOU  746  CG  LEU A 103     6691   4790   7043    152   -956   1624       C  \\nATOM    747  CD1 LEU A 103       0.830  24.874   8.325  1.00 54.15           C  \\nANISOU  747  CD1 LEU A 103     7346   5588   7639    -26   -742   1477       C  \\nATOM    748  CD2 LEU A 103      -0.565  26.660   7.241  1.00 54.90           C  \\nANISOU  748  CD2 LEU A 103     7721   5225   7915    206  -1063   1744       C  \\nATOM    749  N   GLY A 104      -2.326  22.801   3.715  1.00 45.29           N  \\nANISOU  749  N   GLY A 104     6152   4817   6237    146  -1490   1994       N  \\nATOM    750  CA  GLY A 104      -2.259  22.144   2.424  1.00 47.24           C  \\nANISOU  750  CA  GLY A 104     6492   5217   6239     -1  -1593   2096       C  \\nATOM    751  C   GLY A 104      -0.904  21.505   2.170  1.00 48.63           C  \\nANISOU  751  C   GLY A 104     6760   5541   6175   -241  -1386   2029       C  \\nATOM    752  O   GLY A 104      -0.361  21.626   1.079  1.00 52.13           O  \\nANISOU  752  O   GLY A 104     7410   6014   6381   -417  -1418   2145       O  \\nATOM    753  N   LEU A 105      -0.349  20.841   3.182  1.00 39.04           N  \\nANISOU  753  N   LEU A 105     5391   4425   5016   -245  -1172   1842       N  \\nATOM    754  CA  LEU A 105       0.955  20.193   3.043  1.00 40.94           C  \\nANISOU  754  CA  LEU A 105     5669   4822   5063   -426   -972   1760       C  \\nATOM    755  C   LEU A 105       0.835  18.829   2.343  1.00 40.81           C  \\nANISOU  755  C   LEU A 105     5614   5005   4886   -479   -968   1706       C  \\nATOM    756  O   LEU A 105      -0.206  18.182   2.412  1.00 39.41           O  \\nANISOU  756  O   LEU A 105     5318   4866   4789   -378  -1072   1675       O  \\nATOM    757  CB  LEU A 105       1.635  20.061   4.409  1.00 36.04           C  \\nANISOU  757  CB  LEU A 105     4909   4224   4560   -394   -776   1597       C  \\nATOM    758  CG  LEU A 105       1.787  21.374   5.195  1.00 35.18           C  \\nANISOU  758  CG  LEU A 105     4853   3912   4600   -364   -762   1613       C  \\nATOM    759  CD1 LEU A 105       2.720  21.201   6.393  1.00 35.44           C  \\nANISOU  759  CD1 LEU A 105     4778   4015   4672   -399   -568   1455       C  \\nATOM    760  CD2 LEU A 105       2.270  22.519   4.296  1.00 36.37           C  \\nANISOU  760  CD2 LEU A 105     5260   3922   4636   -515   -823   1780       C  \\nATOM    761  N   LYS A 106       1.888  18.412   1.643  1.00 40.58           N  \\nANISOU  761  N   LYS A 106     5689   5102   4627   -649   -842   1686       N  \\nATOM    762  CA  LYS A 106       1.845  17.160   0.886  1.00 38.36           C  \\nANISOU  762  CA  LYS A 106     5420   4983   4171   -707   -823   1619       C  \\nATOM    763  C   LYS A 106       2.750  16.098   1.487  1.00 37.06           C  \\nANISOU  763  C   LYS A 106     5142   4948   3990   -697   -596   1433       C  \\nATOM    764  O   LYS A 106       2.448  14.912   1.387  1.00 34.61           O  \\nANISOU  764  O   LYS A 106     4793   4716   3642   -665   -579   1334       O  \\nATOM    765  CB  LYS A 106       2.231  17.360  -0.588  1.00 40.48           C  \\nANISOU  765  CB  LYS A 106     5920   5310   4151   -899   -860   1730       C  \\nATOM    766  CG  LYS A 106       1.346  18.306  -1.411  1.00 53.75           C  \\nANISOU  766  CG  LYS A 106     7762   6869   5791   -918  -1124   1947       C  \\nATOM    767  CD  LYS A 106      -0.136  18.072  -1.165  1.00 70.50           C  \\nANISOU  767  CD  LYS A 106     9747   8948   8092   -745  -1349   1966       C  \\nATOM    768  CE  LYS A 106      -1.003  19.036  -1.977  1.00 81.67           C  \\nANISOU  768  CE  LYS A 106    11301  10249   9481   -725  -1642   2194       C  \\nATOM    769  NZ  LYS A 106      -2.458  18.898  -1.659  1.00 82.99           N  \\nANISOU  769  NZ  LYS A 106    11271  10400   9861   -533  -1863   2206       N  \\nATOM    770  N   THR A 107       3.869  16.518   2.082  1.00 32.54           N  \\nANISOU  770  N   THR A 107     4526   4396   3440   -728   -434   1388       N  \\nATOM    771  CA  THR A 107       4.829  15.579   2.658  1.00 28.57           C  \\nANISOU  771  CA  THR A 107     3902   4028   2927   -693   -238   1226       C  \\nATOM    772  C   THR A 107       5.255  16.038   4.047  1.00 34.27           C  \\nANISOU  772  C   THR A 107     4481   4711   3829   -619   -171   1175       C  \\nATOM    773  O   THR A 107       5.693  17.175   4.239  1.00 34.02           O  \\nANISOU  773  O   THR A 107     4481   4622   3825   -701   -162   1236       O  \\nATOM    774  CB  THR A 107       6.092  15.416   1.777  1.00 29.25           C  \\nANISOU  774  CB  THR A 107     4052   4271   2790   -842    -74   1192       C  \\nATOM    775  OG1 THR A 107       6.722  16.684   1.609  1.00 30.42           O  \\nANISOU  775  OG1 THR A 107     4265   4396   2898   -992    -49   1290       O  \\nATOM    776  CG2 THR A 107       5.749  14.836   0.399  1.00 30.67           C  \\nANISOU  776  CG2 THR A 107     4396   4507   2748   -930   -114   1211       C  \\nATOM    777  N   VAL A 108       5.112  15.148   5.019  1.00 25.29           N  \\nANISOU  777  N   VAL A 108     3212   3599   2799   -481   -128   1064       N  \\nATOM    778  CA  VAL A 108       5.392  15.473   6.419  1.00 24.44           C  \\nANISOU  778  CA  VAL A 108     2980   3463   2842   -409    -82   1010       C  \\nATOM    779  C   VAL A 108       6.120  14.298   7.079  1.00 24.56           C  \\nANISOU  779  C   VAL A 108     2885   3601   2846   -320     35    885       C  \\nATOM    780  O   VAL A 108       5.701  13.148   6.961  1.00 26.16           O  \\nANISOU  780  O   VAL A 108     3099   3812   3029   -242     34    836       O  \\nATOM    781  CB  VAL A 108       4.067  15.779   7.204  1.00 26.57           C  \\nANISOU  781  CB  VAL A 108     3215   3587   3294   -302   -194   1030       C  \\nATOM    782  CG1 VAL A 108       4.356  16.141   8.646  1.00 22.84           C  \\nANISOU  782  CG1 VAL A 108     2644   3090   2943   -248   -134    961       C  \\nATOM    783  CG2 VAL A 108       3.311  16.925   6.542  1.00 29.45           C  \\nANISOU  783  CG2 VAL A 108     3682   3815   3693   -339   -333   1159       C  \\nATOM    784  N   ALA A 109       7.216  14.580   7.766  1.00 28.99           N  \\nANISOU  784  N   ALA A 109     3348   4251   3417   -334    123    837       N  \\nATOM    785  CA  ALA A 109       7.962  13.526   8.439  1.00 23.97           C  \\nANISOU  785  CA  ALA A 109     2599   3731   2777   -220    205    736       C  \\nATOM    786  C   ALA A 109       7.689  13.518   9.946  1.00 25.45           C  \\nANISOU  786  C   ALA A 109     2713   3868   3087   -130    173    705       C  \\nATOM    787  O   ALA A 109       7.541  14.564  10.563  1.00 24.82           O  \\nANISOU  787  O   ALA A 109     2625   3728   3078   -189    140    727       O  \\nATOM    788  CB  ALA A 109       9.436  13.676   8.171  1.00 24.13           C  \\nANISOU  788  CB  ALA A 109     2520   3937   2710   -285    318    693       C  \\nATOM    789  N   PHE A 110       7.637  12.329  10.529  1.00 23.66           N  \\nANISOU  789  N   PHE A 110     2462   3656   2872      5    188    652       N  \\nATOM    790  CA  PHE A 110       7.390  12.177  11.952  1.00 27.87           C  \\nANISOU  790  CA  PHE A 110     2953   4153   3481     78    164    629       C  \\nATOM    791  C   PHE A 110       8.428  11.208  12.460  1.00 31.76           C  \\nANISOU  791  C   PHE A 110     3375   4763   3930    197    204    578       C  \\nATOM    792  O   PHE A 110       8.922  10.371  11.699  1.00 26.36           O  \\nANISOU  792  O   PHE A 110     2700   4133   3185    265    252    549       O  \\nATOM    793  CB  PHE A 110       6.065  11.449  12.228  1.00 23.30           C  \\nANISOU  793  CB  PHE A 110     2452   3447   2952    133    125    636       C  \\nATOM    794  CG  PHE A 110       4.819  12.294  12.147  1.00 23.54           C  \\nANISOU  794  CG  PHE A 110     2508   3367   3071     72     66    677       C  \\nATOM    795  CD1 PHE A 110       4.265  12.645  10.920  1.00 20.25           C  \\nANISOU  795  CD1 PHE A 110     2145   2908   2642     13     13    730       C  \\nATOM    796  CD2 PHE A 110       4.141  12.647  13.309  1.00 21.20           C  \\nANISOU  796  CD2 PHE A 110     2180   3011   2862     89     62    657       C  \\nATOM    797  CE1 PHE A 110       3.082  13.360  10.863  1.00 29.36           C  \\nANISOU  797  CE1 PHE A 110     3298   3962   3894     -2    -64    772       C  \\nATOM    798  CE2 PHE A 110       2.947  13.363  13.256  1.00 24.45           C  \\nANISOU  798  CE2 PHE A 110     2585   3327   3378     73     17    677       C  \\nATOM    799  CZ  PHE A 110       2.419  13.719  12.038  1.00 27.71           C  \\nANISOU  799  CZ  PHE A 110     3028   3698   3801     41    -56    739       C  \\nATOM    800  N   THR A 111       8.714  11.289  13.759  1.00 24.67           N  \\nANISOU  800  N   THR A 111     2419   3898   3058    234    180    564       N  \\nATOM    801  CA  THR A 111       9.303  10.177  14.485  1.00 24.93           C  \\nANISOU  801  CA  THR A 111     2425   3986   3061    387    172    543       C  \\nATOM    802  C   THR A 111       8.218   9.590  15.399  1.00 26.04           C  \\nANISOU  802  C   THR A 111     2684   3984   3224    425    139    565       C  \\nATOM    803  O   THR A 111       7.074  10.041  15.383  1.00 24.64           O  \\nANISOU  803  O   THR A 111     2566   3699   3097    339    136    579       O  \\nATOM    804  CB  THR A 111      10.527  10.617  15.298  1.00 34.73           C  \\nANISOU  804  CB  THR A 111     3511   5404   4281    391    148    521       C  \\nATOM    805  OG1 THR A 111      10.157  11.678  16.185  1.00 32.26           O  \\nANISOU  805  OG1 THR A 111     3205   5061   3992    264    114    522       O  \\nATOM    806  CG2 THR A 111      11.630  11.095  14.367  1.00 36.95           C  \\nANISOU  806  CG2 THR A 111     3652   5855   4534    331    204    491       C  \\nATOM    807  N   ALA A 112       8.564   8.573  16.177  1.00 26.27           N  \\nANISOU  807  N   ALA A 112     2748   4017   3218    554    115    572       N  \\nATOM    808  CA  ALA A 112       7.585   7.911  17.034  1.00 27.43           C  \\nANISOU  808  CA  ALA A 112     3032   4030   3358    561    101    602       C  \\nATOM    809  C   ALA A 112       7.337   8.753  18.284  1.00 27.32           C  \\nANISOU  809  C   ALA A 112     2992   4048   3340    473     82    601       C  \\nATOM    810  O   ALA A 112       8.008   8.574  19.290  1.00 29.26           O  \\nANISOU  810  O   ALA A 112     3227   4370   3523    523     35    613       O  \\nATOM    811  CB  ALA A 112       8.071   6.491  17.422  1.00 23.05           C  \\nANISOU  811  CB  ALA A 112     2574   3435   2749    730     74    629       C  \\nATOM    812  N   LEU A 113       6.366   9.659  18.200  1.00 30.13           N  \\nANISOU  812  N   LEU A 113     3344   4345   3760    352    114    580       N  \\nATOM    813  CA  LEU A 113       6.114  10.656  19.238  1.00 26.84           C  \\nANISOU  813  CA  LEU A 113     2904   3945   3348    265    123    546       C  \\nATOM    814  C   LEU A 113       5.925  10.047  20.622  1.00 34.93           C  \\nANISOU  814  C   LEU A 113     4016   4970   4287    275    121    555       C  \\nATOM    815  O   LEU A 113       5.099   9.154  20.812  1.00 30.00           O  \\nANISOU  815  O   LEU A 113     3494   4257   3648    282    152    583       O  \\nATOM    816  CB  LEU A 113       4.878  11.472  18.876  1.00 28.22           C  \\nANISOU  816  CB  LEU A 113     3076   4019   3627    186    165    521       C  \\nATOM    817  CG  LEU A 113       4.917  12.303  17.598  1.00 28.26           C  \\nANISOU  817  CG  LEU A 113     3031   4001   3704    153    149    532       C  \\nATOM    818  CD1 LEU A 113       3.579  13.034  17.395  1.00 27.32           C  \\nANISOU  818  CD1 LEU A 113     2910   3772   3699    118    163    519       C  \\nATOM    819  CD2 LEU A 113       6.044  13.288  17.691  1.00 31.64           C  \\nANISOU  819  CD2 LEU A 113     3402   4510   4108     98    134    511       C  \\nATOM    820  N   GLY A 114       6.700  10.525  21.591  1.00 31.07           N  \\nANISOU  820  N   GLY A 114     3498   4584   3722    251     82    534       N  \\nATOM    821  CA  GLY A 114       6.579  10.029  22.942  1.00 29.12           C  \\nANISOU  821  CA  GLY A 114     3356   4352   3356    242     69    551       C  \\nATOM    822  C   GLY A 114       7.323   8.736  23.222  1.00 28.75           C  \\nANISOU  822  C   GLY A 114     3376   4334   3213    376    -17    638       C  \\nATOM    823  O   GLY A 114       7.342   8.273  24.362  1.00 30.82           O  \\nANISOU  823  O   GLY A 114     3751   4609   3348    371    -54    679       O  \\nATOM    824  N   ALA A 115       7.952   8.140  22.212  1.00 28.58           N  \\nANISOU  824  N   ALA A 115     3301   4316   3242    504    -50    668       N  \\nATOM    825  CA  ALA A 115       8.673   6.888  22.476  1.00 34.64           C  \\nANISOU  825  CA  ALA A 115     4140   5083   3938    678   -135    745       C  \\nATOM    826  C   ALA A 115      10.109   7.089  22.972  1.00 42.60           C  \\nANISOU  826  C   ALA A 115     5006   6291   4888    769   -258    755       C  \\nATOM    827  O   ALA A 115      10.705   6.187  23.552  1.00 53.26           O  \\nANISOU  827  O   ALA A 115     6416   7658   6162    921   -362    831       O  \\nATOM    828  CB  ALA A 115       8.641   5.972  21.270  1.00 30.33           C  \\nANISOU  828  CB  ALA A 115     3632   4430   3463    799   -103    756       C  \\nATOM    829  N   TRP A 116      10.674   8.267  22.750  1.00 51.62           N  \\nANISOU  829  N   TRP A 116     5962   7585   6066    672   -257    684       N  \\nATOM    830  CA  TRP A 116      12.020   8.534  23.247  1.00 53.81           C  \\nANISOU  830  CA  TRP A 116     6069   8089   6287    716   -378    681       C  \\nATOM    831  C   TRP A 116      11.958   9.278  24.577  1.00 48.54           C  \\nANISOU  831  C   TRP A 116     5445   7498   5502    557   -436    662       C  \\nATOM    832  O   TRP A 116      12.166   8.690  25.646  1.00 42.89           O  \\nANISOU  832  O   TRP A 116     4823   6818   4655    614   -546    728       O  \\nATOM    833  CB  TRP A 116      12.845   9.310  22.211  1.00 63.08           C  \\nANISOU  833  CB  TRP A 116     7009   9410   7549    674   -343    612       C  \\nATOM    834  CG  TRP A 116      14.196   9.769  22.706  1.00 76.29           C  \\nANISOU  834  CG  TRP A 116     8460  11354   9171    659   -458    589       C  \\nATOM    835  CD1 TRP A 116      14.601  11.059  22.896  1.00 79.44           C  \\nANISOU  835  CD1 TRP A 116     8740  11886   9556    438   -462    516       C  \\nATOM    836  CD2 TRP A 116      15.310   8.942  23.076  1.00 87.53           C  \\nANISOU  836  CD2 TRP A 116     9748  12952  10557    868   -598    636       C  \\nATOM    837  NE1 TRP A 116      15.896  11.089  23.358  1.00 83.74           N  \\nANISOU  837  NE1 TRP A 116     9067  12702  10048    465   -594    509       N  \\nATOM    838  CE2 TRP A 116      16.355   9.804  23.478  1.00 88.41           C  \\nANISOU  838  CE2 TRP A 116     9625  13335  10630    745   -686    585       C  \\nATOM    839  CE3 TRP A 116      15.527   7.560  23.108  1.00 96.71           C  \\nANISOU  839  CE3 TRP A 116    10970  14056  11720   1152   -664    716       C  \\nATOM    840  CZ2 TRP A 116      17.596   9.329  23.905  1.00 94.95           C  \\nANISOU  840  CZ2 TRP A 116    10235  14416  11426    906   -849    613       C  \\nATOM    841  CZ3 TRP A 116      16.766   7.089  23.536  1.00101.24           C  \\nANISOU  841  CZ3 TRP A 116    11350  14848  12268   1345   -824    749       C  \\nATOM    842  CH2 TRP A 116      17.781   7.973  23.928  1.00 99.66           C  \\nANISOU  842  CH2 TRP A 116    10876  14955  12035   1225   -920    699       C  \\nATOM    843  N   VAL A 117      11.638  10.564  24.507  1.00 44.15           N  \\nANISOU  843  N   VAL A 117     4850   6945   4980    357   -362    570       N  \\nATOM    844  CA  VAL A 117      11.627  11.407  25.690  1.00 46.95           C  \\nANISOU  844  CA  VAL A 117     5251   7368   5222    187   -396    514       C  \\nATOM    845  C   VAL A 117      10.470  11.027  26.618  1.00 51.79           C  \\nANISOU  845  C   VAL A 117     6096   7838   5743    150   -339    532       C  \\nATOM    846  O   VAL A 117      10.579  11.140  27.838  1.00 59.01           O  \\nANISOU  846  O   VAL A 117     7099   8827   6495     71   -399    524       O  \\nATOM    847  CB  VAL A 117      11.621  12.923  25.316  1.00 64.39           C  \\nANISOU  847  CB  VAL A 117     7384   9578   7502    -13   -323    399       C  \\nATOM    848  CG1 VAL A 117      10.320  13.321  24.628  1.00 66.97           C  \\nANISOU  848  CG1 VAL A 117     7810   9681   7955    -49   -170    367       C  \\nATOM    849  CG2 VAL A 117      11.868  13.780  26.538  1.00 67.39           C  \\nANISOU  849  CG2 VAL A 117     7808  10047   7749   -192   -373    318       C  \\nATOM    850  N   GLY A 118       9.382  10.537  26.037  1.00 46.38           N  \\nANISOU  850  N   GLY A 118     5505   6970   5147    193   -223    554       N  \\nATOM    851  CA  GLY A 118       8.213  10.147  26.804  1.00 37.41           C  \\nANISOU  851  CA  GLY A 118     4562   5715   3939    135   -137    564       C  \\nATOM    852  C   GLY A 118       8.299   8.786  27.481  1.00 35.85           C  \\nANISOU  852  C   GLY A 118     4526   5495   3602    231   -215    691       C  \\nATOM    853  O   GLY A 118       7.593   8.535  28.445  1.00 36.94           O  \\nANISOU  853  O   GLY A 118     4837   5589   3610    138   -164    706       O  \\nATOM    854  N   GLY A 119       9.154   7.899  26.985  1.00 41.52           N  \\nANISOU  854  N   GLY A 119     5202   6235   4340    417   -328    782       N  \\nATOM    855  CA  GLY A 119       9.297   6.580  27.591  1.00 40.14           C  \\nANISOU  855  CA  GLY A 119     5211   5999   4042    539   -422    918       C  \\nATOM    856  C   GLY A 119       8.310   5.497  27.155  1.00 38.50           C  \\nANISOU  856  C   GLY A 119     5192   5563   3874    570   -328    980       C  \\nATOM    857  O   GLY A 119       8.374   4.377  27.645  1.00 44.92           O  \\nANISOU  857  O   GLY A 119     6205   6281   4582    657   -399   1102       O  \\nATOM    858  N   LEU A 120       7.411   5.808  26.226  1.00 31.59           N  \\nANISOU  858  N   LEU A 120     4264   4594   3144    493   -182    903       N  \\nATOM    859  CA  LEU A 120       6.394   4.843  25.817  1.00 32.00           C  \\nANISOU  859  CA  LEU A 120     4481   4449   3228    472    -93    945       C  \\nATOM    860  C   LEU A 120       6.948   3.809  24.851  1.00 27.84           C  \\nANISOU  860  C   LEU A 120     3987   3820   2772    670   -152    999       C  \\nATOM    861  O   LEU A 120       7.875   4.087  24.106  1.00 32.75           O  \\nANISOU  861  O   LEU A 120     4432   4534   3476    803   -205    963       O  \\nATOM    862  CB  LEU A 120       5.202   5.556  25.177  1.00 26.99           C  \\nANISOU  862  CB  LEU A 120     3757   3777   2722    320     60    842       C  \\nATOM    863  CG  LEU A 120       4.540   6.592  26.064  1.00 29.82           C  \\nANISOU  863  CG  LEU A 120     4081   4212   3038    150    152    760       C  \\nATOM    864  CD1 LEU A 120       3.610   7.478  25.258  1.00 24.45           C  \\nANISOU  864  CD1 LEU A 120     3250   3510   2529     74    264    655       C  \\nATOM    865  CD2 LEU A 120       3.805   5.881  27.182  1.00 29.54           C  \\nANISOU  865  CD2 LEU A 120     4259   4122   2843     31    215    814       C  \\nATOM    866  N   PRO A 121       6.365   2.605  24.844  1.00 35.67           N  \\nANISOU  866  N   PRO A 121     5214   4613   3726    675   -128   1073       N  \\nATOM    867  CA  PRO A 121       6.901   1.618  23.897  1.00 33.71           C  \\nANISOU  867  CA  PRO A 121     5023   4239   3547    876   -172   1100       C  \\nATOM    868  C   PRO A 121       6.566   1.963  22.452  1.00 33.53           C  \\nANISOU  868  C   PRO A 121     4858   4207   3674    849    -82    993       C  \\nATOM    869  O   PRO A 121       5.447   2.389  22.140  1.00 32.15           O  \\nANISOU  869  O   PRO A 121     4660   4008   3548    657     19    939       O  \\nATOM    870  CB  PRO A 121       6.275   0.280  24.348  1.00 34.13           C  \\nANISOU  870  CB  PRO A 121     5415   4046   3505    845   -164   1206       C  \\nATOM    871  CG  PRO A 121       5.230   0.620  25.341  1.00 34.65           C  \\nANISOU  871  CG  PRO A 121     5567   4132   3467    585    -79   1221       C  \\nATOM    872  CD  PRO A 121       5.442   2.023  25.832  1.00 34.22           C  \\nANISOU  872  CD  PRO A 121     5278   4318   3407    516    -74   1145       C  \\nATOM    873  N   ALA A 122       7.552   1.792  21.578  1.00 27.86           N  \\nANISOU  873  N   ALA A 122     4035   3527   3024   1044   -123    961       N  \\nATOM    874  CA  ALA A 122       7.433   2.176  20.174  1.00 33.17           C  \\nANISOU  874  CA  ALA A 122     4576   4221   3804   1022    -49    863       C  \\nATOM    875  C   ALA A 122       6.267   1.494  19.456  1.00 32.69           C  \\nANISOU  875  C   ALA A 122     4689   3965   3768    903     31    843       C  \\nATOM    876  O   ALA A 122       5.566   2.138  18.690  1.00 31.13           O  \\nANISOU  876  O   ALA A 122     4393   3802   3632    762     90    780       O  \\nATOM    877  CB  ALA A 122       8.738   1.894  19.439  1.00 32.75           C  \\nANISOU  877  CB  ALA A 122     4411   4241   3791   1258    -84    829       C  \\nATOM    878  N   GLU A 123       6.073   0.196  19.704  1.00 32.54           N  \\nANISOU  878  N   GLU A 123     4934   3735   3694    955     19    901       N  \\nATOM    879  CA  GLU A 123       5.025  -0.575  19.037  1.00 34.27           C  \\nANISOU  879  CA  GLU A 123     5340   3758   3922    818     87    875       C  \\nATOM    880  C   GLU A 123       3.622  -0.016  19.308  1.00 29.10           C  \\nANISOU  880  C   GLU A 123     4642   3137   3276    531    154    864       C  \\nATOM    881  O   GLU A 123       2.806   0.100  18.394  1.00 27.71           O  \\nANISOU  881  O   GLU A 123     4425   2948   3156    395    200    800       O  \\nATOM    882  CB  GLU A 123       5.091  -2.065  19.418  1.00 32.14           C  \\nANISOU  882  CB  GLU A 123     5407   3224   3582    905     60    950       C  \\nATOM    883  CG  GLU A 123       6.508  -2.636  19.622  1.00 59.70           C  \\nANISOU  883  CG  GLU A 123     8939   6682   7061   1240    -33    993       C  \\nATOM    884  CD  GLU A 123       7.051  -2.441  21.054  1.00 65.39           C  \\nANISOU  884  CD  GLU A 123     9653   7493   7699   1316   -138   1114       C  \\nATOM    885  OE1 GLU A 123       7.699  -1.405  21.316  1.00 49.62           O  \\nANISOU  885  OE1 GLU A 123     7382   5754   5719   1359   -180   1094       O  \\nATOM    886  OE2 GLU A 123       6.832  -3.325  21.918  1.00 71.34           O  \\nANISOU  886  OE2 GLU A 123    10694   8057   8353   1314   -184   1234       O  \\nATOM    887  N   ALA A 124       3.337   0.329  20.559  1.00 30.49           N  \\nANISOU  887  N   ALA A 124     4820   3372   3392    444    159    921       N  \\nATOM    888  CA  ALA A 124       2.038   0.923  20.882  1.00 27.91           C  \\nANISOU  888  CA  ALA A 124     4413   3107   3085    197    246    889       C  \\nATOM    889  C   ALA A 124       1.898   2.319  20.277  1.00 26.44           C  \\nANISOU  889  C   ALA A 124     3936   3097   3012    176    259    804       C  \\nATOM    890  O   ALA A 124       0.836   2.671  19.747  1.00 25.83           O  \\nANISOU  890  O   ALA A 124     3762   3040   3011     32    308    752       O  \\nATOM    891  CB  ALA A 124       1.825   0.962  22.387  1.00 27.36           C  \\nANISOU  891  CB  ALA A 124     4430   3066   2900    109    271    954       C  \\nATOM    892  N   VAL A 125       2.962   3.115  20.360  1.00 23.98           N  \\nANISOU  892  N   VAL A 125     3488   2912   2711    315    206    794       N  \\nATOM    893  CA  VAL A 125       2.961   4.470  19.793  1.00 22.58           C  \\nANISOU  893  CA  VAL A 125     3078   2869   2631    295    212    726       C  \\nATOM    894  C   VAL A 125       2.699   4.445  18.294  1.00 23.36           C  \\nANISOU  894  C   VAL A 125     3131   2939   2807    288    209    687       C  \\nATOM    895  O   VAL A 125       1.870   5.208  17.786  1.00 24.30           O  \\nANISOU  895  O   VAL A 125     3131   3095   3007    189    226    651       O  \\nATOM    896  CB  VAL A 125       4.277   5.242  20.103  1.00 24.22           C  \\nANISOU  896  CB  VAL A 125     3166   3214   2821    415    154    723       C  \\nATOM    897  CG1 VAL A 125       4.442   6.469  19.191  1.00 21.24           C  \\nANISOU  897  CG1 VAL A 125     2602   2931   2539    394    156    665       C  \\nATOM    898  CG2 VAL A 125       4.317   5.658  21.570  1.00 26.26           C  \\nANISOU  898  CG2 VAL A 125     3439   3539   2999    365    154    738       C  \\nATOM    899  N   LEU A 126       3.398   3.562  17.583  1.00 23.60           N  \\nANISOU  899  N   LEU A 126     3261   2900   2806    403    183    692       N  \\nATOM    900  CA  LEU A 126       3.223   3.455  16.141  1.00 22.40           C  \\nANISOU  900  CA  LEU A 126     3100   2723   2687    384    184    645       C  \\nATOM    901  C   LEU A 126       1.790   3.028  15.775  1.00 24.23           C  \\nANISOU  901  C   LEU A 126     3403   2866   2939    202    203    632       C  \\nATOM    902  O   LEU A 126       1.229   3.496  14.801  1.00 27.46           O  \\nANISOU  902  O   LEU A 126     3727   3316   3391    121    184    601       O  \\nATOM    903  CB  LEU A 126       4.257   2.493  15.543  1.00 23.43           C  \\nANISOU  903  CB  LEU A 126     3345   2788   2769    554    179    627       C  \\nATOM    904  CG  LEU A 126       5.705   2.996  15.635  1.00 28.91           C  \\nANISOU  904  CG  LEU A 126     3896   3627   3462    728    161    622       C  \\nATOM    905  CD1 LEU A 126       6.721   1.978  15.101  1.00 34.70           C  \\nANISOU  905  CD1 LEU A 126     4714   4305   4164    934    171    588       C  \\nATOM    906  CD2 LEU A 126       5.856   4.290  14.902  1.00 25.88           C  \\nANISOU  906  CD2 LEU A 126     3318   3397   3118    655    170    589       C  \\nATOM    907  N   ARG A 127       1.207   2.135  16.562  1.00 23.62           N  \\nANISOU  907  N   ARG A 127     3479   2675   2818    125    232    663       N  \\nATOM    908  CA  ARG A 127      -0.176   1.747  16.331  1.00 24.20           C  \\nANISOU  908  CA  ARG A 127     3587   2697   2911    -86    257    645       C  \\nATOM    909  C   ARG A 127      -1.158   2.914  16.494  1.00 24.49           C  \\nANISOU  909  C   ARG A 127     3383   2878   3043   -198    270    623       C  \\nATOM    910  O   ARG A 127      -2.025   3.106  15.641  1.00 25.88           O  \\nANISOU  910  O   ARG A 127     3466   3090   3276   -305    239    591       O  \\nATOM    911  CB  ARG A 127      -0.583   0.586  17.233  1.00 25.70           C  \\nANISOU  911  CB  ARG A 127     4004   2739   3024   -182    303    689       C  \\nATOM    912  CG  ARG A 127      -1.962   0.036  16.901  1.00 28.46           C  \\nANISOU  912  CG  ARG A 127     4391   3038   3383   -437    334    660       C  \\nATOM    913  CD  ARG A 127      -2.241  -1.285  17.609  1.00 32.34           C  \\nANISOU  913  CD  ARG A 127     5176   3336   3775   -557    383    709       C  \\nATOM    914  NE  ARG A 127      -1.383  -2.366  17.136  1.00 32.12           N  \\nANISOU  914  NE  ARG A 127     5429   3093   3683   -422    346    719       N  \\nATOM    915  CZ  ARG A 127      -1.603  -3.077  16.034  1.00 36.39           C  \\nANISOU  915  CZ  ARG A 127     6101   3511   4213   -489    329    655       C  \\nATOM    916  NH1 ARG A 127      -2.654  -2.827  15.271  1.00 30.53           N  \\nANISOU  916  NH1 ARG A 127     5226   2862   3514   -703    320    590       N  \\nATOM    917  NH2 ARG A 127      -0.765  -4.045  15.697  1.00 41.57           N  \\nANISOU  917  NH2 ARG A 127     7025   3955   4815   -331    314    648       N  \\nATOM    918  N   VAL A 128      -1.045   3.703  17.564  1.00 26.22           N  \\nANISOU  918  N   VAL A 128     3503   3182   3279   -167    307    635       N  \\nATOM    919  CA  VAL A 128      -1.965   4.833  17.655  1.00 27.05           C  \\nANISOU  919  CA  VAL A 128     3384   3401   3492   -236    328    595       C  \\nATOM    920  C   VAL A 128      -1.723   5.881  16.569  1.00 28.15           C  \\nANISOU  920  C   VAL A 128     3377   3603   3716   -156    251    580       C  \\nATOM    921  O   VAL A 128      -2.685   6.442  16.055  1.00 24.24           O  \\nANISOU  921  O   VAL A 128     2736   3157   3317   -216    223    559       O  \\nATOM    922  CB  VAL A 128      -2.165   5.462  19.090  1.00 34.56           C  \\nANISOU  922  CB  VAL A 128     4273   4420   4440   -255    416    580       C  \\nATOM    923  CG1 VAL A 128      -1.598   4.599  20.189  1.00 30.71           C  \\nANISOU  923  CG1 VAL A 128     3991   3871   3808   -257    453    631       C  \\nATOM    924  CG2 VAL A 128      -1.690   6.891  19.161  1.00 23.99           C  \\nANISOU  924  CG2 VAL A 128     2783   3161   3170   -150    396    548       C  \\nATOM    925  N   MET A 129      -0.456   6.132  16.217  1.00 24.39           N  \\nANISOU  925  N   MET A 129     2937   3131   3199    -28    213    596       N  \\nATOM    926  CA  MET A 129      -0.139   7.032  15.105  1.00 25.88           C  \\nANISOU  926  CA  MET A 129     3033   3368   3433     15    148    596       C  \\nATOM    927  C   MET A 129      -0.758   6.527  13.812  1.00 24.20           C  \\nANISOU  927  C   MET A 129     2853   3128   3212    -58     90    594       C  \\nATOM    928  O   MET A 129      -1.392   7.286  13.072  1.00 25.69           O  \\nANISOU  928  O   MET A 129     2935   3358   3466    -95     23    604       O  \\nATOM    929  CB  MET A 129       1.377   7.230  14.952  1.00 22.49           C  \\nANISOU  929  CB  MET A 129     2630   2971   2942    132    142    605       C  \\nATOM    930  CG  MET A 129       1.990   7.974  16.154  1.00 22.45           C  \\nANISOU  930  CG  MET A 129     2566   3023   2942    177    169    602       C  \\nATOM    931  SD  MET A 129       3.751   8.254  15.979  1.00 24.28           S  \\nANISOU  931  SD  MET A 129     2770   3343   3112    283    151    605       S  \\nATOM    932  CE  MET A 129       3.777   9.585  14.776  1.00 19.43           C  \\nANISOU  932  CE  MET A 129     2061   2768   2555    226    123    608       C  \\nATOM    933  N   LEU A 130      -0.608   5.235  13.560  1.00 22.65           N  \\nANISOU  933  N   LEU A 130     2823   2851   2931    -82    104    583       N  \\nATOM    934  CA  LEU A 130      -1.185   4.634  12.357  1.00 26.95           C  \\nANISOU  934  CA  LEU A 130     3436   3363   3440   -181     51    561       C  \\nATOM    935  C   LEU A 130      -2.689   4.899  12.260  1.00 25.77           C  \\nANISOU  935  C   LEU A 130     3153   3264   3374   -330      2    559       C  \\nATOM    936  O   LEU A 130      -3.195   5.275  11.195  1.00 27.09           O  \\nANISOU  936  O   LEU A 130     3256   3485   3554   -384    -96    564       O  \\nATOM    937  CB  LEU A 130      -0.906   3.132  12.332  1.00 23.32           C  \\nANISOU  937  CB  LEU A 130     3210   2767   2884   -195     92    533       C  \\nATOM    938  CG  LEU A 130      -1.668   2.342  11.273  1.00 29.08           C  \\nANISOU  938  CG  LEU A 130     4048   3441   3561   -349     46    489       C  \\nATOM    939  CD1 LEU A 130      -1.321   2.880   9.895  1.00 30.22           C  \\nANISOU  939  CD1 LEU A 130     4165   3659   3659   -329    -19    472       C  \\nATOM    940  CD2 LEU A 130      -1.361   0.841  11.367  1.00 25.81           C  \\nANISOU  940  CD2 LEU A 130     3912   2840   3056   -357    100    452       C  \\nATOM    941  N   GLU A 131      -3.390   4.720  13.377  1.00 26.78           N  \\nANISOU  941  N   GLU A 131     3230   3393   3551   -396     68    554       N  \\nATOM    942  CA  GLU A 131      -4.839   4.928  13.437  1.00 26.90           C  \\nANISOU  942  CA  GLU A 131     3069   3489   3662   -533     47    536       C  \\nATOM    943  C   GLU A 131      -5.216   6.378  13.146  1.00 27.04           C  \\nANISOU  943  C   GLU A 131     2855   3610   3807   -448    -23    550       C  \\nATOM    944  O   GLU A 131      -6.196   6.640  12.456  1.00 27.62           O  \\nANISOU  944  O   GLU A 131     2784   3759   3951   -512   -120    550       O  \\nATOM    945  CB  GLU A 131      -5.394   4.495  14.802  1.00 25.07           C  \\nANISOU  945  CB  GLU A 131     2830   3254   3440   -625    173    518       C  \\nATOM    946  CG  GLU A 131      -5.355   2.992  15.017  1.00 29.66           C  \\nANISOU  946  CG  GLU A 131     3659   3706   3902   -756    224    520       C  \\nATOM    947  CD  GLU A 131      -5.790   2.568  16.411  1.00 31.87           C  \\nANISOU  947  CD  GLU A 131     3978   3975   4155   -864    355    525       C  \\nATOM    948  OE1 GLU A 131      -6.278   3.432  17.172  1.00 26.88           O  \\nANISOU  948  OE1 GLU A 131     3146   3464   3602   -854    420    503       O  \\nATOM    949  OE2 GLU A 131      -5.635   1.364  16.738  1.00 31.26           O  \\nANISOU  949  OE2 GLU A 131     4154   3755   3967   -959    399    549       O  \\nATOM    950  N   GLU A 132      -4.430   7.320  13.653  1.00 24.56           N  \\nANISOU  950  N   GLU A 132     2515   3294   3522   -304     11    563       N  \\nATOM    951  CA  GLU A 132      -4.697   8.731  13.369  1.00 26.22           C  \\nANISOU  951  CA  GLU A 132     2559   3551   3853   -211    -56    581       C  \\nATOM    952  C   GLU A 132      -4.324   9.117  11.934  1.00 29.33           C  \\nANISOU  952  C   GLU A 132     2999   3940   4207   -185   -192    638       C  \\nATOM    953  O   GLU A 132      -5.040   9.885  11.287  1.00 23.92           O  \\nANISOU  953  O   GLU A 132     2191   3290   3608   -166   -308    673       O  \\nATOM    954  CB  GLU A 132      -3.996   9.641  14.381  1.00 22.26           C  \\nANISOU  954  CB  GLU A 132     2044   3030   3383   -101     28    566       C  \\nATOM    955  CG  GLU A 132      -4.612   9.557  15.773  1.00 24.41           C  \\nANISOU  955  CG  GLU A 132     2244   3332   3699   -134    159    503       C  \\nATOM    956  CD  GLU A 132      -6.119   9.818  15.766  1.00 31.62           C  \\nANISOU  956  CD  GLU A 132     2939   4321   4755   -174    156    463       C  \\nATOM    957  OE1 GLU A 132      -6.574  10.755  15.075  1.00 37.31           O  \\nANISOU  957  OE1 GLU A 132     3524   5055   5596    -89     53    480       O  \\nATOM    958  OE2 GLU A 132      -6.857   9.083  16.450  1.00 32.93           O  \\nANISOU  958  OE2 GLU A 132     3062   4537   4911   -292    254    419       O  \\nATOM    959  N   ILE A 133      -3.218   8.579  11.432  1.00 28.43           N  \\nANISOU  959  N   ILE A 133     3060   3786   3956   -180   -180    648       N  \\nATOM    960  CA  ILE A 133      -2.754   8.947  10.092  1.00 31.09           C  \\nANISOU  960  CA  ILE A 133     3462   4132   4219   -177   -277    696       C  \\nATOM    961  C   ILE A 133      -3.733   8.479   9.011  1.00 32.03           C  \\nANISOU  961  C   ILE A 133     3579   4286   4304   -292   -404    706       C  \\nATOM    962  O   ILE A 133      -3.937   9.159   8.007  1.00 32.24           O  \\nANISOU  962  O   ILE A 133     3590   4345   4316   -297   -533    768       O  \\nATOM    963  CB  ILE A 133      -1.309   8.430   9.814  1.00 30.71           C  \\nANISOU  963  CB  ILE A 133     3578   4062   4030   -141   -203    679       C  \\nATOM    964  CG1 ILE A 133      -0.314   9.145  10.736  1.00 23.52           C  \\nANISOU  964  CG1 ILE A 133     2630   3155   3152    -41   -122    680       C  \\nATOM    965  CG2 ILE A 133      -0.917   8.661   8.353  1.00 28.05           C  \\nANISOU  965  CG2 ILE A 133     3324   3753   3582   -178   -276    712       C  \\nATOM    966  CD1 ILE A 133       1.020   8.430  10.895  1.00 20.60           C  \\nANISOU  966  CD1 ILE A 133     2358   2789   2678     20    -36    647       C  \\nATOM    967  N   LYS A 134      -4.364   7.334   9.240  1.00 29.08           N  \\nANISOU  967  N   LYS A 134     3232   3906   3910   -401   -377    652       N  \\nATOM    968  CA  LYS A 134      -5.349   6.805   8.307  1.00 28.24           C  \\nANISOU  968  CA  LYS A 134     3115   3849   3766   -550   -501    646       C  \\nATOM    969  C   LYS A 134      -6.550   7.729   8.130  1.00 31.54           C  \\nANISOU  969  C   LYS A 134     3288   4370   4328   -545   -642    692       C  \\nATOM    970  O   LYS A 134      -7.283   7.611   7.153  1.00 31.05           O  \\nANISOU  970  O   LYS A 134     3189   4380   4231   -644   -801    713       O  \\nATOM    971  CB  LYS A 134      -5.833   5.421   8.746  1.00 39.00           C  \\nANISOU  971  CB  LYS A 134     4557   5170   5090   -700   -430    573       C  \\nATOM    972  CG  LYS A 134      -4.820   4.294   8.551  1.00 48.78           C  \\nANISOU  972  CG  LYS A 134     6079   6282   6174   -708   -340    525       C  \\nATOM    973  CD  LYS A 134      -5.299   2.981   9.187  1.00 51.15           C  \\nANISOU  973  CD  LYS A 134     6495   6491   6449   -852   -260    469       C  \\nATOM    974  CE  LYS A 134      -6.540   2.423   8.489  1.00 58.91           C  \\nANISOU  974  CE  LYS A 134     7446   7526   7409  -1092   -367    433       C  \\nATOM    975  NZ  LYS A 134      -7.078   1.185   9.140  1.00 62.17           N  \\nANISOU  975  NZ  LYS A 134     7981   7843   7798  -1280   -281    382       N  \\nATOM    976  N   LYS A 135      -6.758   8.653   9.062  1.00 31.22           N  \\nANISOU  976  N   LYS A 135     3079   4338   4446   -420   -591    702       N  \\nATOM    977  CA  LYS A 135      -7.901   9.563   8.959  1.00 31.82           C  \\nANISOU  977  CA  LYS A 135     2902   4499   4688   -364   -715    735       C  \\nATOM    978  C   LYS A 135      -7.620  10.819   8.128  1.00 33.26           C  \\nANISOU  978  C   LYS A 135     3097   4651   4889   -234   -862    839       C  \\nATOM    979  O   LYS A 135      -8.550  11.512   7.737  1.00 35.28           O  \\nANISOU  979  O   LYS A 135     3178   4964   5263   -172  -1020    890       O  \\nATOM    980  CB  LYS A 135      -8.372   9.984  10.342  1.00 33.84           C  \\nANISOU  980  CB  LYS A 135     2975   4771   5112   -284   -577    676       C  \\nATOM    981  CG  LYS A 135      -8.895   8.845  11.196  1.00 34.82           C  \\nANISOU  981  CG  LYS A 135     3066   4942   5224   -440   -439    590       C  \\nATOM    982  CD  LYS A 135      -9.002   9.296  12.645  1.00 31.75           C  \\nANISOU  982  CD  LYS A 135     2570   4553   4940   -362   -260    529       C  \\nATOM    983  CE  LYS A 135      -9.599   8.210  13.532  1.00 36.28           C  \\nANISOU  983  CE  LYS A 135     3120   5180   5485   -544   -112    457       C  \\nATOM    984  NZ  LYS A 135      -9.650   8.675  14.947  1.00 42.87           N  \\nANISOU  984  NZ  LYS A 135     3878   6025   6384   -480     75    394       N  \\nATOM    985  N   ALA A 136      -6.344  11.118   7.882  1.00 27.23           N  \\nANISOU  985  N   ALA A 136     2534   3799   4012   -194   -812    874       N  \\nATOM    986  CA  ALA A 136      -5.960  12.317   7.140  1.00 29.49           C  \\nANISOU  986  CA  ALA A 136     2880   4035   4291   -107   -927    982       C  \\nATOM    987  C   ALA A 136      -6.188  12.080   5.639  1.00 34.06           C  \\nANISOU  987  C   ALA A 136     3554   4668   4719   -210  -1116   1060       C  \\nATOM    988  O   ALA A 136      -6.241  10.930   5.200  1.00 33.40           O  \\nANISOU  988  O   ALA A 136     3549   4640   4502   -353  -1111   1005       O  \\nATOM    989  CB  ALA A 136      -4.487  12.677   7.443  1.00 26.01           C  \\nANISOU  989  CB  ALA A 136     2602   3510   3769    -72   -787    981       C  \\nATOM    990  N   PRO A 137      -6.346  13.157   4.846  1.00 34.71           N  \\nANISOU  990  N   PRO A 137     3658   4724   4808   -147  -1290   1188       N  \\nATOM    991  CA  PRO A 137      -6.654  12.942   3.425  1.00 37.80           C  \\nANISOU  991  CA  PRO A 137     4148   5185   5029   -259  -1494   1272       C  \\nATOM    992  C   PRO A 137      -5.483  12.307   2.663  1.00 40.30           C  \\nANISOU  992  C   PRO A 137     4733   5501   5078   -399  -1401   1250       C  \\nATOM    993  O   PRO A 137      -4.332  12.434   3.087  1.00 35.47           O  \\nANISOU  993  O   PRO A 137     4219   4825   4431   -370  -1217   1216       O  \\nATOM    994  CB  PRO A 137      -6.922  14.365   2.889  1.00 37.78           C  \\nANISOU  994  CB  PRO A 137     4149   5116   5092   -136  -1688   1436       C  \\nATOM    995  CG  PRO A 137      -6.912  15.274   4.086  1.00 40.05           C  \\nANISOU  995  CG  PRO A 137     4311   5289   5615     50  -1579   1413       C  \\nATOM    996  CD  PRO A 137      -6.121  14.577   5.151  1.00 30.63           C  \\nANISOU  996  CD  PRO A 137     3133   4089   4416     10  -1307   1267       C  \\nATOM    997  N   ASP A 138      -5.781  11.631   1.554  1.00 40.79           N  \\nANISOU  997  N   ASP A 138     4899   5647   4952   -551  -1524   1256       N  \\nATOM    998  CA  ASP A 138      -4.739  10.978   0.761  1.00 51.41           C  \\nANISOU  998  CA  ASP A 138     6499   7001   6033   -679  -1420   1208       C  \\nATOM    999  C   ASP A 138      -3.819  11.940   0.004  1.00 42.83           C  \\nANISOU  999  C   ASP A 138     5589   5883   4801   -683  -1425   1324       C  \\nATOM   1000  O   ASP A 138      -2.865  11.501  -0.620  1.00 38.25           O  \\nANISOU 1000  O   ASP A 138     5199   5328   4005   -781  -1304   1275       O  \\nATOM   1001  CB  ASP A 138      -5.340   9.955  -0.208  1.00 62.34           C  \\nANISOU 1001  CB  ASP A 138     7974   8479   7235   -863  -1540   1158       C  \\nATOM   1002  CG  ASP A 138      -5.593   8.609   0.451  1.00 75.49           C  \\nANISOU 1002  CG  ASP A 138     9604  10137   8942   -929  -1413    993       C  \\nATOM   1003  OD1 ASP A 138      -4.978   8.341   1.509  1.00 71.05           O  \\nANISOU 1003  OD1 ASP A 138     9014   9496   8485   -834  -1204    919       O  \\nATOM   1004  OD2 ASP A 138      -6.401   7.819  -0.094  1.00 83.16           O  \\nANISOU 1004  OD2 ASP A 138    10592  11178   9829  -1090  -1534    944       O  \\nATOM   1005  N   THR A 139      -4.111  13.239   0.059  1.00 33.67           N  \\nANISOU 1005  N   THR A 139     4374   4661   3758   -581  -1554   1471       N  \\nATOM   1006  CA  THR A 139      -3.237  14.252  -0.516  1.00 45.75           C  \\nANISOU 1006  CA  THR A 139     6085   6130   5167   -604  -1545   1594       C  \\nATOM   1007  C   THR A 139      -1.953  14.354   0.299  1.00 42.50           C  \\nANISOU 1007  C   THR A 139     5687   5669   4790   -573  -1273   1509       C  \\nATOM   1008  O   THR A 139      -0.973  14.978  -0.124  1.00 38.91           O  \\nANISOU 1008  O   THR A 139     5382   5194   4208   -641  -1197   1569       O  \\nATOM   1009  CB  THR A 139      -3.901  15.636  -0.515  1.00 42.93           C  \\nANISOU 1009  CB  THR A 139     5684   5668   4961   -479  -1750   1771       C  \\nATOM   1010  OG1 THR A 139      -4.434  15.905   0.789  1.00 48.13           O  \\nANISOU 1010  OG1 THR A 139     6105   6259   5922   -299  -1705   1710       O  \\nATOM   1011  CG2 THR A 139      -5.023  15.702  -1.551  1.00 50.13           C  \\nANISOU 1011  CG2 THR A 139     6605   6645   5795   -511  -2064   1898       C  \\nATOM   1012  N   LEU A 140      -1.967  13.742   1.477  1.00 34.95           N  \\nANISOU 1012  N   LEU A 140     4573   4709   3999   -486  -1135   1374       N  \\nATOM   1013  CA  LEU A 140      -0.851  13.853   2.402  1.00 32.21           C  \\nANISOU 1013  CA  LEU A 140     4202   4330   3708   -437   -914   1297       C  \\nATOM   1014  C   LEU A 140      -0.078  12.547   2.505  1.00 35.42           C  \\nANISOU 1014  C   LEU A 140     4644   4811   4003   -476   -732   1149       C  \\nATOM   1015  O   LEU A 140      -0.671  11.466   2.620  1.00 28.45           O  \\nANISOU 1015  O   LEU A 140     3732   3951   3127   -483   -740   1064       O  \\nATOM   1016  CB  LEU A 140      -1.363  14.247   3.782  1.00 34.78           C  \\nANISOU 1016  CB  LEU A 140     4340   4580   4292   -292   -892   1265       C  \\nATOM   1017  CG  LEU A 140      -0.308  14.291   4.886  1.00 42.33           C  \\nANISOU 1017  CG  LEU A 140     5258   5520   5305   -247   -689   1177       C  \\nATOM   1018  CD1 LEU A 140       0.648  15.456   4.656  1.00 43.78           C  \\nANISOU 1018  CD1 LEU A 140     5540   5655   5441   -292   -654   1254       C  \\nATOM   1019  CD2 LEU A 140      -0.969  14.386   6.244  1.00 39.00           C  \\nANISOU 1019  CD2 LEU A 140     4670   5049   5099   -130   -660   1117       C  \\nATOM   1020  N   GLU A 141       1.248  12.653   2.448  1.00 33.22           N  \\nANISOU 1020  N   GLU A 141     4428   4568   3626   -502   -568   1115       N  \\nATOM   1021  CA  GLU A 141       2.114  11.504   2.672  1.00 30.34           C  \\nANISOU 1021  CA  GLU A 141     4072   4264   3190   -481   -386    971       C  \\nATOM   1022  C   GLU A 141       2.938  11.714   3.931  1.00 33.92           C  \\nANISOU 1022  C   GLU A 141     4400   4711   3775   -378   -252    922       C  \\nATOM   1023  O   GLU A 141       3.668  12.696   4.053  1.00 31.04           O  \\nANISOU 1023  O   GLU A 141     4015   4360   3418   -404   -209    970       O  \\nATOM   1024  CB  GLU A 141       3.042  11.265   1.481  1.00 34.97           C  \\nANISOU 1024  CB  GLU A 141     4807   4946   3533   -589   -291    942       C  \\nATOM   1025  CG  GLU A 141       3.997  10.084   1.669  1.00 32.06           C  \\nANISOU 1025  CG  GLU A 141     4438   4634   3108   -523    -94    779       C  \\nATOM   1026  CD  GLU A 141       5.186  10.136   0.719  1.00 38.18           C  \\nANISOU 1026  CD  GLU A 141     5295   5534   3677   -605     58    735       C  \\nATOM   1027  OE1 GLU A 141       5.968  11.102   0.790  1.00 37.50           O  \\nANISOU 1027  OE1 GLU A 141     5152   5508   3589   -650    120    793       O  \\nATOM   1028  OE2 GLU A 141       5.332   9.212  -0.110  1.00 41.81           O  \\nANISOU 1028  OE2 GLU A 141     5882   6034   3971   -639    128    633       O  \\nATOM   1029  N   VAL A 142       2.794  10.783   4.869  1.00 28.97           N  \\nANISOU 1029  N   VAL A 142     3706   4062   3241   -282   -197    831       N  \\nATOM   1030  CA  VAL A 142       3.562  10.781   6.094  1.00 28.78           C  \\nANISOU 1030  CA  VAL A 142     3577   4047   3313   -183    -89    780       C  \\nATOM   1031  C   VAL A 142       4.767   9.853   5.967  1.00 33.20           C  \\nANISOU 1031  C   VAL A 142     4158   4684   3770   -130     56    681       C  \\nATOM   1032  O   VAL A 142       4.637   8.686   5.588  1.00 29.89           O  \\nANISOU 1032  O   VAL A 142     3830   4245   3282   -105     85    609       O  \\nATOM   1033  CB  VAL A 142       2.696  10.348   7.290  1.00 29.31           C  \\nANISOU 1033  CB  VAL A 142     3570   4039   3526   -108   -116    755       C  \\nATOM   1034  CG1 VAL A 142       3.546  10.202   8.535  1.00 23.06           C  \\nANISOU 1034  CG1 VAL A 142     2703   3268   2790    -14    -17    706       C  \\nATOM   1035  CG2 VAL A 142       1.590  11.370   7.519  1.00 31.57           C  \\nANISOU 1035  CG2 VAL A 142     3787   4267   3941   -125   -234    831       C  \\nATOM   1036  N   THR A 143       5.941  10.394   6.258  1.00 29.73           N  \\nANISOU 1036  N   THR A 143     3634   4336   3325   -113    145    671       N  \\nATOM   1037  CA  THR A 143       7.170   9.615   6.283  1.00 26.39           C  \\nANISOU 1037  CA  THR A 143     3169   4017   2842    -24    282    573       C  \\nATOM   1038  C   THR A 143       7.584   9.410   7.734  1.00 29.92           C  \\nANISOU 1038  C   THR A 143     3494   4467   3407    107    296    548       C  \\nATOM   1039  O   THR A 143       7.753  10.379   8.481  1.00 29.83           O  \\nANISOU 1039  O   THR A 143     3387   4477   3469     72    268    591       O  \\nATOM   1040  CB  THR A 143       8.295  10.363   5.556  1.00 29.93           C  \\nANISOU 1040  CB  THR A 143     3568   4617   3186   -120    377    574       C  \\nATOM   1041  OG1 THR A 143       7.949  10.533   4.175  1.00 28.98           O  \\nANISOU 1041  OG1 THR A 143     3595   4501   2916   -258    365    607       O  \\nATOM   1042  CG2 THR A 143       9.621   9.621   5.678  1.00 26.10           C  \\nANISOU 1042  CG2 THR A 143     2971   4277   2668      1    524    459       C  \\nATOM   1043  N   GLY A 144       7.707   8.154   8.147  1.00 27.37           N  \\nANISOU 1043  N   GLY A 144     3203   4105   3092    251    328    481       N  \\nATOM   1044  CA  GLY A 144       8.317   7.843   9.424  1.00 23.07           C  \\nANISOU 1044  CA  GLY A 144     2559   3585   2620    389    336    465       C  \\nATOM   1045  C   GLY A 144       9.814   7.664   9.228  1.00 28.41           C  \\nANISOU 1045  C   GLY A 144     3109   4433   3253    487    433    398       C  \\nATOM   1046  O   GLY A 144      10.251   6.869   8.406  1.00 32.47           O  \\nANISOU 1046  O   GLY A 144     3670   4970   3696    562    518    322       O  \\nATOM   1047  N   VAL A 145      10.606   8.416   9.979  1.00 34.93           N  \\nANISOU 1047  N   VAL A 145     3762   5390   4120    480    426    414       N  \\nATOM   1048  CA  VAL A 145      12.057   8.373   9.830  1.00 34.71           C  \\nANISOU 1048  CA  VAL A 145     3549   5575   4063    553    512    349       C  \\nATOM   1049  C   VAL A 145      12.697   7.586  10.979  1.00 37.91           C  \\nANISOU 1049  C   VAL A 145     3859   6019   4526    774    469    329       C  \\nATOM   1050  O   VAL A 145      12.458   7.891  12.137  1.00 38.92           O  \\nANISOU 1050  O   VAL A 145     3970   6116   4704    765    373    383       O  \\nATOM   1051  CB  VAL A 145      12.638   9.806   9.806  1.00 28.99           C  \\nANISOU 1051  CB  VAL A 145     2681   5005   3330    356    526    378       C  \\nATOM   1052  CG1 VAL A 145      14.134   9.769   9.592  1.00 32.98           C  \\nANISOU 1052  CG1 VAL A 145     2955   5773   3804    397    627    301       C  \\nATOM   1053  CG2 VAL A 145      11.956  10.642   8.720  1.00 30.84           C  \\nANISOU 1053  CG2 VAL A 145     3050   5167   3501    143    540    431       C  \\nATOM   1054  N   HIS A 146      13.506   6.579  10.655  1.00 35.53           N  \\nANISOU 1054  N   HIS A 146     3506   5781   4211    980    539    249       N  \\nATOM   1055  CA  HIS A 146      14.119   5.727  11.672  1.00 34.96           C  \\nANISOU 1055  CA  HIS A 146     3364   5726   4194   1231    475    246       C  \\nATOM   1056  C   HIS A 146      15.634   5.856  11.689  1.00 43.19           C  \\nANISOU 1056  C   HIS A 146     4101   7058   5252   1345    521    181       C  \\nATOM   1057  O   HIS A 146      16.252   6.137  10.666  1.00 42.50           O  \\nANISOU 1057  O   HIS A 146     3893   7133   5124   1286    657    101       O  \\nATOM   1058  CB  HIS A 146      13.779   4.262  11.415  1.00 39.92           C  \\nANISOU 1058  CB  HIS A 146     4202   6146   4820   1442    496    208       C  \\nATOM   1059  CG  HIS A 146      12.314   3.987  11.325  1.00 41.62           C  \\nANISOU 1059  CG  HIS A 146     4697   6100   5016   1317    459    257       C  \\nATOM   1060  ND1 HIS A 146      11.629   3.289  12.295  1.00 36.59           N  \\nANISOU 1060  ND1 HIS A 146     4229   5267   4405   1382    366    323       N  \\nATOM   1061  CD2 HIS A 146      11.404   4.299  10.372  1.00 41.84           C  \\nANISOU 1061  CD2 HIS A 146     4856   6048   4994   1122    496    249       C  \\nATOM   1062  CE1 HIS A 146      10.358   3.192  11.949  1.00 36.57           C  \\nANISOU 1062  CE1 HIS A 146     4422   5091   4383   1221    361    343       C  \\nATOM   1063  NE2 HIS A 146      10.196   3.793  10.785  1.00 39.03           N  \\nANISOU 1063  NE2 HIS A 146     4707   5471   4652   1074    426    300       N  \\nATOM   1064  N   GLY A 147      16.233   5.625  12.853  1.00 46.97           N  \\nANISOU 1064  N   GLY A 147     4449   7619   5780   1501    405    214       N  \\nATOM   1065  CA  GLY A 147      17.679   5.618  12.964  1.00 45.13           C  \\nANISOU 1065  CA  GLY A 147     3887   7683   5577   1644    420    152       C  \\nATOM   1066  C   GLY A 147      18.300   4.401  12.303  1.00 50.00           C  \\nANISOU 1066  C   GLY A 147     4473   8303   6222   1957    517     52       C  \\nATOM   1067  O   GLY A 147      19.464   4.424  11.899  1.00 55.87           O  \\nANISOU 1067  O   GLY A 147     4941   9298   6990   2038    600    -44       O  \\nATOM   1068  N   THR A 148      17.521   3.332  12.186  1.00 51.81           N  \\nANISOU 1068  N   THR A 148     5007   8227   6450   2106    512     61       N  \\nATOM   1069  CA  THR A 148      18.046   2.061  11.699  1.00 56.32           C  \\nANISOU 1069  CA  THR A 148     5611   8731   7057   2441    591    -38       C  \\nATOM   1070  C   THR A 148      17.188   1.516  10.571  1.00 48.08           C  \\nANISOU 1070  C   THR A 148     4875   7445   5949   2377    724   -110       C  \\nATOM   1071  O   THR A 148      15.984   1.732  10.543  1.00 45.10           O  \\nANISOU 1071  O   THR A 148     4745   6867   5523   2154    683    -39       O  \\nATOM   1072  CB  THR A 148      18.082   1.015  12.836  1.00 63.25           C  \\nANISOU 1072  CB  THR A 148     6608   9430   7996   2746    421     46       C  \\nATOM   1073  OG1 THR A 148      18.760   1.569  13.967  1.00 72.30           O  \\nANISOU 1073  OG1 THR A 148     7537  10761   9171   2700    254    126       O  \\nATOM   1074  CG2 THR A 148      18.810  -0.237  12.402  1.00 72.93           C  \\nANISOU 1074  CG2 THR A 148     7883  10544   9284   3036    475    -60       C  \\nATOM   1075  N   GLU A 149      17.806   0.791   9.651  1.00 46.57           N  \\nANISOU 1075  N   GLU A 149     4659   7280   5755   2576    883   -261       N  \\nATOM   1076  CA  GLU A 149      17.057   0.122   8.598  1.00 54.18           C  \\nANISOU 1076  CA  GLU A 149     5942   8004   6640   2533   1004   -351       C  \\nATOM   1077  C   GLU A 149      16.269  -1.051   9.167  1.00 54.73           C  \\nANISOU 1077  C   GLU A 149     6364   7690   6741   2687    895   -297       C  \\nATOM   1078  O   GLU A 149      15.200  -1.391   8.661  1.00 51.51           O  \\nANISOU 1078  O   GLU A 149     6270   7040   6262   2525    914   -305       O  \\nATOM   1079  CB  GLU A 149      17.995  -0.340   7.472  1.00 56.31           C  \\nANISOU 1079  CB  GLU A 149     6098   8414   6884   2707   1229   -555       C  \\nATOM   1080  CG  GLU A 149      17.297  -1.001   6.286  1.00 55.20           C  \\nANISOU 1080  CG  GLU A 149     6297   8049   6627   2637   1368   -676       C  \\nATOM   1081  CD  GLU A 149      16.280  -0.097   5.592  1.00 64.89           C  \\nANISOU 1081  CD  GLU A 149     7667   9272   7716   2209   1370   -618       C  \\nATOM   1082  OE1 GLU A 149      16.368   1.147   5.717  1.00 65.18           O  \\nANISOU 1082  OE1 GLU A 149     7500   9529   7735   1974   1336   -525       O  \\nATOM   1083  OE2 GLU A 149      15.384  -0.642   4.912  1.00 72.52           O  \\nANISOU 1083  OE2 GLU A 149     8961  10006   8588   2109   1395   -664       O  \\nATOM   1084  N   LYS A 150      16.807  -1.666  10.219  1.00 65.52           N  \\nANISOU 1084  N   LYS A 150     7681   9009   8205   2984    772   -235       N  \\nATOM   1085  CA  LYS A 150      16.130  -2.758  10.919  1.00 69.31           C  \\nANISOU 1085  CA  LYS A 150     8511   9116   8706   3120    655   -153       C  \\nATOM   1086  C   LYS A 150      14.831  -2.242  11.534  1.00 55.55           C  \\nANISOU 1086  C   LYS A 150     6948   7247   6912   2786    538     -2       C  \\nATOM   1087  O   LYS A 150      13.779  -2.873  11.420  1.00 50.99           O  \\nANISOU 1087  O   LYS A 150     6709   6371   6292   2679    532     16       O  \\nATOM   1088  CB  LYS A 150      17.049  -3.367  11.989  1.00 75.11           C  \\nANISOU 1088  CB  LYS A 150     9143   9854   9541   3429    506    -83       C  \\nATOM   1089  CG  LYS A 150      16.369  -4.320  12.964  1.00 79.44           C  \\nANISOU 1089  CG  LYS A 150    10050  10041  10090   3503    350     56       C  \\nATOM   1090  CD  LYS A 150      17.358  -4.819  14.016  1.00 88.07           C  \\nANISOU 1090  CD  LYS A 150    11026  11171  11266   3705    178    138       C  \\nATOM   1091  CE  LYS A 150      16.686  -5.666  15.097  1.00 96.09           C  \\nANISOU 1091  CE  LYS A 150    12400  11856  12253   3729     17    303       C  \\nATOM   1092  NZ  LYS A 150      15.922  -4.859  16.096  1.00 95.65           N  \\nANISOU 1092  NZ  LYS A 150    12369  11857  12117   3500   -100    466       N  \\nATOM   1093  N   SER A 151      14.917  -1.074  12.162  1.00 38.39           N  \\nANISOU 1093  N   SER A 151     4533   5310   4741   2613    459     90       N  \\nATOM   1094  CA  SER A 151      13.743  -0.365  12.662  1.00 39.75           C  \\nANISOU 1094  CA  SER A 151     4813   5417   4874   2294    381    203       C  \\nATOM   1095  C   SER A 151      12.705  -0.056  11.565  1.00 45.37           C  \\nANISOU 1095  C   SER A 151     5672   6046   5522   2015    476    151       C  \\nATOM   1096  O   SER A 151      11.516  -0.371  11.707  1.00 45.77           O  \\nANISOU 1096  O   SER A 151     5969   5875   5546   1863    437    201       O  \\nATOM   1097  CB  SER A 151      14.190   0.920  13.355  1.00 43.54           C  \\nANISOU 1097  CB  SER A 151     4994   6181   5368   2170    309    269       C  \\nATOM   1098  OG  SER A 151      13.092   1.629  13.883  1.00 51.19           O  \\nANISOU 1098  OG  SER A 151     6056   7083   6309   1897    247    360       O  \\nATOM   1099  N   ALA A 152      13.150   0.555  10.472  1.00 47.02           N  \\nANISOU 1099  N   ALA A 152     5726   6445   5695   1935    596     54       N  \\nATOM   1100  CA  ALA A 152      12.241   0.902   9.379  1.00 44.60           C  \\nANISOU 1100  CA  ALA A 152     5554   6085   5305   1674    661     17       C  \\nATOM   1101  C   ALA A 152      11.619  -0.338   8.735  1.00 39.85           C  \\nANISOU 1101  C   ALA A 152     5274   5212   4655   1717    709    -63       C  \\nATOM   1102  O   ALA A 152      10.435  -0.353   8.401  1.00 38.54           O  \\nANISOU 1102  O   ALA A 152     5298   4906   4440   1497    675    -39       O  \\nATOM   1103  CB  ALA A 152      12.965   1.749   8.328  1.00 44.21           C  \\nANISOU 1103  CB  ALA A 152     5303   6298   5198   1580    786    -62       C  \\nATOM   1104  N   GLU A 153      12.417  -1.383   8.574  1.00 41.21           N  \\nANISOU 1104  N   GLU A 153     5504   5308   4846   2005    782   -166       N  \\nATOM   1105  CA  GLU A 153      11.933  -2.614   7.966  1.00 45.30           C  \\nANISOU 1105  CA  GLU A 153     6359   5538   5314   2057    839   -265       C  \\nATOM   1106  C   GLU A 153      10.842  -3.243   8.825  1.00 49.96           C  \\nANISOU 1106  C   GLU A 153     7220   5837   5927   1977    711   -153       C  \\nATOM   1107  O   GLU A 153       9.884  -3.823   8.309  1.00 47.03           O  \\nANISOU 1107  O   GLU A 153     7127   5253   5489   1814    722   -194       O  \\nATOM   1108  CB  GLU A 153      13.083  -3.596   7.783  1.00 46.07           C  \\nANISOU 1108  CB  GLU A 153     6459   5590   5455   2436    941   -397       C  \\nATOM   1109  CG  GLU A 153      12.798  -4.721   6.801  1.00 65.35           C  \\nANISOU 1109  CG  GLU A 153     9236   7772   7822   2483   1058   -564       C  \\nATOM   1110  CD  GLU A 153      14.057  -5.504   6.437  1.00 94.85           C  \\nANISOU 1110  CD  GLU A 153    12919  11513  11606   2880   1201   -733       C  \\nATOM   1111  OE1 GLU A 153      14.268  -6.606   6.996  1.00105.55           O  \\nANISOU 1111  OE1 GLU A 153    14467  12596  13042   3173   1160   -737       O  \\nATOM   1112  OE2 GLU A 153      14.840  -5.015   5.590  1.00101.79           O  \\nANISOU 1112  OE2 GLU A 153    13566  12669  12442   2902   1360   -863       O  \\nATOM   1113  N   ALA A 154      10.995  -3.123  10.139  1.00 44.11           N  \\nANISOU 1113  N   ALA A 154     6395   5102   5261   2069    593    -15       N  \\nATOM   1114  CA  ALA A 154      10.018  -3.661  11.070  1.00 47.25           C  \\nANISOU 1114  CA  ALA A 154     7035   5253   5663   1973    487    104       C  \\nATOM   1115  C   ALA A 154       8.659  -2.961  10.917  1.00 46.03           C  \\nANISOU 1115  C   ALA A 154     6904   5118   5467   1597    456    154       C  \\nATOM   1116  O   ALA A 154       7.623  -3.622  10.796  1.00 49.32           O  \\nANISOU 1116  O   ALA A 154     7579   5313   5846   1433    446    153       O  \\nATOM   1117  CB  ALA A 154      10.538  -3.559  12.519  1.00 40.65           C  \\nANISOU 1117  CB  ALA A 154     6093   4467   4886   2135    367    245       C  \\nATOM   1118  N   ALA A 155       8.666  -1.631  10.920  1.00 37.70           N  \\nANISOU 1118  N   ALA A 155     5575   4324   4426   1461    438    194       N  \\nATOM   1119  CA  ALA A 155       7.435  -0.857  10.750  1.00 30.97           C  \\nANISOU 1119  CA  ALA A 155     4704   3507   3558   1152    400    240       C  \\nATOM   1120  C   ALA A 155       6.820  -1.118   9.365  1.00 41.34           C  \\nANISOU 1120  C   ALA A 155     6156   4760   4790    995    450    140       C  \\nATOM   1121  O   ALA A 155       5.605  -1.212   9.206  1.00 38.95           O  \\nANISOU 1121  O   ALA A 155     5966   4367   4466    772    405    160       O  \\nATOM   1122  CB  ALA A 155       7.724   0.603  10.948  1.00 34.83           C  \\nANISOU 1122  CB  ALA A 155     4900   4253   4079   1081    376    291       C  \\nATOM   1123  N   ARG A 156       7.679  -1.267   8.367  1.00 41.89           N  \\nANISOU 1123  N   ARG A 156     6211   4898   4806   1108    547     25       N  \\nATOM   1124  CA  ARG A 156       7.247  -1.604   7.017  1.00 46.80           C  \\nANISOU 1124  CA  ARG A 156     6998   5469   5316    971    603    -88       C  \\nATOM   1125  C   ARG A 156       6.567  -2.976   6.964  1.00 55.27           C  \\nANISOU 1125  C   ARG A 156     8409   6238   6354    944    601   -147       C  \\nATOM   1126  O   ARG A 156       5.560  -3.162   6.272  1.00 48.92           O  \\nANISOU 1126  O   ARG A 156     7754   5362   5470    703    570   -183       O  \\nATOM   1127  CB  ARG A 156       8.464  -1.609   6.108  1.00 46.41           C  \\nANISOU 1127  CB  ARG A 156     6874   5553   5205   1132    743   -218       C  \\nATOM   1128  CG  ARG A 156       8.240  -1.063   4.744  1.00 47.89           C  \\nANISOU 1128  CG  ARG A 156     7072   5870   5255    928    792   -285       C  \\nATOM   1129  CD  ARG A 156       9.556  -1.073   4.001  1.00 54.42           C  \\nANISOU 1129  CD  ARG A 156     7800   6855   6020   1094    964   -418       C  \\nATOM   1130  NE  ARG A 156      10.409   0.030   4.422  1.00 56.53           N  \\nANISOU 1130  NE  ARG A 156     7742   7384   6352   1139    976   -343       N  \\nATOM   1131  CZ  ARG A 156      11.708  -0.072   4.687  1.00 50.91           C  \\nANISOU 1131  CZ  ARG A 156     6836   6808   5699   1384   1076   -404       C  \\nATOM   1132  NH1 ARG A 156      12.337  -1.242   4.587  1.00 47.90           N  \\nANISOU 1132  NH1 ARG A 156     6550   6313   5335   1657   1177   -541       N  \\nATOM   1133  NH2 ARG A 156      12.376   1.012   5.054  1.00 48.68           N  \\nANISOU 1133  NH2 ARG A 156     6258   6775   5463   1357   1071   -333       N  \\nATOM   1134  N   ARG A 157       7.135  -3.938   7.687  1.00 55.00           N  \\nANISOU 1134  N   ARG A 157     8503   6020   6375   1187    622   -154       N  \\nATOM   1135  CA  ARG A 157       6.536  -5.261   7.802  1.00 54.80           C  \\nANISOU 1135  CA  ARG A 157     8837   5659   6325   1161    617   -191       C  \\nATOM   1136  C   ARG A 157       5.185  -5.172   8.503  1.00 47.02           C  \\nANISOU 1136  C   ARG A 157     7912   4601   5355    876    511    -69       C  \\nATOM   1137  O   ARG A 157       4.215  -5.789   8.062  1.00 44.65           O  \\nANISOU 1137  O   ARG A 157     7838   4137   4991    649    499   -117       O  \\nATOM   1138  CB  ARG A 157       7.467  -6.236   8.544  1.00 65.88           C  \\nANISOU 1138  CB  ARG A 157    10371   6866   7795   1511    639   -189       C  \\nATOM   1139  CG  ARG A 157       8.639  -6.755   7.704  1.00 81.70           C  \\nANISOU 1139  CG  ARG A 157    12401   8858   9785   1806    772   -363       C  \\nATOM   1140  CD  ARG A 157       9.468  -7.813   8.439  1.00 90.35           C  \\nANISOU 1140  CD  ARG A 157    13648   9719  10964   2187    769   -356       C  \\nATOM   1141  NE  ARG A 157      10.215  -7.268   9.575  1.00 93.70           N  \\nANISOU 1141  NE  ARG A 157    13792  10322  11487   2390    679   -209       N  \\nATOM   1142  CZ  ARG A 157      11.528  -7.403   9.745  1.00 94.30           C  \\nANISOU 1142  CZ  ARG A 157    13691  10497  11641   2769    705   -246       C  \\nATOM   1143  NH1 ARG A 157      12.250  -8.070   8.853  1.00 95.73           N  \\nANISOU 1143  NH1 ARG A 157    13940  10608  11823   3012    844   -435       N  \\nATOM   1144  NH2 ARG A 157      12.122  -6.874  10.808  1.00 89.83           N  \\nANISOU 1144  NH2 ARG A 157    12870  10114  11148   2906    594   -106       N  \\nATOM   1145  N   ALA A 158       5.122  -4.396   9.586  1.00 40.39           N  \\nANISOU 1145  N   ALA A 158     6857   3896   4592    875    443     75       N  \\nATOM   1146  CA  ALA A 158       3.872  -4.213  10.331  1.00 46.54           C  \\nANISOU 1146  CA  ALA A 158     7645   4647   5391    617    371    179       C  \\nATOM   1147  C   ALA A 158       2.793  -3.616   9.438  1.00 54.62           C  \\nANISOU 1147  C   ALA A 158     8584   5791   6379    322    338    145       C  \\nATOM   1148  O   ALA A 158       1.637  -4.043   9.469  1.00 53.81           O  \\nANISOU 1148  O   ALA A 158     8602   5588   6256     75    304    150       O  \\nATOM   1149  CB  ALA A 158       4.088  -3.331  11.562  1.00 36.72           C  \\nANISOU 1149  CB  ALA A 158     6166   3564   4220    677    324    311       C  \\nATOM   1150  N   LEU A 159       3.191  -2.630   8.639  1.00 56.05           N  \\nANISOU 1150  N   LEU A 159     8557   6194   6546    341    342    115       N  \\nATOM   1151  CA  LEU A 159       2.280  -1.928   7.751  1.00 54.30           C  \\nANISOU 1151  CA  LEU A 159     8240   6106   6286     99    282    105       C  \\nATOM   1152  C   LEU A 159       1.718  -2.863   6.676  1.00 55.59           C  \\nANISOU 1152  C   LEU A 159     8659   6131   6331    -64    284    -12       C  \\nATOM   1153  O   LEU A 159       0.553  -2.751   6.300  1.00 55.78           O  \\nANISOU 1153  O   LEU A 159     8676   6188   6330   -322    198     -4       O  \\nATOM   1154  CB  LEU A 159       2.995  -0.729   7.124  1.00 61.49           C  \\nANISOU 1154  CB  LEU A 159     8928   7251   7184    168    291    112       C  \\nATOM   1155  CG  LEU A 159       2.165   0.396   6.512  1.00 62.95           C  \\nANISOU 1155  CG  LEU A 159     8954   7603   7361    -29    194    167       C  \\nATOM   1156  CD1 LEU A 159       1.205   0.969   7.541  1.00 58.30           C  \\nANISOU 1156  CD1 LEU A 159     8211   7045   6897   -119    115    272       C  \\nATOM   1157  CD2 LEU A 159       3.091   1.479   5.969  1.00 63.11           C  \\nANISOU 1157  CD2 LEU A 159     8815   7810   7354     51    225    182       C  \\nATOM   1158  N   LEU A 160       2.534  -3.795   6.192  1.00 54.60           N  \\nANISOU 1158  N   LEU A 160     8756   5855   6135     88    380   -130       N  \\nATOM   1159  CA  LEU A 160       2.036  -4.787   5.242  1.00 64.25           C  \\nANISOU 1159  CA  LEU A 160    10273   6906   7231    -72    394   -265       C  \\nATOM   1160  C   LEU A 160       1.099  -5.807   5.906  1.00 69.73           C  \\nANISOU 1160  C   LEU A 160    11197   7350   7946   -240    359   -248       C  \\nATOM   1161  O   LEU A 160       0.116  -6.235   5.298  1.00 65.95           O  \\nANISOU 1161  O   LEU A 160    10860   6812   7385   -524    306   -309       O  \\nATOM   1162  CB  LEU A 160       3.186  -5.503   4.521  1.00 71.08           C  \\nANISOU 1162  CB  LEU A 160    11326   7664   8016    154    532   -425       C  \\nATOM   1163  CG  LEU A 160       2.776  -6.269   3.257  1.00 73.33           C  \\nANISOU 1163  CG  LEU A 160    11902   7829   8129    -28    560   -599       C  \\nATOM   1164  CD1 LEU A 160       2.263  -5.309   2.182  1.00 65.24           C  \\nANISOU 1164  CD1 LEU A 160    10732   7069   6985   -263    484   -598       C  \\nATOM   1165  CD2 LEU A 160       3.916  -7.134   2.719  1.00 77.79           C  \\nANISOU 1165  CD2 LEU A 160    12684   8238   8633    235    729   -781       C  \\nATOM   1166  N   GLU A 161       1.408  -6.189   7.149  1.00 76.78           N  \\nANISOU 1166  N   GLU A 161    12133   8107   8935    -86    382   -160       N  \\nATOM   1167  CA  GLU A 161       0.554  -7.087   7.937  1.00 78.28           C  \\nANISOU 1167  CA  GLU A 161    12544   8062   9137   -263    360   -113       C  \\nATOM   1168  C   GLU A 161      -0.838  -6.491   8.149  1.00 75.82           C  \\nANISOU 1168  C   GLU A 161    12041   7916   8851   -601    272    -40       C  \\nATOM   1169  O   GLU A 161      -1.818  -7.220   8.334  1.00 68.12           O  \\nANISOU 1169  O   GLU A 161    11236   6799   7847   -871    254    -47       O  \\nATOM   1170  CB  GLU A 161       1.175  -7.374   9.309  1.00 77.68           C  \\nANISOU 1170  CB  GLU A 161    12511   7862   9141    -34    383      6       C  \\nATOM   1171  CG  GLU A 161       2.405  -8.271   9.307  1.00 80.37           C  \\nANISOU 1171  CG  GLU A 161    13091   7971   9477    311    451    -52       C  \\nATOM   1172  CD  GLU A 161       2.991  -8.459  10.703  1.00 76.35           C  \\nANISOU 1172  CD  GLU A 161    12600   7374   9037    539    429     95       C  \\nATOM   1173  OE1 GLU A 161       3.724  -9.447  10.914  1.00 84.84           O  \\nANISOU 1173  OE1 GLU A 161    13943   8179  10115    790    455     79       O  \\nATOM   1174  OE2 GLU A 161       2.723  -7.620  11.591  1.00 67.98           O  \\nANISOU 1174  OE2 GLU A 161    11297   6509   8023    475    380    226       O  \\nATOM   1175  N   HIS A 162      -0.905  -5.161   8.127  1.00 77.15           N  \\nANISOU 1175  N   HIS A 162    11851   8382   9079   -581    221     25       N  \\nATOM   1176  CA  HIS A 162      -2.143  -4.416   8.343  1.00 77.01           C  \\nANISOU 1176  CA  HIS A 162    11588   8554   9118   -829    137     92       C  \\nATOM   1177  C   HIS A 162      -3.161  -4.623   7.214  1.00 82.37           C  \\nANISOU 1177  C   HIS A 162    12304   9278   9714  -1126     52      9       C  \\nATOM   1178  O   HIS A 162      -4.353  -4.369   7.397  1.00 79.29           O  \\nANISOU 1178  O   HIS A 162    11753   9004   9369  -1368    -22     45       O  \\nATOM   1179  CB  HIS A 162      -1.829  -2.925   8.509  1.00 69.73           C  \\nANISOU 1179  CB  HIS A 162    10315   7898   8282   -689    102    172       C  \\nATOM   1180  CG  HIS A 162      -2.908  -2.147   9.196  1.00 69.05           C  \\nANISOU 1180  CG  HIS A 162     9966   7970   8298   -839     49    254       C  \\nATOM   1181  ND1 HIS A 162      -3.351  -2.451  10.466  1.00 67.09           N  \\nANISOU 1181  ND1 HIS A 162     9725   7662   8105   -901    100    313       N  \\nATOM   1182  CD2 HIS A 162      -3.627  -1.071   8.794  1.00 66.88           C  \\nANISOU 1182  CD2 HIS A 162     9419   7912   8081   -925    -44    283       C  \\nATOM   1183  CE1 HIS A 162      -4.299  -1.598  10.815  1.00 64.84           C  \\nANISOU 1183  CE1 HIS A 162     9161   7562   7911  -1018     61    353       C  \\nATOM   1184  NE2 HIS A 162      -4.484  -0.750   9.818  1.00 65.93           N  \\nANISOU 1184  NE2 HIS A 162     9123   7862   8066  -1017    -35    339       N  \\nATOM   1185  N   HIS A 163      -2.687  -5.090   6.057  1.00 90.63           N  \\nANISOU 1185  N   HIS A 163    13553  10249  10634  -1107     62   -111       N  \\nATOM   1186  CA  HIS A 163      -3.551  -5.361   4.904  1.00 96.61           C  \\nANISOU 1186  CA  HIS A 163    14389  11045  11272  -1397    -31   -205       C  \\nATOM   1187  C   HIS A 163      -4.133  -6.775   4.915  1.00104.23           C  \\nANISOU 1187  C   HIS A 163    15696  11743  12164  -1635     -6   -299       C  \\nATOM   1188  O   HIS A 163      -4.663  -7.249   3.906  1.00104.93           O  \\nANISOU 1188  O   HIS A 163    15938  11810  12121  -1875    -68   -413       O  \\nATOM   1189  CB  HIS A 163      -2.796  -5.129   3.596  1.00 97.60           C  \\nANISOU 1189  CB  HIS A 163    14588  11238  11259  -1301    -24   -301       C  \\nATOM   1190  CG  HIS A 163      -2.473  -3.693   3.333  1.00 96.52           C  \\nANISOU 1190  CG  HIS A 163    14138  11376  11159  -1176    -78   -206       C  \\nATOM   1191  ND1 HIS A 163      -1.778  -2.911   4.230  1.00 93.68           N  \\nANISOU 1191  ND1 HIS A 163    13571  11089  10935   -931    -24   -100       N  \\nATOM   1192  CD2 HIS A 163      -2.749  -2.897   2.273  1.00102.72           C  \\nANISOU 1192  CD2 HIS A 163    14811  12366  11852  -1276   -191   -196       C  \\nATOM   1193  CE1 HIS A 163      -1.640  -1.694   3.735  1.00 97.08           C  \\nANISOU 1193  CE1 HIS A 163    13783  11740  11363   -894    -88    -35       C  \\nATOM   1194  NE2 HIS A 163      -2.219  -1.659   2.548  1.00103.91           N  \\nANISOU 1194  NE2 HIS A 163    14706  12684  12091  -1092   -192    -81       N  \\nATOM   1195  N   HIS A 164      -4.030  -7.443   6.058  1.00108.04           N  \\nANISOU 1195  N   HIS A 164    16320  12015  12717  -1586     78   -247       N  \\nATOM   1196  CA  HIS A 164      -4.637  -8.756   6.236  1.00113.16           C  \\nANISOU 1196  CA  HIS A 164    17310  12380  13305  -1841    106   -308       C  \\nATOM   1197  C   HIS A 164      -5.488  -8.764   7.516  1.00113.93           C  \\nANISOU 1197  C   HIS A 164    17290  12496  13501  -2018    113   -182       C  \\nATOM   1198  O   HIS A 164      -5.073  -9.319   8.534  1.00110.40           O  \\nANISOU 1198  O   HIS A 164    17030  11833  13085  -1904    200   -111       O  \\nATOM   1199  CB  HIS A 164      -3.545  -9.834   6.305  1.00115.00           C  \\nANISOU 1199  CB  HIS A 164    17959  12246  13489  -1598    226   -381       C  \\nATOM   1200  CG  HIS A 164      -3.953 -11.162   5.738  1.00116.52           C  \\nANISOU 1200  CG  HIS A 164    18592  12122  13557  -1846    247   -526       C  \\nATOM   1201  ND1 HIS A 164      -4.372 -12.213   6.527  1.00117.68           N  \\nANISOU 1201  ND1 HIS A 164    19050  11957  13705  -2013    290   -495       N  \\nATOM   1202  CD2 HIS A 164      -3.986 -11.615   4.462  1.00113.60           C  \\nANISOU 1202  CD2 HIS A 164    18434  11686  13041  -1969    237   -710       C  \\nATOM   1203  CE1 HIS A 164      -4.657 -13.251   5.760  1.00115.48           C  \\nANISOU 1203  CE1 HIS A 164    19154  11421  13301  -2231    302   -655       C  \\nATOM   1204  NE2 HIS A 164      -4.431 -12.914   4.503  1.00113.27           N  \\nANISOU 1204  NE2 HIS A 164    18823  11287  12927  -2208    271   -797       N  \\nATOM   1205  N   HIS A 165      -6.668  -8.143   7.474  1.00117.25           N  \\nANISOU 1205  N   HIS A 165    17397  13186  13966  -2289     22   -151       N  \\nATOM   1206  CA  HIS A 165      -7.167  -7.452   6.285  1.00122.61           C  \\nANISOU 1206  CA  HIS A 165    17853  14127  14605  -2404   -117   -209       C  \\nATOM   1207  C   HIS A 165      -7.290  -5.949   6.530  1.00118.61           C  \\nANISOU 1207  C   HIS A 165    16885  13954  14229  -2250   -183    -97       C  \\nATOM   1208  O   HIS A 165      -7.063  -5.470   7.641  1.00116.85           O  \\nANISOU 1208  O   HIS A 165    16511  13765  14123  -2088   -111      4       O  \\nATOM   1209  CB  HIS A 165      -8.517  -8.027   5.843  1.00128.80           C  \\nANISOU 1209  CB  HIS A 165    18656  14954  15329  -2861   -209   -281       C  \\nATOM   1210  CG  HIS A 165      -8.988  -7.516   4.515  1.00131.62           C  \\nANISOU 1210  CG  HIS A 165    18859  15547  15604  -2992   -381   -346       C  \\nATOM   1211  ND1 HIS A 165     -10.228  -7.825   3.996  1.00134.65           N  \\nANISOU 1211  ND1 HIS A 165    19171  16056  15934  -3398   -514   -407       N  \\nATOM   1212  CD2 HIS A 165      -8.384  -6.723   3.599  1.00130.61           C  \\nANISOU 1212  CD2 HIS A 165    18643  15562  15422  -2788   -452   -352       C  \\nATOM   1213  CE1 HIS A 165     -10.367  -7.242   2.818  1.00134.23           C  \\nANISOU 1213  CE1 HIS A 165    18998  16211  15794  -3420   -679   -441       C  \\nATOM   1214  NE2 HIS A 165      -9.263  -6.567   2.554  1.00132.23           N  \\nANISOU 1214  NE2 HIS A 165    18748  15963  15532  -3059   -637   -404       N  \\nTER    1215      HIS A 165                                                      \\nHETATM 1216 CL    CL A 201      -5.185   5.185  22.315  0.50 37.55          CL  \\nANISOU 1216 CL    CL A 201     4758   4758   4751   -719    649    649      CL  \\nHETATM 1217  O4  PE4 A 202      18.806  11.434   5.330  1.00 67.98           O  \\nANISOU 1217  O4  PE4 A 202     6690  11338   7800   -207   1408     -2       O  \\nHETATM 1218  C7  PE4 A 202      17.805  12.453   5.408  1.00 58.65           C  \\nANISOU 1218  C7  PE4 A 202     5755   9927   6602   -437   1277    148       C  \\nHETATM 1219  C8  PE4 A 202      16.488  11.956   4.727  1.00 53.12           C  \\nANISOU 1219  C8  PE4 A 202     5390   8948   5843   -383   1221    198       C  \\nHETATM 1220  O5  PE4 A 202      15.754  11.049   5.572  1.00 56.63           O  \\nANISOU 1220  O5  PE4 A 202     5902   9204   6412   -118   1086    196       O  \\nHETATM 1221  C9  PE4 A 202      14.331  10.936   5.254  1.00 50.34           C  \\nANISOU 1221  C9  PE4 A 202     5396   8134   5596   -139    975    277       C  \\nHETATM 1222  C10 PE4 A 202      14.025  10.000   3.970  1.00 34.81           C  \\nANISOU 1222  C10 PE4 A 202     3608   6130   3488    -97   1072    204       C  \\nHETATM 1223  O6  PE4 A 202      15.222   9.318   3.472  1.00 48.35           O  \\nANISOU 1223  O6  PE4 A 202     5166   8066   5139     10   1272     51       O  \\nHETATM 1224  C11 PE4 A 202      15.022   8.402   2.387  1.00 46.89           C  \\nANISOU 1224  C11 PE4 A 202     5152   7843   4820     64   1380    -51       C  \\nHETATM 1225  C12 PE4 A 202      14.173   9.037   1.243  1.00 35.99           C  \\nANISOU 1225  C12 PE4 A 202     4036   6381   3256   -213   1349     40       C  \\nHETATM 1226  O7  PE4 A 202      14.951   9.345   0.122  1.00 58.20           O  \\nANISOU 1226  O7  PE4 A 202     6837   9403   5874   -383   1547    -20       O  \\nHETATM 1227  C13 PE4 A 202      14.260  10.034  -0.848  1.00 57.26           C  \\nANISOU 1227  C13 PE4 A 202     6962   9221   5575   -648   1496     92       C  \\nHETATM 1228  C14 PE4 A 202      13.641  11.143  -0.481  1.00 52.40           C  \\nANISOU 1228  C14 PE4 A 202     6404   8490   5017   -801   1319    277       C  \\nHETATM 1229  O8  PE4 A 202      12.280  11.116  -0.572  1.00 54.59           O  \\nANISOU 1229  O8  PE4 A 202     6901   8536   5304   -807   1113    381       O  \\nHETATM 1230  O4  PE5 A 203      -5.402  -5.046  14.054  1.00 73.30           O  \\nANISOU 1230  O4  PE5 A 203    10985   8033   8835  -1493    334    461       O  \\nHETATM 1231  C7  PE5 A 203      -4.562  -6.107  14.235  1.00 76.38           C  \\nANISOU 1231  C7  PE5 A 203    11756   8125   9139  -1372    360    484       C  \\nHETATM 1232  C8  PE5 A 203      -3.254  -5.806  14.116  1.00 83.62           C  \\nANISOU 1232  C8  PE5 A 203    12681   9024  10066  -1018    337    495       C  \\nHETATM 1233  O5  PE5 A 203      -2.359  -6.849  13.981  1.00 87.99           O  \\nANISOU 1233  O5  PE5 A 203    13585   9291  10555   -843    346    487       O  \\nHETATM 1234  C9  PE5 A 203      -1.722  -6.885  12.698  1.00 87.01           C  \\nANISOU 1234  C9  PE5 A 203    13485   9151  10424   -692    319    374       C  \\nHETATM 1235  C10 PE5 A 203      -0.429  -6.037  12.837  1.00 61.06           C  \\nANISOU 1235  C10 PE5 A 203     9994   6018   7189   -327    308    403       C  \\nHETATM 1236  O6  PE5 A 203      -0.688  -4.698  12.376  1.00 64.33           O  \\nANISOU 1236  O6  PE5 A 203    10040   6742   7661   -367    274    386       O  \\nHETATM 1237  C11 PE5 A 203       0.440  -3.842  12.531  1.00 53.33           C  \\nANISOU 1237  C11 PE5 A 203     8450   5502   6311    -89    269    413       C  \\nHETATM 1238  C12 PE5 A 203      -0.019  -2.362  12.502  1.00 52.69           C  \\nANISOU 1238  C12 PE5 A 203     8010   5707   6301   -168    233    438       C  \\nHETATM 1239  O7  PE5 A 203       0.330  -1.773  13.784  1.00 55.44           O  \\nANISOU 1239  O7  PE5 A 203     8234   6139   6692    -67    243    528       O  \\nHETATM 1240  C13 PE5 A 203       1.532  -1.068  13.861  1.00 42.18           C  \\nANISOU 1240  C13 PE5 A 203     6413   4580   5035    168    233    539       C  \\nHETATM 1241  C14 PE5 A 203       2.453  -1.455  14.771  1.00 40.88           C  \\nANISOU 1241  C14 PE5 A 203     6329   4354   4848    360    234    594       C  \\nHETATM 1242  O8  PE5 A 203       3.770  -1.142  14.457  1.00 53.68           O  \\nANISOU 1242  O8  PE5 A 203     7848   6068   6480    601    226    569       O  \\nHETATM 1243  C15 PE5 A 203       4.750  -1.915  15.140  1.00 56.63           C  \\nANISOU 1243  C15 PE5 A 203     8351   6337   6828    834    207    612       C  \\nHETATM 1244  C16 PE5 A 203       6.067  -1.959  14.309  1.00 53.31           C  \\nANISOU 1244  C16 PE5 A 203     7847   5986   6423   1088    226    533       C  \\nHETATM 1245  O52 PE5 A 203       7.151  -1.591  15.111  1.00 56.45           O  \\nANISOU 1245  O52 PE5 A 203     8089   6519   6839   1290    177    586       O  \\nHETATM 1246  C3  PE6 A 204      11.394   8.034  18.595  1.00 63.90           C  \\nANISOU 1246  C3  PE6 A 204     7271   9142   7865    813    -58    591       C  \\nHETATM 1247  O2  PE6 A 204      10.905   9.368  18.469  1.00 58.59           O  \\nANISOU 1247  O2  PE6 A 204     6570   8468   7222    601    -19    561       O  \\nHETATM 1248  C4  PE6 A 204      12.135   7.706  17.252  1.00 44.74           C  \\nANISOU 1248  C4  PE6 A 204     4738   6790   5470    906      7    547       C  \\nHETATM 1249  O5  PE6 A 204      11.154   7.078  16.402  1.00 43.18           O  \\nANISOU 1249  O5  PE6 A 204     4718   6398   5291    913     73    548       O  \\nHETATM 1250  C6  PE6 A 204      11.582   6.993  15.021  1.00 47.62           C  \\nANISOU 1250  C6  PE6 A 204     5224   7012   5859    933    160    493       C  \\nHETATM 1251  C7  PE6 A 204      10.584   6.003  14.316  1.00 41.39           C  \\nANISOU 1251  C7  PE6 A 204     4657   6004   5064    966    200    490       C  \\nHETATM 1252  O8  PE6 A 204       9.560   6.850  13.787  1.00 36.03           O  \\nANISOU 1252  O8  PE6 A 204     4026   5264   4398    761    216    504       O  \\nHETATM 1253  C9  PE6 A 204       8.495   6.171  13.195  1.00 22.76           C  \\nANISOU 1253  C9  PE6 A 204     2523   3414   2712    729    230    504       C  \\nHETATM 1254  C10 PE6 A 204       7.423   7.247  13.009  1.00 24.90           C  \\nANISOU 1254  C10 PE6 A 204     2790   3650   3019    537    206    537       C  \\nHETATM 1255  O11 PE6 A 204       6.477   6.642  12.151  1.00 30.47           O  \\nANISOU 1255  O11 PE6 A 204     3632   4239   3709    486    209    529       O  \\nHETATM 1256  O17 PE6 A 204       3.166   2.354  11.021  1.00 40.16           O  \\nANISOU 1256  O17 PE6 A 204     5547   4884   4827    304    194    445       O  \\nHETATM 1257  C16 PE6 A 204       3.039   3.356  11.961  1.00 31.95           C  \\nANISOU 1257  C16 PE6 A 204     4339   3938   3862    292    172    507       C  \\nHETATM 1258  C15 PE6 A 204       2.980   4.655  11.165  1.00 38.40           C  \\nANISOU 1258  C15 PE6 A 204     5002   4887   4701    227    138    518       C  \\nHETATM 1259  O14 PE6 A 204       3.854   5.543  11.837  1.00 36.07           O  \\nANISOU 1259  O14 PE6 A 204     4569   4694   4441    309    153    543       O  \\nHETATM 1260  C13 PE6 A 204       4.441   6.497  10.965  1.00 29.51           C  \\nANISOU 1260  C13 PE6 A 204     3649   3973   3590    290    153    544       C  \\nHETATM 1261  C12 PE6 A 204       5.337   7.432  11.877  1.00 24.53           C  \\nANISOU 1261  C12 PE6 A 204     2874   3445   2999    342    165    564       C  \\nHETATM 1262  O   HOH A 301       6.272  12.148   3.204  1.00 26.49           O  \\nANISOU 1262  O   HOH A 301     3461   4034   2570   -482    112    835       O  \\nHETATM 1263  O   HOH A 302      11.948  16.088  20.987  1.00 39.62           O  \\nANISOU 1263  O   HOH A 302     4008   6274   4774   -353    -63    300       O  \\nHETATM 1264  O   HOH A 303      12.593  19.007  24.028  1.00 36.11           O  \\nANISOU 1264  O   HOH A 303     3735   5861   4125   -904   -143     14       O  \\nHETATM 1265  O   HOH A 304      -4.755  -0.797  15.247  1.00 34.68           O  \\nANISOU 1265  O   HOH A 304     5109   3840   4228   -986    312    545       O  \\nHETATM 1266  O   HOH A 305      -2.861  17.616   2.601  1.00 48.13           O  \\nANISOU 1266  O   HOH A 305     6133   5993   6160   -145  -1386   1680       O  \\nHETATM 1267  O   HOH A 306      -1.321  22.699  31.175  1.00 38.49           O  \\nANISOU 1267  O   HOH A 306     5033   4677   4915   -282   1623  -1407       O  \\nHETATM 1268  O   HOH A 307      -6.019  12.052  12.830  1.00 26.81           O  \\nANISOU 1268  O   HOH A 307     2244   3653   4290     43   -241    627       O  \\nHETATM 1269  O   HOH A 308      16.673   5.257   3.282  1.00 45.55           O  \\nANISOU 1269  O   HOH A 308     4715   7727   4863    902   1576   -417       O  \\nHETATM 1270  O   HOH A 309      -8.485  11.574   0.972  1.00 54.17           O  \\nANISOU 1270  O   HOH A 309     6229   7545   6807   -594  -1975   1329       O  \\nHETATM 1271  O   HOH A 310      -1.452  20.828  29.409  1.00 38.81           O  \\nANISOU 1271  O   HOH A 310     4787   4869   5089   -173   1394   -999       O  \\nHETATM 1272  O   HOH A 311      14.988   2.770   3.865  1.00 40.92           O  \\nANISOU 1272  O   HOH A 311     4664   6509   4376   1291   1396   -461       O  \\nHETATM 1273  O   HOH A 312       6.549  11.799  30.005  1.00 39.13           O  \\nANISOU 1273  O   HOH A 312     5096   5957   3816   -296     87    325       O  \\nHETATM 1274  O   HOH A 313       9.541  21.274  22.842  1.00 36.54           O  \\nANISOU 1274  O   HOH A 313     4233   5123   4529   -897    127    -62       O  \\nHETATM 1275  O   HOH A 314      16.177  -3.215   4.294  1.00 68.40           O  \\nANISOU 1275  O   HOH A 314     8747   9138   8103   2715   1611   -989       O  \\nHETATM 1276  O   HOH A 315       0.989  -2.340  18.672  1.00 31.82           O  \\nANISOU 1276  O   HOH A 315     5501   3029   3561     59    277    857       O  \\nHETATM 1277  O   HOH A 316      10.721  14.295  15.542  1.00 39.06           O  \\nANISOU 1277  O   HOH A 316     3994   5969   4877    -81    145    505       O  \\nHETATM 1278  O   HOH A 317      -4.942  13.859  27.177  1.00 50.68           O  \\nANISOU 1278  O   HOH A 317     5704   6905   6647   -259   1384   -326       O  \\nHETATM 1279  O   HOH A 318     -10.705   9.125  26.802  1.00 55.25           O  \\nANISOU 1279  O   HOH A 318     5659   7997   7336  -1014   1894   -279       O  \\nHETATM 1280  O   HOH A 319      12.403  13.051  20.420  1.00 40.45           O  \\nANISOU 1280  O   HOH A 319     4003   6515   4850    103   -118    432       O  \\nHETATM 1281  O   HOH A 320       3.146   7.405  -1.008  1.00 55.27           O  \\nANISOU 1281  O   HOH A 320     7843   7600   5558   -747   -109    546       O  \\nHETATM 1282  O   HOH A 321       0.473   3.878   2.272  1.00 43.74           O  \\nANISOU 1282  O   HOH A 321     6315   5697   4607   -583   -199    334       O  \\nHETATM 1283  O   HOH A 322      22.534  13.813   8.897  1.00 56.95           O  \\nANISOU 1283  O   HOH A 322     4372  10553   6714   -521   1102    -63       O  \\nHETATM 1284  O   HOH A 323       2.044   7.517  -3.477  1.00 35.27           O  \\nANISOU 1284  O   HOH A 323     5649   5163   2589  -1102   -355    617       O  \\nHETATM 1285  O   HOH A 324       1.911  -3.531  21.372  1.00 32.96           O  \\nANISOU 1285  O   HOH A 324     6090   2964   3470    202    211   1119       O  \\nHETATM 1286  O   HOH A 325      10.919  17.501  26.317  1.00 41.26           O  \\nANISOU 1286  O   HOH A 325     4640   6440   4596   -677   -133      3       O  \\nHETATM 1287  O   HOH A 326     -11.938   8.081  24.063  1.00 47.82           O  \\nANISOU 1287  O   HOH A 326     4368   7089   6712  -1072   1574   -147       O  \\nHETATM 1288  O   HOH A 327     -12.231  10.217  14.349  1.00 56.72           O  \\nANISOU 1288  O   HOH A 327     4931   8068   8554   -333    -90    332       O  \\nHETATM 1289  O   HOH A 328      15.336  26.153  23.930  1.00 50.05           O  \\nANISOU 1289  O   HOH A 328     5933   7258   5827  -2489    -80   -451       O  \\nHETATM 1290  O   HOH A 329      -0.221  15.105  -2.263  1.00 54.30           O  \\nANISOU 1290  O   HOH A 329     7699   7244   5689   -917  -1242   1689       O  \\nHETATM 1291  O   HOH A 330      -6.494  16.930  26.354  1.00 47.89           O  \\nANISOU 1291  O   HOH A 330     4971   6365   6860    131   1478   -622       O  \\nHETATM 1292  O   HOH A 331      -0.662  11.559  30.723  1.00 44.47           O  \\nANISOU 1292  O   HOH A 331     5826   6301   4771   -612   1110    -22       O  \\nHETATM 1293  O   HOH A 332      -0.267  26.969  14.543  1.00 43.51           O  \\nANISOU 1293  O   HOH A 332     5826   3772   6936    402   -228    657       O  \\nHETATM 1294  O   HOH A 333     -11.541  12.092   1.072  1.00 60.51           O  \\nANISOU 1294  O   HOH A 333     6369   8621   8001   -481  -2435   1401       O  \\nHETATM 1295  O   HOH A 334      17.575  29.772  12.558  1.00 51.60           O  \\nANISOU 1295  O   HOH A 334     6665   6902   6037  -3619    472    550       O  \\nHETATM 1296  O   HOH A 335      15.873  11.022  12.841  1.00 41.15           O  \\nANISOU 1296  O   HOH A 335     3515   7152   4966    376    406    320       O  \\nHETATM 1297  O   HOH A 336       5.886   9.288  30.290  1.00 50.80           O  \\nANISOU 1297  O   HOH A 336     6823   7321   5156   -183     71    575       O  \\nHETATM 1298  O   HOH A 337       9.332  11.420  22.448  1.00 39.65           O  \\nANISOU 1298  O   HOH A 337     4383   6005   4678    233    -77    503       O  \\nCONECT 1217 1218                                                                \\nCONECT 1218 1217 1219                                                           \\nCONECT 1219 1218 1220                                                           \\nCONECT 1220 1219 1221                                                           \\nCONECT 1221 1220 1222                                                           \\nCONECT 1222 1221 1223                                                           \\nCONECT 1223 1222 1224                                                           \\nCONECT 1224 1223 1225                                                           \\nCONECT 1225 1224 1226                                                           \\nCONECT 1226 1225 1227                                                           \\nCONECT 1227 1226 1228                                                           \\nCONECT 1228 1227 1229                                                           \\nCONECT 1229 1228                                                                \\nCONECT 1230 1231                                                                \\nCONECT 1231 1230 1232                                                           \\nCONECT 1232 1231 1233                                                           \\nCONECT 1233 1232 1234                                                           \\nCONECT 1234 1233 1235                                                           \\nCONECT 1235 1234 1236                                                           \\nCONECT 1236 1235 1237                                                           \\nCONECT 1237 1236 1238                                                           \\nCONECT 1238 1237 1239                                                           \\nCONECT 1239 1238 1240                                                           \\nCONECT 1240 1239 1241                                                           \\nCONECT 1241 1240 1242                                                           \\nCONECT 1242 1241 1243                                                           \\nCONECT 1243 1242 1244                                                           \\nCONECT 1244 1243 1245                                                           \\nCONECT 1245 1244                                                                \\nCONECT 1246 1247 1248                                                           \\nCONECT 1247 1246                                                                \\nCONECT 1248 1246 1249                                                           \\nCONECT 1249 1248 1250                                                           \\nCONECT 1250 1249 1251                                                           \\nCONECT 1251 1250 1252                                                           \\nCONECT 1252 1251 1253                                                           \\nCONECT 1253 1252 1254                                                           \\nCONECT 1254 1253 1255                                                           \\nCONECT 1255 1254 1261                                                           \\nCONECT 1256 1257                                                                \\nCONECT 1257 1256 1258                                                           \\nCONECT 1258 1257 1259                                                           \\nCONECT 1259 1258 1260                                                           \\nCONECT 1260 1259 1261                                                           \\nCONECT 1261 1255 1260                                                           \\nMASTER      319    0    4    6    6    0    7    6 1297    1   45   13          \\nEND                                                                             \\n\",\"pdb\");\n\tviewer_17228382996107566.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"#33ff33\"}});\n\tviewer_17228382996107566.zoomTo();\nviewer_17228382996107566.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_17228382996107566\"  style=\"position: relative; width: 640px; height: 480px;\">\n",
              "        <p id=\"3dmolwarning_17228382996107566\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_17228382996107566 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_17228382996107566\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_17228382996107566 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_17228382996107566\"),{backgroundColor:\"white\"});\n",
              "viewer_17228382996107566.zoomTo();\n",
              "\tviewer_17228382996107566.addModel(\"HEADER    DE NOVO PROTEIN                         24-APR-12   4ETJ              \\nTITLE     CRYSTAL STRUCTURE OF E6H VARIANT OF DE NOVO DESIGNED SERINE HYDROLASE \\nTITLE    2 OSH55, NORTHEAST STRUCTURAL GENOMICS CONSORTIUM (NESG) TARGET OR185  \\nCOMPND    MOL_ID: 1;                                                            \\nCOMPND   2 MOLECULE: SERINE HYDROLASE;                                          \\nCOMPND   3 CHAIN: A;                                                            \\nCOMPND   4 ENGINEERED: YES;                                                     \\nCOMPND   5 MUTATION: YES                                                        \\nSOURCE    MOL_ID: 1;                                                            \\nSOURCE   2 ORGANISM_SCIENTIFIC: ARTIFICIAL GENE;                                \\nSOURCE   3 ORGANISM_TAXID: 32630                                                \\nKEYWDS    STRUCTURAL GENOMICS, PSI-BIOLOGY, PROTEIN STRUCTURE INITIATIVE,       \\nKEYWDS   2 NORTHEAST STRUCTURAL GENOMICS CONSORTIUM, NESG, DE NOVO PROTEIN      \\nEXPDTA    X-RAY DIFFRACTION                                                     \\nAUTHOR    A.KUZIN,M.SU,J.SEETHARAMAN,K.KORNHABER,G.KORNHABER,S.RAJAGOPALAN,     \\nAUTHOR   2 D.BAKER,J.K.EVERETT,T.B.ACTON,G.T.MONTELIONE,L.TONG,J.F.HUNT,        \\nAUTHOR   3 NORTHEAST STRUCTURAL GENOMICS CONSORTIUM (NESG)                      \\nREVDAT   4   13-SEP-23 4ETJ    1       REMARK HETSYN                            \\nREVDAT   3   14-MAY-14 4ETJ    1       JRNL                                     \\nREVDAT   2   16-APR-14 4ETJ    1       JRNL                                     \\nREVDAT   1   13-JUN-12 4ETJ    0                                                \\nJRNL        AUTH   S.RAJAGOPALAN,C.WANG,K.YU,A.P.KUZIN,F.RICHTER,S.LEW,         \\nJRNL        AUTH 2 A.E.MIKLOS,M.L.MATTHEWS,J.SEETHARAMAN,M.SU,J.F.HUNT,         \\nJRNL        AUTH 3 B.F.CRAVATT,D.BAKER                                          \\nJRNL        TITL   DESIGN OF ACTIVATED SERINE-CONTAINING CATALYTIC TRIADS WITH  \\nJRNL        TITL 2 ATOMIC-LEVEL ACCURACY.                                       \\nJRNL        REF    NAT.CHEM.BIOL.                V.  10   386 2014              \\nJRNL        REFN                   ISSN 1552-4450                               \\nJRNL        PMID   24705591                                                     \\nJRNL        DOI    10.1038/NCHEMBIO.1498                                        \\nREMARK   2                                                                      \\nREMARK   2 RESOLUTION.    2.20 ANGSTROMS.                                       \\nREMARK   3                                                                      \\nREMARK   3 REFINEMENT.                                                          \\nREMARK   3   PROGRAM     : PHENIX DEV_988                                       \\nREMARK   3   AUTHORS     : PAUL ADAMS,PAVEL AFONINE,VINCENT CHEN,IAN            \\nREMARK   3               : DAVIS,KRESHNA GOPAL,RALF GROSSE-KUNSTLEVE,           \\nREMARK   3               : LI-WEI HUNG,ROBERT IMMORMINO,TOM IOERGER,            \\nREMARK   3               : AIRLIE MCCOY,ERIK MCKEE,NIGEL MORIARTY,              \\nREMARK   3               : REETAL PAI,RANDY READ,JANE RICHARDSON,               \\nREMARK   3               : DAVID RICHARDSON,TOD ROMO,JIM SACCHETTINI,           \\nREMARK   3               : NICHOLAS SAUTER,JACOB SMITH,LAURENT                  \\nREMARK   3               : STORONI,TOM TERWILLIGER,PETER ZWART                  \\nREMARK   3                                                                      \\nREMARK   3    REFINEMENT TARGET : ML                                            \\nREMARK   3                                                                      \\nREMARK   3  DATA USED IN REFINEMENT.                                            \\nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.20                           \\nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 36.96                          \\nREMARK   3   MIN(FOBS/SIGMA_FOBS)              : 1.340                          \\nREMARK   3   COMPLETENESS FOR RANGE        (%) : 99.9                           \\nREMARK   3   NUMBER OF REFLECTIONS             : 10470                          \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \\nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.185                           \\nREMARK   3   R VALUE            (WORKING SET) : 0.183                           \\nREMARK   3   FREE R VALUE                     : 0.226                           \\nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 4.790                           \\nREMARK   3   FREE R VALUE TEST SET COUNT      : 502                             \\nREMARK   3                                                                      \\nREMARK   3  FIT TO DATA USED IN REFINEMENT (IN BINS).                           \\nREMARK   3   BIN  RESOLUTION RANGE  COMPL.    NWORK NFREE   RWORK  RFREE        \\nREMARK   3     1 36.9636 -  3.4958    1.00     2633   122  0.1813 0.2065        \\nREMARK   3     2  3.4958 -  2.7750    1.00     2472   126  0.1753 0.2493        \\nREMARK   3     3  2.7750 -  2.4243    1.00     2437   134  0.1943 0.2220        \\nREMARK   3     4  2.4243 -  2.2027    1.00     2426   120  0.1952 0.2595        \\nREMARK   3                                                                      \\nREMARK   3  BULK SOLVENT MODELLING.                                             \\nREMARK   3   METHOD USED        : FLAT BULK SOLVENT MODEL                       \\nREMARK   3   SOLVENT RADIUS     : 1.00                                          \\nREMARK   3   SHRINKAGE RADIUS   : 0.73                                          \\nREMARK   3   K_SOL              : 0.38                                          \\nREMARK   3   B_SOL              : 49.03                                         \\nREMARK   3                                                                      \\nREMARK   3  ERROR ESTIMATES.                                                    \\nREMARK   3   COORDINATE ERROR (MAXIMUM-LIKELIHOOD BASED)     : 0.210            \\nREMARK   3   PHASE ERROR (DEGREES, MAXIMUM-LIKELIHOOD BASED) : 18.790           \\nREMARK   3                                                                      \\nREMARK   3  B VALUES.                                                           \\nREMARK   3   FROM WILSON PLOT           (A**2) : 36.45                          \\nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : NULL                           \\nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \\nREMARK   3    B11 (A**2) : 1.42180                                              \\nREMARK   3    B22 (A**2) : 1.42180                                              \\nREMARK   3    B33 (A**2) : -2.84360                                             \\nREMARK   3    B12 (A**2) : 0.00000                                              \\nREMARK   3    B13 (A**2) : 0.00000                                              \\nREMARK   3    B23 (A**2) : 0.00000                                              \\nREMARK   3                                                                      \\nREMARK   3  TWINNING INFORMATION.                                               \\nREMARK   3   FRACTION: NULL                                                     \\nREMARK   3   OPERATOR: NULL                                                     \\nREMARK   3                                                                      \\nREMARK   3  DEVIATIONS FROM IDEAL VALUES.                                       \\nREMARK   3                 RMSD          COUNT                                  \\nREMARK   3   BOND      :  0.008           1273                                  \\nREMARK   3   ANGLE     :  1.085           1704                                  \\nREMARK   3   CHIRALITY :  0.070            196                                  \\nREMARK   3   PLANARITY :  0.005            217                                  \\nREMARK   3   DIHEDRAL  : 17.508            485                                  \\nREMARK   3                                                                      \\nREMARK   3  TLS DETAILS                                                         \\nREMARK   3   NUMBER OF TLS GROUPS  : 1                                          \\nREMARK   3   TLS GROUP : 1                                                      \\nREMARK   3    SELECTION: ALL                                                    \\nREMARK   3    ORIGIN FOR THE GROUP (A):   5.6904  13.5459  14.1463              \\nREMARK   3    T TENSOR                                                          \\nREMARK   3      T11:   0.1942 T22:   0.2940                                     \\nREMARK   3      T33:   0.2653 T12:   0.0050                                     \\nREMARK   3      T13:   0.0100 T23:   0.0623                                     \\nREMARK   3    L TENSOR                                                          \\nREMARK   3      L11:   2.4074 L22:   3.1653                                     \\nREMARK   3      L33:   4.5167 L12:   1.0594                                     \\nREMARK   3      L13:   0.0247 L23:   0.4689                                     \\nREMARK   3    S TENSOR                                                          \\nREMARK   3      S11:  -0.0585 S12:   0.0363 S13:   0.1890                       \\nREMARK   3      S21:  -0.1093 S22:   0.0609 S23:   0.0638                       \\nREMARK   3      S31:  -0.0745 S32:   0.3546 S33:  -0.0012                       \\nREMARK   3                                                                      \\nREMARK   3  NCS DETAILS                                                         \\nREMARK   3   NUMBER OF NCS GROUPS : NULL                                        \\nREMARK   3                                                                      \\nREMARK   3  OTHER REFINEMENT REMARKS: NULL                                      \\nREMARK   4                                                                      \\nREMARK   4 4ETJ COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \\nREMARK 100                                                                      \\nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 27-APR-12.                  \\nREMARK 100 THE DEPOSITION ID IS D_1000072050.                                   \\nREMARK 200                                                                      \\nREMARK 200 EXPERIMENTAL DETAILS                                                 \\nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \\nREMARK 200  DATE OF DATA COLLECTION        : 21-MAR-12                          \\nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \\nREMARK 200  PH                             : 7                                  \\nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \\nREMARK 200                                                                      \\nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \\nREMARK 200  RADIATION SOURCE               : NSLS                               \\nREMARK 200  BEAMLINE                       : X4C                                \\nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \\nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : M                                  \\nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.97925                            \\nREMARK 200  MONOCHROMATOR                  : SI 111 CHANNEL                     \\nREMARK 200  OPTICS                         : MIRROR                             \\nREMARK 200                                                                      \\nREMARK 200  DETECTOR TYPE                  : CCD                                \\nREMARK 200  DETECTOR MANUFACTURER          : MAR CCD 165 MM                     \\nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : NULL                               \\nREMARK 200  DATA SCALING SOFTWARE          : NULL                               \\nREMARK 200                                                                      \\nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 19058                              \\nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.200                              \\nREMARK 200  RESOLUTION RANGE LOW       (A) : 50.000                             \\nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : -3.000                             \\nREMARK 200                                                                      \\nREMARK 200 OVERALL.                                                             \\nREMARK 200  COMPLETENESS FOR RANGE     (%) : 100.0                              \\nREMARK 200  DATA REDUNDANCY                : 14.90                              \\nREMARK 200  R MERGE                    (I) : 0.09000                            \\nREMARK 200  R SYM                      (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 57.6000                            \\nREMARK 200                                                                      \\nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : NULL                     \\nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : NULL                     \\nREMARK 200  COMPLETENESS FOR SHELL     (%) : NULL                               \\nREMARK 200  DATA REDUNDANCY IN SHELL       : NULL                               \\nREMARK 200  R MERGE FOR SHELL          (I) : NULL                               \\nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \\nREMARK 200  <I/SIGMA(I)> FOR SHELL         : NULL                               \\nREMARK 200                                                                      \\nREMARK 200 DIFFRACTION PROTOCOL: SINGLE WAVELENGTH                              \\nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: MIR                          \\nREMARK 200 SOFTWARE USED: NULL                                                  \\nREMARK 200 STARTING MODEL: 3V45                                                 \\nREMARK 200                                                                      \\nREMARK 200 REMARK: NULL                                                         \\nREMARK 280                                                                      \\nREMARK 280 CRYSTAL                                                              \\nREMARK 280 SOLVENT CONTENT, VS   (%): 55.15                                     \\nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 2.74                     \\nREMARK 280                                                                      \\nREMARK 280 CRYSTALLIZATION CONDITIONS: PROTEIN SOLUTION: 100MM NACL, 5MM DTT,   \\nREMARK 280  0.02% NAN3, 10MM TRIS-HCL (PH 7.5), RESERVOIR SOLUTION: 0.1M        \\nREMARK 280  MGCL2, 0.1M BIS-TRIS, 80% PEG400, MICROBATCH UNDER OIL,             \\nREMARK 280  TEMPERATURE 293KK                                                   \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \\nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 43 21 2                        \\nREMARK 290                                                                      \\nREMARK 290      SYMOP   SYMMETRY                                                \\nREMARK 290     NNNMMM   OPERATOR                                                \\nREMARK 290       1555   X,Y,Z                                                   \\nREMARK 290       2555   -X,-Y,Z+1/2                                             \\nREMARK 290       3555   -Y+1/2,X+1/2,Z+3/4                                      \\nREMARK 290       4555   Y+1/2,-X+1/2,Z+1/4                                      \\nREMARK 290       5555   -X+1/2,Y+1/2,-Z+3/4                                     \\nREMARK 290       6555   X+1/2,-Y+1/2,-Z+1/4                                     \\nREMARK 290       7555   Y,X,-Z                                                  \\nREMARK 290       8555   -Y,-X,-Z+1/2                                            \\nREMARK 290                                                                      \\nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \\nREMARK 290           MMM -> TRANSLATION VECTOR                                  \\nREMARK 290                                                                      \\nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \\nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \\nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \\nREMARK 290 RELATED MOLECULES.                                                   \\nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       44.63000            \\nREMARK 290   SMTRY1   3  0.000000 -1.000000  0.000000       32.96500            \\nREMARK 290   SMTRY2   3  1.000000  0.000000  0.000000       32.96500            \\nREMARK 290   SMTRY3   3  0.000000  0.000000  1.000000       66.94500            \\nREMARK 290   SMTRY1   4  0.000000  1.000000  0.000000       32.96500            \\nREMARK 290   SMTRY2   4 -1.000000  0.000000  0.000000       32.96500            \\nREMARK 290   SMTRY3   4  0.000000  0.000000  1.000000       22.31500            \\nREMARK 290   SMTRY1   5 -1.000000  0.000000  0.000000       32.96500            \\nREMARK 290   SMTRY2   5  0.000000  1.000000  0.000000       32.96500            \\nREMARK 290   SMTRY3   5  0.000000  0.000000 -1.000000       66.94500            \\nREMARK 290   SMTRY1   6  1.000000  0.000000  0.000000       32.96500            \\nREMARK 290   SMTRY2   6  0.000000 -1.000000  0.000000       32.96500            \\nREMARK 290   SMTRY3   6  0.000000  0.000000 -1.000000       22.31500            \\nREMARK 290   SMTRY1   7  0.000000  1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   7  1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   7  0.000000  0.000000 -1.000000        0.00000            \\nREMARK 290   SMTRY1   8  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 290   SMTRY2   8 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 290   SMTRY3   8  0.000000  0.000000 -1.000000       44.63000            \\nREMARK 290                                                                      \\nREMARK 290 REMARK: NULL                                                         \\nREMARK 300                                                                      \\nREMARK 300 BIOMOLECULE: 1                                                       \\nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \\nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \\nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \\nREMARK 300 BURIED SURFACE AREA.                                                 \\nREMARK 300 REMARK: MONOMER,17.05 KD,95.7%, DIMER,34.77 KD,4.68%                 \\nREMARK 350                                                                      \\nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \\nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \\nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \\nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \\nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \\nREMARK 350                                                                      \\nREMARK 350 BIOMOLECULE: 1                                                       \\nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: DIMERIC                           \\nREMARK 350 SOFTWARE DETERMINED QUATERNARY STRUCTURE: DIMERIC                    \\nREMARK 350 SOFTWARE USED: PISA                                                  \\nREMARK 350 TOTAL BURIED SURFACE AREA: 4310 ANGSTROM**2                          \\nREMARK 350 SURFACE AREA OF THE COMPLEX: 15060 ANGSTROM**2                       \\nREMARK 350 CHANGE IN SOLVENT FREE ENERGY: -22.0 KCAL/MOL                        \\nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \\nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \\nREMARK 350   BIOMT1   2  0.000000 -1.000000  0.000000        0.00000            \\nREMARK 350   BIOMT2   2 -1.000000  0.000000  0.000000        0.00000            \\nREMARK 350   BIOMT3   2  0.000000  0.000000 -1.000000       44.63000            \\nREMARK 375                                                                      \\nREMARK 375 SPECIAL POSITION                                                     \\nREMARK 375 THE FOLLOWING ATOMS ARE FOUND TO BE WITHIN 0.15 ANGSTROMS            \\nREMARK 375 OF A SYMMETRY RELATED ATOM AND ARE ASSUMED TO BE ON SPECIAL          \\nREMARK 375 POSITIONS.                                                           \\nREMARK 375                                                                      \\nREMARK 375 ATOM RES CSSEQI                                                      \\nREMARK 375 CL    CL A 201  LIES ON A SPECIAL POSITION.                          \\nREMARK 465                                                                      \\nREMARK 465 MISSING RESIDUES                                                     \\nREMARK 465 THE FOLLOWING RESIDUES WERE NOT LOCATED IN THE                       \\nREMARK 465 EXPERIMENT. (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \\nREMARK 465 IDENTIFIER; SSSEQ=SEQUENCE NUMBER; I=INSERTION CODE.)                \\nREMARK 465                                                                      \\nREMARK 465   M RES C SSSEQI                                                     \\nREMARK 465     MET A     1                                                      \\nREMARK 465     HIS A   166                                                      \\nREMARK 465     HIS A   167                                                      \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \\nREMARK 500                                                                      \\nREMARK 500 THE FOLLOWING ATOMS THAT ARE RELATED BY CRYSTALLOGRAPHIC             \\nREMARK 500 SYMMETRY ARE IN CLOSE CONTACT.  AN ATOM LOCATED WITHIN 0.15          \\nREMARK 500 ANGSTROMS OF A SYMMETRY RELATED ATOM IS ASSUMED TO BE ON A           \\nREMARK 500 SPECIAL POSITION AND IS, THEREFORE, LISTED IN REMARK 375             \\nREMARK 500 INSTEAD OF REMARK 500.  ATOMS WITH NON-BLANK ALTERNATE               \\nREMARK 500 LOCATION INDICATORS ARE NOT INCLUDED IN THE CALCULATIONS.            \\nREMARK 500                                                                      \\nREMARK 500 DISTANCE CUTOFF:                                                     \\nREMARK 500 2.2 ANGSTROMS FOR CONTACTS NOT INVOLVING HYDROGEN ATOMS              \\nREMARK 500 1.6 ANGSTROMS FOR CONTACTS INVOLVING HYDROGEN ATOMS                  \\nREMARK 500                                                                      \\nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI  SSYMOP   DISTANCE          \\nREMARK 500   O8   PE4 A   202     O8   PE4 A   202     7555     2.00            \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 500                                                                      \\nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \\nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \\nREMARK 500                                                                      \\nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \\nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \\nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \\nREMARK 500                                                                      \\nREMARK 500 STANDARD TABLE:                                                      \\nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \\nREMARK 500                                                                      \\nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \\nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \\nREMARK 500                                                                      \\nREMARK 500  M RES CSSEQI        PSI       PHI                                   \\nREMARK 500    ILE A  55     -162.58   -110.53                                   \\nREMARK 500    TRP A 116      -75.21    -96.81                                   \\nREMARK 500                                                                      \\nREMARK 500 REMARK: NULL                                                         \\nREMARK 610                                                                      \\nREMARK 610 MISSING HETEROATOM                                                   \\nREMARK 610 THE FOLLOWING RESIDUES HAVE MISSING ATOMS (M=MODEL NUMBER;           \\nREMARK 610 RES=RESIDUE NAME; C=CHAIN IDENTIFIER; SSEQ=SEQUENCE NUMBER;          \\nREMARK 610 I=INSERTION CODE):                                                   \\nREMARK 610   M RES C SSEQI                                                      \\nREMARK 610     PE4 A  202                                                       \\nREMARK 610     PE5 A  203                                                       \\nREMARK 610     PE6 A  204                                                       \\nREMARK 800                                                                      \\nREMARK 800 SITE                                                                 \\nREMARK 800 SITE_IDENTIFIER: AC1                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE CL A 201                  \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC2                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE PE4 A 202                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC3                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE PE5 A 203                 \\nREMARK 800                                                                      \\nREMARK 800 SITE_IDENTIFIER: AC4                                                 \\nREMARK 800 EVIDENCE_CODE: SOFTWARE                                              \\nREMARK 800 SITE_DESCRIPTION: BINDING SITE FOR RESIDUE PE6 A 204                 \\nREMARK 900                                                                      \\nREMARK 900 RELATED ENTRIES                                                      \\nREMARK 900 RELATED ID: 3V45   RELATED DB: PDB                                   \\nREMARK 900 99.4% IDENTITY                                                       \\nREMARK 900 RELATED ID: NESG-OR185   RELATED DB: TARGETTRACK                     \\nDBREF  4ETJ A    1   167  PDB    4ETJ     4ETJ             1    167             \\nSEQRES   1 A  167  MET ALA ARG ILE ARG HIS VAL GLN GLY ASP ILE THR GLU          \\nSEQRES   2 A  167  PHE GLN GLY ASP ALA ILE VAL ASN ALA ALA ASN ASN TYR          \\nSEQRES   3 A  167  LEU LYS LEU GLY ALA GLY VAL ALA GLY ALA ILE LEU ARG          \\nSEQRES   4 A  167  LYS GLY GLY PRO SER ILE GLN GLU GLU CYS ASP ARG ILE          \\nSEQRES   5 A  167  GLY LYS ILE ARG VAL GLY GLU ALA ALA VAL THR GLY ALA          \\nSEQRES   6 A  167  GLY ASN LEU PRO VAL ARG TYR VAL ILE HIS ALA ALA VAL          \\nSEQRES   7 A  167  LEU GLY ASP GLU PRO ALA SER LEU GLU THR VAL ARG LYS          \\nSEQRES   8 A  167  ALA THR LYS SER ALA LEU GLU LYS ALA VAL GLU LEU GLY          \\nSEQRES   9 A  167  LEU LYS THR VAL ALA PHE THR ALA LEU GLY ALA TRP VAL          \\nSEQRES  10 A  167  GLY GLY LEU PRO ALA GLU ALA VAL LEU ARG VAL MET LEU          \\nSEQRES  11 A  167  GLU GLU ILE LYS LYS ALA PRO ASP THR LEU GLU VAL THR          \\nSEQRES  12 A  167  GLY VAL HIS GLY THR GLU LYS SER ALA GLU ALA ALA ARG          \\nSEQRES  13 A  167  ARG ALA LEU LEU GLU HIS HIS HIS HIS HIS HIS                  \\nHET     CL  A 201       1                                                       \\nHET    PE4  A 202      13                                                       \\nHET    PE5  A 203      16                                                       \\nHET    PE6  A 204      16                                                       \\nHETNAM      CL CHLORIDE ION                                                     \\nHETNAM     PE4 2-{2-[2-(2-{2-[2-(2-ETHOXY-ETHOXY)-ETHOXY]-ETHOXY}-              \\nHETNAM   2 PE4  ETHOXY)-ETHOXY]-ETHOXY}-ETHANOL                                 \\nHETNAM     PE5 3,6,9,12,15,18,21,24-OCTAOXAHEXACOSAN-1-OL                       \\nHETNAM     PE6 PHOSPHORYL-HEXAETHYLENE GLYCOL                                   \\nHETSYN     PE4 POLYETHYLENE GLYCOL PEG4000                                      \\nHETSYN     PE5 2-(2-{2-[2-(2-{2-[2-(2-ETHOXY-ETHOXY)-ETHOXY]-ETHOXY}-           \\nHETSYN   2 PE5  ETHOXY)-ETHOXY]-ETHOXY}-ETHOXY)-ETHANOL; POLYETHYLENE           \\nHETSYN   3 PE5  GLYCOL PEG400                                                   \\nFORMUL   2   CL    CL 1-                                                        \\nFORMUL   3  PE4    C16 H34 O8                                                   \\nFORMUL   4  PE5    C18 H38 O9                                                   \\nFORMUL   5  PE6    C12 H27 O10 P                                                \\nFORMUL   6  HOH   *37(H2 O)                                                     \\nHELIX    1   1 ASP A   10  PHE A   14  5                                   5    \\nHELIX    2   2 GLY A   32  GLY A   42  1                                  11    \\nHELIX    3   3 PRO A   43  GLY A   53  1                                  11    \\nHELIX    4   4 SER A   85  LEU A  103  1                                  19    \\nHELIX    5   5 PRO A  121  LYS A  134  1                                  14    \\nHELIX    6   6 THR A  148  HIS A  162  1                                  15    \\nSHEET    1   A 6 ARG A   3  GLN A   8  0                                        \\nSHEET    2   A 6 GLU A 141  HIS A 146  1  O  GLY A 144   N  ARG A   5           \\nSHEET    3   A 6 THR A 107  PHE A 110  1  N  PHE A 110   O  THR A 143           \\nSHEET    4   A 6 ALA A  18  ASN A  24  1  N  VAL A  20   O  ALA A 109           \\nSHEET    5   A 6 TYR A  72  LEU A  79  1  O  ILE A  74   N  ASN A  21           \\nSHEET    6   A 6 ALA A  60  GLY A  64 -1  N  ALA A  61   O  HIS A  75           \\nCISPEP   1 HIS A  164    HIS A  165          0        -0.83                     \\nSITE     1 AC1  1 ARG A  90                                                     \\nSITE     1 AC2  7 ARG A   5  VAL A   7  GLY A   9  ASP A  10                    \\nSITE     2 AC2  7 GLU A  13  PHE A  14  THR A 107                               \\nSITE     1 AC3  6 GLY A 119  LEU A 120  LEU A 126  ARG A 127                    \\nSITE     2 AC3  6 LEU A 130  ALA A 158                                          \\nSITE     1 AC4  7 HIS A   6  THR A 111  ALA A 112  LEU A 126                    \\nSITE     2 AC4  7 GLY A 144  VAL A 145  HIS A 146                               \\nCRYST1   65.930   65.930   89.260  90.00  90.00  90.00 P 43 21 2     8          \\nORIGX1      1.000000  0.000000  0.000000        0.00000                         \\nORIGX2      0.000000  1.000000  0.000000        0.00000                         \\nORIGX3      0.000000  0.000000  1.000000        0.00000                         \\nSCALE1      0.015168  0.000000  0.000000        0.00000                         \\nSCALE2      0.000000  0.015168  0.000000        0.00000                         \\nSCALE3      0.000000  0.000000  0.011203        0.00000                         \\nATOM      1  N   ALA A   2      -4.297   9.078   4.794  1.00 42.80           N  \\nANISOU    1  N   ALA A   2     5133   5774   5353   -498   -855    869       N  \\nATOM      2  CA  ALA A   2      -2.998   9.524   4.298  1.00 45.02           C  \\nANISOU    2  CA  ALA A   2     5564   6033   5509   -462   -784    899       C  \\nATOM      3  C   ALA A   2      -2.069   8.349   4.013  1.00 50.16           C  \\nANISOU    3  C   ALA A   2     6397   6670   5993   -509   -643    794       C  \\nATOM      4  O   ALA A   2      -2.135   7.320   4.690  1.00 49.71           O  \\nANISOU    4  O   ALA A   2     6357   6567   5965   -509   -550    705       O  \\nATOM      5  CB  ALA A   2      -2.352  10.475   5.292  1.00 29.16           C  \\nANISOU    5  CB  ALA A   2     3474   3977   3629   -329   -686    924       C  \\nATOM      6  N   ARG A   3      -1.200   8.513   3.016  1.00 39.44           N  \\nANISOU    6  N   ARG A   3     5184   5344   4459   -544   -617    806       N  \\nATOM      7  CA  ARG A   3      -0.190   7.510   2.703  1.00 37.63           C  \\nANISOU    7  CA  ARG A   3     5111   5106   4080   -548   -458    691       C  \\nATOM      8  C   ARG A   3       0.922   7.478   3.746  1.00 35.42           C  \\nANISOU    8  C   ARG A   3     4772   4802   3883   -401   -283    647       C  \\nATOM      9  O   ARG A   3       1.173   8.474   4.427  1.00 27.61           O  \\nANISOU    9  O   ARG A   3     3658   3824   3009   -333   -278    714       O  \\nATOM     10  CB  ARG A   3       0.427   7.786   1.341  1.00 42.67           C  \\nANISOU   10  CB  ARG A   3     5900   5817   4497   -636   -458    706       C  \\nATOM     11  CG  ARG A   3      -0.558   7.879   0.223  1.00 37.51           C  \\nANISOU   11  CG  ARG A   3     5329   5206   3719   -791   -655    764       C  \\nATOM     12  CD  ARG A   3       0.073   8.660  -0.895  1.00 50.80           C  \\nANISOU   12  CD  ARG A   3     7136   6963   5205   -870   -667    841       C  \\nATOM     13  NE  ARG A   3      -0.298  10.069  -0.871  1.00 50.27           N  \\nANISOU   13  NE  ARG A   3     6985   6891   5225   -852   -817   1020       N  \\nATOM     14  CZ  ARG A   3       0.500  11.057  -1.258  1.00 42.75           C  \\nANISOU   14  CZ  ARG A   3     6095   5957   4192   -876   -775   1112       C  \\nATOM     15  NH1 ARG A   3       1.734  10.800  -1.677  1.00 42.14           N  \\nANISOU   15  NH1 ARG A   3     6119   5944   3947   -923   -573   1033       N  \\nATOM     16  NH2 ARG A   3       0.064  12.307  -1.219  1.00 50.69           N  \\nANISOU   16  NH2 ARG A   3     7059   6912   5288   -853   -929   1279       N  \\nATOM     17  N   ILE A   4       1.573   6.323   3.873  1.00 34.63           N  \\nANISOU   17  N   ILE A   4     4771   4663   3723   -347   -152    532       N  \\nATOM     18  CA  ILE A   4       2.642   6.131   4.849  1.00 31.66           C  \\nANISOU   18  CA  ILE A   4     4335   4276   3417   -189    -12    490       C  \\nATOM     19  C   ILE A   4       3.853   5.432   4.230  1.00 32.50           C  \\nANISOU   19  C   ILE A   4     4545   4415   3389   -121    134    384       C  \\nATOM     20  O   ILE A   4       3.716   4.460   3.492  1.00 36.11           O  \\nANISOU   20  O   ILE A   4     5172   4819   3730   -160    162    292       O  \\nATOM     21  CB  ILE A   4       2.173   5.293   6.045  1.00 34.39           C  \\nANISOU   21  CB  ILE A   4     4675   4514   3878   -128      1    463       C  \\nATOM     22  CG1 ILE A   4       1.005   5.975   6.743  1.00 40.16           C  \\nANISOU   22  CG1 ILE A   4     5272   5240   4748   -186   -105    543       C  \\nATOM     23  CG2 ILE A   4       3.294   5.109   7.062  1.00 28.10           C  \\nANISOU   23  CG2 ILE A   4     3824   3717   3135     44    111    439       C  \\nATOM     24  CD1 ILE A   4       0.560   5.239   7.977  1.00 40.30           C  \\nANISOU   24  CD1 ILE A   4     5283   5171   4856   -158    -69    522       C  \\nATOM     25  N   ARG A   5       5.041   5.932   4.537  1.00 27.75           N  \\nANISOU   25  N   ARG A   5     3832   3905   2804    -22    234    382       N  \\nATOM     26  CA  ARG A   5       6.262   5.239   4.159  1.00 29.32           C  \\nANISOU   26  CA  ARG A   5     4066   4158   2916     90    392    266       C  \\nATOM     27  C   ARG A   5       7.198   5.262   5.348  1.00 30.26           C  \\nANISOU   27  C   ARG A   5     4023   4317   3156    268    452    262       C  \\nATOM     28  O   ARG A   5       7.180   6.203   6.157  1.00 25.25           O  \\nANISOU   28  O   ARG A   5     3246   3726   2623    251    396    351       O  \\nATOM     29  CB  ARG A   5       6.931   5.912   2.948  1.00 31.36           C  \\nANISOU   29  CB  ARG A   5     4329   4569   3016    -15    466    255       C  \\nATOM     30  CG  ARG A   5       7.344   7.357   3.233  1.00 29.80           C  \\nANISOU   30  CG  ARG A   5     3968   4485   2869    -80    445    367       C  \\nATOM     31  CD  ARG A   5       7.739   8.108   1.992  1.00 28.89           C  \\nANISOU   31  CD  ARG A   5     3907   4492   2578   -247    492    395       C  \\nATOM     32  NE  ARG A   5       9.058   7.740   1.496  1.00 30.53           N  \\nANISOU   32  NE  ARG A   5     4071   4853   2677   -201    698    272       N  \\nATOM     33  CZ  ARG A   5      10.134   8.508   1.598  1.00 31.06           C  \\nANISOU   33  CZ  ARG A   5     3977   5084   2742   -232    802    282       C  \\nATOM     34  NH1 ARG A   5      10.051   9.686   2.204  1.00 31.62           N  \\nANISOU   34  NH1 ARG A   5     3953   5153   2909   -316    714    409       N  \\nATOM     35  NH2 ARG A   5      11.285   8.106   1.098  1.00 32.94           N  \\nANISOU   35  NH2 ARG A   5     4144   5488   2884   -188   1002    154       N  \\nATOM     36  N   HIS A   6       8.016   4.222   5.449  1.00 31.14           N  \\nANISOU   36  N   HIS A   6     4165   4410   3256    447    558    156       N  \\nATOM     37  CA  HIS A   6       9.012   4.125   6.501  1.00 29.49           C  \\nANISOU   37  CA  HIS A   6     3797   4261   3146    642    596    150       C  \\nATOM     38  C   HIS A   6      10.409   4.014   5.922  1.00 33.46           C  \\nANISOU   38  C   HIS A   6     4187   4937   3591    758    748     46       C  \\nATOM     39  O   HIS A   6      10.712   3.100   5.155  1.00 32.55           O  \\nANISOU   39  O   HIS A   6     4188   4784   3396    847    853    -77       O  \\nATOM     40  CB  HIS A   6       8.722   2.926   7.404  1.00 27.85           C  \\nANISOU   40  CB  HIS A   6     3709   3863   3008    800    560    138       C  \\nATOM     41  CG  HIS A   6       7.572   3.147   8.328  1.00 30.27           C  \\nANISOU   41  CG  HIS A   6     4050   4060   3393    697    438    242       C  \\nATOM     42  ND1 HIS A   6       7.671   3.929   9.457  1.00 27.37           N  \\nANISOU   42  ND1 HIS A   6     3527   3761   3113    703    381    327       N  \\nATOM     43  CD2 HIS A   6       6.286   2.728   8.269  1.00 29.80           C  \\nANISOU   43  CD2 HIS A   6     4146   3847   3329    566    373    263       C  \\nATOM     44  CE1 HIS A   6       6.502   3.958  10.072  1.00 27.00           C  \\nANISOU   44  CE1 HIS A   6     3542   3604   3114    600    304    389       C  \\nATOM     45  NE2 HIS A   6       5.643   3.247   9.366  1.00 25.26           N  \\nANISOU   45  NE2 HIS A   6     3492   3259   2846    512    296    355       N  \\nATOM     46  N   VAL A   7      11.271   4.940   6.310  1.00 31.89           N  \\nANISOU   46  N   VAL A   7     3753   4933   3432    752    769     81       N  \\nATOM     47  CA  VAL A   7      12.626   4.922   5.794  1.00 35.73           C  \\nANISOU   47  CA  VAL A   7     4073   5633   3871    840    924    -21       C  \\nATOM     48  C   VAL A   7      13.640   5.216   6.895  1.00 31.69           C  \\nANISOU   48  C   VAL A   7     3291   5279   3472    975    903     -3       C  \\nATOM     49  O   VAL A   7      13.310   5.764   7.936  1.00 34.46           O  \\nANISOU   49  O   VAL A   7     3591   5598   3903    931    775    100       O  \\nATOM     50  CB  VAL A   7      12.790   5.936   4.635  1.00 39.39           C  \\nANISOU   50  CB  VAL A   7     4516   6253   4198    591   1007    -15       C  \\nATOM     51  CG1 VAL A   7      12.113   5.428   3.345  1.00 40.49           C  \\nANISOU   51  CG1 VAL A   7     4912   6293   4179    492   1054    -73       C  \\nATOM     52  CG2 VAL A   7      12.219   7.269   5.037  1.00 33.34           C  \\nANISOU   52  CG2 VAL A   7     3717   5479   3470    379    881    136       C  \\nATOM     53  N   GLN A   8      14.880   4.834   6.652  1.00 35.53           N  \\nANISOU   53  N   GLN A   8     3595   5951   3955   1140   1032   -115       N  \\nATOM     54  CA  GLN A   8      15.973   5.195   7.536  1.00 43.70           C  \\nANISOU   54  CA  GLN A   8     4322   7204   5080   1243   1010   -108       C  \\nATOM     55  C   GLN A   8      16.491   6.568   7.130  1.00 49.93           C  \\nANISOU   55  C   GLN A   8     4924   8234   5811    969   1074    -85       C  \\nATOM     56  O   GLN A   8      16.609   6.873   5.936  1.00 42.97           O  \\nANISOU   56  O   GLN A   8     4081   7437   4809    811   1216   -136       O  \\nATOM     57  CB  GLN A   8      17.102   4.163   7.430  1.00 45.62           C  \\nANISOU   57  CB  GLN A   8     4411   7562   5362   1565   1118   -245       C  \\nATOM     58  CG  GLN A   8      18.152   4.282   8.523  1.00 62.26           C  \\nANISOU   58  CG  GLN A   8     6198   9878   7581   1734   1038   -227       C  \\nATOM     59  CD  GLN A   8      19.270   5.238   8.162  1.00 72.38           C  \\nANISOU   59  CD  GLN A   8     7131  11532   8838   1587   1152   -281       C  \\nATOM     60  OE1 GLN A   8      19.688   5.308   7.006  1.00 81.18           O  \\nANISOU   60  OE1 GLN A   8     8195  12785   9866   1513   1352   -391       O  \\nATOM     61  NE2 GLN A   8      19.758   5.985   9.149  1.00 69.60           N  \\nANISOU   61  NE2 GLN A   8     6547  11352   8546   1516   1030   -211       N  \\nATOM     62  N   GLY A   9      16.803   7.394   8.122  1.00 50.85           N  \\nANISOU   62  N   GLY A   9     4866   8456   5998    891    970    -10       N  \\nATOM     63  CA  GLY A   9      17.429   8.674   7.857  1.00 47.46           C  \\nANISOU   63  CA  GLY A   9     4258   8252   5524    624   1029      5       C  \\nATOM     64  C   GLY A   9      17.530   9.576   9.068  1.00 45.69           C  \\nANISOU   64  C   GLY A   9     3916   8072   5373    515    885     88       C  \\nATOM     65  O   GLY A   9      17.388   9.152  10.221  1.00 44.74           O  \\nANISOU   65  O   GLY A   9     3783   7881   5336    681    746    121       O  \\nATOM     66  N   ASP A  10      17.780  10.842   8.787  1.00 45.52           N  \\nANISOU   66  N   ASP A  10     3836   8159   5302    214    922    122       N  \\nATOM     67  CA  ASP A  10      17.893  11.860   9.811  1.00 45.61           C  \\nANISOU   67  CA  ASP A  10     3765   8202   5361     53    805    181       C  \\nATOM     68  C   ASP A  10      16.771  12.837   9.528  1.00 38.29           C  \\nANISOU   68  C   ASP A  10     3111   7032   4406   -180    760    283       C  \\nATOM     69  O   ASP A  10      16.756  13.488   8.481  1.00 42.39           O  \\nANISOU   69  O   ASP A  10     3712   7558   4835   -393    853    307       O  \\nATOM     70  CB  ASP A  10      19.260  12.542   9.699  1.00 48.13           C  \\nANISOU   70  CB  ASP A  10     3774   8860   5652   -119    896    121       C  \\nATOM     71  CG  ASP A  10      19.416  13.724  10.643  1.00 54.24           C  \\nANISOU   71  CG  ASP A  10     4492   9664   6451   -354    786    168       C  \\nATOM     72  OD1 ASP A  10      18.526  13.951  11.490  1.00 54.81           O  \\nANISOU   72  OD1 ASP A  10     4748   9506   6570   -340    644    234       O  \\nATOM     73  OD2 ASP A  10      20.450  14.422  10.542  1.00 58.67           O  \\nANISOU   73  OD2 ASP A  10     4891  10411   6988   -553    830    122       O  \\nATOM     74  N   ILE A  11      15.818  12.931  10.446  1.00 37.84           N  \\nANISOU   74  N   ILE A  11     3199   6759   4420   -133    618    344       N  \\nATOM     75  CA  ILE A  11      14.607  13.703  10.165  1.00 28.38           C  \\nANISOU   75  CA  ILE A  11     2253   5309   3222   -280    567    433       C  \\nATOM     76  C   ILE A  11      14.906  15.182   9.903  1.00 34.07           C  \\nANISOU   76  C   ILE A  11     2987   6051   3907   -581    589    476       C  \\nATOM     77  O   ILE A  11      14.165  15.850   9.184  1.00 36.46           O  \\nANISOU   77  O   ILE A  11     3492   6184   4179   -715    580    554       O  \\nATOM     78  CB  ILE A  11      13.521  13.507  11.262  1.00 28.33           C  \\nANISOU   78  CB  ILE A  11     2368   5091   3306   -164    436    470       C  \\nATOM     79  CG1 ILE A  11      12.214  14.188  10.849  1.00 28.19           C  \\nANISOU   79  CG1 ILE A  11     2573   4830   3308   -267    388    549       C  \\nATOM     80  CG2 ILE A  11      14.024  13.972  12.648  1.00 28.63           C  \\nANISOU   80  CG2 ILE A  11     2280   5209   3388   -190    362    448       C  \\nATOM     81  CD1 ILE A  11      11.068  13.937  11.796  1.00 25.86           C  \\nANISOU   81  CD1 ILE A  11     2374   4350   3100   -158    295    568       C  \\nATOM     82  N   THR A  12      16.019  15.682  10.446  1.00 35.00           N  \\nANISOU   82  N   THR A  12     2895   6378   4023   -696    609    428       N  \\nATOM     83  CA  THR A  12      16.407  17.079  10.244  1.00 40.07           C  \\nANISOU   83  CA  THR A  12     3564   7037   4625  -1020    639    460       C  \\nATOM     84  C   THR A  12      16.910  17.367   8.826  1.00 40.68           C  \\nANISOU   84  C   THR A  12     3655   7224   4578  -1211    788    478       C  \\nATOM     85  O   THR A  12      17.111  18.528   8.449  1.00 46.83           O  \\nANISOU   85  O   THR A  12     4525   7969   5301  -1508    821    532       O  \\nATOM     86  CB  THR A  12      17.475  17.539  11.265  1.00 43.90           C  \\nANISOU   86  CB  THR A  12     3813   7736   5131  -1134    610    393       C  \\nATOM     87  OG1 THR A  12      18.667  16.766  11.096  1.00 46.75           O  \\nANISOU   87  OG1 THR A  12     3862   8435   5466  -1044    691    307       O  \\nATOM     88  CG2 THR A  12      16.960  17.376  12.688  1.00 39.56           C  \\nANISOU   88  CG2 THR A  12     3291   7075   4664   -989    463    380       C  \\nATOM     89  N   GLU A  13      17.119  16.315   8.043  1.00 36.37           N  \\nANISOU   89  N   GLU A  13     3044   6797   3977  -1054    886    430       N  \\nATOM     90  CA  GLU A  13      17.549  16.477   6.659  1.00 39.43           C  \\nANISOU   90  CA  GLU A  13     3465   7301   4216  -1229   1049    433       C  \\nATOM     91  C   GLU A  13      16.413  16.140   5.687  1.00 42.12           C  \\nANISOU   91  C   GLU A  13     4101   7417   4487  -1171   1028    506       C  \\nATOM     92  O   GLU A  13      16.602  16.151   4.476  1.00 36.31           O  \\nANISOU   92  O   GLU A  13     3446   6753   3596  -1300   1151    514       O  \\nATOM     93  CB  GLU A  13      18.794  15.632   6.367  1.00 40.85           C  \\nANISOU   93  CB  GLU A  13     3332   7831   4358  -1130   1212    294       C  \\nATOM     94  CG  GLU A  13      20.041  16.067   7.126  1.00 53.55           C  \\nANISOU   94  CG  GLU A  13     4680   9629   6037  -1198   1191    216       C  \\nATOM     95  CD  GLU A  13      20.461  17.499   6.814  1.00 66.41           C  \\nANISOU   95  CD  GLU A  13     6400  11231   7601  -1575   1213    263       C  \\nATOM     96  OE1 GLU A  13      20.308  17.935   5.653  1.00 69.17           O  \\nANISOU   96  OE1 GLU A  13     6941  11515   7827  -1760   1309    316       O  \\nATOM     97  OE2 GLU A  13      20.939  18.194   7.737  1.00 72.13           O  \\nANISOU   97  OE2 GLU A  13     7029  11991   8387  -1691   1126    249       O  \\nATOM     98  N   PHE A  14      15.234  15.845   6.237  1.00 41.85           N  \\nANISOU   98  N   PHE A  14     4217   7130   4553   -994    872    554       N  \\nATOM     99  CA  PHE A  14      14.032  15.597   5.444  1.00 37.50           C  \\nANISOU   99  CA  PHE A  14     3927   6367   3955   -951    809    629       C  \\nATOM    100  C   PHE A  14      13.780  16.761   4.507  1.00 40.01           C  \\nANISOU  100  C   PHE A  14     4454   6590   4157  -1226    804    756       C  \\nATOM    101  O   PHE A  14      13.855  17.915   4.922  1.00 37.63           O  \\nANISOU  101  O   PHE A  14     4193   6207   3899  -1395    756    824       O  \\nATOM    102  CB  PHE A  14      12.823  15.421   6.359  1.00 35.54           C  \\nANISOU  102  CB  PHE A  14     3769   5880   3854   -781    638    669       C  \\nATOM    103  CG  PHE A  14      11.566  15.009   5.642  1.00 38.00           C  \\nANISOU  103  CG  PHE A  14     4291   6012   4136   -718    557    729       C  \\nATOM    104  CD1 PHE A  14      11.471  13.759   5.043  1.00 28.42           C  \\nANISOU  104  CD1 PHE A  14     3100   4846   2853   -588    610    657       C  \\nATOM    105  CD2 PHE A  14      10.466  15.858   5.602  1.00 33.27           C  \\nANISOU  105  CD2 PHE A  14     3861   5193   3586   -781    419    848       C  \\nATOM    106  CE1 PHE A  14      10.308  13.377   4.394  1.00 27.91           C  \\nANISOU  106  CE1 PHE A  14     3222   4632   2750   -563    520    704       C  \\nATOM    107  CE2 PHE A  14       9.296  15.479   4.974  1.00 27.28           C  \\nANISOU  107  CE2 PHE A  14     3255   4303   2808   -724    319    903       C  \\nATOM    108  CZ  PHE A  14       9.214  14.240   4.367  1.00 33.17           C  \\nANISOU  108  CZ  PHE A  14     4020   5116   3467   -635    366    832       C  \\nATOM    109  N   GLN A  15      13.456  16.461   3.253  1.00 33.44           N  \\nANISOU  109  N   GLN A  15     3785   5751   3170  -1276    843    790       N  \\nATOM    110  CA  GLN A  15      13.376  17.494   2.242  1.00 38.91           C  \\nANISOU  110  CA  GLN A  15     4691   6385   3707  -1551    848    923       C  \\nATOM    111  C   GLN A  15      11.952  17.688   1.714  1.00 47.06           C  \\nANISOU  111  C   GLN A  15     5999   7158   4724  -1517    661   1062       C  \\nATOM    112  O   GLN A  15      11.740  17.746   0.511  1.00 66.16           O  \\nANISOU  112  O   GLN A  15     8609   9579   6951  -1644    668   1136       O  \\nATOM    113  CB  GLN A  15      14.327  17.140   1.089  1.00 45.47           C  \\nANISOU  113  CB  GLN A  15     5492   7470   4314  -1699   1064    857       C  \\nATOM    114  CG  GLN A  15      14.878  18.327   0.334  1.00 70.07           C  \\nANISOU  114  CG  GLN A  15     8734  10630   7260  -2059   1146    964       C  \\nATOM    115  CD  GLN A  15      16.301  18.681   0.736  1.00 74.60           C  \\nANISOU  115  CD  GLN A  15     9049  11409   7886  -2158   1287    842       C  \\nATOM    116  OE1 GLN A  15      16.859  18.109   1.676  1.00 72.04           O  \\nANISOU  116  OE1 GLN A  15     8431  11247   7693  -2003   1332    718       O  \\nATOM    117  NE2 GLN A  15      16.894  19.632   0.020  1.00 72.94           N  \\nANISOU  117  NE2 GLN A  15     8949  11191   7573  -2411   1337    881       N  \\nATOM    118  N   GLY A  16      10.979  17.798   2.610  1.00 42.64           N  \\nANISOU  118  N   GLY A  16     5449   6395   4357  -1352    492   1096       N  \\nATOM    119  CA  GLY A  16       9.598  18.006   2.204  1.00 39.94           C  \\nANISOU  119  CA  GLY A  16     5309   5832   4035  -1296    300   1221       C  \\nATOM    120  C   GLY A  16       8.975  19.264   2.783  1.00 38.32           C  \\nANISOU  120  C   GLY A  16     5200   5387   3975  -1317    155   1339       C  \\nATOM    121  O   GLY A  16       9.661  20.242   3.058  1.00 41.27           O  \\nANISOU  121  O   GLY A  16     5592   5735   4354  -1478    206   1369       O  \\nATOM    122  N   ASP A  17       7.664  19.229   2.993  1.00 38.86           N  \\nANISOU  122  N   ASP A  17     5322   5276   4167  -1152    -21   1394       N  \\nATOM    123  CA  ASP A  17       6.940  20.374   3.554  1.00 38.38           C  \\nANISOU  123  CA  ASP A  17     5346   4967   4269  -1113   -158   1488       C  \\nATOM    124  C   ASP A  17       7.150  20.654   5.051  1.00 37.56           C  \\nANISOU  124  C   ASP A  17     5095   4818   4357  -1032   -112   1381       C  \\nATOM    125  O   ASP A  17       7.258  21.806   5.452  1.00 40.79           O  \\nANISOU  125  O   ASP A  17     5595   5068   4837  -1110   -135   1425       O  \\nATOM    126  CB  ASP A  17       5.445  20.235   3.257  1.00 41.40           C  \\nANISOU  126  CB  ASP A  17     5790   5209   4731   -949   -358   1570       C  \\nATOM    127  CG  ASP A  17       5.154  20.269   1.774  1.00 42.91           C  \\nANISOU  127  CG  ASP A  17     6177   5409   4719  -1058   -455   1710       C  \\nATOM    128  OD1 ASP A  17       5.790  21.088   1.070  1.00 55.57           O  \\nANISOU  128  OD1 ASP A  17     7961   6983   6171  -1262   -426   1820       O  \\nATOM    129  OD2 ASP A  17       4.309  19.477   1.311  1.00 35.78           O  \\nANISOU  129  OD2 ASP A  17     5259   4546   3787   -966   -558   1711       O  \\nATOM    130  N   ALA A  18       7.179  19.615   5.878  1.00 29.25           N  \\nANISOU  130  N   ALA A  18     3849   3887   3377   -883    -56   1243       N  \\nATOM    131  CA  ALA A  18       7.267  19.816   7.316  1.00 27.26           C  \\nANISOU  131  CA  ALA A  18     3479   3599   3280   -806    -30   1147       C  \\nATOM    132  C   ALA A  18       7.883  18.616   7.996  1.00 29.70           C  \\nANISOU  132  C   ALA A  18     3594   4113   3580   -717     67   1013       C  \\nATOM    133  O   ALA A  18       7.847  17.511   7.462  1.00 32.56           O  \\nANISOU  133  O   ALA A  18     3920   4582   3868   -647     93    986       O  \\nATOM    134  CB  ALA A  18       5.871  20.069   7.903  1.00 26.68           C  \\nANISOU  134  CB  ALA A  18     3427   3328   3384   -633   -152   1163       C  \\nATOM    135  N   ILE A  19       8.439  18.835   9.182  1.00 28.26           N  \\nANISOU  135  N   ILE A  19     3305   3968   3463   -717    111    928       N  \\nATOM    136  CA  ILE A  19       8.765  17.729  10.087  1.00 28.40           C  \\nANISOU  136  CA  ILE A  19     3159   4130   3502   -583    155    821       C  \\nATOM    137  C   ILE A  19       8.018  17.960  11.395  1.00 28.77           C  \\nANISOU  137  C   ILE A  19     3197   4054   3681   -486    107    777       C  \\nATOM    138  O   ILE A  19       7.703  19.097  11.753  1.00 28.52           O  \\nANISOU  138  O   ILE A  19     3247   3870   3719   -550     75    791       O  \\nATOM    139  CB  ILE A  19      10.292  17.579  10.347  1.00 26.34           C  \\nANISOU  139  CB  ILE A  19     2742   4101   3164   -667    251    750       C  \\nATOM    140  CG1 ILE A  19      10.839  18.742  11.172  1.00 28.19           C  \\nANISOU  140  CG1 ILE A  19     2961   4317   3432   -820    248    725       C  \\nATOM    141  CG2 ILE A  19      11.063  17.461   9.024  1.00 27.29           C  \\nANISOU  141  CG2 ILE A  19     2860   4363   3146   -784    337    776       C  \\nATOM    142  CD1 ILE A  19      12.378  18.738  11.279  1.00 30.31           C  \\nANISOU  142  CD1 ILE A  19     3047   4850   3619   -951    329    662       C  \\nATOM    143  N   VAL A  20       7.693  16.887  12.093  1.00 27.48           N  \\nANISOU  143  N   VAL A  20     2959   3938   3544   -338    109    721       N  \\nATOM    144  CA  VAL A  20       6.964  17.032  13.337  1.00 28.29           C  \\nANISOU  144  CA  VAL A  20     3057   3947   3743   -264     88    673       C  \\nATOM    145  C   VAL A  20       7.939  16.828  14.490  1.00 37.76           C  \\nANISOU  145  C   VAL A  20     4162   5286   4901   -276    121    595       C  \\nATOM    146  O   VAL A  20       8.679  15.844  14.524  1.00 31.21           O  \\nANISOU  146  O   VAL A  20     3242   4612   4002   -216    140    577       O  \\nATOM    147  CB  VAL A  20       5.778  16.051  13.421  1.00 26.49           C  \\nANISOU  147  CB  VAL A  20     2834   3667   3564   -126     64    675       C  \\nATOM    148  CG1 VAL A  20       5.075  16.161  14.770  1.00 23.98           C  \\nANISOU  148  CG1 VAL A  20     2500   3285   3324    -70     76    612       C  \\nATOM    149  CG2 VAL A  20       4.803  16.328  12.292  1.00 21.73           C  \\nANISOU  149  CG2 VAL A  20     2303   2953   3000   -124     -1    753       C  \\nATOM    150  N   ASN A  21       7.961  17.797  15.401  1.00 28.05           N  \\nANISOU  150  N   ASN A  21     2958   3992   3707   -350    118    546       N  \\nATOM    151  CA  ASN A  21       8.798  17.737  16.588  1.00 32.04           C  \\nANISOU  151  CA  ASN A  21     3387   4628   4157   -386    123    471       C  \\nATOM    152  C   ASN A  21       8.002  17.214  17.777  1.00 29.03           C  \\nANISOU  152  C   ASN A  21     3030   4206   3796   -277    122    423       C  \\nATOM    153  O   ASN A  21       6.908  17.698  18.047  1.00 32.04           O  \\nANISOU  153  O   ASN A  21     3490   4424   4260   -252    139    402       O  \\nATOM    154  CB  ASN A  21       9.304  19.145  16.900  1.00 36.32           C  \\nANISOU  154  CB  ASN A  21     3978   5121   4702   -573    125    429       C  \\nATOM    155  CG  ASN A  21      10.131  19.200  18.155  1.00 40.47           C  \\nANISOU  155  CG  ASN A  21     4432   5791   5155   -644    109    342       C  \\nATOM    156  OD1 ASN A  21      10.873  18.265  18.456  1.00 41.43           O  \\nANISOU  156  OD1 ASN A  21     4417   6119   5203   -582     84    339       O  \\nATOM    157  ND2 ASN A  21      10.019  20.308  18.898  1.00 36.78           N  \\nANISOU  157  ND2 ASN A  21     4064   5209   4703   -769    113    267       N  \\nATOM    158  N   ALA A  22       8.530  16.214  18.470  1.00 30.31           N  \\nANISOU  158  N   ALA A  22     3124   4515   3879   -208    106    410       N  \\nATOM    159  CA  ALA A  22       7.971  15.799  19.747  1.00 27.08           C  \\nANISOU  159  CA  ALA A  22     2760   4088   3442   -155    109    369       C  \\nATOM    160  C   ALA A  22       8.268  16.875  20.796  1.00 33.07           C  \\nANISOU  160  C   ALA A  22     3551   4848   4165   -285    108    280       C  \\nATOM    161  O   ALA A  22       9.370  16.944  21.336  1.00 26.15           O  \\nANISOU  161  O   ALA A  22     2609   4132   3195   -357     58    254       O  \\nATOM    162  CB  ALA A  22       8.564  14.465  20.173  1.00 32.12           C  \\nANISOU  162  CB  ALA A  22     3351   4866   3987    -47     68    402       C  \\nATOM    163  N   ALA A  23       7.284  17.718  21.083  1.00 27.63           N  \\nANISOU  163  N   ALA A  23     2958   3988   3554   -313    162    223       N  \\nATOM    164  CA  ALA A  23       7.519  18.876  21.946  1.00 27.66           C  \\nANISOU  164  CA  ALA A  23     3031   3947   3531   -448    178    116       C  \\nATOM    165  C   ALA A  23       6.942  18.724  23.352  1.00 27.59           C  \\nANISOU  165  C   ALA A  23     3092   3937   3454   -437    226     24       C  \\nATOM    166  O   ALA A  23       6.069  17.880  23.598  1.00 29.02           O  \\nANISOU  166  O   ALA A  23     3279   4106   3641   -329    269     45       O  \\nATOM    167  CB  ALA A  23       6.949  20.136  21.275  1.00 28.66           C  \\nANISOU  167  CB  ALA A  23     3246   3850   3791   -487    213     98       C  \\nATOM    168  N   ASN A  24       7.434  19.538  24.280  1.00 33.09           N  \\nANISOU  168  N   ASN A  24     3854   4653   4067   -577    225    -85       N  \\nATOM    169  CA  ASN A  24       6.721  19.760  25.531  1.00 33.67           C  \\nANISOU  169  CA  ASN A  24     4037   4676   4081   -595    307   -207       C  \\nATOM    170  C   ASN A  24       5.722  20.905  25.337  1.00 32.36           C  \\nANISOU  170  C   ASN A  24     3963   4260   4073   -573    412   -303       C  \\nATOM    171  O   ASN A  24       5.823  21.645  24.358  1.00 28.12           O  \\nANISOU  171  O   ASN A  24     3434   3592   3659   -583    388   -267       O  \\nATOM    172  CB  ASN A  24       7.682  20.040  26.682  1.00 30.93           C  \\nANISOU  172  CB  ASN A  24     3737   4473   3543   -760    252   -293       C  \\nATOM    173  CG  ASN A  24       8.619  21.189  26.398  1.00 39.97           C  \\nANISOU  173  CG  ASN A  24     4893   5597   4695   -940    202   -347       C  \\nATOM    174  OD1 ASN A  24       8.200  22.347  26.336  1.00 38.10           O  \\nANISOU  174  OD1 ASN A  24     4780   5151   4547  -1005    275   -447       O  \\nATOM    175  ND2 ASN A  24       9.907  20.879  26.250  1.00 39.44           N  \\nANISOU  175  ND2 ASN A  24     4698   5749   4537  -1025     78   -286       N  \\nATOM    176  N   ASN A  25       4.755  21.059  26.239  1.00 32.91           N  \\nANISOU  176  N   ASN A  25     4105   4260   4140   -535    531   -423       N  \\nATOM    177  CA  ASN A  25       3.712  22.062  26.015  1.00 29.87           C  \\nANISOU  177  CA  ASN A  25     3778   3635   3937   -452    634   -517       C  \\nATOM    178  C   ASN A  25       4.169  23.512  26.251  1.00 34.77           C  \\nANISOU  178  C   ASN A  25     4552   4086   4571   -576    646   -645       C  \\nATOM    179  O   ASN A  25       3.437  24.454  25.971  1.00 32.91           O  \\nANISOU  179  O   ASN A  25     4391   3611   4503   -491    711   -714       O  \\nATOM    180  CB  ASN A  25       2.418  21.732  26.780  1.00 30.45           C  \\nANISOU  180  CB  ASN A  25     3838   3699   4033   -348    787   -615       C  \\nATOM    181  CG  ASN A  25       2.585  21.822  28.281  1.00 41.87           C  \\nANISOU  181  CG  ASN A  25     5400   5229   5277   -472    876   -772       C  \\nATOM    182  OD1 ASN A  25       3.524  21.261  28.839  1.00 43.26           O  \\nANISOU  182  OD1 ASN A  25     5604   5584   5246   -597    791   -731       O  \\nATOM    183  ND2 ASN A  25       1.682  22.539  28.942  1.00 33.85           N  \\nANISOU  183  ND2 ASN A  25     4455   4092   4316   -431   1044   -957       N  \\nATOM    184  N   TYR A  26       5.389  23.680  26.750  1.00 34.58           N  \\nANISOU  184  N   TYR A  26     4579   4185   4377   -777    572   -675       N  \\nATOM    185  CA  TYR A  26       6.009  25.004  26.861  1.00 36.43           C  \\nANISOU  185  CA  TYR A  26     4968   4269   4606   -953    562   -783       C  \\nATOM    186  C   TYR A  26       6.767  25.390  25.593  1.00 37.31           C  \\nANISOU  186  C   TYR A  26     5047   4331   4796  -1029    459   -647       C  \\nATOM    187  O   TYR A  26       7.260  26.511  25.478  1.00 37.46           O  \\nANISOU  187  O   TYR A  26     5208   4194   4829  -1194    449   -708       O  \\nATOM    188  CB  TYR A  26       6.929  25.069  28.085  1.00 35.78           C  \\nANISOU  188  CB  TYR A  26     4951   4360   4282  -1175    529   -901       C  \\nATOM    189  CG  TYR A  26       6.156  24.938  29.373  1.00 36.85           C  \\nANISOU  189  CG  TYR A  26     5182   4507   4314  -1142    656  -1062       C  \\nATOM    190  CD1 TYR A  26       5.908  26.044  30.173  1.00 45.94           C  \\nANISOU  190  CD1 TYR A  26     6541   5481   5434  -1238    766  -1287       C  \\nATOM    191  CD2 TYR A  26       5.632  23.711  29.767  1.00 46.14           C  \\nANISOU  191  CD2 TYR A  26     6261   5855   5417  -1021    686   -995       C  \\nATOM    192  CE1 TYR A  26       5.170  25.929  31.354  1.00 46.14           C  \\nANISOU  192  CE1 TYR A  26     6657   5528   5344  -1215    914  -1453       C  \\nATOM    193  CE2 TYR A  26       4.899  23.583  30.939  1.00 42.11           C  \\nANISOU  193  CE2 TYR A  26     5845   5367   4788  -1018    827  -1140       C  \\nATOM    194  CZ  TYR A  26       4.670  24.693  31.727  1.00 39.40           C  \\nANISOU  194  CZ  TYR A  26     5691   4875   4406  -1112    947  -1373       C  \\nATOM    195  OH  TYR A  26       3.943  24.558  32.888  1.00 40.92           O  \\nANISOU  195  OH  TYR A  26     5978   5109   4461  -1118   1113  -1532       O  \\nATOM    196  N   LEU A  27       6.825  24.464  24.639  1.00 32.87           N  \\nANISOU  196  N   LEU A  27     4321   3893   4276   -922    395   -470       N  \\nATOM    197  CA  LEU A  27       7.533  24.667  23.376  1.00 33.50           C  \\nANISOU  197  CA  LEU A  27     4357   3968   4401   -995    317   -333       C  \\nATOM    198  C   LEU A  27       8.978  25.092  23.596  1.00 33.35           C  \\nANISOU  198  C   LEU A  27     4333   4094   4246  -1268    252   -365       C  \\nATOM    199  O   LEU A  27       9.514  25.921  22.854  1.00 38.35           O  \\nANISOU  199  O   LEU A  27     5033   4626   4914  -1424    232   -329       O  \\nATOM    200  CB  LEU A  27       6.797  25.667  22.473  1.00 34.18           C  \\nANISOU  200  CB  LEU A  27     4581   3736   4671   -928    340   -299       C  \\nATOM    201  CG  LEU A  27       5.356  25.307  22.088  1.00 36.53           C  \\nANISOU  201  CG  LEU A  27     4839   3913   5127   -654    378   -255       C  \\nATOM    202  CD1 LEU A  27       4.732  26.375  21.214  1.00 32.69           C  \\nANISOU  202  CD1 LEU A  27     4493   3112   4816   -579    363   -208       C  \\nATOM    203  CD2 LEU A  27       5.274  23.939  21.403  1.00 29.13           C  \\nANISOU  203  CD2 LEU A  27     3714   3186   4170   -546    330   -110       C  \\nATOM    204  N   LYS A  28       9.596  24.534  24.636  1.00 33.08           N  \\nANISOU  204  N   LYS A  28     4220   4301   4046  -1340    213   -429       N  \\nATOM    205  CA  LYS A  28      11.012  24.758  24.901  1.00 38.01           C  \\nANISOU  205  CA  LYS A  28     4771   5140   4532  -1592    125   -456       C  \\nATOM    206  C   LYS A  28      11.804  23.619  24.264  1.00 41.87           C  \\nANISOU  206  C   LYS A  28     5005   5919   4983  -1524     44   -312       C  \\nATOM    207  O   LYS A  28      11.629  22.462  24.645  1.00 37.52           O  \\nANISOU  207  O   LYS A  28     4354   5518   4382  -1348     14   -265       O  \\nATOM    208  CB  LYS A  28      11.287  24.820  26.411  1.00 35.98           C  \\nANISOU  208  CB  LYS A  28     4571   5001   4099  -1710     98   -605       C  \\nATOM    209  CG  LYS A  28      10.612  25.991  27.149  1.00 37.78           C  \\nANISOU  209  CG  LYS A  28     5068   4948   4338  -1798    198   -792       C  \\nATOM    210  CD  LYS A  28      10.990  27.344  26.570  1.00 52.62           C  \\nANISOU  210  CD  LYS A  28     7099   6602   6294  -2011    213   -838       C  \\nATOM    211  CE  LYS A  28      10.088  28.449  27.111  1.00 64.68           C  \\nANISOU  211  CE  LYS A  28     8917   7769   7889  -2007    333  -1015       C  \\nATOM    212  NZ  LYS A  28      10.280  29.738  26.382  1.00 66.39           N  \\nANISOU  212  NZ  LYS A  28     9330   7681   8214  -2167    350  -1025       N  \\nATOM    213  N   LEU A  29      12.651  23.941  23.283  1.00 39.05           N  \\nANISOU  213  N   LEU A  29     4558   5629   4651  -1660     22   -245       N  \\nATOM    214  CA  LEU A  29      13.387  22.912  22.542  1.00 36.08           C  \\nANISOU  214  CA  LEU A  29     3935   5516   4256  -1576    -24   -127       C  \\nATOM    215  C   LEU A  29      14.623  22.451  23.300  1.00 37.18           C  \\nANISOU  215  C   LEU A  29     3866   6005   4254  -1668   -132   -164       C  \\nATOM    216  O   LEU A  29      15.514  23.243  23.598  1.00 42.04           O  \\nANISOU  216  O   LEU A  29     4448   6726   4800  -1940   -172   -240       O  \\nATOM    217  CB  LEU A  29      13.767  23.387  21.134  1.00 42.19           C  \\nANISOU  217  CB  LEU A  29     4686   6246   5096  -1684     20    -44       C  \\nATOM    218  CG  LEU A  29      12.694  23.401  20.034  1.00 45.68           C  \\nANISOU  218  CG  LEU A  29     5257   6434   5666  -1526     84     56       C  \\nATOM    219  CD1 LEU A  29      11.804  24.621  20.132  1.00 49.81           C  \\nANISOU  219  CD1 LEU A  29     6051   6599   6277  -1580    124     11       C  \\nATOM    220  CD2 LEU A  29      13.361  23.365  18.682  1.00 54.43           C  \\nANISOU  220  CD2 LEU A  29     6273   7639   6769  -1612    110    158       C  \\nATOM    221  N   GLY A  30      14.672  21.162  23.622  1.00 47.01           N  \\nANISOU  221  N   GLY A  30     4977   7428   5458  -1444   -192   -106       N  \\nATOM    222  CA  GLY A  30      15.769  20.639  24.418  1.00 50.89           C  \\nANISOU  222  CA  GLY A  30     5271   8248   5818  -1477   -327   -123       C  \\nATOM    223  C   GLY A  30      16.109  19.185  24.150  1.00 53.65           C  \\nANISOU  223  C   GLY A  30     5424   8790   6169  -1204   -386    -15       C  \\nATOM    224  O   GLY A  30      16.982  18.877  23.340  1.00 50.75           O  \\nANISOU  224  O   GLY A  30     4830   8614   5837  -1184   -393     26       O  \\nATOM    225  N   ALA A  31      15.418  18.287  24.841  1.00 61.31           N  \\nANISOU  225  N   ALA A  31     6494   9703   7098   -998   -417     24       N  \\nATOM    226  CA  ALA A  31      15.739  16.865  24.779  1.00 64.96           C  \\nANISOU  226  CA  ALA A  31     6823  10312   7547   -733   -492    126       C  \\nATOM    227  C   ALA A  31      15.191  16.180  23.526  1.00 55.16           C  \\nANISOU  227  C   ALA A  31     5588   8933   6437   -540   -384    204       C  \\nATOM    228  O   ALA A  31      14.068  16.455  23.085  1.00 54.99           O  \\nANISOU  228  O   ALA A  31     5743   8658   6493   -537   -273    206       O  \\nATOM    229  CB  ALA A  31      15.252  16.145  26.053  1.00 60.83           C  \\nANISOU  229  CB  ALA A  31     6443   9770   6899   -621   -573    152       C  \\nATOM    230  N   GLY A  32      16.001  15.289  22.960  1.00 44.48           N  \\nANISOU  230  N   GLY A  32     4034   7760   5106   -375   -423    259       N  \\nATOM    231  CA  GLY A  32      15.567  14.442  21.864  1.00 47.32           C  \\nANISOU  231  CA  GLY A  32     4412   8009   5557   -178   -334    321       C  \\nATOM    232  C   GLY A  32      15.819  15.063  20.507  1.00 46.61           C  \\nANISOU  232  C   GLY A  32     4243   7922   5545   -292   -219    302       C  \\nATOM    233  O   GLY A  32      16.821  15.744  20.304  1.00 53.00           O  \\nANISOU  233  O   GLY A  32     4872   8927   6337   -462   -223    257       O  \\nATOM    234  N   VAL A  33      14.902  14.833  19.574  1.00 41.95           N  \\nANISOU  234  N   VAL A  33     3792   7125   5023   -222   -119    340       N  \\nATOM    235  CA  VAL A  33      15.026  15.410  18.235  1.00 45.34           C  \\nANISOU  235  CA  VAL A  33     4196   7534   5496   -340    -13    340       C  \\nATOM    236  C   VAL A  33      14.988  16.940  18.271  1.00 36.79           C  \\nANISOU  236  C   VAL A  33     3186   6378   4415   -628     10    305       C  \\nATOM    237  O   VAL A  33      15.628  17.596  17.459  1.00 43.05           O  \\nANISOU  237  O   VAL A  33     3903   7252   5201   -802     69    299       O  \\nATOM    238  CB  VAL A  33      13.925  14.889  17.288  1.00 46.07           C  \\nANISOU  238  CB  VAL A  33     4453   7410   5642   -221     59    393       C  \\nATOM    239  CG1 VAL A  33      14.190  13.438  16.925  1.00 49.42           C  \\nANISOU  239  CG1 VAL A  33     4811   7906   6061     26     64    411       C  \\nATOM    240  CG2 VAL A  33      12.545  15.044  17.934  1.00 54.38           C  \\nANISOU  240  CG2 VAL A  33     5716   8215   6729   -203     41    407       C  \\nATOM    241  N   ALA A  34      14.232  17.493  19.219  1.00 35.40           N  \\nANISOU  241  N   ALA A  34     3173   6036   4240   -683    -25    277       N  \\nATOM    242  CA  ALA A  34      14.087  18.937  19.371  1.00 34.71           C  \\nANISOU  242  CA  ALA A  34     3206   5818   4163   -930     -4    229       C  \\nATOM    243  C   ALA A  34      15.434  19.619  19.453  1.00 32.95           C  \\nANISOU  243  C   ALA A  34     2821   5821   3878  -1170    -27    178       C  \\nATOM    244  O   ALA A  34      15.634  20.676  18.860  1.00 47.63           O  \\nANISOU  244  O   ALA A  34     4743   7605   5748  -1400     27    171       O  \\nATOM    245  CB  ALA A  34      13.242  19.271  20.596  1.00 38.41           C  \\nANISOU  245  CB  ALA A  34     3842   6126   4624   -925    -31    171       C  \\nATOM    246  N   GLY A  35      16.366  19.006  20.177  1.00 36.76           N  \\nANISOU  246  N   GLY A  35     3094   6583   4290  -1126   -116    148       N  \\nATOM    247  CA  GLY A  35      17.717  19.531  20.272  1.00 34.72           C  \\nANISOU  247  CA  GLY A  35     2614   6605   3972  -1353   -153     93       C  \\nATOM    248  C   GLY A  35      18.414  19.593  18.926  1.00 46.44           C  \\nANISOU  248  C   GLY A  35     3940   8223   5484  -1433    -49    121       C  \\nATOM    249  O   GLY A  35      19.074  20.583  18.584  1.00 37.32           O  \\nANISOU  249  O   GLY A  35     2737   7140   4301  -1739     -5     87       O  \\nATOM    250  N   ALA A  36      18.276  18.520  18.155  1.00 40.45           N  \\nANISOU  250  N   ALA A  36     3113   7494   4762  -1178      2    178       N  \\nATOM    251  CA  ALA A  36      18.828  18.491  16.808  1.00 42.87           C  \\nANISOU  251  CA  ALA A  36     3299   7917   5073  -1237    128    195       C  \\nATOM    252  C   ALA A  36      18.148  19.551  15.931  1.00 40.70           C  \\nANISOU  252  C   ALA A  36     3285   7373   4808  -1453    223    241       C  \\nATOM    253  O   ALA A  36      18.799  20.221  15.131  1.00 36.37           O  \\nANISOU  253  O   ALA A  36     2681   6919   4220  -1701    314    242       O  \\nATOM    254  CB  ALA A  36      18.659  17.106  16.207  1.00 45.04           C  \\nANISOU  254  CB  ALA A  36     3513   8223   5378   -906    167    229       C  \\nATOM    255  N   ILE A  37      16.833  19.694  16.094  1.00 38.42           N  \\nANISOU  255  N   ILE A  37     3275   6753   4568  -1358    199    284       N  \\nATOM    256  CA  ILE A  37      16.061  20.675  15.334  1.00 41.55           C  \\nANISOU  256  CA  ILE A  37     3936   6862   4990  -1505    252    343       C  \\nATOM    257  C   ILE A  37      16.565  22.093  15.592  1.00 41.32           C  \\nANISOU  257  C   ILE A  37     3980   6789   4932  -1852    256    307       C  \\nATOM    258  O   ILE A  37      16.807  22.865  14.658  1.00 39.74           O  \\nANISOU  258  O   ILE A  37     3869   6531   4700  -2078    329    356       O  \\nATOM    259  CB  ILE A  37      14.543  20.538  15.627  1.00 37.23           C  \\nANISOU  259  CB  ILE A  37     3623   6000   4523  -1306    209    379       C  \\nATOM    260  CG1 ILE A  37      13.996  19.323  14.868  1.00 30.02           C  \\nANISOU  260  CG1 ILE A  37     2694   5088   3626  -1053    232    437       C  \\nATOM    261  CG2 ILE A  37      13.803  21.822  15.261  1.00 32.46           C  \\nANISOU  261  CG2 ILE A  37     3285   5086   3964  -1458    220    421       C  \\nATOM    262  CD1 ILE A  37      12.587  18.949  15.178  1.00 28.25           C  \\nANISOU  262  CD1 ILE A  37     2627   4630   3476   -858    191    463       C  \\nATOM    263  N   LEU A  38      16.758  22.417  16.865  1.00 38.38           N  \\nANISOU  263  N   LEU A  38     3588   6445   4550  -1916    177    219       N  \\nATOM    264  CA  LEU A  38      17.285  23.723  17.245  1.00 41.56           C  \\nANISOU  264  CA  LEU A  38     4070   6805   4915  -2267    172    158       C  \\nATOM    265  C   LEU A  38      18.745  23.982  16.801  1.00 44.09           C  \\nANISOU  265  C   LEU A  38     4141   7455   5155  -2556    218    132       C  \\nATOM    266  O   LEU A  38      19.096  25.112  16.455  1.00 46.01           O  \\nANISOU  266  O   LEU A  38     4505   7612   5365  -2896    268    130       O  \\nATOM    267  CB  LEU A  38      17.134  23.920  18.755  1.00 47.77           C  \\nANISOU  267  CB  LEU A  38     4902   7565   5685  -2268     76     51       C  \\nATOM    268  CG  LEU A  38      17.534  25.286  19.311  1.00 54.54           C  \\nANISOU  268  CG  LEU A  38     5900   8326   6499  -2630     63    -42       C  \\nATOM    269  CD1 LEU A  38      16.687  26.386  18.680  1.00 54.18           C  \\nANISOU  269  CD1 LEU A  38     6208   7855   6523  -2726    131      5       C  \\nATOM    270  CD2 LEU A  38      17.413  25.290  20.826  1.00 52.20           C  \\nANISOU  270  CD2 LEU A  38     5633   8050   6152  -2610    -32   -162       C  \\nATOM    271  N   ARG A  39      19.604  22.963  16.819  1.00 50.25           N  \\nANISOU  271  N   ARG A  39     4574   8611   5908  -2429    207    109       N  \\nATOM    272  CA  ARG A  39      20.997  23.148  16.369  1.00 51.24           C  \\nANISOU  272  CA  ARG A  39     4506   8993   5972  -2564    252     66       C  \\nATOM    273  C   ARG A  39      21.075  23.400  14.859  1.00 50.12           C  \\nANISOU  273  C   ARG A  39     4452   8782   5808  -2649    400    137       C  \\nATOM    274  O   ARG A  39      21.784  24.291  14.399  1.00 56.26           O  \\nANISOU  274  O   ARG A  39     5279   9571   6527  -2916    459    122       O  \\nATOM    275  CB  ARG A  39      21.852  21.907  16.634  1.00 58.92           C  \\nANISOU  275  CB  ARG A  39     5135  10309   6943  -2294    205     25       C  \\nATOM    276  CG  ARG A  39      22.070  21.459  18.057  1.00 63.00           C  \\nANISOU  276  CG  ARG A  39     5516  10972   7448  -2175     39    -30       C  \\nATOM    277  CD  ARG A  39      22.489  19.995  17.988  1.00 61.50           C  \\nANISOU  277  CD  ARG A  39     5077  11000   7290  -1803      9    -17       C  \\nATOM    278  NE  ARG A  39      23.185  19.739  16.725  1.00 69.30           N  \\nANISOU  278  NE  ARG A  39     5940  12102   8289  -1777    152    -17       N  \\nATOM    279  CZ  ARG A  39      23.172  18.585  16.058  1.00 66.22           C  \\nANISOU  279  CZ  ARG A  39     5452  11768   7941  -1476    213      3       C  \\nATOM    280  NH1 ARG A  39      22.499  17.541  16.523  1.00 73.56           N  \\nANISOU  280  NH1 ARG A  39     6400  12638   8912  -1162    136     41       N  \\nATOM    281  NH2 ARG A  39      23.844  18.470  14.921  1.00 52.22           N  \\nANISOU  281  NH2 ARG A  39     3578  10103   6160  -1498    356    -20       N  \\nATOM    282  N   LYS A  40      20.366  22.573  14.093  1.00 45.55           N  \\nANISOU  282  N   LYS A  40     3899   8148   5261  -2427    458    212       N  \\nATOM    283  CA  LYS A  40      20.464  22.596  12.642  1.00 51.13           C  \\nANISOU  283  CA  LYS A  40     4676   8835   5919  -2476    592    274       C  \\nATOM    284  C   LYS A  40      19.550  23.640  12.013  1.00 48.72           C  \\nANISOU  284  C   LYS A  40     4741   8171   5600  -2673    619    384       C  \\nATOM    285  O   LYS A  40      19.806  24.102  10.905  1.00 48.77           O  \\nANISOU  285  O   LYS A  40     4864   8136   5530  -2825    710    437       O  \\nATOM    286  CB  LYS A  40      20.132  21.220  12.059  1.00 49.30           C  \\nANISOU  286  CB  LYS A  40     4329   8699   5704  -2155    640    297       C  \\nATOM    287  CG  LYS A  40      21.182  20.143  12.286  1.00 49.00           C  \\nANISOU  287  CG  LYS A  40     3954   8989   5676  -1933    642    200       C  \\nATOM    288  CD  LYS A  40      21.095  19.124  11.165  1.00 60.33           C  \\nANISOU  288  CD  LYS A  40     5350  10481   7091  -1725    759    211       C  \\nATOM    289  CE  LYS A  40      21.798  17.832  11.496  1.00 64.36           C  \\nANISOU  289  CE  LYS A  40     5581  11225   7649  -1404    741    127       C  \\nATOM    290  NZ  LYS A  40      21.552  16.840  10.415  1.00 65.36           N  \\nANISOU  290  NZ  LYS A  40     5725  11353   7754  -1201    864    124       N  \\nATOM    291  N   GLY A  41      18.471  23.986  12.712  1.00 43.32           N  \\nANISOU  291  N   GLY A  41     4251   7220   4988  -2656    535    421       N  \\nATOM    292  CA  GLY A  41      17.512  24.946  12.201  1.00 42.32           C  \\nANISOU  292  CA  GLY A  41     4509   6690   4881  -2751    526    527       C  \\nATOM    293  C   GLY A  41      17.847  26.336  12.683  1.00 45.01           C  \\nANISOU  293  C   GLY A  41     5025   6864   5210  -3068    505    491       C  \\nATOM    294  O   GLY A  41      17.571  27.333  12.007  1.00 43.53           O  \\nANISOU  294  O   GLY A  41     5139   6398   5002  -3226    522    577       O  \\nATOM    295  N   GLY A  42      18.443  26.391  13.871  1.00 42.81           N  \\nANISOU  295  N   GLY A  42     4585   6740   4940  -3119    449    359       N  \\nATOM    296  CA  GLY A  42      18.941  27.632  14.435  1.00 45.37           C  \\nANISOU  296  CA  GLY A  42     5056   6947   5235  -3399    424    286       C  \\nATOM    297  C   GLY A  42      17.963  28.308  15.367  1.00 44.74           C  \\nANISOU  297  C   GLY A  42     5250   6514   5236  -3420    353    250       C  \\nATOM    298  O   GLY A  42      16.841  27.844  15.531  1.00 42.31           O  \\nANISOU  298  O   GLY A  42     5035   6025   5017  -3127    316    283       O  \\nATOM    299  N   PRO A  43      18.385  29.434  15.958  1.00 47.27           N  \\nANISOU  299  N   PRO A  43     5729   6707   5525  -3663    332    162       N  \\nATOM    300  CA  PRO A  43      17.715  30.149  17.053  1.00 51.32           C  \\nANISOU  300  CA  PRO A  43     6489   6918   6093  -3708    276     64       C  \\nATOM    301  C   PRO A  43      16.321  30.666  16.707  1.00 53.24           C  \\nANISOU  301  C   PRO A  43     7105   6667   6455  -3577    281    148       C  \\nATOM    302  O   PRO A  43      15.496  30.813  17.609  1.00 45.84           O  \\nANISOU  302  O   PRO A  43     6306   5516   5595  -3389    242     56       O  \\nATOM    303  CB  PRO A  43      18.646  31.336  17.317  1.00 58.07           C  \\nANISOU  303  CB  PRO A  43     7456   7743   6863  -4023    281    -18       C  \\nATOM    304  CG  PRO A  43      19.958  30.961  16.704  1.00 52.61           C  \\nANISOU  304  CG  PRO A  43     6456   7465   6069  -4130    320      2       C  \\nATOM    305  CD  PRO A  43      19.636  30.096  15.545  1.00 50.54           C  \\nANISOU  305  CD  PRO A  43     6091   7280   5833  -3930    377    140       C  \\nATOM    306  N   SER A  44      16.068  30.948  15.429  1.00 49.10           N  \\nANISOU  306  N   SER A  44     6738   5979   5941  -3580    319    312       N  \\nATOM    307  CA  SER A  44      14.796  31.547  15.020  1.00 48.91           C  \\nANISOU  307  CA  SER A  44     7073   5477   6031  -3409    291    411       C  \\nATOM    308  C   SER A  44      13.615  30.639  15.341  1.00 43.62           C  \\nANISOU  308  C   SER A  44     6316   4779   5478  -2939    244    405       C  \\nATOM    309  O   SER A  44      12.518  31.112  15.625  1.00 51.16           O  \\nANISOU  309  O   SER A  44     7496   5384   6559  -2736    210    394       O  \\nATOM    310  CB  SER A  44      14.806  31.929  13.530  1.00 51.34           C  \\nANISOU  310  CB  SER A  44     7544   5670   6295  -3474    313    607       C  \\nATOM    311  OG  SER A  44      14.619  30.793  12.696  1.00 48.63           O  \\nANISOU  311  OG  SER A  44     7004   5551   5922  -3309    327    720       O  \\nATOM    312  N   ILE A  45      13.847  29.331  15.314  1.00 46.48           N  \\nANISOU  312  N   ILE A  45     6348   5511   5802  -2769    247    404       N  \\nATOM    313  CA  ILE A  45      12.802  28.361  15.651  1.00 38.55           C  \\nANISOU  313  CA  ILE A  45     5248   4512   4889  -2368    210    395       C  \\nATOM    314  C   ILE A  45      12.299  28.508  17.099  1.00 39.87           C  \\nANISOU  314  C   ILE A  45     5454   4580   5115  -2258    192    231       C  \\nATOM    315  O   ILE A  45      11.090  28.620  17.334  1.00 37.61           O  \\nANISOU  315  O   ILE A  45     5299   4041   4949  -2011    180    221       O  \\nATOM    316  CB  ILE A  45      13.273  26.913  15.383  1.00 41.19           C  \\nANISOU  316  CB  ILE A  45     5249   5244   5157  -2235    222    417       C  \\nATOM    317  CG1 ILE A  45      13.634  26.744  13.899  1.00 43.60           C  \\nANISOU  317  CG1 ILE A  45     5539   5634   5394  -2321    265    562       C  \\nATOM    318  CG2 ILE A  45      12.206  25.917  15.809  1.00 35.59           C  \\nANISOU  318  CG2 ILE A  45     4469   4524   4531  -1867    188    404       C  \\nATOM    319  CD1 ILE A  45      13.924  25.320  13.480  1.00 35.27           C  \\nANISOU  319  CD1 ILE A  45     4207   4904   4291  -2143    290    578       C  \\nATOM    320  N   GLN A  46      13.228  28.515  18.056  1.00 40.44           N  \\nANISOU  320  N   GLN A  46     5402   4870   5092  -2448    190     98       N  \\nATOM    321  CA  GLN A  46      12.897  28.776  19.458  1.00 39.58           C  \\nANISOU  321  CA  GLN A  46     5367   4680   4991  -2416    179    -72       C  \\nATOM    322  C   GLN A  46      12.301  30.165  19.669  1.00 41.70           C  \\nANISOU  322  C   GLN A  46     5995   4508   5340  -2500    206   -137       C  \\nATOM    323  O   GLN A  46      11.381  30.338  20.462  1.00 41.57           O  \\nANISOU  323  O   GLN A  46     6099   4297   5397  -2316    226   -243       O  \\nATOM    324  CB  GLN A  46      14.124  28.646  20.356  1.00 40.78           C  \\nANISOU  324  CB  GLN A  46     5339   5159   4997  -2659    145   -191       C  \\nATOM    325  CG  GLN A  46      13.778  28.810  21.827  1.00 41.25           C  \\nANISOU  325  CG  GLN A  46     5485   5167   5022  -2631    129   -367       C  \\nATOM    326  CD  GLN A  46      12.845  27.701  22.311  1.00 38.74           C  \\nANISOU  326  CD  GLN A  46     5070   4909   4739  -2265    128   -361       C  \\nATOM    327  OE1 GLN A  46      13.262  26.552  22.449  1.00 38.05           O  \\nANISOU  327  OE1 GLN A  46     4731   5141   4584  -2160     80   -316       O  \\nATOM    328  NE2 GLN A  46      11.574  28.035  22.533  1.00 38.33           N  \\nANISOU  328  NE2 GLN A  46     5218   4547   4801  -2068    187   -402       N  \\nATOM    329  N   GLU A  47      12.834  31.157  18.965  1.00 43.97           N  \\nANISOU  329  N   GLU A  47     6465   4630   5610  -2782    216    -79       N  \\nATOM    330  CA  GLU A  47      12.320  32.519  19.097  1.00 51.53           C  \\nANISOU  330  CA  GLU A  47     7808   5119   6650  -2863    236   -130       C  \\nATOM    331  C   GLU A  47      10.848  32.574  18.711  1.00 51.23           C  \\nANISOU  331  C   GLU A  47     7917   4756   6792  -2477    230    -56       C  \\nATOM    332  O   GLU A  47      10.036  33.180  19.409  1.00 49.29           O  \\nANISOU  332  O   GLU A  47     7867   4210   6650  -2335    255   -179       O  \\nATOM    333  CB  GLU A  47      13.126  33.495  18.246  1.00 49.20           C  \\nANISOU  333  CB  GLU A  47     7708   4688   6297  -3241    245    -38       C  \\nATOM    334  CG  GLU A  47      14.516  33.761  18.777  1.00 68.48           C  \\nANISOU  334  CG  GLU A  47    10047   7391   8581  -3678    255   -152       C  \\nATOM    335  CD  GLU A  47      15.304  34.710  17.893  1.00 73.98           C  \\nANISOU  335  CD  GLU A  47    10867   8033   9211  -3907    271    -53       C  \\nATOM    336  OE1 GLU A  47      16.277  35.314  18.397  1.00 81.43           O  \\nANISOU  336  OE1 GLU A  47    11796   9098  10046  -4175    275   -162       O  \\nATOM    337  OE2 GLU A  47      14.955  34.844  16.696  1.00 69.41           O  \\nANISOU  337  OE2 GLU A  47    10402   7299   8671  -3830    277    133       O  \\nATOM    338  N   GLU A  48      10.511  31.921  17.606  1.00 43.82           N  \\nANISOU  338  N   GLU A  48     6867   3898   5886  -2306    196    133       N  \\nATOM    339  CA  GLU A  48       9.131  31.871  17.147  1.00 47.31           C  \\nANISOU  339  CA  GLU A  48     7391   4093   6493  -1944    163    219       C  \\nATOM    340  C   GLU A  48       8.253  31.097  18.136  1.00 43.00           C  \\nANISOU  340  C   GLU A  48     6672   3647   6020  -1637    190     87       C  \\nATOM    341  O   GLU A  48       7.130  31.502  18.422  1.00 42.15           O  \\nANISOU  341  O   GLU A  48     6681   3268   6065  -1391    201     35       O  \\nATOM    342  CB  GLU A  48       9.056  31.261  15.743  1.00 46.56           C  \\nANISOU  342  CB  GLU A  48     7204   4111   6374  -1875    110    443       C  \\nATOM    343  CG  GLU A  48       7.690  31.366  15.087  1.00 47.94           C  \\nANISOU  343  CG  GLU A  48     7486   4023   6708  -1548     37    562       C  \\nATOM    344  CD  GLU A  48       7.725  31.076  13.591  1.00 49.55           C  \\nANISOU  344  CD  GLU A  48     7700   4280   6849  -1567    -31    794       C  \\nATOM    345  OE1 GLU A  48       8.826  31.079  12.995  1.00 51.62           O  \\nANISOU  345  OE1 GLU A  48     7958   4705   6950  -1867      1    863       O  \\nATOM    346  OE2 GLU A  48       6.640  30.851  13.013  1.00 44.98           O  \\nANISOU  346  OE2 GLU A  48     7124   3594   6373  -1290   -116    899       O  \\nATOM    347  N   CYS A  49       8.776  29.994  18.665  1.00 45.54           N  \\nANISOU  347  N   CYS A  49     6717   4357   6231  -1652    206     34       N  \\nATOM    348  CA  CYS A  49       8.096  29.251  19.731  1.00 41.42           C  \\nANISOU  348  CA  CYS A  49     6057   3949   5732  -1431    244    -94       C  \\nATOM    349  C   CYS A  49       7.775  30.123  20.942  1.00 45.18           C  \\nANISOU  349  C   CYS A  49     6720   4210   6236  -1457    309   -304       C  \\nATOM    350  O   CYS A  49       6.733  29.955  21.578  1.00 44.88           O  \\nANISOU  350  O   CYS A  49     6673   4094   6287  -1217    365   -400       O  \\nATOM    351  CB  CYS A  49       8.940  28.059  20.183  1.00 35.91           C  \\nANISOU  351  CB  CYS A  49     5088   3675   4880  -1496    233   -111       C  \\nATOM    352  SG  CYS A  49       8.793  26.592  19.138  1.00 33.27           S  \\nANISOU  352  SG  CYS A  49     4506   3593   4543  -1304    193     68       S  \\nATOM    353  N   ASP A  50       8.691  31.033  21.266  1.00 47.28           N  \\nANISOU  353  N   ASP A  50     7150   4398   6415  -1769    313   -388       N  \\nATOM    354  CA  ASP A  50       8.512  31.947  22.389  1.00 51.60           C  \\nANISOU  354  CA  ASP A  50     7920   4722   6962  -1843    380   -609       C  \\nATOM    355  C   ASP A  50       7.377  32.922  22.101  1.00 53.58           C  \\nANISOU  355  C   ASP A  50     8433   4505   7419  -1628    418   -624       C  \\nATOM    356  O   ASP A  50       6.638  33.298  23.008  1.00 54.89           O  \\nANISOU  356  O   ASP A  50     8708   4498   7650  -1486    505   -812       O  \\nATOM    357  CB  ASP A  50       9.804  32.718  22.693  1.00 52.79           C  \\nANISOU  357  CB  ASP A  50     8198   4894   6966  -2270    361   -690       C  \\nATOM    358  CG  ASP A  50      10.939  31.814  23.160  1.00 53.93           C  \\nANISOU  358  CG  ASP A  50     8056   5522   6914  -2462    309   -704       C  \\nATOM    359  OD1 ASP A  50      10.671  30.666  23.578  1.00 57.14           O  \\nANISOU  359  OD1 ASP A  50     8229   6197   7283  -2259    301   -697       O  \\nATOM    360  OD2 ASP A  50      12.103  32.260  23.113  1.00 53.53           O  \\nANISOU  360  OD2 ASP A  50     8012   5579   6747  -2820    272   -720       O  \\nATOM    361  N   ARG A  51       7.241  33.325  20.838  1.00 49.94           N  \\nANISOU  361  N   ARG A  51     8075   3845   7056  -1596    352   -428       N  \\nATOM    362  CA  ARG A  51       6.148  34.208  20.433  1.00 57.18           C  \\nANISOU  362  CA  ARG A  51     9228   4314   8182  -1347    350   -402       C  \\nATOM    363  C   ARG A  51       4.802  33.504  20.642  1.00 55.84           C  \\nANISOU  363  C   ARG A  51     8859   4198   8161   -929    379   -427       C  \\nATOM    364  O   ARG A  51       3.843  34.108  21.124  1.00 55.19           O  \\nANISOU  364  O   ARG A  51     8895   3836   8237   -695    443   -558       O  \\nATOM    365  CB  ARG A  51       6.317  34.699  18.982  1.00 49.38           C  \\nANISOU  365  CB  ARG A  51     8397   3133   7234  -1412    245   -150       C  \\nATOM    366  CG  ARG A  51       7.612  35.499  18.738  1.00 60.48           C  \\nANISOU  366  CG  ARG A  51    10022   4463   8495  -1866    237   -124       C  \\nATOM    367  CD  ARG A  51       7.581  36.360  17.462  1.00 65.05           C  \\nANISOU  367  CD  ARG A  51    10895   4697   9125  -1932    156    100       C  \\nATOM    368  NE  ARG A  51       7.471  35.597  16.214  1.00 70.83           N  \\nANISOU  368  NE  ARG A  51    11456   5627   9830  -1840     73    350       N  \\nATOM    369  CZ  ARG A  51       8.485  35.328  15.388  1.00 72.65           C  \\nANISOU  369  CZ  ARG A  51    11626   6088   9889  -2144     59    495       C  \\nATOM    370  NH1 ARG A  51       9.718  35.741  15.671  1.00 73.52           N  \\nANISOU  370  NH1 ARG A  51    11795   6289   9850  -2570    118    423       N  \\nATOM    371  NH2 ARG A  51       8.266  34.639  14.273  1.00 64.16           N  \\nANISOU  371  NH2 ARG A  51    10421   5170   8785  -2033     -6    700       N  \\nATOM    372  N   ILE A  52       4.747  32.221  20.304  1.00 49.52           N  \\nANISOU  372  N   ILE A  52     7750   3758   7308   -845    344   -316       N  \\nATOM    373  CA  ILE A  52       3.553  31.408  20.543  1.00 49.58           C  \\nANISOU  373  CA  ILE A  52     7537   3876   7426   -512    378   -343       C  \\nATOM    374  C   ILE A  52       3.255  31.252  22.038  1.00 47.18           C  \\nANISOU  374  C   ILE A  52     7189   3653   7083   -474    519   -597       C  \\nATOM    375  O   ILE A  52       2.124  31.440  22.475  1.00 51.34           O  \\nANISOU  375  O   ILE A  52     7702   4045   7759   -214    602   -711       O  \\nATOM    376  CB  ILE A  52       3.687  30.003  19.906  1.00 43.73           C  \\nANISOU  376  CB  ILE A  52     6509   3503   6604   -485    315   -182       C  \\nATOM    377  CG1 ILE A  52       3.881  30.111  18.395  1.00 44.61           C  \\nANISOU  377  CG1 ILE A  52     6668   3550   6732   -515    189     59       C  \\nATOM    378  CG2 ILE A  52       2.464  29.148  20.208  1.00 42.53           C  \\nANISOU  378  CG2 ILE A  52     6138   3468   6553   -194    357   -218       C  \\nATOM    379  CD1 ILE A  52       3.995  28.764  17.715  1.00 48.85           C  \\nANISOU  379  CD1 ILE A  52     6955   4419   7187   -486    138    196       C  \\nATOM    380  N   GLY A  53       4.274  30.909  22.820  1.00 48.34           N  \\nANISOU  380  N   GLY A  53     7307   4038   7021   -736    547   -687       N  \\nATOM    381  CA  GLY A  53       4.095  30.717  24.250  1.00 43.61           C  \\nANISOU  381  CA  GLY A  53     6694   3546   6331   -744    668   -914       C  \\nATOM    382  C   GLY A  53       3.538  29.346  24.590  1.00 48.09           C  \\nANISOU  382  C   GLY A  53     6987   4427   6857   -586    703   -890       C  \\nATOM    383  O   GLY A  53       3.363  28.502  23.720  1.00 48.39           O  \\nANISOU  383  O   GLY A  53     6844   4607   6935   -484    626   -705       O  \\nATOM    384  N   LYS A  54       3.247  29.138  25.868  1.00 47.40           N  \\nANISOU  384  N   LYS A  54     6900   4436   6675   -585    825  -1082       N  \\nATOM    385  CA  LYS A  54       2.770  27.863  26.374  1.00 48.16           C  \\nANISOU  385  CA  LYS A  54     6786   4819   6694   -488    875  -1072       C  \\nATOM    386  C   LYS A  54       1.491  27.377  25.697  1.00 51.58           C  \\nANISOU  386  C   LYS A  54     7045   5225   7329   -199    899   -982       C  \\nATOM    387  O   LYS A  54       0.503  28.106  25.615  1.00 54.30           O  \\nANISOU  387  O   LYS A  54     7429   5335   7869     -2    975  -1067       O  \\nATOM    388  CB  LYS A  54       2.534  27.963  27.887  1.00 47.76           C  \\nANISOU  388  CB  LYS A  54     6823   4820   6505   -545   1029  -1315       C  \\nATOM    389  CG  LYS A  54       1.956  26.700  28.514  1.00 46.23           C  \\nANISOU  389  CG  LYS A  54     6455   4898   6212   -469   1104  -1310       C  \\nATOM    390  CD  LYS A  54       1.833  26.847  30.030  1.00 51.25           C  \\nANISOU  390  CD  LYS A  54     7216   5597   6660   -572   1259  -1547       C  \\nATOM    391  CE  LYS A  54       1.205  25.604  30.662  1.00 51.20           C  \\nANISOU  391  CE  LYS A  54     7068   5846   6539   -521   1349  -1529       C  \\nATOM    392  NZ  LYS A  54       1.075  25.694  32.154  1.00 56.00           N  \\nANISOU  392  NZ  LYS A  54     7818   6539   6922   -644   1510  -1750       N  \\nATOM    393  N   ILE A  55       1.509  26.137  25.218  1.00 43.06           N  \\nANISOU  393  N   ILE A  55     5769   4386   6205   -171    830   -817       N  \\nATOM    394  CA  ILE A  55       0.276  25.508  24.778  1.00 41.20           C  \\nANISOU  394  CA  ILE A  55     5350   4184   6120     55    860   -758       C  \\nATOM    395  C   ILE A  55      -0.166  24.369  25.700  1.00 41.50           C  \\nANISOU  395  C   ILE A  55     5263   4469   6035     51    968   -814       C  \\nATOM    396  O   ILE A  55       0.263  24.250  26.848  1.00 41.71           O  \\nANISOU  396  O   ILE A  55     5374   4601   5874    -87   1045   -935       O  \\nATOM    397  CB  ILE A  55       0.339  25.021  23.317  1.00 41.47           C  \\nANISOU  397  CB  ILE A  55     5278   4246   6233    112    703   -524       C  \\nATOM    398  CG1 ILE A  55       1.358  23.883  23.167  1.00 38.47           C  \\nANISOU  398  CG1 ILE A  55     4830   4126   5660    -40    623   -402       C  \\nATOM    399  CG2 ILE A  55       0.616  26.206  22.375  1.00 41.36           C  \\nANISOU  399  CG2 ILE A  55     5415   3962   6337    118    606   -453       C  \\nATOM    400  CD1 ILE A  55       1.440  23.306  21.779  1.00 33.93           C  \\nANISOU  400  CD1 ILE A  55     4163   3597   5131      2    495   -200       C  \\nATOM    401  N   ARG A  56      -1.050  23.540  25.180  1.00 37.24           N  \\nANISOU  401  N   ARG A  56     4538   4020   5592    185    966   -722       N  \\nATOM    402  CA  ARG A  56      -1.658  22.497  25.966  1.00 38.87           C  \\nANISOU  402  CA  ARG A  56     4636   4427   5705    176   1083   -768       C  \\nATOM    403  C   ARG A  56      -1.095  21.142  25.521  1.00 33.32           C  \\nANISOU  403  C   ARG A  56     3869   3919   4874     93    968   -582       C  \\nATOM    404  O   ARG A  56      -0.866  20.930  24.336  1.00 31.26           O  \\nANISOU  404  O   ARG A  56     3555   3636   4687    132    830   -427       O  \\nATOM    405  CB  ARG A  56      -3.151  22.591  25.706  1.00 50.72           C  \\nANISOU  405  CB  ARG A  56     5962   5881   7428    377   1174   -818       C  \\nATOM    406  CG  ARG A  56      -3.988  21.721  26.521  1.00 57.43           C  \\nANISOU  406  CG  ARG A  56     6691   6917   8214    359   1337   -895       C  \\nATOM    407  CD  ARG A  56      -4.408  22.357  27.837  1.00 59.88           C  \\nANISOU  407  CD  ARG A  56     7070   7205   8476    352   1563  -1145       C  \\nATOM    408  NE  ARG A  56      -5.508  21.532  28.298  1.00 52.45           N  \\nANISOU  408  NE  ARG A  56     5948   6446   7536    363   1726  -1196       N  \\nATOM    409  CZ  ARG A  56      -5.349  20.320  28.807  1.00 38.82           C  \\nANISOU  409  CZ  ARG A  56     4234   4921   5596    192   1757  -1126       C  \\nATOM    410  NH1 ARG A  56      -4.129  19.818  28.998  1.00 35.12           N  \\nANISOU  410  NH1 ARG A  56     3940   4503   4901     33   1633  -1015       N  \\nATOM    411  NH2 ARG A  56      -6.416  19.627  29.155  1.00 40.95           N  \\nANISOU  411  NH2 ARG A  56     4341   5341   5877    177   1915  -1171       N  \\nATOM    412  N   VAL A  57      -0.821  20.240  26.457  1.00 42.67           N  \\nANISOU  412  N   VAL A  57     5082   5278   5851    -21   1021   -596       N  \\nATOM    413  CA  VAL A  57      -0.412  18.886  26.070  1.00 36.22           C  \\nANISOU  413  CA  VAL A  57     4219   4612   4932    -62    923   -427       C  \\nATOM    414  C   VAL A  57      -1.515  18.305  25.192  1.00 38.37           C  \\nANISOU  414  C   VAL A  57     4333   4880   5366     48    925   -351       C  \\nATOM    415  O   VAL A  57      -2.688  18.377  25.563  1.00 34.99           O  \\nANISOU  415  O   VAL A  57     3812   4455   5028    102   1061   -447       O  \\nATOM    416  CB  VAL A  57      -0.194  17.955  27.290  1.00 35.42           C  \\nANISOU  416  CB  VAL A  57     4195   4673   4591   -179    985   -444       C  \\nATOM    417  CG1 VAL A  57      -0.117  16.502  26.828  1.00 30.76           C  \\nANISOU  417  CG1 VAL A  57     3564   4181   3943   -178    908   -275       C  \\nATOM    418  CG2 VAL A  57       1.075  18.349  28.081  1.00 29.06           C  \\nANISOU  418  CG2 VAL A  57     3533   3919   3590   -305    923   -486       C  \\nATOM    419  N   GLY A  58      -1.145  17.755  24.030  1.00 33.16           N  \\nANISOU  419  N   GLY A  58     3632   4229   4740     72    782   -193       N  \\nATOM    420  CA  GLY A  58      -2.112  17.248  23.067  1.00 25.78           C  \\nANISOU  420  CA  GLY A  58     2559   3293   3945    152    750   -118       C  \\nATOM    421  C   GLY A  58      -2.357  18.198  21.891  1.00 37.99           C  \\nANISOU  421  C   GLY A  58     4055   4701   5678    263    649    -75       C  \\nATOM    422  O   GLY A  58      -2.958  17.823  20.882  1.00 37.19           O  \\nANISOU  422  O   GLY A  58     3854   4606   5673    320    569     14       O  \\nATOM    423  N   GLU A  59      -1.901  19.441  22.017  1.00 34.12           N  \\nANISOU  423  N   GLU A  59     3657   4077   5230    282    645   -134       N  \\nATOM    424  CA  GLU A  59      -2.021  20.414  20.933  1.00 30.53           C  \\nANISOU  424  CA  GLU A  59     3214   3456   4932    376    537    -72       C  \\nATOM    425  C   GLU A  59      -0.754  20.417  20.074  1.00 29.62           C  \\nANISOU  425  C   GLU A  59     3201   3337   4717    277    409     57       C  \\nATOM    426  O   GLU A  59       0.234  19.766  20.420  1.00 28.43           O  \\nANISOU  426  O   GLU A  59     3089   3313   4399    161    410     74       O  \\nATOM    427  CB  GLU A  59      -2.304  21.824  21.495  1.00 29.56           C  \\nANISOU  427  CB  GLU A  59     3166   3142   4923    452    612   -211       C  \\nATOM    428  CG  GLU A  59      -3.789  22.105  21.671  1.00 38.06           C  \\nANISOU  428  CG  GLU A  59     4088   4175   6197    637    694   -304       C  \\nATOM    429  CD  GLU A  59      -4.089  23.266  22.612  1.00 56.76           C  \\nANISOU  429  CD  GLU A  59     6534   6383   8647    717    834   -501       C  \\nATOM    430  OE1 GLU A  59      -3.182  24.082  22.906  1.00 49.79           O  \\nANISOU  430  OE1 GLU A  59     5859   5362   7696    630    831   -549       O  \\nATOM    431  OE2 GLU A  59      -5.247  23.348  23.080  1.00 66.75           O  \\nANISOU  431  OE2 GLU A  59     7648   7670  10044    856    959   -625       O  \\nATOM    432  N   ALA A  60      -0.800  21.147  18.960  1.00 32.03           N  \\nANISOU  432  N   ALA A  60     3545   3505   5122    327    300    151       N  \\nATOM    433  CA  ALA A  60       0.342  21.296  18.063  1.00 29.83           C  \\nANISOU  433  CA  ALA A  60     3364   3220   4750    215    202    267       C  \\nATOM    434  C   ALA A  60       0.463  22.758  17.669  1.00 36.60           C  \\nANISOU  434  C   ALA A  60     4366   3842   5698    220    157    282       C  \\nATOM    435  O   ALA A  60      -0.528  23.478  17.662  1.00 35.04           O  \\nANISOU  435  O   ALA A  60     4168   3477   5667    371    151    252       O  \\nATOM    436  CB  ALA A  60       0.167  20.446  16.829  1.00 32.14           C  \\nANISOU  436  CB  ALA A  60     3592   3599   5022    232    102    409       C  \\nATOM    437  N   ALA A  61       1.675  23.198  17.349  1.00 32.60           N  \\nANISOU  437  N   ALA A  61     3981   3319   5086     56    126    328       N  \\nATOM    438  CA  ALA A  61       1.886  24.566  16.884  1.00 38.93           C  \\nANISOU  438  CA  ALA A  61     4966   3875   5950     15     79    365       C  \\nATOM    439  C   ALA A  61       2.905  24.595  15.759  1.00 35.48           C  \\nANISOU  439  C   ALA A  61     4603   3483   5394   -154      5    512       C  \\nATOM    440  O   ALA A  61       3.850  23.809  15.738  1.00 39.24           O  \\nANISOU  440  O   ALA A  61     5002   4184   5725   -279     28    520       O  \\nATOM    441  CB  ALA A  61       2.339  25.459  18.022  1.00 38.92           C  \\nANISOU  441  CB  ALA A  61     5088   3761   5939    -77    169    208       C  \\nATOM    442  N   VAL A  62       2.715  25.517  14.829  1.00 41.66           N  \\nANISOU  442  N   VAL A  62     5542   4050   6237   -151    -82    627       N  \\nATOM    443  CA  VAL A  62       3.568  25.576  13.659  1.00 40.86           C  \\nANISOU  443  CA  VAL A  62     5527   3991   6007   -326   -140    776       C  \\nATOM    444  C   VAL A  62       4.492  26.794  13.704  1.00 33.29           C  \\nANISOU  444  C   VAL A  62     4782   2864   5002   -545   -117    776       C  \\nATOM    445  O   VAL A  62       4.062  27.903  14.024  1.00 35.39           O  \\nANISOU  445  O   VAL A  62     5223   2838   5386   -498   -129    744       O  \\nATOM    446  CB  VAL A  62       2.721  25.584  12.368  1.00 46.08           C  \\nANISOU  446  CB  VAL A  62     6231   4561   6715   -206   -276    949       C  \\nATOM    447  CG1 VAL A  62       3.605  25.371  11.142  1.00 41.48           C  \\nANISOU  447  CG1 VAL A  62     5720   4089   5950   -401   -310   1092       C  \\nATOM    448  CG2 VAL A  62       1.643  24.506  12.448  1.00 42.23           C  \\nANISOU  448  CG2 VAL A  62     5533   4214   6299     -1   -302    925       C  \\nATOM    449  N   THR A  63       5.771  26.563  13.414  1.00 33.38           N  \\nANISOU  449  N   THR A  63     4773   3064   4846   -789    -75    798       N  \\nATOM    450  CA  THR A  63       6.727  27.642  13.195  1.00 34.64           C  \\nANISOU  450  CA  THR A  63     5126   3102   4933  -1062    -56    827       C  \\nATOM    451  C   THR A  63       7.437  27.432  11.860  1.00 44.64           C  \\nANISOU  451  C   THR A  63     6416   4497   6048  -1238    -72    984       C  \\nATOM    452  O   THR A  63       7.304  26.372  11.239  1.00 45.21           O  \\nANISOU  452  O   THR A  63     6339   4776   6062  -1147    -85   1038       O  \\nATOM    453  CB  THR A  63       7.799  27.702  14.313  1.00 43.86           C  \\nANISOU  453  CB  THR A  63     6218   4424   6023  -1260     36    663       C  \\nATOM    454  OG1 THR A  63       8.663  26.556  14.234  1.00 33.04           O  \\nANISOU  454  OG1 THR A  63     4603   3428   4522  -1324     74    652       O  \\nATOM    455  CG2 THR A  63       7.147  27.778  15.674  1.00 34.42           C  \\nANISOU  455  CG2 THR A  63     5001   3147   4931  -1106     69    493       C  \\nATOM    456  N   GLY A  64       8.198  28.438  11.431  1.00 40.92           N  \\nANISOU  456  N   GLY A  64     6146   3902   5501  -1511    -56   1047       N  \\nATOM    457  CA  GLY A  64       9.088  28.298  10.292  1.00 41.01           C  \\nANISOU  457  CA  GLY A  64     6171   4078   5333  -1748    -24   1167       C  \\nATOM    458  C   GLY A  64      10.221  27.369  10.683  1.00 46.30           C  \\nANISOU  458  C   GLY A  64     6549   5151   5893  -1863     85   1050       C  \\nATOM    459  O   GLY A  64      10.458  27.148  11.867  1.00 41.65           O  \\nANISOU  459  O   GLY A  64     5816   4660   5350  -1828    116    896       O  \\nATOM    460  N   ALA A  65      10.928  26.819   9.702  1.00 36.66           N  \\nANISOU  460  N   ALA A  65     5240   4170   4520  -1990    141   1119       N  \\nATOM    461  CA  ALA A  65      11.937  25.816  10.018  1.00 36.30           C  \\nANISOU  461  CA  ALA A  65     4884   4516   4394  -2027    237   1006       C  \\nATOM    462  C   ALA A  65      13.369  26.319   9.980  1.00 37.74           C  \\nANISOU  462  C   ALA A  65     4997   4886   4456  -2382    340    966       C  \\nATOM    463  O   ALA A  65      14.294  25.516   9.914  1.00 43.66           O  \\nANISOU  463  O   ALA A  65     5477   5987   5124  -2421    424    899       O  \\nATOM    464  CB  ALA A  65      11.782  24.605   9.124  1.00 36.69           C  \\nANISOU  464  CB  ALA A  65     4810   4762   4371  -1874    255   1052       C  \\nATOM    465  N   GLY A  66      13.555  27.636  10.041  1.00 40.04           N  \\nANISOU  465  N   GLY A  66     5524   4945   4744  -2639    335   1000       N  \\nATOM    466  CA  GLY A  66      14.893  28.205  10.052  1.00 42.45           C  \\nANISOU  466  CA  GLY A  66     5769   5424   4937  -3031    433    956       C  \\nATOM    467  C   GLY A  66      15.742  27.733   8.883  1.00 62.93           C  \\nANISOU  467  C   GLY A  66     8224   8313   7373  -3123    537    992       C  \\nATOM    468  O   GLY A  66      15.320  27.809   7.723  1.00 43.94           O  \\nANISOU  468  O   GLY A  66     6003   5795   4898  -3103    533   1127       O  \\nATOM    469  N   ASN A  67      16.931  27.224   9.195  1.00 43.99           N  \\nANISOU  469  N   ASN A  67     5497   6294   4922  -3188    618    856       N  \\nATOM    470  CA  ASN A  67      17.875  26.772   8.172  1.00 45.42           C  \\nANISOU  470  CA  ASN A  67     5517   6773   4969  -3251    734    842       C  \\nATOM    471  C   ASN A  67      17.674  25.312   7.749  1.00 54.16           C  \\nANISOU  471  C   ASN A  67     6411   8106   6060  -2993    779    835       C  \\nATOM    472  O   ASN A  67      18.470  24.770   6.981  1.00 51.71           O  \\nANISOU  472  O   ASN A  67     5938   8058   5651  -3002    890    790       O  \\nATOM    473  CB  ASN A  67      19.320  26.988   8.633  1.00 47.77           C  \\nANISOU  473  CB  ASN A  67     5563   7363   5224  -3434    795    698       C  \\nATOM    474  CG  ASN A  67      19.676  28.449   8.750  1.00 58.96           C  \\nANISOU  474  CG  ASN A  67     7212   8582   6607  -3740    780    707       C  \\nATOM    475  OD1 ASN A  67      19.367  29.252   7.861  1.00 52.23           O  \\nANISOU  475  OD1 ASN A  67     6667   7489   5691  -3872    790    827       O  \\nATOM    476  ND2 ASN A  67      20.322  28.813   9.856  1.00 54.91           N  \\nANISOU  476  ND2 ASN A  67     6570   8164   6128  -3854    746    583       N  \\nATOM    477  N   LEU A  68      16.617  24.677   8.253  1.00 46.62           N  \\nANISOU  477  N   LEU A  68     5467   7046   5202  -2765    702    871       N  \\nATOM    478  CA  LEU A  68      16.264  23.330   7.803  1.00 44.93           C  \\nANISOU  478  CA  LEU A  68     5121   6973   4976  -2479    723    864       C  \\nATOM    479  C   LEU A  68      15.641  23.366   6.399  1.00 54.09           C  \\nANISOU  479  C   LEU A  68     6529   7997   6024  -2503    738   1005       C  \\nATOM    480  O   LEU A  68      14.997  24.347   6.012  1.00 39.77           O  \\nANISOU  480  O   LEU A  68     5027   5883   4200  -2620    663   1140       O  \\nATOM    481  CB  LEU A  68      15.333  22.642   8.810  1.00 43.86           C  \\nANISOU  481  CB  LEU A  68     4953   6716   4995  -2125    594    813       C  \\nATOM    482  CG  LEU A  68      15.918  22.376  10.209  1.00 37.62           C  \\nANISOU  482  CG  LEU A  68     3920   6091   4282  -2064    564    678       C  \\nATOM    483  CD1 LEU A  68      14.833  22.018  11.181  1.00 38.37           C  \\nANISOU  483  CD1 LEU A  68     4083   5991   4504  -1789    444    656       C  \\nATOM    484  CD2 LEU A  68      16.972  21.282  10.174  1.00 38.87           C  \\nANISOU  484  CD2 LEU A  68     3736   6638   4393  -1976    649    583       C  \\nATOM    485  N   PRO A  69      15.836  22.295   5.622  1.00 53.09           N  \\nANISOU  485  N   PRO A  69     6281   8085   5804  -2386    827    975       N  \\nATOM    486  CA  PRO A  69      15.320  22.297   4.251  1.00 52.93           C  \\nANISOU  486  CA  PRO A  69     6501   7974   5636  -2441    842   1099       C  \\nATOM    487  C   PRO A  69      13.816  22.015   4.167  1.00 50.79           C  \\nANISOU  487  C   PRO A  69     6429   7421   5447  -2178    663   1185       C  \\nATOM    488  O   PRO A  69      13.325  21.670   3.100  1.00 63.81           O  \\nANISOU  488  O   PRO A  69     8226   9045   6975  -2157    650   1261       O  \\nATOM    489  CB  PRO A  69      16.116  21.170   3.592  1.00 52.32           C  \\nANISOU  489  CB  PRO A  69     6204   8223   5451  -2361   1002    975       C  \\nATOM    490  CG  PRO A  69      16.356  20.204   4.713  1.00 49.30           C  \\nANISOU  490  CG  PRO A  69     5516   8006   5211  -2110    999    837       C  \\nATOM    491  CD  PRO A  69      16.574  21.058   5.937  1.00 49.13           C  \\nANISOU  491  CD  PRO A  69     5436   7912   5318  -2202    918    822       C  \\nATOM    492  N   VAL A  70      13.096  22.151   5.273  1.00 47.31           N  \\nANISOU  492  N   VAL A  70     5983   6793   5199  -1994    531   1162       N  \\nATOM    493  CA  VAL A  70      11.641  22.054   5.245  1.00 40.29           C  \\nANISOU  493  CA  VAL A  70     5258   5644   4406  -1773    365   1242       C  \\nATOM    494  C   VAL A  70      11.007  23.426   5.355  1.00 37.00           C  \\nANISOU  494  C   VAL A  70     5097   4899   4062  -1859    245   1366       C  \\nATOM    495  O   VAL A  70      11.645  24.393   5.751  1.00 41.88           O  \\nANISOU  495  O   VAL A  70     5766   5458   4687  -2068    286   1363       O  \\nATOM    496  CB  VAL A  70      11.073  21.159   6.385  1.00 38.71           C  \\nANISOU  496  CB  VAL A  70     4883   5451   4375  -1475    308   1128       C  \\nATOM    497  CG1 VAL A  70      11.098  19.705   5.980  1.00 38.59           C  \\nANISOU  497  CG1 VAL A  70     4737   5625   4299  -1312    360   1058       C  \\nATOM    498  CG2 VAL A  70      11.838  21.371   7.667  1.00 34.55           C  \\nANISOU  498  CG2 VAL A  70     4182   5014   3932  -1507    353   1011       C  \\nATOM    499  N   ARG A  71       9.733  23.494   5.012  1.00 42.02           N  \\nANISOU  499  N   ARG A  71     5892   5315   4760  -1691     91   1469       N  \\nATOM    500  CA  ARG A  71       8.972  24.731   5.097  1.00 51.92           C  \\nANISOU  500  CA  ARG A  71     7389   6225   6113  -1693    -45   1589       C  \\nATOM    501  C   ARG A  71       8.577  25.040   6.551  1.00 47.74           C  \\nANISOU  501  C   ARG A  71     6776   5557   5804  -1538    -75   1474       C  \\nATOM    502  O   ARG A  71       8.675  26.193   7.015  1.00 39.56           O  \\nANISOU  502  O   ARG A  71     5893   4299   4837  -1641    -90   1488       O  \\nATOM    503  CB  ARG A  71       7.727  24.589   4.218  1.00 59.09           C  \\nANISOU  503  CB  ARG A  71     8441   6993   7017  -1531   -215   1732       C  \\nATOM    504  CG  ARG A  71       7.040  25.865   3.812  1.00 66.82           C  \\nANISOU  504  CG  ARG A  71     9720   7630   8040  -1548   -374   1914       C  \\nATOM    505  CD  ARG A  71       6.170  25.595   2.587  1.00 71.41           C  \\nANISOU  505  CD  ARG A  71    10432   8182   8519  -1472   -536   2079       C  \\nATOM    506  NE  ARG A  71       5.299  26.720   2.285  1.00 77.35           N  \\nANISOU  506  NE  ARG A  71    11444   8592   9355  -1395   -738   2262       N  \\nATOM    507  CZ  ARG A  71       5.706  27.818   1.663  1.00 88.61           C  \\nANISOU  507  CZ  ARG A  71    13103   9876  10688  -1546   -732   2334       C  \\nATOM    508  NH1 ARG A  71       6.969  27.925   1.285  1.00 88.67           N  \\nANISOU  508  NH1 ARG A  71    13141  10043  10506  -1841   -552   2295       N  \\nATOM    509  NH2 ARG A  71       4.853  28.807   1.420  1.00 97.17           N  \\nANISOU  509  NH2 ARG A  71    14372  10671  11876  -1393   -901   2442       N  \\nATOM    510  N   TYR A  72       8.137  24.005   7.264  1.00 32.61           N  \\nANISOU  510  N   TYR A  72     4644   3762   3983  -1307    -74   1356       N  \\nATOM    511  CA  TYR A  72       7.570  24.190   8.590  1.00 31.65           C  \\nANISOU  511  CA  TYR A  72     4456   3519   4049  -1143   -100   1249       C  \\nATOM    512  C   TYR A  72       8.056  23.164   9.609  1.00 31.17           C  \\nANISOU  512  C   TYR A  72     4142   3700   4001  -1068    -11   1086       C  \\nATOM    513  O   TYR A  72       8.298  22.002   9.272  1.00 37.33           O  \\nANISOU  513  O   TYR A  72     4783   4698   4705  -1010     29   1061       O  \\nATOM    514  CB  TYR A  72       6.039  24.095   8.522  1.00 35.83           C  \\nANISOU  514  CB  TYR A  72     5021   3873   4719   -884   -235   1300       C  \\nATOM    515  CG  TYR A  72       5.356  25.210   7.766  1.00 46.52           C  \\nANISOU  515  CG  TYR A  72     6625   4938   6113   -881   -368   1463       C  \\nATOM    516  CD1 TYR A  72       5.645  26.546   8.037  1.00 57.49           C  \\nANISOU  516  CD1 TYR A  72     8217   6077   7547   -999   -368   1490       C  \\nATOM    517  CD2 TYR A  72       4.419  24.928   6.782  1.00 49.96           C  \\nANISOU  517  CD2 TYR A  72     7108   5338   6535   -760   -509   1593       C  \\nATOM    518  CE1 TYR A  72       5.013  27.573   7.347  1.00 60.28           C  \\nANISOU  518  CE1 TYR A  72     8833   6128   7944   -970   -506   1656       C  \\nATOM    519  CE2 TYR A  72       3.782  25.947   6.082  1.00 59.71           C  \\nANISOU  519  CE2 TYR A  72     8576   6306   7804   -731   -664   1764       C  \\nATOM    520  CZ  TYR A  72       4.082  27.266   6.370  1.00 62.44           C  \\nANISOU  520  CZ  TYR A  72     9138   6381   8206   -822   -662   1801       C  \\nATOM    521  OH  TYR A  72       3.452  28.276   5.678  1.00 71.87           O  \\nANISOU  521  OH  TYR A  72    10594   7275   9440   -769   -830   1986       O  \\nATOM    522  N   VAL A  73       8.166  23.587  10.864  1.00 34.99           N  \\nANISOU  522  N   VAL A  73     4591   4130   4575  -1063     13    976       N  \\nATOM    523  CA  VAL A  73       8.248  22.630  11.968  1.00 36.94           C  \\nANISOU  523  CA  VAL A  73     4637   4548   4851   -935     53    843       C  \\nATOM    524  C   VAL A  73       6.924  22.614  12.737  1.00 30.71           C  \\nANISOU  524  C   VAL A  73     3863   3591   4214   -720      5    800       C  \\nATOM    525  O   VAL A  73       6.421  23.656  13.167  1.00 31.49           O  \\nANISOU  525  O   VAL A  73     4090   3461   4415   -712    -18    781       O  \\nATOM    526  CB  VAL A  73       9.409  22.911  12.947  1.00 33.63           C  \\nANISOU  526  CB  VAL A  73     4130   4263   4385  -1094    116    731       C  \\nATOM    527  CG1 VAL A  73       9.323  21.940  14.130  1.00 27.04           C  \\nANISOU  527  CG1 VAL A  73     3131   3571   3573   -937    126    620       C  \\nATOM    528  CG2 VAL A  73      10.775  22.782  12.240  1.00 29.51           C  \\nANISOU  528  CG2 VAL A  73     3514   3977   3721  -1300    183    752       C  \\nATOM    529  N   ILE A  74       6.351  21.431  12.882  1.00 36.64           N  \\nANISOU  529  N   ILE A  74     4491   4452   4981   -552      0    777       N  \\nATOM    530  CA  ILE A  74       5.089  21.278  13.602  1.00 28.20           C  \\nANISOU  530  CA  ILE A  74     3396   3274   4043   -369    -21    727       C  \\nATOM    531  C   ILE A  74       5.423  20.701  14.958  1.00 25.29           C  \\nANISOU  531  C   ILE A  74     2923   3034   3651   -348     48    601       C  \\nATOM    532  O   ILE A  74       5.942  19.588  15.065  1.00 24.07           O  \\nANISOU  532  O   ILE A  74     2664   3072   3409   -331     71    588       O  \\nATOM    533  CB  ILE A  74       4.098  20.370  12.847  1.00 30.28           C  \\nANISOU  533  CB  ILE A  74     3609   3565   4333   -232    -79    794       C  \\nATOM    534  CG1 ILE A  74       3.804  20.946  11.457  1.00 28.33           C  \\nANISOU  534  CG1 ILE A  74     3482   3207   4074   -265   -175    935       C  \\nATOM    535  CG2 ILE A  74       2.789  20.210  13.641  1.00 27.64           C  \\nANISOU  535  CG2 ILE A  74     3208   3155   4139    -68    -82    730       C  \\nATOM    536  CD1 ILE A  74       3.027  19.977  10.534  1.00 29.93           C  \\nANISOU  536  CD1 ILE A  74     3641   3480   4251   -183   -249   1002       C  \\nATOM    537  N   HIS A  75       5.169  21.491  15.994  1.00 26.51           N  \\nANISOU  537  N   HIS A  75     3128   3072   3872   -351     79    506       N  \\nATOM    538  CA  HIS A  75       5.541  21.106  17.344  1.00 25.84           C  \\nANISOU  538  CA  HIS A  75     2982   3106   3731   -365    136    388       C  \\nATOM    539  C   HIS A  75       4.355  20.436  18.035  1.00 30.00           C  \\nANISOU  539  C   HIS A  75     3455   3621   4322   -206    170    336       C  \\nATOM    540  O   HIS A  75       3.395  21.086  18.450  1.00 33.10           O  \\nANISOU  540  O   HIS A  75     3889   3858   4828   -130    202    275       O  \\nATOM    541  CB  HIS A  75       6.031  22.320  18.133  1.00 26.19           C  \\nANISOU  541  CB  HIS A  75     3130   3050   3770   -502    164    291       C  \\nATOM    542  CG  HIS A  75       7.189  23.022  17.503  1.00 30.82           C  \\nANISOU  542  CG  HIS A  75     3769   3652   4289   -708    141    336       C  \\nATOM    543  ND1 HIS A  75       8.498  22.746  17.835  1.00 29.74           N  \\nANISOU  543  ND1 HIS A  75     3535   3743   4023   -861    145    303       N  \\nATOM    544  CD2 HIS A  75       7.237  24.003  16.566  1.00 33.47           C  \\nANISOU  544  CD2 HIS A  75     4244   3809   4662   -801    115    416       C  \\nATOM    545  CE1 HIS A  75       9.301  23.538  17.145  1.00 33.20           C  \\nANISOU  545  CE1 HIS A  75     4029   4160   4424  -1060    141    346       C  \\nATOM    546  NE2 HIS A  75       8.562  24.307  16.363  1.00 34.77           N  \\nANISOU  546  NE2 HIS A  75     4395   4101   4713  -1037    125    422       N  \\nATOM    547  N   ALA A  76       4.439  19.123  18.150  1.00 26.99           N  \\nANISOU  547  N   ALA A  76     2983   3405   3867   -156    174    357       N  \\nATOM    548  CA  ALA A  76       3.351  18.332  18.676  1.00 27.48           C  \\nANISOU  548  CA  ALA A  76     2999   3475   3968    -47    214    328       C  \\nATOM    549  C   ALA A  76       3.630  18.053  20.133  1.00 28.22           C  \\nANISOU  549  C   ALA A  76     3101   3655   3968    -81    275    232       C  \\nATOM    550  O   ALA A  76       4.517  17.265  20.450  1.00 26.08           O  \\nANISOU  550  O   ALA A  76     2808   3530   3571   -110    251    253       O  \\nATOM    551  CB  ALA A  76       3.235  17.029  17.898  1.00 28.62           C  \\nANISOU  551  CB  ALA A  76     3090   3711   4075      5    179    413       C  \\nATOM    552  N   ALA A  77       2.868  18.690  21.018  1.00 27.67           N  \\nANISOU  552  N   ALA A  77     3065   3496   3953    -66    351    125       N  \\nATOM    553  CA  ALA A  77       3.139  18.623  22.457  1.00 27.94           C  \\nANISOU  553  CA  ALA A  77     3144   3606   3867   -129    415     21       C  \\nATOM    554  C   ALA A  77       2.627  17.311  23.057  1.00 29.96           C  \\nANISOU  554  C   ALA A  77     3371   3970   4045    -91    458     38       C  \\nATOM    555  O   ALA A  77       1.422  17.109  23.164  1.00 25.00           O  \\nANISOU  555  O   ALA A  77     2704   3297   3497    -33    536      8       O  \\nATOM    556  CB  ALA A  77       2.501  19.813  23.157  1.00 28.05           C  \\nANISOU  556  CB  ALA A  77     3226   3474   3956   -132    506   -123       C  \\nATOM    557  N   VAL A  78       3.547  16.433  23.456  1.00 31.98           N  \\nANISOU  557  N   VAL A  78     3642   4366   4144   -128    404     89       N  \\nATOM    558  CA  VAL A  78       3.186  15.098  23.940  1.00 28.67           C  \\nANISOU  558  CA  VAL A  78     3242   4018   3633    -99    426    137       C  \\nATOM    559  C   VAL A  78       3.356  14.977  25.437  1.00 29.98           C  \\nANISOU  559  C   VAL A  78     3500   4263   3629   -175    469     73       C  \\nATOM    560  O   VAL A  78       2.931  13.998  26.040  1.00 32.98           O  \\nANISOU  560  O   VAL A  78     3938   4682   3914   -180    508    107       O  \\nATOM    561  CB  VAL A  78       4.003  13.957  23.243  1.00 33.61           C  \\nANISOU  561  CB  VAL A  78     3849   4718   4205    -43    324    263       C  \\nATOM    562  CG1 VAL A  78       3.608  13.837  21.796  1.00 40.61           C  \\nANISOU  562  CG1 VAL A  78     4674   5534   5222     16    303    321       C  \\nATOM    563  CG2 VAL A  78       5.521  14.183  23.370  1.00 25.50           C  \\nANISOU  563  CG2 VAL A  78     2797   3808   3084    -70    227    276       C  \\nATOM    564  N   LEU A  79       3.973  15.977  26.043  1.00 29.90           N  \\nANISOU  564  N   LEU A  79     3527   4271   3563   -256    460    -18       N  \\nATOM    565  CA  LEU A  79       4.218  15.925  27.473  1.00 38.16           C  \\nANISOU  565  CA  LEU A  79     4679   5408   4411   -351    483    -85       C  \\nATOM    566  C   LEU A  79       4.513  17.313  28.015  1.00 38.08           C  \\nANISOU  566  C   LEU A  79     4722   5362   4385   -458    514   -236       C  \\nATOM    567  O   LEU A  79       4.557  18.290  27.264  1.00 27.57           O  \\nANISOU  567  O   LEU A  79     3356   3915   3203   -453    513   -274       O  \\nATOM    568  CB  LEU A  79       5.364  14.957  27.792  1.00 42.03           C  \\nANISOU  568  CB  LEU A  79     5185   6054   4731   -348    336     29       C  \\nATOM    569  CG  LEU A  79       6.832  15.365  27.582  1.00 49.10           C  \\nANISOU  569  CG  LEU A  79     6007   7065   5585   -385    188     47       C  \\nATOM    570  CD1 LEU A  79       7.719  14.142  27.744  1.00 51.84           C  \\nANISOU  570  CD1 LEU A  79     6332   7556   5810   -303     46    179       C  \\nATOM    571  CD2 LEU A  79       7.086  16.005  26.237  1.00 49.76           C  \\nANISOU  571  CD2 LEU A  79     5976   7080   5849   -361    175     58       C  \\nATOM    572  N   GLY A  80       4.708  17.393  29.324  1.00 40.31           N  \\nANISOU  572  N   GLY A  80     5119   5730   4469   -567    539   -321       N  \\nATOM    573  CA  GLY A  80       5.023  18.656  29.966  1.00 38.12           C  \\nANISOU  573  CA  GLY A  80     4928   5416   4139   -697    569   -487       C  \\nATOM    574  C   GLY A  80       4.388  18.715  31.333  1.00 40.73           C  \\nANISOU  574  C   GLY A  80     5403   5775   4299   -780    707   -624       C  \\nATOM    575  O   GLY A  80       4.950  18.184  32.289  1.00 38.99           O  \\nANISOU  575  O   GLY A  80     5270   5713   3831   -877    637   -600       O  \\nATOM    576  N   ASP A  81       3.220  19.348  31.441  1.00 33.38           N  \\nANISOU  576  N   ASP A  81     4493   4699   3489   -738    902   -767       N  \\nATOM    577  CA  ASP A  81       2.543  19.406  32.738  1.00 35.40           C  \\nANISOU  577  CA  ASP A  81     4879   4995   3576   -821   1077   -922       C  \\nATOM    578  C   ASP A  81       1.985  18.042  33.073  1.00 35.04           C  \\nANISOU  578  C   ASP A  81     4825   5068   3420   -806   1122   -803       C  \\nATOM    579  O   ASP A  81       1.829  17.696  34.228  1.00 36.69           O  \\nANISOU  579  O   ASP A  81     5171   5382   3386   -923   1202   -855       O  \\nATOM    580  CB  ASP A  81       1.443  20.465  32.768  1.00 39.88           C  \\nANISOU  580  CB  ASP A  81     5448   5385   4322   -753   1292  -1129       C  \\nATOM    581  CG  ASP A  81       1.995  21.866  32.672  1.00 44.01           C  \\nANISOU  581  CG  ASP A  81     6061   5751   4910   -801   1263  -1268       C  \\nATOM    582  OD1 ASP A  81       2.990  22.169  33.364  1.00 49.58           O  \\nANISOU  582  OD1 ASP A  81     6900   6536   5404   -981   1175  -1322       O  \\nATOM    583  OD2 ASP A  81       1.441  22.663  31.895  1.00 47.33           O  \\nANISOU  583  OD2 ASP A  81     6428   5967   5590   -668   1315  -1317       O  \\nATOM    584  N   GLU A  82       1.706  17.259  32.038  1.00 33.51           N  \\nANISOU  584  N   GLU A  82     4495   4850   3389   -681   1067   -641       N  \\nATOM    585  CA  GLU A  82       1.302  15.877  32.226  1.00 38.93           C  \\nANISOU  585  CA  GLU A  82     5197   5621   3974   -685   1081   -504       C  \\nATOM    586  C   GLU A  82       2.342  14.988  31.570  1.00 32.21           C  \\nANISOU  586  C   GLU A  82     4321   4817   3100   -627    853   -295       C  \\nATOM    587  O   GLU A  82       2.965  15.378  30.582  1.00 32.47           O  \\nANISOU  587  O   GLU A  82     4248   4805   3284   -550    735   -254       O  \\nATOM    588  CB  GLU A  82      -0.077  15.614  31.604  1.00 42.35           C  \\nANISOU  588  CB  GLU A  82     5493   5982   4616   -599   1234   -514       C  \\nATOM    589  CG  GLU A  82      -1.223  16.386  32.238  1.00 46.38           C  \\nANISOU  589  CG  GLU A  82     5980   6467   5176   -620   1485   -730       C  \\nATOM    590  CD  GLU A  82      -1.272  17.844  31.787  1.00 41.56           C  \\nANISOU  590  CD  GLU A  82     5305   5709   4774   -520   1507   -879       C  \\nATOM    591  OE1 GLU A  82      -1.128  18.107  30.582  1.00 45.79           O  \\nANISOU  591  OE1 GLU A  82     5729   6144   5526   -399   1385   -795       O  \\nATOM    592  OE2 GLU A  82      -1.436  18.731  32.643  1.00 47.09           O  \\nANISOU  592  OE2 GLU A  82     6098   6384   5411   -569   1645  -1080       O  \\nATOM    593  N   PRO A  83       2.539  13.789  32.118  1.00 31.69           N  \\nANISOU  593  N   PRO A  83     4365   4836   2840   -662    797   -163       N  \\nATOM    594  CA  PRO A  83       3.440  12.818  31.479  1.00 32.87           C  \\nANISOU  594  CA  PRO A  83     4491   5010   2986   -563    595     31       C  \\nATOM    595  C   PRO A  83       2.875  12.279  30.146  1.00 39.03           C  \\nANISOU  595  C   PRO A  83     5148   5684   3999   -442    611    110       C  \\nATOM    596  O   PRO A  83       1.665  12.320  29.895  1.00 33.66           O  \\nANISOU  596  O   PRO A  83     4417   4934   3438   -454    767     54       O  \\nATOM    597  CB  PRO A  83       3.572  11.715  32.534  1.00 36.05           C  \\nANISOU  597  CB  PRO A  83     5091   5488   3118   -628    555    143       C  \\nATOM    598  CG  PRO A  83       2.346  11.860  33.410  1.00 37.10           C  \\nANISOU  598  CG  PRO A  83     5322   5616   3158   -770    792     28       C  \\nATOM    599  CD  PRO A  83       1.950  13.297  33.379  1.00 33.64           C  \\nANISOU  599  CD  PRO A  83     4782   5149   2849   -795    925   -189       C  \\nATOM    600  N   ALA A  84       3.758  11.815  29.276  1.00 27.65           N  \\nANISOU  600  N   ALA A  84     3643   4244   2618   -331    451    227       N  \\nATOM    601  CA  ALA A  84       3.337  11.275  27.995  1.00 29.75           C  \\nANISOU  601  CA  ALA A  84     3820   4417   3068   -231    452    295       C  \\nATOM    602  C   ALA A  84       2.549   9.992  28.185  1.00 28.51           C  \\nANISOU  602  C   ALA A  84     3776   4207   2850   -253    511    382       C  \\nATOM    603  O   ALA A  84       2.862   9.187  29.059  1.00 32.27           O  \\nANISOU  603  O   ALA A  84     4416   4714   3130   -285    468    464       O  \\nATOM    604  CB  ALA A  84       4.542  11.028  27.084  1.00 25.30           C  \\nANISOU  604  CB  ALA A  84     3177   3882   2553   -113    289    382       C  \\nATOM    605  N   SER A  85       1.531   9.806  27.353  1.00 29.18           N  \\nANISOU  605  N   SER A  85     3782   4210   3094   -248    596    371       N  \\nATOM    606  CA  SER A  85       0.687   8.619  27.427  1.00 34.03           C  \\nANISOU  606  CA  SER A  85     4496   4768   3667   -311    664    440       C  \\nATOM    607  C   SER A  85       0.211   8.320  26.007  1.00 25.17           C  \\nANISOU  607  C   SER A  85     3262   3566   2735   -253    644    463       C  \\nATOM    608  O   SER A  85       0.420   9.142  25.121  1.00 28.45           O  \\nANISOU  608  O   SER A  85     3533   3981   3298   -174    598    421       O  \\nATOM    609  CB  SER A  85      -0.508   8.911  28.318  1.00 31.98           C  \\nANISOU  609  CB  SER A  85     4239   4546   3366   -461    860    340       C  \\nATOM    610  OG  SER A  85      -1.373   9.821  27.662  1.00 33.92           O  \\nANISOU  610  OG  SER A  85     4275   4785   3827   -437    947    223       O  \\nATOM    611  N   LEU A  86      -0.422   7.173  25.773  1.00 25.62           N  \\nANISOU  611  N   LEU A  86     3406   3552   2775   -313    674    530       N  \\nATOM    612  CA  LEU A  86      -0.949   6.885  24.426  1.00 26.05           C  \\nANISOU  612  CA  LEU A  86     3364   3543   2989   -289    650    537       C  \\nATOM    613  C   LEU A  86      -2.033   7.887  24.042  1.00 24.61           C  \\nANISOU  613  C   LEU A  86     2962   3409   2979   -324    735    428       C  \\nATOM    614  O   LEU A  86      -2.263   8.154  22.871  1.00 27.04           O  \\nANISOU  614  O   LEU A  86     3148   3695   3432   -269    675    423       O  \\nATOM    615  CB  LEU A  86      -1.499   5.460  24.319  1.00 29.03           C  \\nANISOU  615  CB  LEU A  86     3899   3826   3306   -388    673    613       C  \\nATOM    616  CG  LEU A  86      -0.505   4.325  24.581  1.00 34.07           C  \\nANISOU  616  CG  LEU A  86     4786   4365   3795   -318    576    735       C  \\nATOM    617  CD1 LEU A  86      -1.182   2.983  24.427  1.00 27.50           C  \\nANISOU  617  CD1 LEU A  86     4139   3395   2914   -441    613    799       C  \\nATOM    618  CD2 LEU A  86       0.723   4.465  23.655  1.00 25.07           C  \\nANISOU  618  CD2 LEU A  86     3596   3214   2716   -111    435    755       C  \\nATOM    619  N   GLU A  87      -2.673   8.460  25.044  1.00 25.69           N  \\nANISOU  619  N   GLU A  87     3053   3614   3093   -402    870    339       N  \\nATOM    620  CA  GLU A  87      -3.732   9.430  24.807  1.00 26.81           C  \\nANISOU  620  CA  GLU A  87     2974   3800   3412   -398    960    223       C  \\nATOM    621  C   GLU A  87      -3.157  10.780  24.343  1.00 25.16           C  \\nANISOU  621  C   GLU A  87     2669   3569   3320   -254    885    172       C  \\nATOM    622  O   GLU A  87      -3.708  11.423  23.452  1.00 31.56           O  \\nANISOU  622  O   GLU A  87     3321   4358   4314   -183    853    144       O  \\nATOM    623  CB  GLU A  87      -4.602   9.604  26.072  1.00 27.99           C  \\nANISOU  623  CB  GLU A  87     3103   4034   3496   -520   1164    118       C  \\nATOM    624  CG  GLU A  87      -5.946  10.246  25.773  1.00 37.91           C  \\nANISOU  624  CG  GLU A  87     4099   5351   4956   -513   1275      1       C  \\nATOM    625  CD  GLU A  87      -6.921  10.246  26.956  1.00 46.62           C  \\nANISOU  625  CD  GLU A  87     5153   6566   5994   -651   1514   -117       C  \\nATOM    626  OE1 GLU A  87      -6.482  10.150  28.126  1.00 39.01           O  \\nANISOU  626  OE1 GLU A  87     4374   5629   4821   -736   1603   -135       O  \\nATOM    627  OE2 GLU A  87      -8.141  10.353  26.699  1.00 61.20           O  \\nANISOU  627  OE2 GLU A  87     6764   8493   7995   -678   1614   -195       O  \\nATOM    628  N   THR A  88      -2.072  11.228  24.966  1.00 31.22           N  \\nANISOU  628  N   THR A  88     3543   4342   3977   -227    849    165       N  \\nATOM    629  CA  THR A  88      -1.480  12.502  24.554  1.00 27.62           C  \\nANISOU  629  CA  THR A  88     3025   3851   3617   -135    785    118       C  \\nATOM    630  C   THR A  88      -0.834  12.334  23.165  1.00 27.88           C  \\nANISOU  630  C   THR A  88     3033   3841   3720    -58    631    220       C  \\nATOM    631  O   THR A  88      -0.848  13.243  22.338  1.00 29.26           O  \\nANISOU  631  O   THR A  88     3126   3964   4028      4    581    207       O  \\nATOM    632  CB  THR A  88      -0.479  13.049  25.606  1.00 24.77           C  \\nANISOU  632  CB  THR A  88     2777   3529   3106   -167    783     69       C  \\nATOM    633  OG1 THR A  88       0.729  12.283  25.583  1.00 30.39           O  \\nANISOU  633  OG1 THR A  88     3588   4279   3680   -159    655    180       O  \\nATOM    634  CG2 THR A  88      -1.075  12.999  27.002  1.00 29.73           C  \\nANISOU  634  CG2 THR A  88     3475   4215   3607   -269    941    -24       C  \\nATOM    635  N   VAL A  89      -0.297  11.147  22.911  1.00 24.60           N  \\nANISOU  635  N   VAL A  89     2707   3435   3205    -63    565    319       N  \\nATOM    636  CA  VAL A  89       0.295  10.835  21.616  1.00 27.49           C  \\nANISOU  636  CA  VAL A  89     3061   3771   3612      3    451    397       C  \\nATOM    637  C   VAL A  89      -0.794  10.839  20.546  1.00 29.86           C  \\nANISOU  637  C   VAL A  89     3262   4031   4054      1    443    401       C  \\nATOM    638  O   VAL A  89      -0.596  11.350  19.433  1.00 28.17           O  \\nANISOU  638  O   VAL A  89     2995   3792   3918     49    363    426       O  \\nATOM    639  CB  VAL A  89       1.026   9.465  21.646  1.00 26.76           C  \\nANISOU  639  CB  VAL A  89     3101   3678   3390     24    399    483       C  \\nATOM    640  CG1 VAL A  89       1.443   9.036  20.258  1.00 24.90           C  \\nANISOU  640  CG1 VAL A  89     2856   3409   3195     89    319    533       C  \\nATOM    641  CG2 VAL A  89       2.226   9.517  22.586  1.00 22.38           C  \\nANISOU  641  CG2 VAL A  89     2614   3188   2701     54    358    496       C  \\nATOM    642  N   ARG A  90      -1.954  10.283  20.875  1.00 28.56           N  \\nANISOU  642  N   ARG A  90     3067   3874   3910    -73    521    378       N  \\nATOM    643  CA  ARG A  90      -3.049  10.280  19.907  1.00 28.14           C  \\nANISOU  643  CA  ARG A  90     2885   3815   3990    -87    494    377       C  \\nATOM    644  C   ARG A  90      -3.481  11.716  19.567  1.00 27.95           C  \\nANISOU  644  C   ARG A  90     2710   3781   4128     -2    474    327       C  \\nATOM    645  O   ARG A  90      -3.575  12.082  18.404  1.00 26.70           O  \\nANISOU  645  O   ARG A  90     2497   3593   4053     48    365    372       O  \\nATOM    646  CB  ARG A  90      -4.238   9.470  20.418  1.00 25.81           C  \\nANISOU  646  CB  ARG A  90     2553   3561   3692   -212    595    349       C  \\nATOM    647  CG  ARG A  90      -5.372   9.399  19.401  1.00 25.15           C  \\nANISOU  647  CG  ARG A  90     2306   3506   3743   -245    543    346       C  \\nATOM    648  CD  ARG A  90      -6.531   8.501  19.850  1.00 25.84           C  \\nANISOU  648  CD  ARG A  90     2339   3659   3821   -415    645    315       C  \\nATOM    649  NE  ARG A  90      -6.143   7.098  19.947  1.00 27.22           N  \\nANISOU  649  NE  ARG A  90     2739   3768   3837   -535    649    380       N  \\nATOM    650  CZ  ARG A  90      -6.126   6.245  18.927  1.00 26.30           C  \\nANISOU  650  CZ  ARG A  90     2704   3595   3695   -589    551    430       C  \\nATOM    651  NH1 ARG A  90      -6.477   6.644  17.715  1.00 27.65           N  \\nANISOU  651  NH1 ARG A  90     2742   3792   3970   -551    431    431       N  \\nATOM    652  NH2 ARG A  90      -5.756   4.986  19.117  1.00 25.95           N  \\nANISOU  652  NH2 ARG A  90     2897   3453   3511   -680    567    480       N  \\nATOM    653  N   LYS A  91      -3.721  12.523  20.597  1.00 27.24           N  \\nANISOU  653  N   LYS A  91     2578   3701   4069     16    580    234       N  \\nATOM    654  CA  LYS A  91      -4.169  13.903  20.428  1.00 32.06           C  \\nANISOU  654  CA  LYS A  91     3074   4263   4844    120    578    171       C  \\nATOM    655  C   LYS A  91      -3.144  14.763  19.677  1.00 23.43           C  \\nANISOU  655  C   LYS A  91     2058   3082   3763    185    459    226       C  \\nATOM    656  O   LYS A  91      -3.492  15.516  18.775  1.00 31.44           O  \\nANISOU  656  O   LYS A  91     3010   4029   4906    264    370    258       O  \\nATOM    657  CB  LYS A  91      -4.498  14.525  21.791  1.00 36.37           C  \\nANISOU  657  CB  LYS A  91     3604   4824   5390    119    742     34       C  \\nATOM    658  CG  LYS A  91      -5.768  13.971  22.454  1.00 37.96           C  \\nANISOU  658  CG  LYS A  91     3677   5127   5618     54    892    -43       C  \\nATOM    659  CD  LYS A  91      -5.868  14.402  23.925  1.00 43.64           C  \\nANISOU  659  CD  LYS A  91     4438   5881   6263     18   1083   -184       C  \\nATOM    660  CE  LYS A  91      -6.990  13.663  24.683  1.00 45.59           C  \\nANISOU  660  CE  LYS A  91     4585   6258   6478   -101   1264   -254       C  \\nATOM    661  NZ  LYS A  91      -7.069  14.039  26.145  1.00 40.63           N  \\nANISOU  661  NZ  LYS A  91     4024   5680   5734   -161   1472   -399       N  \\nATOM    662  N   ALA A  92      -1.879  14.633  20.041  1.00 23.56           N  \\nANISOU  662  N   ALA A  92     2208   3108   3637    142    450    245       N  \\nATOM    663  CA  ALA A  92      -0.832  15.418  19.411  1.00 22.22           C  \\nANISOU  663  CA  ALA A  92     2101   2884   3460    159    361    288       C  \\nATOM    664  C   ALA A  92      -0.630  15.030  17.959  1.00 25.36           C  \\nANISOU  664  C   ALA A  92     2497   3277   3861    168    245    399       C  \\nATOM    665  O   ALA A  92      -0.398  15.901  17.126  1.00 31.83           O  \\nANISOU  665  O   ALA A  92     3332   4027   4735    189    171    443       O  \\nATOM    666  CB  ALA A  92       0.478  15.292  20.180  1.00 21.61           C  \\nANISOU  666  CB  ALA A  92     2120   2864   3227     99    375    274       C  \\nATOM    667  N   THR A  93      -0.703  13.732  17.656  1.00 26.27           N  \\nANISOU  667  N   THR A  93     2624   3454   3903    139    232    443       N  \\nATOM    668  CA  THR A  93      -0.572  13.264  16.275  1.00 22.58           C  \\nANISOU  668  CA  THR A  93     2175   2988   3416    135    136    525       C  \\nATOM    669  C   THR A  93      -1.734  13.819  15.462  1.00 28.82           C  \\nANISOU  669  C   THR A  93     2874   3736   4340    166     63    550       C  \\nATOM    670  O   THR A  93      -1.555  14.362  14.374  1.00 30.00           O  \\nANISOU  670  O   THR A  93     3048   3848   4502    177    -36    618       O  \\nATOM    671  CB  THR A  93      -0.577  11.717  16.180  1.00 27.03           C  \\nANISOU  671  CB  THR A  93     2795   3593   3883     98    148    544       C  \\nATOM    672  OG1 THR A  93       0.487  11.184  16.971  1.00 25.96           O  \\nANISOU  672  OG1 THR A  93     2742   3490   3632    106    193    536       O  \\nATOM    673  CG2 THR A  93      -0.382  11.255  14.734  1.00 23.70           C  \\nANISOU  673  CG2 THR A  93     2417   3169   3419     85     63    603       C  \\nATOM    674  N   LYS A  94      -2.933  13.704  16.016  1.00 22.08           N  \\nANISOU  674  N   LYS A  94     1908   2900   3581    176    109    497       N  \\nATOM    675  CA  LYS A  94      -4.110  14.271  15.381  1.00 24.55           C  \\nANISOU  675  CA  LYS A  94     2084   3197   4046    234     29    512       C  \\nATOM    676  C   LYS A  94      -3.991  15.799  15.216  1.00 31.34           C  \\nANISOU  676  C   LYS A  94     2949   3943   5014    341    -19    522       C  \\nATOM    677  O   LYS A  94      -4.378  16.338  14.177  1.00 28.09           O  \\nANISOU  677  O   LYS A  94     2516   3484   4673    395   -156    602       O  \\nATOM    678  CB  LYS A  94      -5.357  13.912  16.187  1.00 26.19           C  \\nANISOU  678  CB  LYS A  94     2130   3476   4343    224    122    427       C  \\nATOM    679  CG  LYS A  94      -6.653  14.387  15.581  1.00 34.44           C  \\nANISOU  679  CG  LYS A  94     2974   4547   5563    299     32    433       C  \\nATOM    680  CD  LYS A  94      -7.695  14.543  16.676  1.00 47.54           C  \\nANISOU  680  CD  LYS A  94     4445   6274   7344    330    179    309       C  \\nATOM    681  CE  LYS A  94      -9.097  14.351  16.158  1.00 53.95           C  \\nANISOU  681  CE  LYS A  94     5001   7199   8297    343    107    304       C  \\nATOM    682  NZ  LYS A  94      -9.376  12.898  15.990  1.00 65.85           N  \\nANISOU  682  NZ  LYS A  94     6512   8819   9690    140    117    320       N  \\nATOM    683  N   SER A  95      -3.471  16.501  16.229  1.00 27.42           N  \\nANISOU  683  N   SER A  95     2505   3391   4521    361     84    445       N  \\nATOM    684  CA  SER A  95      -3.316  17.961  16.118  1.00 29.96           C  \\nANISOU  684  CA  SER A  95     2878   3563   4942    445     48    445       C  \\nATOM    685  C   SER A  95      -2.291  18.325  15.044  1.00 35.21           C  \\nANISOU  685  C   SER A  95     3688   4170   5520    389    -62    563       C  \\nATOM    686  O   SER A  95      -2.490  19.282  14.282  1.00 36.31           O  \\nANISOU  686  O   SER A  95     3873   4183   5739    448   -167    637       O  \\nATOM    687  CB  SER A  95      -2.915  18.614  17.450  1.00 29.38           C  \\nANISOU  687  CB  SER A  95     2860   3438   4864    443    187    317       C  \\nATOM    688  OG  SER A  95      -3.952  18.554  18.412  1.00 33.81           O  \\nANISOU  688  OG  SER A  95     3295   4037   5516    503    309    194       O  \\nATOM    689  N   ALA A  96      -1.206  17.552  14.984  1.00 30.23           N  \\nANISOU  689  N   ALA A  96     3130   3634   4723    279    -35    583       N  \\nATOM    690  CA  ALA A  96      -0.158  17.748  13.979  1.00 30.54           C  \\nANISOU  690  CA  ALA A  96     3281   3667   4656    202   -103    677       C  \\nATOM    691  C   ALA A  96      -0.686  17.543  12.565  1.00 31.63           C  \\nANISOU  691  C   ALA A  96     3425   3804   4789    209   -233    790       C  \\nATOM    692  O   ALA A  96      -0.382  18.318  11.664  1.00 34.24           O  \\nANISOU  692  O   ALA A  96     3853   4056   5099    181   -317    883       O  \\nATOM    693  CB  ALA A  96       1.014  16.801  14.242  1.00 21.55           C  \\nANISOU  693  CB  ALA A  96     2168   2661   3359    120    -37    656       C  \\nATOM    694  N   LEU A  97      -1.461  16.481  12.376  1.00 31.02           N  \\nANISOU  694  N   LEU A  97     3261   3814   4710    221   -254    783       N  \\nATOM    695  CA  LEU A  97      -2.084  16.204  11.093  1.00 37.50           C  \\nANISOU  695  CA  LEU A  97     4081   4656   5512    211   -393    874       C  \\nATOM    696  C   LEU A  97      -3.048  17.319  10.690  1.00 34.67           C  \\nANISOU  696  C   LEU A  97     3677   4192   5306    315   -523    939       C  \\nATOM    697  O   LEU A  97      -3.084  17.716   9.534  1.00 36.62           O  \\nANISOU  697  O   LEU A  97     4004   4400   5508    301   -665   1057       O  \\nATOM    698  CB  LEU A  97      -2.803  14.843  11.117  1.00 35.00           C  \\nANISOU  698  CB  LEU A  97     3678   4447   5171    176   -385    833       C  \\nATOM    699  CG  LEU A  97      -1.884  13.623  11.251  1.00 32.70           C  \\nANISOU  699  CG  LEU A  97     3475   4226   4723     96   -290    793       C  \\nATOM    700  CD1 LEU A  97      -2.700  12.362  11.474  1.00 28.50           C  \\nANISOU  700  CD1 LEU A  97     2891   3752   4185     49   -270    745       C  \\nATOM    701  CD2 LEU A  97      -0.963  13.475  10.033  1.00 24.27           C  \\nANISOU  701  CD2 LEU A  97     2541   3179   3502     33   -334    854       C  \\nATOM    702  N   GLU A  98      -3.823  17.826  11.648  1.00 39.19           N  \\nANISOU  702  N   GLU A  98     4126   4716   6051    430   -474    860       N  \\nATOM    703  CA  GLU A  98      -4.780  18.892  11.361  1.00 37.46           C  \\nANISOU  703  CA  GLU A  98     3840   4382   6009    583   -594    907       C  \\nATOM    704  C   GLU A  98      -4.085  20.165  10.892  1.00 34.44           C  \\nANISOU  704  C   GLU A  98     3657   3811   5620    600   -659    999       C  \\nATOM    705  O   GLU A  98      -4.595  20.856  10.032  1.00 34.74           O  \\nANISOU  705  O   GLU A  98     3730   3749   5718    684   -830   1119       O  \\nATOM    706  CB  GLU A  98      -5.689  19.182  12.565  1.00 40.51           C  \\nANISOU  706  CB  GLU A  98     4047   4760   6585    716   -486    770       C  \\nATOM    707  CG  GLU A  98      -6.794  18.143  12.761  1.00 49.27           C  \\nANISOU  707  CG  GLU A  98     4925   6051   7745    707   -469    711       C  \\nATOM    708  CD  GLU A  98      -7.491  18.243  14.113  1.00 58.83           C  \\nANISOU  708  CD  GLU A  98     5966   7296   9090    783   -292    549       C  \\nATOM    709  OE1 GLU A  98      -8.577  17.640  14.264  1.00 59.23           O  \\nANISOU  709  OE1 GLU A  98     5791   7491   9221    786   -278    498       O  \\nATOM    710  OE2 GLU A  98      -6.953  18.912  15.027  1.00 63.08           O  \\nANISOU  710  OE2 GLU A  98     6598   7731   9639    816   -160    463       O  \\nATOM    711  N   LYS A  99      -2.925  20.475  11.458  1.00 34.13           N  \\nANISOU  711  N   LYS A  99     3750   3720   5497    507   -534    951       N  \\nATOM    712  CA  LYS A  99      -2.173  21.644  11.012  1.00 36.00           C  \\nANISOU  712  CA  LYS A  99     4196   3780   5705    463   -580   1037       C  \\nATOM    713  C   LYS A  99      -1.651  21.425   9.590  1.00 35.65           C  \\nANISOU  713  C   LYS A  99     4280   3780   5487    337   -693   1194       C  \\nATOM    714  O   LYS A  99      -1.700  22.329   8.766  1.00 39.59           O  \\nANISOU  714  O   LYS A  99     4929   4129   5986    343   -821   1329       O  \\nATOM    715  CB  LYS A  99      -1.017  21.956  11.963  1.00 33.22           C  \\nANISOU  715  CB  LYS A  99     3931   3404   5289    352   -421    935       C  \\nATOM    716  CG  LYS A  99      -1.456  22.372  13.363  1.00 33.73           C  \\nANISOU  716  CG  LYS A  99     3925   3398   5491    454   -304    774       C  \\nATOM    717  CD  LYS A  99      -2.133  23.744  13.369  1.00 35.21           C  \\nANISOU  717  CD  LYS A  99     4192   3323   5863    608   -364    784       C  \\nATOM    718  CE  LYS A  99      -2.436  24.192  14.798  1.00 47.52           C  \\nANISOU  718  CE  LYS A  99     5710   4811   7535    692   -210    590       C  \\nATOM    719  NZ  LYS A  99      -2.916  25.602  14.852  1.00 58.93           N  \\nANISOU  719  NZ  LYS A  99     7277   5957   9158    849   -247    575       N  \\nATOM    720  N   ALA A 100      -1.174  20.216   9.310  1.00 30.88           N  \\nANISOU  720  N   ALA A 100     3635   3372   4727    225   -643   1173       N  \\nATOM    721  CA  ALA A 100      -0.625  19.889   8.000  1.00 33.32           C  \\nANISOU  721  CA  ALA A 100     4065   3751   4845     93   -711   1287       C  \\nATOM    722  C   ALA A 100      -1.687  20.069   6.924  1.00 37.19           C  \\nANISOU  722  C   ALA A 100     4571   4202   5356    158   -923   1420       C  \\nATOM    723  O   ALA A 100      -1.432  20.654   5.884  1.00 38.42           O  \\nANISOU  723  O   ALA A 100     4902   4291   5405     86  -1030   1563       O  \\nATOM    724  CB  ALA A 100      -0.097  18.461   7.988  1.00 29.22           C  \\nANISOU  724  CB  ALA A 100     3485   3431   4186     10   -611   1208       C  \\nATOM    725  N   VAL A 101      -2.883  19.561   7.192  1.00 42.17           N  \\nANISOU  725  N   VAL A 101     5016   4892   6116    280   -986   1375       N  \\nATOM    726  CA  VAL A 101      -3.982  19.659   6.248  1.00 38.12           C  \\nANISOU  726  CA  VAL A 101     4463   4383   5636    351  -1212   1491       C  \\nATOM    727  C   VAL A 101      -4.421  21.107   6.120  1.00 41.49           C  \\nANISOU  727  C   VAL A 101     4962   4593   6211    502  -1348   1602       C  \\nATOM    728  O   VAL A 101      -4.737  21.573   5.020  1.00 53.32           O  \\nANISOU  728  O   VAL A 101     6574   6034   7652    512  -1557   1773       O  \\nATOM    729  CB  VAL A 101      -5.160  18.762   6.690  1.00 38.72           C  \\nANISOU  729  CB  VAL A 101     4278   4601   5833    425  -1229   1396       C  \\nATOM    730  CG1 VAL A 101      -6.377  18.941   5.781  1.00 38.16           C  \\nANISOU  730  CG1 VAL A 101     4113   4562   5823    509  -1490   1510       C  \\nATOM    731  CG2 VAL A 101      -4.706  17.312   6.740  1.00 36.10           C  \\nANISOU  731  CG2 VAL A 101     3937   4435   5342    267  -1107   1302       C  \\nATOM    732  N   GLU A 102      -4.411  21.835   7.235  1.00 36.55           N  \\nANISOU  732  N   GLU A 102     4298   3828   5760    618  -1233   1508       N  \\nATOM    733  CA  GLU A 102      -4.831  23.241   7.214  1.00 45.37           C  \\nANISOU  733  CA  GLU A 102     5507   4689   7041    790  -1345   1591       C  \\nATOM    734  C   GLU A 102      -3.902  24.099   6.348  1.00 48.47           C  \\nANISOU  734  C   GLU A 102     6226   4911   7278    656  -1413   1759       C  \\nATOM    735  O   GLU A 102      -4.308  25.131   5.810  1.00 54.55           O  \\nANISOU  735  O   GLU A 102     7139   5465   8121    771  -1588   1907       O  \\nATOM    736  CB  GLU A 102      -4.902  23.812   8.632  1.00 52.62           C  \\nANISOU  736  CB  GLU A 102     6354   5486   8152    915  -1173   1422       C  \\nATOM    737  CG  GLU A 102      -6.233  23.571   9.335  1.00 68.38           C  \\nANISOU  737  CG  GLU A 102     8046   7558  10375   1133  -1166   1303       C  \\nATOM    738  CD  GLU A 102      -6.159  23.821  10.836  1.00 77.05           C  \\nANISOU  738  CD  GLU A 102     9076   8606  11593   1193   -933   1094       C  \\nATOM    739  OE1 GLU A 102      -5.237  24.543  11.279  1.00 76.30           O  \\nANISOU  739  OE1 GLU A 102     9191   8343  11457   1121   -832   1060       O  \\nATOM    740  OE2 GLU A 102      -7.020  23.285  11.570  1.00 78.89           O  \\nANISOU  740  OE2 GLU A 102     9049   8981  11946   1286   -847    960       O  \\nATOM    741  N   LEU A 103      -2.656  23.662   6.203  1.00 42.86           N  \\nANISOU  741  N   LEU A 103     5634   4299   6353    414  -1273   1738       N  \\nATOM    742  CA  LEU A 103      -1.681  24.409   5.416  1.00 46.28           C  \\nANISOU  742  CA  LEU A 103     6361   4609   6614    232  -1294   1880       C  \\nATOM    743  C   LEU A 103      -1.540  23.832   4.014  1.00 46.47           C  \\nANISOU  743  C   LEU A 103     6483   4778   6397     85  -1408   2022       C  \\nATOM    744  O   LEU A 103      -0.736  24.303   3.226  1.00 49.18           O  \\nANISOU  744  O   LEU A 103     7068   5067   6551   -102  -1412   2146       O  \\nATOM    745  CB  LEU A 103      -0.335  24.410   6.136  1.00 43.50           C  \\nANISOU  745  CB  LEU A 103     6057   4294   6178     45  -1059   1759       C  \\nATOM    746  CG  LEU A 103      -0.393  25.153   7.474  1.00 48.75           C  \\nANISOU  746  CG  LEU A 103     6691   4790   7043    152   -956   1624       C  \\nATOM    747  CD1 LEU A 103       0.830  24.874   8.325  1.00 54.15           C  \\nANISOU  747  CD1 LEU A 103     7346   5588   7639    -26   -742   1477       C  \\nATOM    748  CD2 LEU A 103      -0.565  26.660   7.241  1.00 54.90           C  \\nANISOU  748  CD2 LEU A 103     7721   5225   7915    206  -1063   1744       C  \\nATOM    749  N   GLY A 104      -2.326  22.801   3.715  1.00 45.29           N  \\nANISOU  749  N   GLY A 104     6152   4817   6237    146  -1490   1994       N  \\nATOM    750  CA  GLY A 104      -2.259  22.144   2.424  1.00 47.24           C  \\nANISOU  750  CA  GLY A 104     6492   5217   6239     -1  -1593   2096       C  \\nATOM    751  C   GLY A 104      -0.904  21.505   2.170  1.00 48.63           C  \\nANISOU  751  C   GLY A 104     6760   5541   6175   -241  -1386   2029       C  \\nATOM    752  O   GLY A 104      -0.361  21.626   1.079  1.00 52.13           O  \\nANISOU  752  O   GLY A 104     7410   6014   6381   -417  -1418   2145       O  \\nATOM    753  N   LEU A 105      -0.349  20.841   3.182  1.00 39.04           N  \\nANISOU  753  N   LEU A 105     5391   4425   5016   -245  -1172   1842       N  \\nATOM    754  CA  LEU A 105       0.955  20.193   3.043  1.00 40.94           C  \\nANISOU  754  CA  LEU A 105     5669   4822   5063   -426   -972   1760       C  \\nATOM    755  C   LEU A 105       0.835  18.829   2.343  1.00 40.81           C  \\nANISOU  755  C   LEU A 105     5614   5005   4886   -479   -968   1706       C  \\nATOM    756  O   LEU A 105      -0.206  18.182   2.412  1.00 39.41           O  \\nANISOU  756  O   LEU A 105     5318   4866   4789   -378  -1072   1675       O  \\nATOM    757  CB  LEU A 105       1.635  20.061   4.409  1.00 36.04           C  \\nANISOU  757  CB  LEU A 105     4909   4224   4560   -394   -776   1597       C  \\nATOM    758  CG  LEU A 105       1.787  21.374   5.195  1.00 35.18           C  \\nANISOU  758  CG  LEU A 105     4853   3912   4600   -364   -762   1613       C  \\nATOM    759  CD1 LEU A 105       2.720  21.201   6.393  1.00 35.44           C  \\nANISOU  759  CD1 LEU A 105     4778   4015   4672   -399   -568   1455       C  \\nATOM    760  CD2 LEU A 105       2.270  22.519   4.296  1.00 36.37           C  \\nANISOU  760  CD2 LEU A 105     5260   3922   4636   -515   -823   1780       C  \\nATOM    761  N   LYS A 106       1.888  18.412   1.643  1.00 40.58           N  \\nANISOU  761  N   LYS A 106     5689   5102   4627   -649   -842   1686       N  \\nATOM    762  CA  LYS A 106       1.845  17.160   0.886  1.00 38.36           C  \\nANISOU  762  CA  LYS A 106     5420   4983   4171   -707   -823   1619       C  \\nATOM    763  C   LYS A 106       2.750  16.098   1.487  1.00 37.06           C  \\nANISOU  763  C   LYS A 106     5142   4948   3990   -697   -596   1433       C  \\nATOM    764  O   LYS A 106       2.448  14.912   1.387  1.00 34.61           O  \\nANISOU  764  O   LYS A 106     4793   4716   3642   -665   -579   1334       O  \\nATOM    765  CB  LYS A 106       2.231  17.360  -0.588  1.00 40.48           C  \\nANISOU  765  CB  LYS A 106     5920   5310   4151   -899   -860   1730       C  \\nATOM    766  CG  LYS A 106       1.346  18.306  -1.411  1.00 53.75           C  \\nANISOU  766  CG  LYS A 106     7762   6869   5791   -918  -1124   1947       C  \\nATOM    767  CD  LYS A 106      -0.136  18.072  -1.165  1.00 70.50           C  \\nANISOU  767  CD  LYS A 106     9747   8948   8092   -745  -1349   1966       C  \\nATOM    768  CE  LYS A 106      -1.003  19.036  -1.977  1.00 81.67           C  \\nANISOU  768  CE  LYS A 106    11301  10249   9481   -725  -1642   2194       C  \\nATOM    769  NZ  LYS A 106      -2.458  18.898  -1.659  1.00 82.99           N  \\nANISOU  769  NZ  LYS A 106    11271  10400   9861   -533  -1863   2206       N  \\nATOM    770  N   THR A 107       3.869  16.518   2.082  1.00 32.54           N  \\nANISOU  770  N   THR A 107     4526   4396   3440   -728   -434   1388       N  \\nATOM    771  CA  THR A 107       4.829  15.579   2.658  1.00 28.57           C  \\nANISOU  771  CA  THR A 107     3902   4028   2927   -693   -238   1226       C  \\nATOM    772  C   THR A 107       5.255  16.038   4.047  1.00 34.27           C  \\nANISOU  772  C   THR A 107     4481   4711   3829   -619   -171   1175       C  \\nATOM    773  O   THR A 107       5.693  17.175   4.239  1.00 34.02           O  \\nANISOU  773  O   THR A 107     4481   4622   3825   -701   -162   1236       O  \\nATOM    774  CB  THR A 107       6.092  15.416   1.777  1.00 29.25           C  \\nANISOU  774  CB  THR A 107     4052   4271   2790   -842    -74   1192       C  \\nATOM    775  OG1 THR A 107       6.722  16.684   1.609  1.00 30.42           O  \\nANISOU  775  OG1 THR A 107     4265   4396   2898   -992    -49   1290       O  \\nATOM    776  CG2 THR A 107       5.749  14.836   0.399  1.00 30.67           C  \\nANISOU  776  CG2 THR A 107     4396   4507   2748   -930   -114   1211       C  \\nATOM    777  N   VAL A 108       5.112  15.148   5.019  1.00 25.29           N  \\nANISOU  777  N   VAL A 108     3212   3599   2799   -481   -128   1064       N  \\nATOM    778  CA  VAL A 108       5.392  15.473   6.419  1.00 24.44           C  \\nANISOU  778  CA  VAL A 108     2980   3463   2842   -409    -82   1010       C  \\nATOM    779  C   VAL A 108       6.120  14.298   7.079  1.00 24.56           C  \\nANISOU  779  C   VAL A 108     2885   3601   2846   -320     35    885       C  \\nATOM    780  O   VAL A 108       5.701  13.148   6.961  1.00 26.16           O  \\nANISOU  780  O   VAL A 108     3099   3812   3029   -242     34    836       O  \\nATOM    781  CB  VAL A 108       4.067  15.779   7.204  1.00 26.57           C  \\nANISOU  781  CB  VAL A 108     3215   3587   3294   -302   -194   1030       C  \\nATOM    782  CG1 VAL A 108       4.356  16.141   8.646  1.00 22.84           C  \\nANISOU  782  CG1 VAL A 108     2644   3090   2943   -248   -134    961       C  \\nATOM    783  CG2 VAL A 108       3.311  16.925   6.542  1.00 29.45           C  \\nANISOU  783  CG2 VAL A 108     3682   3815   3693   -339   -333   1159       C  \\nATOM    784  N   ALA A 109       7.216  14.580   7.766  1.00 28.99           N  \\nANISOU  784  N   ALA A 109     3348   4251   3417   -334    123    837       N  \\nATOM    785  CA  ALA A 109       7.962  13.526   8.439  1.00 23.97           C  \\nANISOU  785  CA  ALA A 109     2599   3731   2777   -220    205    736       C  \\nATOM    786  C   ALA A 109       7.689  13.518   9.946  1.00 25.45           C  \\nANISOU  786  C   ALA A 109     2713   3868   3087   -130    173    705       C  \\nATOM    787  O   ALA A 109       7.541  14.564  10.563  1.00 24.82           O  \\nANISOU  787  O   ALA A 109     2625   3728   3078   -189    140    727       O  \\nATOM    788  CB  ALA A 109       9.436  13.676   8.171  1.00 24.13           C  \\nANISOU  788  CB  ALA A 109     2520   3937   2710   -285    318    693       C  \\nATOM    789  N   PHE A 110       7.637  12.329  10.529  1.00 23.66           N  \\nANISOU  789  N   PHE A 110     2462   3656   2872      5    188    652       N  \\nATOM    790  CA  PHE A 110       7.390  12.177  11.952  1.00 27.87           C  \\nANISOU  790  CA  PHE A 110     2953   4153   3481     78    164    629       C  \\nATOM    791  C   PHE A 110       8.428  11.208  12.460  1.00 31.76           C  \\nANISOU  791  C   PHE A 110     3375   4763   3930    197    204    578       C  \\nATOM    792  O   PHE A 110       8.922  10.371  11.699  1.00 26.36           O  \\nANISOU  792  O   PHE A 110     2700   4133   3185    265    252    549       O  \\nATOM    793  CB  PHE A 110       6.065  11.449  12.228  1.00 23.30           C  \\nANISOU  793  CB  PHE A 110     2452   3447   2952    133    125    636       C  \\nATOM    794  CG  PHE A 110       4.819  12.294  12.147  1.00 23.54           C  \\nANISOU  794  CG  PHE A 110     2508   3367   3071     72     66    677       C  \\nATOM    795  CD1 PHE A 110       4.265  12.645  10.920  1.00 20.25           C  \\nANISOU  795  CD1 PHE A 110     2145   2908   2642     13     13    730       C  \\nATOM    796  CD2 PHE A 110       4.141  12.647  13.309  1.00 21.20           C  \\nANISOU  796  CD2 PHE A 110     2180   3011   2862     89     62    657       C  \\nATOM    797  CE1 PHE A 110       3.082  13.360  10.863  1.00 29.36           C  \\nANISOU  797  CE1 PHE A 110     3298   3962   3894     -2    -64    772       C  \\nATOM    798  CE2 PHE A 110       2.947  13.363  13.256  1.00 24.45           C  \\nANISOU  798  CE2 PHE A 110     2585   3327   3378     73     17    677       C  \\nATOM    799  CZ  PHE A 110       2.419  13.719  12.038  1.00 27.71           C  \\nANISOU  799  CZ  PHE A 110     3028   3698   3801     41    -56    739       C  \\nATOM    800  N   THR A 111       8.714  11.289  13.759  1.00 24.67           N  \\nANISOU  800  N   THR A 111     2419   3898   3058    234    180    564       N  \\nATOM    801  CA  THR A 111       9.303  10.177  14.485  1.00 24.93           C  \\nANISOU  801  CA  THR A 111     2425   3986   3061    387    172    543       C  \\nATOM    802  C   THR A 111       8.218   9.590  15.399  1.00 26.04           C  \\nANISOU  802  C   THR A 111     2684   3984   3224    425    139    565       C  \\nATOM    803  O   THR A 111       7.074  10.041  15.383  1.00 24.64           O  \\nANISOU  803  O   THR A 111     2566   3699   3097    339    136    579       O  \\nATOM    804  CB  THR A 111      10.527  10.617  15.298  1.00 34.73           C  \\nANISOU  804  CB  THR A 111     3511   5404   4281    391    148    521       C  \\nATOM    805  OG1 THR A 111      10.157  11.678  16.185  1.00 32.26           O  \\nANISOU  805  OG1 THR A 111     3205   5061   3992    264    114    522       O  \\nATOM    806  CG2 THR A 111      11.630  11.095  14.367  1.00 36.95           C  \\nANISOU  806  CG2 THR A 111     3652   5855   4534    331    204    491       C  \\nATOM    807  N   ALA A 112       8.564   8.573  16.177  1.00 26.27           N  \\nANISOU  807  N   ALA A 112     2748   4017   3218    554    115    572       N  \\nATOM    808  CA  ALA A 112       7.585   7.911  17.034  1.00 27.43           C  \\nANISOU  808  CA  ALA A 112     3032   4030   3358    561    101    602       C  \\nATOM    809  C   ALA A 112       7.337   8.753  18.284  1.00 27.32           C  \\nANISOU  809  C   ALA A 112     2992   4048   3340    473     82    601       C  \\nATOM    810  O   ALA A 112       8.008   8.574  19.290  1.00 29.26           O  \\nANISOU  810  O   ALA A 112     3227   4370   3523    523     35    613       O  \\nATOM    811  CB  ALA A 112       8.071   6.491  17.422  1.00 23.05           C  \\nANISOU  811  CB  ALA A 112     2574   3435   2749    730     74    629       C  \\nATOM    812  N   LEU A 113       6.366   9.659  18.200  1.00 30.13           N  \\nANISOU  812  N   LEU A 113     3344   4345   3760    352    114    580       N  \\nATOM    813  CA  LEU A 113       6.114  10.656  19.238  1.00 26.84           C  \\nANISOU  813  CA  LEU A 113     2904   3945   3348    265    123    546       C  \\nATOM    814  C   LEU A 113       5.925  10.047  20.622  1.00 34.93           C  \\nANISOU  814  C   LEU A 113     4016   4970   4287    275    121    555       C  \\nATOM    815  O   LEU A 113       5.099   9.154  20.812  1.00 30.00           O  \\nANISOU  815  O   LEU A 113     3494   4257   3648    282    152    583       O  \\nATOM    816  CB  LEU A 113       4.878  11.472  18.876  1.00 28.22           C  \\nANISOU  816  CB  LEU A 113     3076   4019   3627    186    165    521       C  \\nATOM    817  CG  LEU A 113       4.917  12.303  17.598  1.00 28.26           C  \\nANISOU  817  CG  LEU A 113     3031   4001   3704    153    149    532       C  \\nATOM    818  CD1 LEU A 113       3.579  13.034  17.395  1.00 27.32           C  \\nANISOU  818  CD1 LEU A 113     2910   3772   3699    118    163    519       C  \\nATOM    819  CD2 LEU A 113       6.044  13.288  17.691  1.00 31.64           C  \\nANISOU  819  CD2 LEU A 113     3402   4510   4108     98    134    511       C  \\nATOM    820  N   GLY A 114       6.700  10.525  21.591  1.00 31.07           N  \\nANISOU  820  N   GLY A 114     3498   4584   3722    251     82    534       N  \\nATOM    821  CA  GLY A 114       6.579  10.029  22.942  1.00 29.12           C  \\nANISOU  821  CA  GLY A 114     3356   4352   3356    242     69    551       C  \\nATOM    822  C   GLY A 114       7.323   8.736  23.222  1.00 28.75           C  \\nANISOU  822  C   GLY A 114     3376   4334   3213    376    -17    638       C  \\nATOM    823  O   GLY A 114       7.342   8.273  24.362  1.00 30.82           O  \\nANISOU  823  O   GLY A 114     3751   4609   3348    371    -54    679       O  \\nATOM    824  N   ALA A 115       7.952   8.140  22.212  1.00 28.58           N  \\nANISOU  824  N   ALA A 115     3301   4316   3242    504    -50    668       N  \\nATOM    825  CA  ALA A 115       8.673   6.888  22.476  1.00 34.64           C  \\nANISOU  825  CA  ALA A 115     4140   5083   3938    678   -135    745       C  \\nATOM    826  C   ALA A 115      10.109   7.089  22.972  1.00 42.60           C  \\nANISOU  826  C   ALA A 115     5006   6291   4888    769   -258    755       C  \\nATOM    827  O   ALA A 115      10.705   6.187  23.552  1.00 53.26           O  \\nANISOU  827  O   ALA A 115     6416   7658   6162    921   -362    831       O  \\nATOM    828  CB  ALA A 115       8.641   5.972  21.270  1.00 30.33           C  \\nANISOU  828  CB  ALA A 115     3632   4430   3463    799   -103    756       C  \\nATOM    829  N   TRP A 116      10.674   8.267  22.750  1.00 51.62           N  \\nANISOU  829  N   TRP A 116     5962   7585   6066    672   -257    684       N  \\nATOM    830  CA  TRP A 116      12.020   8.534  23.247  1.00 53.81           C  \\nANISOU  830  CA  TRP A 116     6069   8089   6287    716   -378    681       C  \\nATOM    831  C   TRP A 116      11.958   9.278  24.577  1.00 48.54           C  \\nANISOU  831  C   TRP A 116     5445   7498   5502    557   -436    662       C  \\nATOM    832  O   TRP A 116      12.166   8.690  25.646  1.00 42.89           O  \\nANISOU  832  O   TRP A 116     4823   6818   4655    614   -546    728       O  \\nATOM    833  CB  TRP A 116      12.845   9.310  22.211  1.00 63.08           C  \\nANISOU  833  CB  TRP A 116     7009   9410   7549    674   -343    612       C  \\nATOM    834  CG  TRP A 116      14.196   9.769  22.706  1.00 76.29           C  \\nANISOU  834  CG  TRP A 116     8460  11354   9171    659   -458    589       C  \\nATOM    835  CD1 TRP A 116      14.601  11.059  22.896  1.00 79.44           C  \\nANISOU  835  CD1 TRP A 116     8740  11886   9556    438   -462    516       C  \\nATOM    836  CD2 TRP A 116      15.310   8.942  23.076  1.00 87.53           C  \\nANISOU  836  CD2 TRP A 116     9748  12952  10557    868   -598    636       C  \\nATOM    837  NE1 TRP A 116      15.896  11.089  23.358  1.00 83.74           N  \\nANISOU  837  NE1 TRP A 116     9067  12702  10048    465   -594    509       N  \\nATOM    838  CE2 TRP A 116      16.355   9.804  23.478  1.00 88.41           C  \\nANISOU  838  CE2 TRP A 116     9625  13335  10630    745   -686    585       C  \\nATOM    839  CE3 TRP A 116      15.527   7.560  23.108  1.00 96.71           C  \\nANISOU  839  CE3 TRP A 116    10970  14056  11720   1152   -664    716       C  \\nATOM    840  CZ2 TRP A 116      17.596   9.329  23.905  1.00 94.95           C  \\nANISOU  840  CZ2 TRP A 116    10235  14416  11426    906   -849    613       C  \\nATOM    841  CZ3 TRP A 116      16.766   7.089  23.536  1.00101.24           C  \\nANISOU  841  CZ3 TRP A 116    11350  14848  12268   1345   -824    749       C  \\nATOM    842  CH2 TRP A 116      17.781   7.973  23.928  1.00 99.66           C  \\nANISOU  842  CH2 TRP A 116    10876  14955  12035   1225   -920    699       C  \\nATOM    843  N   VAL A 117      11.638  10.564  24.507  1.00 44.15           N  \\nANISOU  843  N   VAL A 117     4850   6945   4980    357   -362    570       N  \\nATOM    844  CA  VAL A 117      11.627  11.407  25.690  1.00 46.95           C  \\nANISOU  844  CA  VAL A 117     5251   7368   5222    187   -396    514       C  \\nATOM    845  C   VAL A 117      10.470  11.027  26.618  1.00 51.79           C  \\nANISOU  845  C   VAL A 117     6096   7838   5743    150   -339    532       C  \\nATOM    846  O   VAL A 117      10.579  11.140  27.838  1.00 59.01           O  \\nANISOU  846  O   VAL A 117     7099   8827   6495     71   -399    524       O  \\nATOM    847  CB  VAL A 117      11.621  12.923  25.316  1.00 64.39           C  \\nANISOU  847  CB  VAL A 117     7384   9578   7502    -13   -323    399       C  \\nATOM    848  CG1 VAL A 117      10.320  13.321  24.628  1.00 66.97           C  \\nANISOU  848  CG1 VAL A 117     7810   9681   7955    -49   -170    367       C  \\nATOM    849  CG2 VAL A 117      11.868  13.780  26.538  1.00 67.39           C  \\nANISOU  849  CG2 VAL A 117     7808  10047   7749   -192   -373    318       C  \\nATOM    850  N   GLY A 118       9.382  10.537  26.037  1.00 46.38           N  \\nANISOU  850  N   GLY A 118     5505   6970   5147    193   -223    554       N  \\nATOM    851  CA  GLY A 118       8.213  10.147  26.804  1.00 37.41           C  \\nANISOU  851  CA  GLY A 118     4562   5715   3939    135   -137    564       C  \\nATOM    852  C   GLY A 118       8.299   8.786  27.481  1.00 35.85           C  \\nANISOU  852  C   GLY A 118     4526   5495   3602    231   -215    691       C  \\nATOM    853  O   GLY A 118       7.593   8.535  28.445  1.00 36.94           O  \\nANISOU  853  O   GLY A 118     4837   5589   3610    138   -164    706       O  \\nATOM    854  N   GLY A 119       9.154   7.899  26.985  1.00 41.52           N  \\nANISOU  854  N   GLY A 119     5202   6235   4340    417   -328    782       N  \\nATOM    855  CA  GLY A 119       9.297   6.580  27.591  1.00 40.14           C  \\nANISOU  855  CA  GLY A 119     5211   5999   4042    539   -422    918       C  \\nATOM    856  C   GLY A 119       8.310   5.497  27.155  1.00 38.50           C  \\nANISOU  856  C   GLY A 119     5192   5563   3874    570   -328    980       C  \\nATOM    857  O   GLY A 119       8.374   4.377  27.645  1.00 44.92           O  \\nANISOU  857  O   GLY A 119     6205   6281   4582    657   -399   1102       O  \\nATOM    858  N   LEU A 120       7.411   5.808  26.226  1.00 31.59           N  \\nANISOU  858  N   LEU A 120     4264   4594   3144    493   -182    903       N  \\nATOM    859  CA  LEU A 120       6.394   4.843  25.817  1.00 32.00           C  \\nANISOU  859  CA  LEU A 120     4481   4449   3228    472    -93    945       C  \\nATOM    860  C   LEU A 120       6.948   3.809  24.851  1.00 27.84           C  \\nANISOU  860  C   LEU A 120     3987   3820   2772    670   -152    999       C  \\nATOM    861  O   LEU A 120       7.875   4.087  24.106  1.00 32.75           O  \\nANISOU  861  O   LEU A 120     4432   4534   3476    803   -205    963       O  \\nATOM    862  CB  LEU A 120       5.202   5.556  25.177  1.00 26.99           C  \\nANISOU  862  CB  LEU A 120     3757   3777   2722    320     60    842       C  \\nATOM    863  CG  LEU A 120       4.540   6.592  26.064  1.00 29.82           C  \\nANISOU  863  CG  LEU A 120     4081   4212   3038    150    152    760       C  \\nATOM    864  CD1 LEU A 120       3.610   7.478  25.258  1.00 24.45           C  \\nANISOU  864  CD1 LEU A 120     3250   3510   2529     74    264    655       C  \\nATOM    865  CD2 LEU A 120       3.805   5.881  27.182  1.00 29.54           C  \\nANISOU  865  CD2 LEU A 120     4259   4122   2843     31    215    814       C  \\nATOM    866  N   PRO A 121       6.365   2.605  24.844  1.00 35.67           N  \\nANISOU  866  N   PRO A 121     5214   4613   3726    675   -128   1073       N  \\nATOM    867  CA  PRO A 121       6.901   1.618  23.897  1.00 33.71           C  \\nANISOU  867  CA  PRO A 121     5023   4239   3547    876   -172   1100       C  \\nATOM    868  C   PRO A 121       6.566   1.963  22.452  1.00 33.53           C  \\nANISOU  868  C   PRO A 121     4858   4207   3674    849    -82    993       C  \\nATOM    869  O   PRO A 121       5.447   2.389  22.140  1.00 32.15           O  \\nANISOU  869  O   PRO A 121     4660   4008   3548    657     19    939       O  \\nATOM    870  CB  PRO A 121       6.275   0.280  24.348  1.00 34.13           C  \\nANISOU  870  CB  PRO A 121     5415   4046   3505    845   -164   1206       C  \\nATOM    871  CG  PRO A 121       5.230   0.620  25.341  1.00 34.65           C  \\nANISOU  871  CG  PRO A 121     5567   4132   3467    585    -79   1221       C  \\nATOM    872  CD  PRO A 121       5.442   2.023  25.832  1.00 34.22           C  \\nANISOU  872  CD  PRO A 121     5278   4318   3407    516    -74   1145       C  \\nATOM    873  N   ALA A 122       7.552   1.792  21.578  1.00 27.86           N  \\nANISOU  873  N   ALA A 122     4035   3527   3024   1044   -123    961       N  \\nATOM    874  CA  ALA A 122       7.433   2.176  20.174  1.00 33.17           C  \\nANISOU  874  CA  ALA A 122     4576   4221   3804   1022    -49    863       C  \\nATOM    875  C   ALA A 122       6.267   1.494  19.456  1.00 32.69           C  \\nANISOU  875  C   ALA A 122     4689   3965   3768    903     31    843       C  \\nATOM    876  O   ALA A 122       5.566   2.138  18.690  1.00 31.13           O  \\nANISOU  876  O   ALA A 122     4393   3802   3632    762     90    780       O  \\nATOM    877  CB  ALA A 122       8.738   1.894  19.439  1.00 32.75           C  \\nANISOU  877  CB  ALA A 122     4411   4241   3791   1258    -84    829       C  \\nATOM    878  N   GLU A 123       6.073   0.196  19.704  1.00 32.54           N  \\nANISOU  878  N   GLU A 123     4934   3735   3694    955     19    901       N  \\nATOM    879  CA  GLU A 123       5.025  -0.575  19.037  1.00 34.27           C  \\nANISOU  879  CA  GLU A 123     5340   3758   3922    818     87    875       C  \\nATOM    880  C   GLU A 123       3.622  -0.016  19.308  1.00 29.10           C  \\nANISOU  880  C   GLU A 123     4642   3137   3276    531    154    864       C  \\nATOM    881  O   GLU A 123       2.806   0.100  18.394  1.00 27.71           O  \\nANISOU  881  O   GLU A 123     4425   2948   3156    395    200    800       O  \\nATOM    882  CB  GLU A 123       5.091  -2.065  19.418  1.00 32.14           C  \\nANISOU  882  CB  GLU A 123     5407   3224   3582    905     60    950       C  \\nATOM    883  CG  GLU A 123       6.508  -2.636  19.622  1.00 59.70           C  \\nANISOU  883  CG  GLU A 123     8939   6682   7061   1240    -33    993       C  \\nATOM    884  CD  GLU A 123       7.051  -2.441  21.054  1.00 65.39           C  \\nANISOU  884  CD  GLU A 123     9653   7493   7699   1316   -138   1114       C  \\nATOM    885  OE1 GLU A 123       7.699  -1.405  21.316  1.00 49.62           O  \\nANISOU  885  OE1 GLU A 123     7382   5754   5719   1359   -180   1094       O  \\nATOM    886  OE2 GLU A 123       6.832  -3.325  21.918  1.00 71.34           O  \\nANISOU  886  OE2 GLU A 123    10694   8057   8353   1314   -184   1234       O  \\nATOM    887  N   ALA A 124       3.337   0.329  20.559  1.00 30.49           N  \\nANISOU  887  N   ALA A 124     4820   3372   3392    444    159    921       N  \\nATOM    888  CA  ALA A 124       2.038   0.923  20.882  1.00 27.91           C  \\nANISOU  888  CA  ALA A 124     4413   3107   3085    197    246    889       C  \\nATOM    889  C   ALA A 124       1.898   2.319  20.277  1.00 26.44           C  \\nANISOU  889  C   ALA A 124     3936   3097   3012    176    259    804       C  \\nATOM    890  O   ALA A 124       0.836   2.671  19.747  1.00 25.83           O  \\nANISOU  890  O   ALA A 124     3762   3040   3011     32    308    752       O  \\nATOM    891  CB  ALA A 124       1.825   0.962  22.387  1.00 27.36           C  \\nANISOU  891  CB  ALA A 124     4430   3066   2900    109    271    954       C  \\nATOM    892  N   VAL A 125       2.962   3.115  20.360  1.00 23.98           N  \\nANISOU  892  N   VAL A 125     3488   2912   2711    315    206    794       N  \\nATOM    893  CA  VAL A 125       2.961   4.470  19.793  1.00 22.58           C  \\nANISOU  893  CA  VAL A 125     3078   2869   2631    295    212    726       C  \\nATOM    894  C   VAL A 125       2.699   4.445  18.294  1.00 23.36           C  \\nANISOU  894  C   VAL A 125     3131   2939   2807    288    209    687       C  \\nATOM    895  O   VAL A 125       1.870   5.208  17.786  1.00 24.30           O  \\nANISOU  895  O   VAL A 125     3131   3095   3007    189    226    651       O  \\nATOM    896  CB  VAL A 125       4.277   5.242  20.103  1.00 24.22           C  \\nANISOU  896  CB  VAL A 125     3166   3214   2821    415    154    723       C  \\nATOM    897  CG1 VAL A 125       4.442   6.469  19.191  1.00 21.24           C  \\nANISOU  897  CG1 VAL A 125     2602   2931   2539    394    156    665       C  \\nATOM    898  CG2 VAL A 125       4.317   5.658  21.570  1.00 26.26           C  \\nANISOU  898  CG2 VAL A 125     3439   3539   2999    365    154    738       C  \\nATOM    899  N   LEU A 126       3.398   3.562  17.583  1.00 23.60           N  \\nANISOU  899  N   LEU A 126     3261   2900   2806    403    183    692       N  \\nATOM    900  CA  LEU A 126       3.223   3.455  16.141  1.00 22.40           C  \\nANISOU  900  CA  LEU A 126     3100   2723   2687    384    184    645       C  \\nATOM    901  C   LEU A 126       1.790   3.028  15.775  1.00 24.23           C  \\nANISOU  901  C   LEU A 126     3403   2866   2939    202    203    632       C  \\nATOM    902  O   LEU A 126       1.229   3.496  14.801  1.00 27.46           O  \\nANISOU  902  O   LEU A 126     3727   3316   3391    121    184    601       O  \\nATOM    903  CB  LEU A 126       4.257   2.493  15.543  1.00 23.43           C  \\nANISOU  903  CB  LEU A 126     3345   2788   2769    554    179    627       C  \\nATOM    904  CG  LEU A 126       5.705   2.996  15.635  1.00 28.91           C  \\nANISOU  904  CG  LEU A 126     3896   3627   3462    728    161    622       C  \\nATOM    905  CD1 LEU A 126       6.721   1.978  15.101  1.00 34.70           C  \\nANISOU  905  CD1 LEU A 126     4714   4305   4164    934    171    588       C  \\nATOM    906  CD2 LEU A 126       5.856   4.290  14.902  1.00 25.88           C  \\nANISOU  906  CD2 LEU A 126     3318   3397   3118    655    170    589       C  \\nATOM    907  N   ARG A 127       1.207   2.135  16.562  1.00 23.62           N  \\nANISOU  907  N   ARG A 127     3479   2675   2818    125    232    663       N  \\nATOM    908  CA  ARG A 127      -0.176   1.747  16.331  1.00 24.20           C  \\nANISOU  908  CA  ARG A 127     3587   2697   2911    -86    257    645       C  \\nATOM    909  C   ARG A 127      -1.158   2.914  16.494  1.00 24.49           C  \\nANISOU  909  C   ARG A 127     3383   2878   3043   -198    270    623       C  \\nATOM    910  O   ARG A 127      -2.025   3.106  15.641  1.00 25.88           O  \\nANISOU  910  O   ARG A 127     3466   3090   3276   -305    239    591       O  \\nATOM    911  CB  ARG A 127      -0.583   0.586  17.233  1.00 25.70           C  \\nANISOU  911  CB  ARG A 127     4004   2739   3024   -182    303    689       C  \\nATOM    912  CG  ARG A 127      -1.962   0.036  16.901  1.00 28.46           C  \\nANISOU  912  CG  ARG A 127     4391   3038   3383   -437    334    660       C  \\nATOM    913  CD  ARG A 127      -2.241  -1.285  17.609  1.00 32.34           C  \\nANISOU  913  CD  ARG A 127     5176   3336   3775   -557    383    709       C  \\nATOM    914  NE  ARG A 127      -1.383  -2.366  17.136  1.00 32.12           N  \\nANISOU  914  NE  ARG A 127     5429   3093   3683   -422    346    719       N  \\nATOM    915  CZ  ARG A 127      -1.603  -3.077  16.034  1.00 36.39           C  \\nANISOU  915  CZ  ARG A 127     6101   3511   4213   -489    329    655       C  \\nATOM    916  NH1 ARG A 127      -2.654  -2.827  15.271  1.00 30.53           N  \\nANISOU  916  NH1 ARG A 127     5226   2862   3514   -703    320    590       N  \\nATOM    917  NH2 ARG A 127      -0.765  -4.045  15.697  1.00 41.57           N  \\nANISOU  917  NH2 ARG A 127     7025   3955   4815   -331    314    648       N  \\nATOM    918  N   VAL A 128      -1.045   3.703  17.564  1.00 26.22           N  \\nANISOU  918  N   VAL A 128     3503   3182   3279   -167    307    635       N  \\nATOM    919  CA  VAL A 128      -1.965   4.833  17.655  1.00 27.05           C  \\nANISOU  919  CA  VAL A 128     3384   3401   3492   -236    328    595       C  \\nATOM    920  C   VAL A 128      -1.723   5.881  16.569  1.00 28.15           C  \\nANISOU  920  C   VAL A 128     3377   3603   3716   -156    251    580       C  \\nATOM    921  O   VAL A 128      -2.685   6.442  16.055  1.00 24.24           O  \\nANISOU  921  O   VAL A 128     2736   3157   3317   -216    223    559       O  \\nATOM    922  CB  VAL A 128      -2.165   5.462  19.090  1.00 34.56           C  \\nANISOU  922  CB  VAL A 128     4273   4420   4440   -255    416    580       C  \\nATOM    923  CG1 VAL A 128      -1.598   4.599  20.189  1.00 30.71           C  \\nANISOU  923  CG1 VAL A 128     3991   3871   3808   -257    453    631       C  \\nATOM    924  CG2 VAL A 128      -1.690   6.891  19.161  1.00 23.99           C  \\nANISOU  924  CG2 VAL A 128     2783   3161   3170   -150    396    548       C  \\nATOM    925  N   MET A 129      -0.456   6.132  16.217  1.00 24.39           N  \\nANISOU  925  N   MET A 129     2937   3131   3199    -28    213    596       N  \\nATOM    926  CA  MET A 129      -0.139   7.032  15.105  1.00 25.88           C  \\nANISOU  926  CA  MET A 129     3033   3368   3433     15    148    596       C  \\nATOM    927  C   MET A 129      -0.758   6.527  13.812  1.00 24.20           C  \\nANISOU  927  C   MET A 129     2853   3128   3212    -58     90    594       C  \\nATOM    928  O   MET A 129      -1.392   7.286  13.072  1.00 25.69           O  \\nANISOU  928  O   MET A 129     2935   3358   3466    -95     23    604       O  \\nATOM    929  CB  MET A 129       1.377   7.230  14.952  1.00 22.49           C  \\nANISOU  929  CB  MET A 129     2630   2971   2942    132    142    605       C  \\nATOM    930  CG  MET A 129       1.990   7.974  16.154  1.00 22.45           C  \\nANISOU  930  CG  MET A 129     2566   3023   2942    177    169    602       C  \\nATOM    931  SD  MET A 129       3.751   8.254  15.979  1.00 24.28           S  \\nANISOU  931  SD  MET A 129     2770   3343   3112    283    151    605       S  \\nATOM    932  CE  MET A 129       3.777   9.585  14.776  1.00 19.43           C  \\nANISOU  932  CE  MET A 129     2061   2768   2555    226    123    608       C  \\nATOM    933  N   LEU A 130      -0.608   5.235  13.560  1.00 22.65           N  \\nANISOU  933  N   LEU A 130     2823   2851   2931    -82    104    583       N  \\nATOM    934  CA  LEU A 130      -1.185   4.634  12.357  1.00 26.95           C  \\nANISOU  934  CA  LEU A 130     3436   3363   3440   -181     51    561       C  \\nATOM    935  C   LEU A 130      -2.689   4.899  12.260  1.00 25.77           C  \\nANISOU  935  C   LEU A 130     3153   3264   3374   -330      2    559       C  \\nATOM    936  O   LEU A 130      -3.195   5.275  11.195  1.00 27.09           O  \\nANISOU  936  O   LEU A 130     3256   3485   3554   -384    -96    564       O  \\nATOM    937  CB  LEU A 130      -0.906   3.132  12.332  1.00 23.32           C  \\nANISOU  937  CB  LEU A 130     3210   2767   2884   -195     92    533       C  \\nATOM    938  CG  LEU A 130      -1.668   2.342  11.273  1.00 29.08           C  \\nANISOU  938  CG  LEU A 130     4048   3441   3561   -349     46    489       C  \\nATOM    939  CD1 LEU A 130      -1.321   2.880   9.895  1.00 30.22           C  \\nANISOU  939  CD1 LEU A 130     4165   3659   3659   -329    -19    472       C  \\nATOM    940  CD2 LEU A 130      -1.361   0.841  11.367  1.00 25.81           C  \\nANISOU  940  CD2 LEU A 130     3912   2840   3056   -357    100    452       C  \\nATOM    941  N   GLU A 131      -3.390   4.720  13.377  1.00 26.78           N  \\nANISOU  941  N   GLU A 131     3230   3393   3551   -396     68    554       N  \\nATOM    942  CA  GLU A 131      -4.839   4.928  13.437  1.00 26.90           C  \\nANISOU  942  CA  GLU A 131     3069   3489   3662   -533     47    536       C  \\nATOM    943  C   GLU A 131      -5.216   6.378  13.146  1.00 27.04           C  \\nANISOU  943  C   GLU A 131     2855   3610   3807   -448    -23    550       C  \\nATOM    944  O   GLU A 131      -6.196   6.640  12.456  1.00 27.62           O  \\nANISOU  944  O   GLU A 131     2784   3759   3951   -512   -120    550       O  \\nATOM    945  CB  GLU A 131      -5.394   4.495  14.802  1.00 25.07           C  \\nANISOU  945  CB  GLU A 131     2830   3254   3440   -625    173    518       C  \\nATOM    946  CG  GLU A 131      -5.355   2.992  15.017  1.00 29.66           C  \\nANISOU  946  CG  GLU A 131     3659   3706   3902   -756    224    520       C  \\nATOM    947  CD  GLU A 131      -5.790   2.568  16.411  1.00 31.87           C  \\nANISOU  947  CD  GLU A 131     3978   3975   4155   -864    355    525       C  \\nATOM    948  OE1 GLU A 131      -6.278   3.432  17.172  1.00 26.88           O  \\nANISOU  948  OE1 GLU A 131     3146   3464   3602   -854    420    503       O  \\nATOM    949  OE2 GLU A 131      -5.635   1.364  16.738  1.00 31.26           O  \\nANISOU  949  OE2 GLU A 131     4154   3755   3967   -959    399    549       O  \\nATOM    950  N   GLU A 132      -4.430   7.320  13.653  1.00 24.56           N  \\nANISOU  950  N   GLU A 132     2515   3294   3522   -304     11    563       N  \\nATOM    951  CA  GLU A 132      -4.697   8.731  13.369  1.00 26.22           C  \\nANISOU  951  CA  GLU A 132     2559   3551   3853   -211    -56    581       C  \\nATOM    952  C   GLU A 132      -4.324   9.117  11.934  1.00 29.33           C  \\nANISOU  952  C   GLU A 132     2999   3940   4207   -185   -192    638       C  \\nATOM    953  O   GLU A 132      -5.040   9.885  11.287  1.00 23.92           O  \\nANISOU  953  O   GLU A 132     2191   3290   3608   -166   -308    673       O  \\nATOM    954  CB  GLU A 132      -3.996   9.641  14.381  1.00 22.26           C  \\nANISOU  954  CB  GLU A 132     2044   3030   3383   -101     28    566       C  \\nATOM    955  CG  GLU A 132      -4.612   9.557  15.773  1.00 24.41           C  \\nANISOU  955  CG  GLU A 132     2244   3332   3699   -134    159    503       C  \\nATOM    956  CD  GLU A 132      -6.119   9.818  15.766  1.00 31.62           C  \\nANISOU  956  CD  GLU A 132     2939   4321   4755   -174    156    463       C  \\nATOM    957  OE1 GLU A 132      -6.574  10.755  15.075  1.00 37.31           O  \\nANISOU  957  OE1 GLU A 132     3524   5055   5596    -89     53    480       O  \\nATOM    958  OE2 GLU A 132      -6.857   9.083  16.450  1.00 32.93           O  \\nANISOU  958  OE2 GLU A 132     3062   4537   4911   -292    254    419       O  \\nATOM    959  N   ILE A 133      -3.218   8.579  11.432  1.00 28.43           N  \\nANISOU  959  N   ILE A 133     3060   3786   3956   -180   -180    648       N  \\nATOM    960  CA  ILE A 133      -2.754   8.947  10.092  1.00 31.09           C  \\nANISOU  960  CA  ILE A 133     3462   4132   4219   -177   -277    696       C  \\nATOM    961  C   ILE A 133      -3.733   8.479   9.011  1.00 32.03           C  \\nANISOU  961  C   ILE A 133     3579   4286   4304   -292   -404    706       C  \\nATOM    962  O   ILE A 133      -3.937   9.159   8.007  1.00 32.24           O  \\nANISOU  962  O   ILE A 133     3590   4345   4316   -297   -533    768       O  \\nATOM    963  CB  ILE A 133      -1.309   8.430   9.814  1.00 30.71           C  \\nANISOU  963  CB  ILE A 133     3578   4062   4030   -141   -203    679       C  \\nATOM    964  CG1 ILE A 133      -0.314   9.145  10.736  1.00 23.52           C  \\nANISOU  964  CG1 ILE A 133     2630   3155   3152    -41   -122    680       C  \\nATOM    965  CG2 ILE A 133      -0.917   8.661   8.353  1.00 28.05           C  \\nANISOU  965  CG2 ILE A 133     3324   3753   3582   -178   -276    712       C  \\nATOM    966  CD1 ILE A 133       1.020   8.430  10.895  1.00 20.60           C  \\nANISOU  966  CD1 ILE A 133     2358   2789   2678     20    -36    647       C  \\nATOM    967  N   LYS A 134      -4.364   7.334   9.240  1.00 29.08           N  \\nANISOU  967  N   LYS A 134     3232   3906   3910   -401   -377    652       N  \\nATOM    968  CA  LYS A 134      -5.349   6.805   8.307  1.00 28.24           C  \\nANISOU  968  CA  LYS A 134     3115   3849   3766   -550   -501    646       C  \\nATOM    969  C   LYS A 134      -6.550   7.729   8.130  1.00 31.54           C  \\nANISOU  969  C   LYS A 134     3288   4370   4328   -545   -642    692       C  \\nATOM    970  O   LYS A 134      -7.283   7.611   7.153  1.00 31.05           O  \\nANISOU  970  O   LYS A 134     3189   4380   4231   -644   -801    713       O  \\nATOM    971  CB  LYS A 134      -5.833   5.421   8.746  1.00 39.00           C  \\nANISOU  971  CB  LYS A 134     4557   5170   5090   -700   -430    573       C  \\nATOM    972  CG  LYS A 134      -4.820   4.294   8.551  1.00 48.78           C  \\nANISOU  972  CG  LYS A 134     6079   6282   6174   -708   -340    525       C  \\nATOM    973  CD  LYS A 134      -5.299   2.981   9.187  1.00 51.15           C  \\nANISOU  973  CD  LYS A 134     6495   6491   6449   -852   -260    469       C  \\nATOM    974  CE  LYS A 134      -6.540   2.423   8.489  1.00 58.91           C  \\nANISOU  974  CE  LYS A 134     7446   7526   7409  -1092   -367    433       C  \\nATOM    975  NZ  LYS A 134      -7.078   1.185   9.140  1.00 62.17           N  \\nANISOU  975  NZ  LYS A 134     7981   7843   7798  -1280   -281    382       N  \\nATOM    976  N   LYS A 135      -6.758   8.653   9.062  1.00 31.22           N  \\nANISOU  976  N   LYS A 135     3079   4338   4446   -420   -591    702       N  \\nATOM    977  CA  LYS A 135      -7.901   9.563   8.959  1.00 31.82           C  \\nANISOU  977  CA  LYS A 135     2902   4499   4688   -364   -715    735       C  \\nATOM    978  C   LYS A 135      -7.620  10.819   8.128  1.00 33.26           C  \\nANISOU  978  C   LYS A 135     3097   4651   4889   -234   -862    839       C  \\nATOM    979  O   LYS A 135      -8.550  11.512   7.737  1.00 35.28           O  \\nANISOU  979  O   LYS A 135     3178   4964   5263   -172  -1020    890       O  \\nATOM    980  CB  LYS A 135      -8.372   9.984  10.342  1.00 33.84           C  \\nANISOU  980  CB  LYS A 135     2975   4771   5112   -284   -577    676       C  \\nATOM    981  CG  LYS A 135      -8.895   8.845  11.196  1.00 34.82           C  \\nANISOU  981  CG  LYS A 135     3066   4942   5224   -440   -439    590       C  \\nATOM    982  CD  LYS A 135      -9.002   9.296  12.645  1.00 31.75           C  \\nANISOU  982  CD  LYS A 135     2570   4553   4940   -362   -260    529       C  \\nATOM    983  CE  LYS A 135      -9.599   8.210  13.532  1.00 36.28           C  \\nANISOU  983  CE  LYS A 135     3120   5180   5485   -544   -112    457       C  \\nATOM    984  NZ  LYS A 135      -9.650   8.675  14.947  1.00 42.87           N  \\nANISOU  984  NZ  LYS A 135     3878   6025   6384   -480     75    394       N  \\nATOM    985  N   ALA A 136      -6.344  11.118   7.882  1.00 27.23           N  \\nANISOU  985  N   ALA A 136     2534   3799   4012   -194   -812    874       N  \\nATOM    986  CA  ALA A 136      -5.960  12.317   7.140  1.00 29.49           C  \\nANISOU  986  CA  ALA A 136     2880   4035   4291   -107   -927    982       C  \\nATOM    987  C   ALA A 136      -6.188  12.080   5.639  1.00 34.06           C  \\nANISOU  987  C   ALA A 136     3554   4668   4719   -210  -1116   1060       C  \\nATOM    988  O   ALA A 136      -6.241  10.930   5.200  1.00 33.40           O  \\nANISOU  988  O   ALA A 136     3549   4640   4502   -353  -1111   1005       O  \\nATOM    989  CB  ALA A 136      -4.487  12.677   7.443  1.00 26.01           C  \\nANISOU  989  CB  ALA A 136     2602   3510   3769    -72   -787    981       C  \\nATOM    990  N   PRO A 137      -6.346  13.157   4.846  1.00 34.71           N  \\nANISOU  990  N   PRO A 137     3658   4724   4808   -147  -1290   1188       N  \\nATOM    991  CA  PRO A 137      -6.654  12.942   3.425  1.00 37.80           C  \\nANISOU  991  CA  PRO A 137     4148   5185   5029   -259  -1494   1272       C  \\nATOM    992  C   PRO A 137      -5.483  12.307   2.663  1.00 40.30           C  \\nANISOU  992  C   PRO A 137     4733   5501   5078   -399  -1401   1250       C  \\nATOM    993  O   PRO A 137      -4.332  12.434   3.087  1.00 35.47           O  \\nANISOU  993  O   PRO A 137     4219   4825   4431   -370  -1217   1216       O  \\nATOM    994  CB  PRO A 137      -6.922  14.365   2.889  1.00 37.78           C  \\nANISOU  994  CB  PRO A 137     4149   5116   5092   -136  -1688   1436       C  \\nATOM    995  CG  PRO A 137      -6.912  15.274   4.086  1.00 40.05           C  \\nANISOU  995  CG  PRO A 137     4311   5289   5615     50  -1579   1413       C  \\nATOM    996  CD  PRO A 137      -6.121  14.577   5.151  1.00 30.63           C  \\nANISOU  996  CD  PRO A 137     3133   4089   4416     10  -1307   1267       C  \\nATOM    997  N   ASP A 138      -5.781  11.631   1.554  1.00 40.79           N  \\nANISOU  997  N   ASP A 138     4899   5647   4952   -551  -1524   1256       N  \\nATOM    998  CA  ASP A 138      -4.739  10.978   0.761  1.00 51.41           C  \\nANISOU  998  CA  ASP A 138     6499   7001   6033   -679  -1420   1208       C  \\nATOM    999  C   ASP A 138      -3.819  11.940   0.004  1.00 42.83           C  \\nANISOU  999  C   ASP A 138     5589   5883   4801   -683  -1425   1324       C  \\nATOM   1000  O   ASP A 138      -2.865  11.501  -0.620  1.00 38.25           O  \\nANISOU 1000  O   ASP A 138     5199   5328   4005   -781  -1304   1275       O  \\nATOM   1001  CB  ASP A 138      -5.340   9.955  -0.208  1.00 62.34           C  \\nANISOU 1001  CB  ASP A 138     7974   8479   7235   -863  -1540   1158       C  \\nATOM   1002  CG  ASP A 138      -5.593   8.609   0.451  1.00 75.49           C  \\nANISOU 1002  CG  ASP A 138     9604  10137   8942   -929  -1413    993       C  \\nATOM   1003  OD1 ASP A 138      -4.978   8.341   1.509  1.00 71.05           O  \\nANISOU 1003  OD1 ASP A 138     9014   9496   8485   -834  -1204    919       O  \\nATOM   1004  OD2 ASP A 138      -6.401   7.819  -0.094  1.00 83.16           O  \\nANISOU 1004  OD2 ASP A 138    10592  11178   9829  -1090  -1534    944       O  \\nATOM   1005  N   THR A 139      -4.111  13.239   0.059  1.00 33.67           N  \\nANISOU 1005  N   THR A 139     4374   4661   3758   -581  -1554   1471       N  \\nATOM   1006  CA  THR A 139      -3.237  14.252  -0.516  1.00 45.75           C  \\nANISOU 1006  CA  THR A 139     6085   6130   5167   -604  -1545   1594       C  \\nATOM   1007  C   THR A 139      -1.953  14.354   0.299  1.00 42.50           C  \\nANISOU 1007  C   THR A 139     5687   5669   4790   -573  -1273   1509       C  \\nATOM   1008  O   THR A 139      -0.973  14.978  -0.124  1.00 38.91           O  \\nANISOU 1008  O   THR A 139     5382   5194   4208   -641  -1197   1569       O  \\nATOM   1009  CB  THR A 139      -3.901  15.636  -0.515  1.00 42.93           C  \\nANISOU 1009  CB  THR A 139     5684   5668   4961   -479  -1750   1771       C  \\nATOM   1010  OG1 THR A 139      -4.434  15.905   0.789  1.00 48.13           O  \\nANISOU 1010  OG1 THR A 139     6105   6259   5922   -299  -1705   1710       O  \\nATOM   1011  CG2 THR A 139      -5.023  15.702  -1.551  1.00 50.13           C  \\nANISOU 1011  CG2 THR A 139     6605   6645   5795   -511  -2064   1898       C  \\nATOM   1012  N   LEU A 140      -1.967  13.742   1.477  1.00 34.95           N  \\nANISOU 1012  N   LEU A 140     4573   4709   3999   -486  -1135   1374       N  \\nATOM   1013  CA  LEU A 140      -0.851  13.853   2.402  1.00 32.21           C  \\nANISOU 1013  CA  LEU A 140     4202   4330   3708   -437   -914   1297       C  \\nATOM   1014  C   LEU A 140      -0.078  12.547   2.505  1.00 35.42           C  \\nANISOU 1014  C   LEU A 140     4644   4811   4003   -476   -732   1149       C  \\nATOM   1015  O   LEU A 140      -0.671  11.466   2.620  1.00 28.45           O  \\nANISOU 1015  O   LEU A 140     3732   3951   3127   -483   -740   1064       O  \\nATOM   1016  CB  LEU A 140      -1.363  14.247   3.782  1.00 34.78           C  \\nANISOU 1016  CB  LEU A 140     4340   4580   4292   -292   -892   1265       C  \\nATOM   1017  CG  LEU A 140      -0.308  14.291   4.886  1.00 42.33           C  \\nANISOU 1017  CG  LEU A 140     5258   5520   5305   -247   -689   1177       C  \\nATOM   1018  CD1 LEU A 140       0.648  15.456   4.656  1.00 43.78           C  \\nANISOU 1018  CD1 LEU A 140     5540   5655   5441   -292   -654   1254       C  \\nATOM   1019  CD2 LEU A 140      -0.969  14.386   6.244  1.00 39.00           C  \\nANISOU 1019  CD2 LEU A 140     4670   5049   5099   -130   -660   1117       C  \\nATOM   1020  N   GLU A 141       1.248  12.653   2.448  1.00 33.22           N  \\nANISOU 1020  N   GLU A 141     4428   4568   3626   -502   -568   1115       N  \\nATOM   1021  CA  GLU A 141       2.114  11.504   2.672  1.00 30.34           C  \\nANISOU 1021  CA  GLU A 141     4072   4264   3190   -481   -386    971       C  \\nATOM   1022  C   GLU A 141       2.938  11.714   3.931  1.00 33.92           C  \\nANISOU 1022  C   GLU A 141     4400   4711   3775   -378   -252    922       C  \\nATOM   1023  O   GLU A 141       3.668  12.696   4.053  1.00 31.04           O  \\nANISOU 1023  O   GLU A 141     4015   4360   3418   -404   -209    970       O  \\nATOM   1024  CB  GLU A 141       3.042  11.265   1.481  1.00 34.97           C  \\nANISOU 1024  CB  GLU A 141     4807   4946   3533   -589   -291    942       C  \\nATOM   1025  CG  GLU A 141       3.997  10.084   1.669  1.00 32.06           C  \\nANISOU 1025  CG  GLU A 141     4438   4634   3108   -523    -94    779       C  \\nATOM   1026  CD  GLU A 141       5.186  10.136   0.719  1.00 38.18           C  \\nANISOU 1026  CD  GLU A 141     5295   5534   3677   -605     58    735       C  \\nATOM   1027  OE1 GLU A 141       5.968  11.102   0.790  1.00 37.50           O  \\nANISOU 1027  OE1 GLU A 141     5152   5508   3589   -650    120    793       O  \\nATOM   1028  OE2 GLU A 141       5.332   9.212  -0.110  1.00 41.81           O  \\nANISOU 1028  OE2 GLU A 141     5882   6034   3971   -639    128    633       O  \\nATOM   1029  N   VAL A 142       2.794  10.783   4.869  1.00 28.97           N  \\nANISOU 1029  N   VAL A 142     3706   4062   3241   -282   -197    831       N  \\nATOM   1030  CA  VAL A 142       3.562  10.781   6.094  1.00 28.78           C  \\nANISOU 1030  CA  VAL A 142     3577   4047   3313   -183    -89    780       C  \\nATOM   1031  C   VAL A 142       4.767   9.853   5.967  1.00 33.20           C  \\nANISOU 1031  C   VAL A 142     4158   4684   3770   -130     56    681       C  \\nATOM   1032  O   VAL A 142       4.637   8.686   5.588  1.00 29.89           O  \\nANISOU 1032  O   VAL A 142     3830   4245   3282   -105     85    609       O  \\nATOM   1033  CB  VAL A 142       2.696  10.348   7.290  1.00 29.31           C  \\nANISOU 1033  CB  VAL A 142     3570   4039   3526   -108   -116    755       C  \\nATOM   1034  CG1 VAL A 142       3.546  10.202   8.535  1.00 23.06           C  \\nANISOU 1034  CG1 VAL A 142     2703   3268   2790    -14    -17    706       C  \\nATOM   1035  CG2 VAL A 142       1.590  11.370   7.519  1.00 31.57           C  \\nANISOU 1035  CG2 VAL A 142     3787   4267   3941   -125   -234    831       C  \\nATOM   1036  N   THR A 143       5.941  10.394   6.258  1.00 29.73           N  \\nANISOU 1036  N   THR A 143     3634   4336   3325   -113    145    671       N  \\nATOM   1037  CA  THR A 143       7.170   9.615   6.283  1.00 26.39           C  \\nANISOU 1037  CA  THR A 143     3169   4017   2842    -24    282    573       C  \\nATOM   1038  C   THR A 143       7.584   9.410   7.734  1.00 29.92           C  \\nANISOU 1038  C   THR A 143     3494   4467   3407    107    296    548       C  \\nATOM   1039  O   THR A 143       7.753  10.379   8.481  1.00 29.83           O  \\nANISOU 1039  O   THR A 143     3387   4477   3469     72    268    591       O  \\nATOM   1040  CB  THR A 143       8.295  10.363   5.556  1.00 29.93           C  \\nANISOU 1040  CB  THR A 143     3568   4617   3186   -120    377    574       C  \\nATOM   1041  OG1 THR A 143       7.949  10.533   4.175  1.00 28.98           O  \\nANISOU 1041  OG1 THR A 143     3595   4501   2916   -258    365    607       O  \\nATOM   1042  CG2 THR A 143       9.621   9.621   5.678  1.00 26.10           C  \\nANISOU 1042  CG2 THR A 143     2971   4277   2668      1    524    459       C  \\nATOM   1043  N   GLY A 144       7.707   8.154   8.147  1.00 27.37           N  \\nANISOU 1043  N   GLY A 144     3203   4105   3092    251    328    481       N  \\nATOM   1044  CA  GLY A 144       8.317   7.843   9.424  1.00 23.07           C  \\nANISOU 1044  CA  GLY A 144     2559   3585   2620    389    336    465       C  \\nATOM   1045  C   GLY A 144       9.814   7.664   9.228  1.00 28.41           C  \\nANISOU 1045  C   GLY A 144     3109   4433   3253    487    433    398       C  \\nATOM   1046  O   GLY A 144      10.251   6.869   8.406  1.00 32.47           O  \\nANISOU 1046  O   GLY A 144     3670   4970   3696    562    518    322       O  \\nATOM   1047  N   VAL A 145      10.606   8.416   9.979  1.00 34.93           N  \\nANISOU 1047  N   VAL A 145     3762   5390   4120    480    426    414       N  \\nATOM   1048  CA  VAL A 145      12.057   8.373   9.830  1.00 34.71           C  \\nANISOU 1048  CA  VAL A 145     3549   5575   4063    553    512    349       C  \\nATOM   1049  C   VAL A 145      12.697   7.586  10.979  1.00 37.91           C  \\nANISOU 1049  C   VAL A 145     3859   6019   4526    774    469    329       C  \\nATOM   1050  O   VAL A 145      12.458   7.891  12.137  1.00 38.92           O  \\nANISOU 1050  O   VAL A 145     3970   6116   4704    765    373    383       O  \\nATOM   1051  CB  VAL A 145      12.638   9.806   9.806  1.00 28.99           C  \\nANISOU 1051  CB  VAL A 145     2681   5005   3330    356    526    378       C  \\nATOM   1052  CG1 VAL A 145      14.134   9.769   9.592  1.00 32.98           C  \\nANISOU 1052  CG1 VAL A 145     2955   5773   3804    397    627    301       C  \\nATOM   1053  CG2 VAL A 145      11.956  10.642   8.720  1.00 30.84           C  \\nANISOU 1053  CG2 VAL A 145     3050   5167   3501    143    540    431       C  \\nATOM   1054  N   HIS A 146      13.506   6.579  10.655  1.00 35.53           N  \\nANISOU 1054  N   HIS A 146     3506   5781   4211    980    539    249       N  \\nATOM   1055  CA  HIS A 146      14.119   5.727  11.672  1.00 34.96           C  \\nANISOU 1055  CA  HIS A 146     3364   5726   4194   1231    475    246       C  \\nATOM   1056  C   HIS A 146      15.634   5.856  11.689  1.00 43.19           C  \\nANISOU 1056  C   HIS A 146     4101   7058   5252   1345    521    181       C  \\nATOM   1057  O   HIS A 146      16.252   6.137  10.666  1.00 42.50           O  \\nANISOU 1057  O   HIS A 146     3893   7133   5124   1286    657    101       O  \\nATOM   1058  CB  HIS A 146      13.779   4.262  11.415  1.00 39.92           C  \\nANISOU 1058  CB  HIS A 146     4202   6146   4820   1442    496    208       C  \\nATOM   1059  CG  HIS A 146      12.314   3.987  11.325  1.00 41.62           C  \\nANISOU 1059  CG  HIS A 146     4697   6100   5016   1317    459    257       C  \\nATOM   1060  ND1 HIS A 146      11.629   3.289  12.295  1.00 36.59           N  \\nANISOU 1060  ND1 HIS A 146     4229   5267   4405   1382    366    323       N  \\nATOM   1061  CD2 HIS A 146      11.404   4.299  10.372  1.00 41.84           C  \\nANISOU 1061  CD2 HIS A 146     4856   6048   4994   1122    496    249       C  \\nATOM   1062  CE1 HIS A 146      10.358   3.192  11.949  1.00 36.57           C  \\nANISOU 1062  CE1 HIS A 146     4422   5091   4383   1221    361    343       C  \\nATOM   1063  NE2 HIS A 146      10.196   3.793  10.785  1.00 39.03           N  \\nANISOU 1063  NE2 HIS A 146     4707   5471   4652   1074    426    300       N  \\nATOM   1064  N   GLY A 147      16.233   5.625  12.853  1.00 46.97           N  \\nANISOU 1064  N   GLY A 147     4449   7619   5780   1501    405    214       N  \\nATOM   1065  CA  GLY A 147      17.679   5.618  12.964  1.00 45.13           C  \\nANISOU 1065  CA  GLY A 147     3887   7683   5577   1644    420    152       C  \\nATOM   1066  C   GLY A 147      18.300   4.401  12.303  1.00 50.00           C  \\nANISOU 1066  C   GLY A 147     4473   8303   6222   1957    517     52       C  \\nATOM   1067  O   GLY A 147      19.464   4.424  11.899  1.00 55.87           O  \\nANISOU 1067  O   GLY A 147     4941   9298   6990   2038    600    -44       O  \\nATOM   1068  N   THR A 148      17.521   3.332  12.186  1.00 51.81           N  \\nANISOU 1068  N   THR A 148     5007   8227   6450   2106    512     61       N  \\nATOM   1069  CA  THR A 148      18.046   2.061  11.699  1.00 56.32           C  \\nANISOU 1069  CA  THR A 148     5611   8731   7057   2441    591    -38       C  \\nATOM   1070  C   THR A 148      17.188   1.516  10.571  1.00 48.08           C  \\nANISOU 1070  C   THR A 148     4875   7445   5949   2377    724   -110       C  \\nATOM   1071  O   THR A 148      15.984   1.732  10.543  1.00 45.10           O  \\nANISOU 1071  O   THR A 148     4745   6867   5523   2154    683    -39       O  \\nATOM   1072  CB  THR A 148      18.082   1.015  12.836  1.00 63.25           C  \\nANISOU 1072  CB  THR A 148     6608   9430   7996   2746    421     46       C  \\nATOM   1073  OG1 THR A 148      18.760   1.569  13.967  1.00 72.30           O  \\nANISOU 1073  OG1 THR A 148     7537  10761   9171   2700    254    126       O  \\nATOM   1074  CG2 THR A 148      18.810  -0.237  12.402  1.00 72.93           C  \\nANISOU 1074  CG2 THR A 148     7883  10544   9284   3036    475    -60       C  \\nATOM   1075  N   GLU A 149      17.806   0.791   9.651  1.00 46.57           N  \\nANISOU 1075  N   GLU A 149     4659   7280   5755   2576    883   -261       N  \\nATOM   1076  CA  GLU A 149      17.057   0.122   8.598  1.00 54.18           C  \\nANISOU 1076  CA  GLU A 149     5942   8004   6640   2533   1004   -351       C  \\nATOM   1077  C   GLU A 149      16.269  -1.051   9.167  1.00 54.73           C  \\nANISOU 1077  C   GLU A 149     6364   7690   6741   2687    895   -297       C  \\nATOM   1078  O   GLU A 149      15.200  -1.391   8.661  1.00 51.51           O  \\nANISOU 1078  O   GLU A 149     6270   7040   6262   2525    914   -305       O  \\nATOM   1079  CB  GLU A 149      17.995  -0.340   7.472  1.00 56.31           C  \\nANISOU 1079  CB  GLU A 149     6098   8414   6884   2707   1229   -555       C  \\nATOM   1080  CG  GLU A 149      17.297  -1.001   6.286  1.00 55.20           C  \\nANISOU 1080  CG  GLU A 149     6297   8049   6627   2637   1368   -676       C  \\nATOM   1081  CD  GLU A 149      16.280  -0.097   5.592  1.00 64.89           C  \\nANISOU 1081  CD  GLU A 149     7667   9272   7716   2209   1370   -618       C  \\nATOM   1082  OE1 GLU A 149      16.368   1.147   5.717  1.00 65.18           O  \\nANISOU 1082  OE1 GLU A 149     7500   9529   7735   1974   1336   -525       O  \\nATOM   1083  OE2 GLU A 149      15.384  -0.642   4.912  1.00 72.52           O  \\nANISOU 1083  OE2 GLU A 149     8961  10006   8588   2109   1395   -664       O  \\nATOM   1084  N   LYS A 150      16.807  -1.666  10.219  1.00 65.52           N  \\nANISOU 1084  N   LYS A 150     7681   9009   8205   2984    772   -235       N  \\nATOM   1085  CA  LYS A 150      16.130  -2.758  10.919  1.00 69.31           C  \\nANISOU 1085  CA  LYS A 150     8511   9116   8706   3120    655   -153       C  \\nATOM   1086  C   LYS A 150      14.831  -2.242  11.534  1.00 55.55           C  \\nANISOU 1086  C   LYS A 150     6948   7247   6912   2786    538     -2       C  \\nATOM   1087  O   LYS A 150      13.779  -2.873  11.420  1.00 50.99           O  \\nANISOU 1087  O   LYS A 150     6709   6371   6292   2679    532     16       O  \\nATOM   1088  CB  LYS A 150      17.049  -3.367  11.989  1.00 75.11           C  \\nANISOU 1088  CB  LYS A 150     9143   9854   9541   3429    506    -83       C  \\nATOM   1089  CG  LYS A 150      16.369  -4.320  12.964  1.00 79.44           C  \\nANISOU 1089  CG  LYS A 150    10050  10041  10090   3503    350     56       C  \\nATOM   1090  CD  LYS A 150      17.358  -4.819  14.016  1.00 88.07           C  \\nANISOU 1090  CD  LYS A 150    11026  11171  11266   3705    178    138       C  \\nATOM   1091  CE  LYS A 150      16.686  -5.666  15.097  1.00 96.09           C  \\nANISOU 1091  CE  LYS A 150    12400  11856  12253   3729     17    303       C  \\nATOM   1092  NZ  LYS A 150      15.922  -4.859  16.096  1.00 95.65           N  \\nANISOU 1092  NZ  LYS A 150    12369  11857  12117   3500   -100    466       N  \\nATOM   1093  N   SER A 151      14.917  -1.074  12.162  1.00 38.39           N  \\nANISOU 1093  N   SER A 151     4533   5310   4741   2613    459     90       N  \\nATOM   1094  CA  SER A 151      13.743  -0.365  12.662  1.00 39.75           C  \\nANISOU 1094  CA  SER A 151     4813   5417   4874   2294    381    203       C  \\nATOM   1095  C   SER A 151      12.705  -0.056  11.565  1.00 45.37           C  \\nANISOU 1095  C   SER A 151     5672   6046   5522   2015    476    151       C  \\nATOM   1096  O   SER A 151      11.516  -0.371  11.707  1.00 45.77           O  \\nANISOU 1096  O   SER A 151     5969   5875   5546   1863    437    201       O  \\nATOM   1097  CB  SER A 151      14.190   0.920  13.355  1.00 43.54           C  \\nANISOU 1097  CB  SER A 151     4994   6181   5368   2170    309    269       C  \\nATOM   1098  OG  SER A 151      13.092   1.629  13.883  1.00 51.19           O  \\nANISOU 1098  OG  SER A 151     6056   7083   6309   1897    247    360       O  \\nATOM   1099  N   ALA A 152      13.150   0.555  10.472  1.00 47.02           N  \\nANISOU 1099  N   ALA A 152     5726   6445   5695   1935    596     54       N  \\nATOM   1100  CA  ALA A 152      12.241   0.902   9.379  1.00 44.60           C  \\nANISOU 1100  CA  ALA A 152     5554   6085   5305   1674    661     17       C  \\nATOM   1101  C   ALA A 152      11.619  -0.338   8.735  1.00 39.85           C  \\nANISOU 1101  C   ALA A 152     5274   5212   4655   1717    709    -63       C  \\nATOM   1102  O   ALA A 152      10.435  -0.353   8.401  1.00 38.54           O  \\nANISOU 1102  O   ALA A 152     5298   4906   4440   1497    675    -39       O  \\nATOM   1103  CB  ALA A 152      12.965   1.749   8.328  1.00 44.21           C  \\nANISOU 1103  CB  ALA A 152     5303   6298   5198   1580    786    -62       C  \\nATOM   1104  N   GLU A 153      12.417  -1.383   8.574  1.00 41.21           N  \\nANISOU 1104  N   GLU A 153     5504   5308   4846   2005    782   -166       N  \\nATOM   1105  CA  GLU A 153      11.933  -2.614   7.966  1.00 45.30           C  \\nANISOU 1105  CA  GLU A 153     6359   5538   5314   2057    839   -265       C  \\nATOM   1106  C   GLU A 153      10.842  -3.243   8.825  1.00 49.96           C  \\nANISOU 1106  C   GLU A 153     7220   5837   5927   1977    711   -153       C  \\nATOM   1107  O   GLU A 153       9.884  -3.823   8.309  1.00 47.03           O  \\nANISOU 1107  O   GLU A 153     7127   5253   5489   1814    722   -194       O  \\nATOM   1108  CB  GLU A 153      13.083  -3.596   7.783  1.00 46.07           C  \\nANISOU 1108  CB  GLU A 153     6459   5590   5455   2436    941   -397       C  \\nATOM   1109  CG  GLU A 153      12.798  -4.721   6.801  1.00 65.35           C  \\nANISOU 1109  CG  GLU A 153     9236   7772   7822   2483   1058   -564       C  \\nATOM   1110  CD  GLU A 153      14.057  -5.504   6.437  1.00 94.85           C  \\nANISOU 1110  CD  GLU A 153    12919  11513  11606   2880   1201   -733       C  \\nATOM   1111  OE1 GLU A 153      14.268  -6.606   6.996  1.00105.55           O  \\nANISOU 1111  OE1 GLU A 153    14467  12596  13042   3173   1160   -737       O  \\nATOM   1112  OE2 GLU A 153      14.840  -5.015   5.590  1.00101.79           O  \\nANISOU 1112  OE2 GLU A 153    13566  12669  12442   2902   1360   -863       O  \\nATOM   1113  N   ALA A 154      10.995  -3.123  10.139  1.00 44.11           N  \\nANISOU 1113  N   ALA A 154     6395   5102   5261   2069    593    -15       N  \\nATOM   1114  CA  ALA A 154      10.018  -3.661  11.070  1.00 47.25           C  \\nANISOU 1114  CA  ALA A 154     7035   5253   5663   1973    487    104       C  \\nATOM   1115  C   ALA A 154       8.659  -2.961  10.917  1.00 46.03           C  \\nANISOU 1115  C   ALA A 154     6904   5118   5467   1597    456    154       C  \\nATOM   1116  O   ALA A 154       7.623  -3.622  10.796  1.00 49.32           O  \\nANISOU 1116  O   ALA A 154     7579   5313   5846   1433    446    153       O  \\nATOM   1117  CB  ALA A 154      10.538  -3.559  12.519  1.00 40.65           C  \\nANISOU 1117  CB  ALA A 154     6093   4467   4886   2135    367    245       C  \\nATOM   1118  N   ALA A 155       8.666  -1.631  10.920  1.00 37.70           N  \\nANISOU 1118  N   ALA A 155     5575   4324   4426   1461    438    194       N  \\nATOM   1119  CA  ALA A 155       7.435  -0.857  10.750  1.00 30.97           C  \\nANISOU 1119  CA  ALA A 155     4704   3507   3558   1152    400    240       C  \\nATOM   1120  C   ALA A 155       6.820  -1.118   9.365  1.00 41.34           C  \\nANISOU 1120  C   ALA A 155     6156   4760   4790    995    450    140       C  \\nATOM   1121  O   ALA A 155       5.605  -1.212   9.206  1.00 38.95           O  \\nANISOU 1121  O   ALA A 155     5966   4367   4466    772    405    160       O  \\nATOM   1122  CB  ALA A 155       7.724   0.603  10.948  1.00 34.83           C  \\nANISOU 1122  CB  ALA A 155     4900   4253   4079   1081    376    291       C  \\nATOM   1123  N   ARG A 156       7.679  -1.267   8.367  1.00 41.89           N  \\nANISOU 1123  N   ARG A 156     6211   4898   4806   1108    547     25       N  \\nATOM   1124  CA  ARG A 156       7.247  -1.604   7.017  1.00 46.80           C  \\nANISOU 1124  CA  ARG A 156     6998   5469   5316    971    603    -88       C  \\nATOM   1125  C   ARG A 156       6.567  -2.976   6.964  1.00 55.27           C  \\nANISOU 1125  C   ARG A 156     8409   6238   6354    944    601   -147       C  \\nATOM   1126  O   ARG A 156       5.560  -3.162   6.272  1.00 48.92           O  \\nANISOU 1126  O   ARG A 156     7754   5362   5470    703    570   -183       O  \\nATOM   1127  CB  ARG A 156       8.464  -1.609   6.108  1.00 46.41           C  \\nANISOU 1127  CB  ARG A 156     6874   5553   5205   1132    743   -218       C  \\nATOM   1128  CG  ARG A 156       8.240  -1.063   4.744  1.00 47.89           C  \\nANISOU 1128  CG  ARG A 156     7072   5870   5255    928    792   -285       C  \\nATOM   1129  CD  ARG A 156       9.556  -1.073   4.001  1.00 54.42           C  \\nANISOU 1129  CD  ARG A 156     7800   6855   6020   1094    964   -418       C  \\nATOM   1130  NE  ARG A 156      10.409   0.030   4.422  1.00 56.53           N  \\nANISOU 1130  NE  ARG A 156     7742   7384   6352   1139    976   -343       N  \\nATOM   1131  CZ  ARG A 156      11.708  -0.072   4.687  1.00 50.91           C  \\nANISOU 1131  CZ  ARG A 156     6836   6808   5699   1384   1076   -404       C  \\nATOM   1132  NH1 ARG A 156      12.337  -1.242   4.587  1.00 47.90           N  \\nANISOU 1132  NH1 ARG A 156     6550   6313   5335   1657   1177   -541       N  \\nATOM   1133  NH2 ARG A 156      12.376   1.012   5.054  1.00 48.68           N  \\nANISOU 1133  NH2 ARG A 156     6258   6775   5463   1357   1071   -333       N  \\nATOM   1134  N   ARG A 157       7.135  -3.938   7.687  1.00 55.00           N  \\nANISOU 1134  N   ARG A 157     8503   6020   6375   1187    622   -154       N  \\nATOM   1135  CA  ARG A 157       6.536  -5.261   7.802  1.00 54.80           C  \\nANISOU 1135  CA  ARG A 157     8837   5659   6325   1161    617   -191       C  \\nATOM   1136  C   ARG A 157       5.185  -5.172   8.503  1.00 47.02           C  \\nANISOU 1136  C   ARG A 157     7912   4601   5355    876    511    -69       C  \\nATOM   1137  O   ARG A 157       4.215  -5.789   8.062  1.00 44.65           O  \\nANISOU 1137  O   ARG A 157     7838   4137   4991    649    499   -117       O  \\nATOM   1138  CB  ARG A 157       7.467  -6.236   8.544  1.00 65.88           C  \\nANISOU 1138  CB  ARG A 157    10371   6866   7795   1511    639   -189       C  \\nATOM   1139  CG  ARG A 157       8.639  -6.755   7.704  1.00 81.70           C  \\nANISOU 1139  CG  ARG A 157    12401   8858   9785   1806    772   -363       C  \\nATOM   1140  CD  ARG A 157       9.468  -7.813   8.439  1.00 90.35           C  \\nANISOU 1140  CD  ARG A 157    13648   9719  10964   2187    769   -356       C  \\nATOM   1141  NE  ARG A 157      10.215  -7.268   9.575  1.00 93.70           N  \\nANISOU 1141  NE  ARG A 157    13792  10322  11487   2390    679   -209       N  \\nATOM   1142  CZ  ARG A 157      11.528  -7.403   9.745  1.00 94.30           C  \\nANISOU 1142  CZ  ARG A 157    13691  10497  11641   2769    705   -246       C  \\nATOM   1143  NH1 ARG A 157      12.250  -8.070   8.853  1.00 95.73           N  \\nANISOU 1143  NH1 ARG A 157    13940  10608  11823   3012    844   -435       N  \\nATOM   1144  NH2 ARG A 157      12.122  -6.874  10.808  1.00 89.83           N  \\nANISOU 1144  NH2 ARG A 157    12870  10114  11148   2906    594   -106       N  \\nATOM   1145  N   ALA A 158       5.122  -4.396   9.586  1.00 40.39           N  \\nANISOU 1145  N   ALA A 158     6857   3896   4592    875    443     75       N  \\nATOM   1146  CA  ALA A 158       3.872  -4.213  10.331  1.00 46.54           C  \\nANISOU 1146  CA  ALA A 158     7645   4647   5391    617    371    179       C  \\nATOM   1147  C   ALA A 158       2.793  -3.616   9.438  1.00 54.62           C  \\nANISOU 1147  C   ALA A 158     8584   5791   6379    322    338    145       C  \\nATOM   1148  O   ALA A 158       1.637  -4.043   9.469  1.00 53.81           O  \\nANISOU 1148  O   ALA A 158     8602   5588   6256     75    304    150       O  \\nATOM   1149  CB  ALA A 158       4.088  -3.331  11.562  1.00 36.72           C  \\nANISOU 1149  CB  ALA A 158     6166   3564   4220    677    324    311       C  \\nATOM   1150  N   LEU A 159       3.191  -2.630   8.639  1.00 56.05           N  \\nANISOU 1150  N   LEU A 159     8557   6194   6546    341    342    115       N  \\nATOM   1151  CA  LEU A 159       2.280  -1.928   7.751  1.00 54.30           C  \\nANISOU 1151  CA  LEU A 159     8240   6106   6286     99    282    105       C  \\nATOM   1152  C   LEU A 159       1.718  -2.863   6.676  1.00 55.59           C  \\nANISOU 1152  C   LEU A 159     8659   6131   6331    -64    284    -12       C  \\nATOM   1153  O   LEU A 159       0.553  -2.751   6.300  1.00 55.78           O  \\nANISOU 1153  O   LEU A 159     8676   6188   6330   -322    198     -4       O  \\nATOM   1154  CB  LEU A 159       2.995  -0.729   7.124  1.00 61.49           C  \\nANISOU 1154  CB  LEU A 159     8928   7251   7184    168    291    112       C  \\nATOM   1155  CG  LEU A 159       2.165   0.396   6.512  1.00 62.95           C  \\nANISOU 1155  CG  LEU A 159     8954   7603   7361    -29    194    167       C  \\nATOM   1156  CD1 LEU A 159       1.205   0.969   7.541  1.00 58.30           C  \\nANISOU 1156  CD1 LEU A 159     8211   7045   6897   -119    115    272       C  \\nATOM   1157  CD2 LEU A 159       3.091   1.479   5.969  1.00 63.11           C  \\nANISOU 1157  CD2 LEU A 159     8815   7810   7354     51    225    182       C  \\nATOM   1158  N   LEU A 160       2.534  -3.795   6.192  1.00 54.60           N  \\nANISOU 1158  N   LEU A 160     8756   5855   6135     88    380   -130       N  \\nATOM   1159  CA  LEU A 160       2.036  -4.787   5.242  1.00 64.25           C  \\nANISOU 1159  CA  LEU A 160    10273   6906   7231    -72    394   -265       C  \\nATOM   1160  C   LEU A 160       1.099  -5.807   5.906  1.00 69.73           C  \\nANISOU 1160  C   LEU A 160    11197   7350   7946   -240    359   -248       C  \\nATOM   1161  O   LEU A 160       0.116  -6.235   5.298  1.00 65.95           O  \\nANISOU 1161  O   LEU A 160    10860   6812   7385   -524    306   -309       O  \\nATOM   1162  CB  LEU A 160       3.186  -5.503   4.521  1.00 71.08           C  \\nANISOU 1162  CB  LEU A 160    11326   7664   8016    154    532   -425       C  \\nATOM   1163  CG  LEU A 160       2.776  -6.269   3.257  1.00 73.33           C  \\nANISOU 1163  CG  LEU A 160    11902   7829   8129    -28    560   -599       C  \\nATOM   1164  CD1 LEU A 160       2.263  -5.309   2.182  1.00 65.24           C  \\nANISOU 1164  CD1 LEU A 160    10732   7069   6985   -263    484   -598       C  \\nATOM   1165  CD2 LEU A 160       3.916  -7.134   2.719  1.00 77.79           C  \\nANISOU 1165  CD2 LEU A 160    12684   8238   8633    235    729   -781       C  \\nATOM   1166  N   GLU A 161       1.408  -6.189   7.149  1.00 76.78           N  \\nANISOU 1166  N   GLU A 161    12133   8107   8935    -86    382   -160       N  \\nATOM   1167  CA  GLU A 161       0.554  -7.087   7.937  1.00 78.28           C  \\nANISOU 1167  CA  GLU A 161    12544   8062   9137   -263    360   -113       C  \\nATOM   1168  C   GLU A 161      -0.838  -6.491   8.149  1.00 75.82           C  \\nANISOU 1168  C   GLU A 161    12041   7916   8851   -601    272    -40       C  \\nATOM   1169  O   GLU A 161      -1.818  -7.220   8.334  1.00 68.12           O  \\nANISOU 1169  O   GLU A 161    11236   6799   7847   -871    254    -47       O  \\nATOM   1170  CB  GLU A 161       1.175  -7.374   9.309  1.00 77.68           C  \\nANISOU 1170  CB  GLU A 161    12511   7862   9141    -34    383      6       C  \\nATOM   1171  CG  GLU A 161       2.405  -8.271   9.307  1.00 80.37           C  \\nANISOU 1171  CG  GLU A 161    13091   7971   9477    311    451    -52       C  \\nATOM   1172  CD  GLU A 161       2.991  -8.459  10.703  1.00 76.35           C  \\nANISOU 1172  CD  GLU A 161    12600   7374   9037    539    429     95       C  \\nATOM   1173  OE1 GLU A 161       3.724  -9.447  10.914  1.00 84.84           O  \\nANISOU 1173  OE1 GLU A 161    13943   8179  10115    790    455     79       O  \\nATOM   1174  OE2 GLU A 161       2.723  -7.620  11.591  1.00 67.98           O  \\nANISOU 1174  OE2 GLU A 161    11297   6509   8023    475    380    226       O  \\nATOM   1175  N   HIS A 162      -0.905  -5.161   8.127  1.00 77.15           N  \\nANISOU 1175  N   HIS A 162    11851   8382   9079   -581    221     25       N  \\nATOM   1176  CA  HIS A 162      -2.143  -4.416   8.343  1.00 77.01           C  \\nANISOU 1176  CA  HIS A 162    11588   8554   9118   -829    137     92       C  \\nATOM   1177  C   HIS A 162      -3.161  -4.623   7.214  1.00 82.37           C  \\nANISOU 1177  C   HIS A 162    12304   9278   9714  -1126     52      9       C  \\nATOM   1178  O   HIS A 162      -4.353  -4.369   7.397  1.00 79.29           O  \\nANISOU 1178  O   HIS A 162    11753   9004   9369  -1368    -22     45       O  \\nATOM   1179  CB  HIS A 162      -1.829  -2.925   8.509  1.00 69.73           C  \\nANISOU 1179  CB  HIS A 162    10315   7898   8282   -689    102    172       C  \\nATOM   1180  CG  HIS A 162      -2.908  -2.147   9.196  1.00 69.05           C  \\nANISOU 1180  CG  HIS A 162     9966   7970   8298   -839     49    254       C  \\nATOM   1181  ND1 HIS A 162      -3.351  -2.451  10.466  1.00 67.09           N  \\nANISOU 1181  ND1 HIS A 162     9725   7662   8105   -901    100    313       N  \\nATOM   1182  CD2 HIS A 162      -3.627  -1.071   8.794  1.00 66.88           C  \\nANISOU 1182  CD2 HIS A 162     9419   7912   8081   -925    -44    283       C  \\nATOM   1183  CE1 HIS A 162      -4.299  -1.598  10.815  1.00 64.84           C  \\nANISOU 1183  CE1 HIS A 162     9161   7562   7911  -1018     61    353       C  \\nATOM   1184  NE2 HIS A 162      -4.484  -0.750   9.818  1.00 65.93           N  \\nANISOU 1184  NE2 HIS A 162     9123   7862   8066  -1017    -35    339       N  \\nATOM   1185  N   HIS A 163      -2.687  -5.090   6.057  1.00 90.63           N  \\nANISOU 1185  N   HIS A 163    13553  10249  10634  -1107     62   -111       N  \\nATOM   1186  CA  HIS A 163      -3.551  -5.361   4.904  1.00 96.61           C  \\nANISOU 1186  CA  HIS A 163    14389  11045  11272  -1397    -31   -205       C  \\nATOM   1187  C   HIS A 163      -4.133  -6.775   4.915  1.00104.23           C  \\nANISOU 1187  C   HIS A 163    15696  11743  12164  -1635     -6   -299       C  \\nATOM   1188  O   HIS A 163      -4.663  -7.249   3.906  1.00104.93           O  \\nANISOU 1188  O   HIS A 163    15938  11810  12121  -1875    -68   -413       O  \\nATOM   1189  CB  HIS A 163      -2.796  -5.129   3.596  1.00 97.60           C  \\nANISOU 1189  CB  HIS A 163    14588  11238  11259  -1301    -24   -301       C  \\nATOM   1190  CG  HIS A 163      -2.473  -3.693   3.333  1.00 96.52           C  \\nANISOU 1190  CG  HIS A 163    14138  11376  11159  -1176    -78   -206       C  \\nATOM   1191  ND1 HIS A 163      -1.778  -2.911   4.230  1.00 93.68           N  \\nANISOU 1191  ND1 HIS A 163    13571  11089  10935   -931    -24   -100       N  \\nATOM   1192  CD2 HIS A 163      -2.749  -2.897   2.273  1.00102.72           C  \\nANISOU 1192  CD2 HIS A 163    14811  12366  11852  -1276   -191   -196       C  \\nATOM   1193  CE1 HIS A 163      -1.640  -1.694   3.735  1.00 97.08           C  \\nANISOU 1193  CE1 HIS A 163    13783  11740  11363   -894    -88    -35       C  \\nATOM   1194  NE2 HIS A 163      -2.219  -1.659   2.548  1.00103.91           N  \\nANISOU 1194  NE2 HIS A 163    14706  12684  12091  -1092   -192    -81       N  \\nATOM   1195  N   HIS A 164      -4.030  -7.443   6.058  1.00108.04           N  \\nANISOU 1195  N   HIS A 164    16320  12015  12717  -1586     78   -247       N  \\nATOM   1196  CA  HIS A 164      -4.637  -8.756   6.236  1.00113.16           C  \\nANISOU 1196  CA  HIS A 164    17310  12380  13305  -1841    106   -308       C  \\nATOM   1197  C   HIS A 164      -5.488  -8.764   7.516  1.00113.93           C  \\nANISOU 1197  C   HIS A 164    17290  12496  13501  -2018    113   -182       C  \\nATOM   1198  O   HIS A 164      -5.073  -9.319   8.534  1.00110.40           O  \\nANISOU 1198  O   HIS A 164    17030  11833  13085  -1904    200   -111       O  \\nATOM   1199  CB  HIS A 164      -3.545  -9.834   6.305  1.00115.00           C  \\nANISOU 1199  CB  HIS A 164    17959  12246  13489  -1598    226   -381       C  \\nATOM   1200  CG  HIS A 164      -3.953 -11.162   5.738  1.00116.52           C  \\nANISOU 1200  CG  HIS A 164    18592  12122  13557  -1846    247   -526       C  \\nATOM   1201  ND1 HIS A 164      -4.372 -12.213   6.527  1.00117.68           N  \\nANISOU 1201  ND1 HIS A 164    19050  11957  13705  -2013    290   -495       N  \\nATOM   1202  CD2 HIS A 164      -3.986 -11.615   4.462  1.00113.60           C  \\nANISOU 1202  CD2 HIS A 164    18434  11686  13041  -1969    237   -710       C  \\nATOM   1203  CE1 HIS A 164      -4.657 -13.251   5.760  1.00115.48           C  \\nANISOU 1203  CE1 HIS A 164    19154  11421  13301  -2231    302   -655       C  \\nATOM   1204  NE2 HIS A 164      -4.431 -12.914   4.503  1.00113.27           N  \\nANISOU 1204  NE2 HIS A 164    18823  11287  12927  -2208    271   -797       N  \\nATOM   1205  N   HIS A 165      -6.668  -8.143   7.474  1.00117.25           N  \\nANISOU 1205  N   HIS A 165    17397  13186  13966  -2289     22   -151       N  \\nATOM   1206  CA  HIS A 165      -7.167  -7.452   6.285  1.00122.61           C  \\nANISOU 1206  CA  HIS A 165    17853  14127  14605  -2404   -117   -209       C  \\nATOM   1207  C   HIS A 165      -7.290  -5.949   6.530  1.00118.61           C  \\nANISOU 1207  C   HIS A 165    16885  13954  14229  -2250   -183    -97       C  \\nATOM   1208  O   HIS A 165      -7.063  -5.470   7.641  1.00116.85           O  \\nANISOU 1208  O   HIS A 165    16511  13765  14123  -2088   -111      4       O  \\nATOM   1209  CB  HIS A 165      -8.517  -8.027   5.843  1.00128.80           C  \\nANISOU 1209  CB  HIS A 165    18656  14954  15329  -2861   -209   -281       C  \\nATOM   1210  CG  HIS A 165      -8.988  -7.516   4.515  1.00131.62           C  \\nANISOU 1210  CG  HIS A 165    18859  15547  15604  -2992   -381   -346       C  \\nATOM   1211  ND1 HIS A 165     -10.228  -7.825   3.996  1.00134.65           N  \\nANISOU 1211  ND1 HIS A 165    19171  16056  15934  -3398   -514   -407       N  \\nATOM   1212  CD2 HIS A 165      -8.384  -6.723   3.599  1.00130.61           C  \\nANISOU 1212  CD2 HIS A 165    18643  15562  15422  -2788   -452   -352       C  \\nATOM   1213  CE1 HIS A 165     -10.367  -7.242   2.818  1.00134.23           C  \\nANISOU 1213  CE1 HIS A 165    18998  16211  15794  -3420   -679   -441       C  \\nATOM   1214  NE2 HIS A 165      -9.263  -6.567   2.554  1.00132.23           N  \\nANISOU 1214  NE2 HIS A 165    18748  15963  15532  -3059   -637   -404       N  \\nTER    1215      HIS A 165                                                      \\nHETATM 1216 CL    CL A 201      -5.185   5.185  22.315  0.50 37.55          CL  \\nANISOU 1216 CL    CL A 201     4758   4758   4751   -719    649    649      CL  \\nHETATM 1217  O4  PE4 A 202      18.806  11.434   5.330  1.00 67.98           O  \\nANISOU 1217  O4  PE4 A 202     6690  11338   7800   -207   1408     -2       O  \\nHETATM 1218  C7  PE4 A 202      17.805  12.453   5.408  1.00 58.65           C  \\nANISOU 1218  C7  PE4 A 202     5755   9927   6602   -437   1277    148       C  \\nHETATM 1219  C8  PE4 A 202      16.488  11.956   4.727  1.00 53.12           C  \\nANISOU 1219  C8  PE4 A 202     5390   8948   5843   -383   1221    198       C  \\nHETATM 1220  O5  PE4 A 202      15.754  11.049   5.572  1.00 56.63           O  \\nANISOU 1220  O5  PE4 A 202     5902   9204   6412   -118   1086    196       O  \\nHETATM 1221  C9  PE4 A 202      14.331  10.936   5.254  1.00 50.34           C  \\nANISOU 1221  C9  PE4 A 202     5396   8134   5596   -139    975    277       C  \\nHETATM 1222  C10 PE4 A 202      14.025  10.000   3.970  1.00 34.81           C  \\nANISOU 1222  C10 PE4 A 202     3608   6130   3488    -97   1072    204       C  \\nHETATM 1223  O6  PE4 A 202      15.222   9.318   3.472  1.00 48.35           O  \\nANISOU 1223  O6  PE4 A 202     5166   8066   5139     10   1272     51       O  \\nHETATM 1224  C11 PE4 A 202      15.022   8.402   2.387  1.00 46.89           C  \\nANISOU 1224  C11 PE4 A 202     5152   7843   4820     64   1380    -51       C  \\nHETATM 1225  C12 PE4 A 202      14.173   9.037   1.243  1.00 35.99           C  \\nANISOU 1225  C12 PE4 A 202     4036   6381   3256   -213   1349     40       C  \\nHETATM 1226  O7  PE4 A 202      14.951   9.345   0.122  1.00 58.20           O  \\nANISOU 1226  O7  PE4 A 202     6837   9403   5874   -383   1547    -20       O  \\nHETATM 1227  C13 PE4 A 202      14.260  10.034  -0.848  1.00 57.26           C  \\nANISOU 1227  C13 PE4 A 202     6962   9221   5575   -648   1496     92       C  \\nHETATM 1228  C14 PE4 A 202      13.641  11.143  -0.481  1.00 52.40           C  \\nANISOU 1228  C14 PE4 A 202     6404   8490   5017   -801   1319    277       C  \\nHETATM 1229  O8  PE4 A 202      12.280  11.116  -0.572  1.00 54.59           O  \\nANISOU 1229  O8  PE4 A 202     6901   8536   5304   -807   1113    381       O  \\nHETATM 1230  O4  PE5 A 203      -5.402  -5.046  14.054  1.00 73.30           O  \\nANISOU 1230  O4  PE5 A 203    10985   8033   8835  -1493    334    461       O  \\nHETATM 1231  C7  PE5 A 203      -4.562  -6.107  14.235  1.00 76.38           C  \\nANISOU 1231  C7  PE5 A 203    11756   8125   9139  -1372    360    484       C  \\nHETATM 1232  C8  PE5 A 203      -3.254  -5.806  14.116  1.00 83.62           C  \\nANISOU 1232  C8  PE5 A 203    12681   9024  10066  -1018    337    495       C  \\nHETATM 1233  O5  PE5 A 203      -2.359  -6.849  13.981  1.00 87.99           O  \\nANISOU 1233  O5  PE5 A 203    13585   9291  10555   -843    346    487       O  \\nHETATM 1234  C9  PE5 A 203      -1.722  -6.885  12.698  1.00 87.01           C  \\nANISOU 1234  C9  PE5 A 203    13485   9151  10424   -692    319    374       C  \\nHETATM 1235  C10 PE5 A 203      -0.429  -6.037  12.837  1.00 61.06           C  \\nANISOU 1235  C10 PE5 A 203     9994   6018   7189   -327    308    403       C  \\nHETATM 1236  O6  PE5 A 203      -0.688  -4.698  12.376  1.00 64.33           O  \\nANISOU 1236  O6  PE5 A 203    10040   6742   7661   -367    274    386       O  \\nHETATM 1237  C11 PE5 A 203       0.440  -3.842  12.531  1.00 53.33           C  \\nANISOU 1237  C11 PE5 A 203     8450   5502   6311    -89    269    413       C  \\nHETATM 1238  C12 PE5 A 203      -0.019  -2.362  12.502  1.00 52.69           C  \\nANISOU 1238  C12 PE5 A 203     8010   5707   6301   -168    233    438       C  \\nHETATM 1239  O7  PE5 A 203       0.330  -1.773  13.784  1.00 55.44           O  \\nANISOU 1239  O7  PE5 A 203     8234   6139   6692    -67    243    528       O  \\nHETATM 1240  C13 PE5 A 203       1.532  -1.068  13.861  1.00 42.18           C  \\nANISOU 1240  C13 PE5 A 203     6413   4580   5035    168    233    539       C  \\nHETATM 1241  C14 PE5 A 203       2.453  -1.455  14.771  1.00 40.88           C  \\nANISOU 1241  C14 PE5 A 203     6329   4354   4848    360    234    594       C  \\nHETATM 1242  O8  PE5 A 203       3.770  -1.142  14.457  1.00 53.68           O  \\nANISOU 1242  O8  PE5 A 203     7848   6068   6480    601    226    569       O  \\nHETATM 1243  C15 PE5 A 203       4.750  -1.915  15.140  1.00 56.63           C  \\nANISOU 1243  C15 PE5 A 203     8351   6337   6828    834    207    612       C  \\nHETATM 1244  C16 PE5 A 203       6.067  -1.959  14.309  1.00 53.31           C  \\nANISOU 1244  C16 PE5 A 203     7847   5986   6423   1088    226    533       C  \\nHETATM 1245  O52 PE5 A 203       7.151  -1.591  15.111  1.00 56.45           O  \\nANISOU 1245  O52 PE5 A 203     8089   6519   6839   1290    177    586       O  \\nHETATM 1246  C3  PE6 A 204      11.394   8.034  18.595  1.00 63.90           C  \\nANISOU 1246  C3  PE6 A 204     7271   9142   7865    813    -58    591       C  \\nHETATM 1247  O2  PE6 A 204      10.905   9.368  18.469  1.00 58.59           O  \\nANISOU 1247  O2  PE6 A 204     6570   8468   7222    601    -19    561       O  \\nHETATM 1248  C4  PE6 A 204      12.135   7.706  17.252  1.00 44.74           C  \\nANISOU 1248  C4  PE6 A 204     4738   6790   5470    906      7    547       C  \\nHETATM 1249  O5  PE6 A 204      11.154   7.078  16.402  1.00 43.18           O  \\nANISOU 1249  O5  PE6 A 204     4718   6398   5291    913     73    548       O  \\nHETATM 1250  C6  PE6 A 204      11.582   6.993  15.021  1.00 47.62           C  \\nANISOU 1250  C6  PE6 A 204     5224   7012   5859    933    160    493       C  \\nHETATM 1251  C7  PE6 A 204      10.584   6.003  14.316  1.00 41.39           C  \\nANISOU 1251  C7  PE6 A 204     4657   6004   5064    966    200    490       C  \\nHETATM 1252  O8  PE6 A 204       9.560   6.850  13.787  1.00 36.03           O  \\nANISOU 1252  O8  PE6 A 204     4026   5264   4398    761    216    504       O  \\nHETATM 1253  C9  PE6 A 204       8.495   6.171  13.195  1.00 22.76           C  \\nANISOU 1253  C9  PE6 A 204     2523   3414   2712    729    230    504       C  \\nHETATM 1254  C10 PE6 A 204       7.423   7.247  13.009  1.00 24.90           C  \\nANISOU 1254  C10 PE6 A 204     2790   3650   3019    537    206    537       C  \\nHETATM 1255  O11 PE6 A 204       6.477   6.642  12.151  1.00 30.47           O  \\nANISOU 1255  O11 PE6 A 204     3632   4239   3709    486    209    529       O  \\nHETATM 1256  O17 PE6 A 204       3.166   2.354  11.021  1.00 40.16           O  \\nANISOU 1256  O17 PE6 A 204     5547   4884   4827    304    194    445       O  \\nHETATM 1257  C16 PE6 A 204       3.039   3.356  11.961  1.00 31.95           C  \\nANISOU 1257  C16 PE6 A 204     4339   3938   3862    292    172    507       C  \\nHETATM 1258  C15 PE6 A 204       2.980   4.655  11.165  1.00 38.40           C  \\nANISOU 1258  C15 PE6 A 204     5002   4887   4701    227    138    518       C  \\nHETATM 1259  O14 PE6 A 204       3.854   5.543  11.837  1.00 36.07           O  \\nANISOU 1259  O14 PE6 A 204     4569   4694   4441    309    153    543       O  \\nHETATM 1260  C13 PE6 A 204       4.441   6.497  10.965  1.00 29.51           C  \\nANISOU 1260  C13 PE6 A 204     3649   3973   3590    290    153    544       C  \\nHETATM 1261  C12 PE6 A 204       5.337   7.432  11.877  1.00 24.53           C  \\nANISOU 1261  C12 PE6 A 204     2874   3445   2999    342    165    564       C  \\nHETATM 1262  O   HOH A 301       6.272  12.148   3.204  1.00 26.49           O  \\nANISOU 1262  O   HOH A 301     3461   4034   2570   -482    112    835       O  \\nHETATM 1263  O   HOH A 302      11.948  16.088  20.987  1.00 39.62           O  \\nANISOU 1263  O   HOH A 302     4008   6274   4774   -353    -63    300       O  \\nHETATM 1264  O   HOH A 303      12.593  19.007  24.028  1.00 36.11           O  \\nANISOU 1264  O   HOH A 303     3735   5861   4125   -904   -143     14       O  \\nHETATM 1265  O   HOH A 304      -4.755  -0.797  15.247  1.00 34.68           O  \\nANISOU 1265  O   HOH A 304     5109   3840   4228   -986    312    545       O  \\nHETATM 1266  O   HOH A 305      -2.861  17.616   2.601  1.00 48.13           O  \\nANISOU 1266  O   HOH A 305     6133   5993   6160   -145  -1386   1680       O  \\nHETATM 1267  O   HOH A 306      -1.321  22.699  31.175  1.00 38.49           O  \\nANISOU 1267  O   HOH A 306     5033   4677   4915   -282   1623  -1407       O  \\nHETATM 1268  O   HOH A 307      -6.019  12.052  12.830  1.00 26.81           O  \\nANISOU 1268  O   HOH A 307     2244   3653   4290     43   -241    627       O  \\nHETATM 1269  O   HOH A 308      16.673   5.257   3.282  1.00 45.55           O  \\nANISOU 1269  O   HOH A 308     4715   7727   4863    902   1576   -417       O  \\nHETATM 1270  O   HOH A 309      -8.485  11.574   0.972  1.00 54.17           O  \\nANISOU 1270  O   HOH A 309     6229   7545   6807   -594  -1975   1329       O  \\nHETATM 1271  O   HOH A 310      -1.452  20.828  29.409  1.00 38.81           O  \\nANISOU 1271  O   HOH A 310     4787   4869   5089   -173   1394   -999       O  \\nHETATM 1272  O   HOH A 311      14.988   2.770   3.865  1.00 40.92           O  \\nANISOU 1272  O   HOH A 311     4664   6509   4376   1291   1396   -461       O  \\nHETATM 1273  O   HOH A 312       6.549  11.799  30.005  1.00 39.13           O  \\nANISOU 1273  O   HOH A 312     5096   5957   3816   -296     87    325       O  \\nHETATM 1274  O   HOH A 313       9.541  21.274  22.842  1.00 36.54           O  \\nANISOU 1274  O   HOH A 313     4233   5123   4529   -897    127    -62       O  \\nHETATM 1275  O   HOH A 314      16.177  -3.215   4.294  1.00 68.40           O  \\nANISOU 1275  O   HOH A 314     8747   9138   8103   2715   1611   -989       O  \\nHETATM 1276  O   HOH A 315       0.989  -2.340  18.672  1.00 31.82           O  \\nANISOU 1276  O   HOH A 315     5501   3029   3561     59    277    857       O  \\nHETATM 1277  O   HOH A 316      10.721  14.295  15.542  1.00 39.06           O  \\nANISOU 1277  O   HOH A 316     3994   5969   4877    -81    145    505       O  \\nHETATM 1278  O   HOH A 317      -4.942  13.859  27.177  1.00 50.68           O  \\nANISOU 1278  O   HOH A 317     5704   6905   6647   -259   1384   -326       O  \\nHETATM 1279  O   HOH A 318     -10.705   9.125  26.802  1.00 55.25           O  \\nANISOU 1279  O   HOH A 318     5659   7997   7336  -1014   1894   -279       O  \\nHETATM 1280  O   HOH A 319      12.403  13.051  20.420  1.00 40.45           O  \\nANISOU 1280  O   HOH A 319     4003   6515   4850    103   -118    432       O  \\nHETATM 1281  O   HOH A 320       3.146   7.405  -1.008  1.00 55.27           O  \\nANISOU 1281  O   HOH A 320     7843   7600   5558   -747   -109    546       O  \\nHETATM 1282  O   HOH A 321       0.473   3.878   2.272  1.00 43.74           O  \\nANISOU 1282  O   HOH A 321     6315   5697   4607   -583   -199    334       O  \\nHETATM 1283  O   HOH A 322      22.534  13.813   8.897  1.00 56.95           O  \\nANISOU 1283  O   HOH A 322     4372  10553   6714   -521   1102    -63       O  \\nHETATM 1284  O   HOH A 323       2.044   7.517  -3.477  1.00 35.27           O  \\nANISOU 1284  O   HOH A 323     5649   5163   2589  -1102   -355    617       O  \\nHETATM 1285  O   HOH A 324       1.911  -3.531  21.372  1.00 32.96           O  \\nANISOU 1285  O   HOH A 324     6090   2964   3470    202    211   1119       O  \\nHETATM 1286  O   HOH A 325      10.919  17.501  26.317  1.00 41.26           O  \\nANISOU 1286  O   HOH A 325     4640   6440   4596   -677   -133      3       O  \\nHETATM 1287  O   HOH A 326     -11.938   8.081  24.063  1.00 47.82           O  \\nANISOU 1287  O   HOH A 326     4368   7089   6712  -1072   1574   -147       O  \\nHETATM 1288  O   HOH A 327     -12.231  10.217  14.349  1.00 56.72           O  \\nANISOU 1288  O   HOH A 327     4931   8068   8554   -333    -90    332       O  \\nHETATM 1289  O   HOH A 328      15.336  26.153  23.930  1.00 50.05           O  \\nANISOU 1289  O   HOH A 328     5933   7258   5827  -2489    -80   -451       O  \\nHETATM 1290  O   HOH A 329      -0.221  15.105  -2.263  1.00 54.30           O  \\nANISOU 1290  O   HOH A 329     7699   7244   5689   -917  -1242   1689       O  \\nHETATM 1291  O   HOH A 330      -6.494  16.930  26.354  1.00 47.89           O  \\nANISOU 1291  O   HOH A 330     4971   6365   6860    131   1478   -622       O  \\nHETATM 1292  O   HOH A 331      -0.662  11.559  30.723  1.00 44.47           O  \\nANISOU 1292  O   HOH A 331     5826   6301   4771   -612   1110    -22       O  \\nHETATM 1293  O   HOH A 332      -0.267  26.969  14.543  1.00 43.51           O  \\nANISOU 1293  O   HOH A 332     5826   3772   6936    402   -228    657       O  \\nHETATM 1294  O   HOH A 333     -11.541  12.092   1.072  1.00 60.51           O  \\nANISOU 1294  O   HOH A 333     6369   8621   8001   -481  -2435   1401       O  \\nHETATM 1295  O   HOH A 334      17.575  29.772  12.558  1.00 51.60           O  \\nANISOU 1295  O   HOH A 334     6665   6902   6037  -3619    472    550       O  \\nHETATM 1296  O   HOH A 335      15.873  11.022  12.841  1.00 41.15           O  \\nANISOU 1296  O   HOH A 335     3515   7152   4966    376    406    320       O  \\nHETATM 1297  O   HOH A 336       5.886   9.288  30.290  1.00 50.80           O  \\nANISOU 1297  O   HOH A 336     6823   7321   5156   -183     71    575       O  \\nHETATM 1298  O   HOH A 337       9.332  11.420  22.448  1.00 39.65           O  \\nANISOU 1298  O   HOH A 337     4383   6005   4678    233    -77    503       O  \\nCONECT 1217 1218                                                                \\nCONECT 1218 1217 1219                                                           \\nCONECT 1219 1218 1220                                                           \\nCONECT 1220 1219 1221                                                           \\nCONECT 1221 1220 1222                                                           \\nCONECT 1222 1221 1223                                                           \\nCONECT 1223 1222 1224                                                           \\nCONECT 1224 1223 1225                                                           \\nCONECT 1225 1224 1226                                                           \\nCONECT 1226 1225 1227                                                           \\nCONECT 1227 1226 1228                                                           \\nCONECT 1228 1227 1229                                                           \\nCONECT 1229 1228                                                                \\nCONECT 1230 1231                                                                \\nCONECT 1231 1230 1232                                                           \\nCONECT 1232 1231 1233                                                           \\nCONECT 1233 1232 1234                                                           \\nCONECT 1234 1233 1235                                                           \\nCONECT 1235 1234 1236                                                           \\nCONECT 1236 1235 1237                                                           \\nCONECT 1237 1236 1238                                                           \\nCONECT 1238 1237 1239                                                           \\nCONECT 1239 1238 1240                                                           \\nCONECT 1240 1239 1241                                                           \\nCONECT 1241 1240 1242                                                           \\nCONECT 1242 1241 1243                                                           \\nCONECT 1243 1242 1244                                                           \\nCONECT 1244 1243 1245                                                           \\nCONECT 1245 1244                                                                \\nCONECT 1246 1247 1248                                                           \\nCONECT 1247 1246                                                                \\nCONECT 1248 1246 1249                                                           \\nCONECT 1249 1248 1250                                                           \\nCONECT 1250 1249 1251                                                           \\nCONECT 1251 1250 1252                                                           \\nCONECT 1252 1251 1253                                                           \\nCONECT 1253 1252 1254                                                           \\nCONECT 1254 1253 1255                                                           \\nCONECT 1255 1254 1261                                                           \\nCONECT 1256 1257                                                                \\nCONECT 1257 1256 1258                                                           \\nCONECT 1258 1257 1259                                                           \\nCONECT 1259 1258 1260                                                           \\nCONECT 1260 1259 1261                                                           \\nCONECT 1261 1255 1260                                                           \\nMASTER      319    0    4    6    6    0    7    6 1297    1   45   13          \\nEND                                                                             \\n\",\"pdb\");\n",
              "\tviewer_17228382996107566.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"#33ff33\"}});\n",
              "\tviewer_17228382996107566.zoomTo();\n",
              "viewer_17228382996107566.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Backbone visualization of 4etj.pdb (164 amino acids)\n"
          ]
        }
      ],
      "source": [
        "#@title **3.3.1: Upload .pdb/.cif structure file**\n",
        "\n",
        "#@markdown After clicking the run button, an upload button will appear for you to upload your .pdb/.cif structure file.\n",
        "\n",
        "#@markdown <font face=\"Consolas\" size=2 color='gray'>NoteÔºösince you may not know the AA type, you can simply populate your .pdb/.cif file with any random AA. If you want to predit partial positions given some accurate AA information in other positions, just input the accurate AA in these positions and any random AA in unknown positions.</fonte>\n",
        "\n",
        "#@markdown After uploading is finished, the .pdb/.cif structure will be transformed into the corresponding AA Sequence and Structure (3Di) Sequence.\n",
        "\n",
        "data_type = \"Single PDB/CIF Structure\"\n",
        "# raw_data = input_raw_data_by_data_type(data_type)\n",
        "\n",
        "def get_structure_file():\n",
        "  print(\"Please provide the structure type, chain and your structure file.\")\n",
        "\n",
        "  dropdown_type = ipywidgets.Dropdown(\n",
        "    value=\"PDB\",\n",
        "    options=[\"PDB\", \"AF2\"],\n",
        "    disabled=False)\n",
        "  dropdown_type.layout.width = '500px'\n",
        "  print(Fore.BLUE+\"Structure type:\"+Style.RESET_ALL)\n",
        "  display(dropdown_type)\n",
        "\n",
        "  input_chain = ipywidgets.Text(\n",
        "    value=\"A\",\n",
        "    placeholder=f'Enter the name of chain here',\n",
        "    disabled=False)\n",
        "  input_chain.layout.width = '500px'\n",
        "  print(Fore.BLUE+\"Chain:\"+Style.RESET_ALL)\n",
        "  display(input_chain)\n",
        "\n",
        "  print(Fore.BLUE+\"Please upload a .pdb/.cif file\"+Style.RESET_ALL)\n",
        "  pdb_file_path = upload_file(STRUCTURE_HOME)\n",
        "  return pdb_file_path, pdb_file_path.stem, dropdown_type, input_chain\n",
        "\n",
        "\n",
        "backbone_path, stem, dropdown_type, input_chain = get_structure_file()\n",
        "raw_data = (stem, dropdown_type, input_chain)\n",
        "\n",
        "sa_seq = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "\n",
        "aa_seq = sa_seq[0::2]\n",
        "struc_seq = sa_seq[1::2]\n",
        "\n",
        "# masked_sa_seq = ''\n",
        "# for s in sa_seq[1::2]:\n",
        "#   masked_sa_seq += '#' + s\n",
        "\n",
        "clear_output(wait=True)\n",
        "\n",
        "################################################################################\n",
        "################################################################################\n",
        "################################################################################\n",
        "\n",
        "input_aa_seq = ipywidgets.Text(\n",
        "      value=aa_seq,\n",
        "      placeholder='Enter Amino Acid Sequence here',\n",
        "      disabled=False)\n",
        "print(Fore.BLUE+\"Amino Acid Sequence:\"+Style.RESET_ALL)\n",
        "input_aa_seq.layout.width = '500px'\n",
        "display(input_aa_seq)\n",
        "\n",
        "input_struc_seq = ipywidgets.Text(\n",
        "  value=struc_seq,\n",
        "  placeholder='Enter Structure Sequence here',\n",
        "  disabled=False)\n",
        "print(Fore.BLUE+\"Structure Sequence:\"+Style.RESET_ALL)\n",
        "input_struc_seq.layout.width = '500px'\n",
        "display(input_struc_seq)\n",
        "\n",
        "# print(Fore.RED+\"If you want to mask all amino acids and make prediction, simply clear the 'Amino Acid Sequence' box.\")\n",
        "\n",
        "backbone_name = os.path.basename(backbone_path)\n",
        "show_pdb(backbone_path, color=\"chain\").show()\n",
        "print(f\"Backbone visualization of {backbone_name} ({len(struc_seq)} amino acids)\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "baiH-BrBl2Ge"
      },
      "outputs": [],
      "source": [
        "#@title **3.3.2: Predict Amino Acid Sequence**\n",
        "\n",
        "#@markdown You can **mask partial or all amino acids** in the AA sequence with '#' at certain positions, allowing the model to make predictions for those masked amino acids.\n",
        "\n",
        "#@markdown <font color=\"red\">If you want to **mask all amino acids** and make prediction, simply clear the 'masked_aa_seq' box.</font>\n",
        "\n",
        "#@markdown | Original AA Sequence | Masked AA Sequence | Description                                |\n",
        "#@markdown | -------------------- | ------------------ | ------------------------------------------ |\n",
        "#@markdown | MEETMKLATMEDTVEYCL   | ME#T#KL#TMEDTVEYCL | Predict the 3rd, 5th, and 8th amino acids. |\n",
        "#@markdown | MEETMKLATMEDTVEYCL   |                    | Predict all amino acids.                   |\n",
        "\n",
        "#@markdown <br>\n",
        "\n",
        "# #@markdown Click the run button to get the predicted Amino Acid Sequence\n",
        "masked_aa_seq = \"\" # @param {type:\"string\", placeholder:\"mask the amino acids with `#` and then paste the sequence here\"}\n",
        "method = \"multinomial\" # @param [\"argmax\", \"multinomial\"]\n",
        "num_samples = 10 # @param {type:\"integer\"}\n",
        "\n",
        "#@markdown - `method` refers to the prediction method. It could be either \"argmax\" or \"multinomial\".\n",
        "#@markdown   - `argmax` selects the amino acid with the highest probability.\n",
        "#@markdown   - `multinomial` samples an amino acid from the multinomial distribution.\n",
        "\n",
        "\n",
        "#@markdown - `num_samples` refers to the number of output amino acid sequences.\n",
        "\n",
        "save_name = \"predicted_seq\" # @param {type:\"string\"}\n",
        "\n",
        "\n",
        "\n",
        "################################################################################\n",
        "############################### Dataset ########################################\n",
        "################################################################################\n",
        "\n",
        "# masked_aa_seq = input_aa_seq.value\n",
        "if masked_aa_seq.strip() == \"\":\n",
        "  masked_aa_seq = \"#\" * len(input_struc_seq.value)\n",
        "\n",
        "masked_struc_seq = input_struc_seq.value\n",
        "\n",
        "# assert len(masked_aa_seq) == len(masked_struc_seq), f\"Please make sure that the amino acid sequence ({len(masked_aa_seq)}) and the structure sequence ({len(masked_struc_seq)}) have the same length.\"\n",
        "# masked_sa_seq = ''.join(a + b for a, b in zip(masked_aa_seq, masked_struc_seq))\n",
        "\n",
        "\n",
        "# if num_samples == 1:\n",
        "#   method = \"argmax\"\n",
        "# elif num_samples > 1:\n",
        "#   method = \"multinomial\"\n",
        "# else:\n",
        "#   raise BaseException(\"\\\"num_samples\\\" should be an integer greater than or equal to 1.\")\n",
        "\n",
        "################################################################################\n",
        "############################### Model ##########################################\n",
        "################################################################################\n",
        "# base_model = \"westlake-repl/SaProt_650M_AF2\"\n",
        "base_model = \"westlake-repl/SaProt_650M_AF2_inverse_folding\"\n",
        "\n",
        "config = {\n",
        "    \"config_path\": base_model,\n",
        "    \"load_pretrained\": True,\n",
        "}\n",
        "from saprot.model.saprot.saprot_if_model import SaProtIFModel\n",
        "try:\n",
        "  saprot_if_model\n",
        "except Exception:\n",
        "  saprot_if_model = SaProtIFModel(**config)\n",
        "  tokenizer = saprot_if_model.tokenizer\n",
        "  device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "  saprot_if_model.to(device)\n",
        "\n",
        "################################################################################\n",
        "############################### Predict ########################################\n",
        "################################################################################\n",
        "\n",
        "pred_aa_seqs = saprot_if_model.predict(masked_aa_seq, masked_struc_seq, method=method, num_samples=num_samples)\n",
        "\n",
        "clear_output(wait=True)\n",
        "print('='*100)\n",
        "print(Fore.BLUE+\"Outputs:\"+Style.RESET_ALL)\n",
        "save_path = f\"{root_dir}/SaprotHub/output/{save_name}.fasta\"\n",
        "with open(save_path, \"w\") as w:\n",
        "  for i, aa_seq in enumerate(pred_aa_seqs):\n",
        "    print(aa_seq)\n",
        "    w.write(f\">predicted_seq_{i}\\n{aa_seq}\")\n",
        "\n",
        "file_download(save_path)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "idDDKW2pl2Gf"
      },
      "outputs": [],
      "source": [
        "#@title **3.3.3: Predict the structure of generated sequence**\n",
        "\n",
        "#@markdown <font color=\"red\">**Warning: Please make sure you have enough RAM to run this cell. (Do not use Colab's T4 or local server with less than 64G RAM).**\n",
        "#@markdown\n",
        "#@markdown Otherwise, it will cause an out-of-memory error, and you will have to restart the notebook. We recommend you connect to a runtime\n",
        "#@markdown with more RAM to run the cell properly.</font>\n",
        "\n",
        "#@markdown Click the run button to predict the structure of generated sequence using ESMFold\n",
        "\n",
        "protein_sequence = \"QQVGSQLDLQEESVEYQIFPTQTHQNDTKNVKERLESILERINSIFIPYSQDYVWQEKELSFMISLGLQQGRPHLMGSTHFGDNIDDEWFLVNLLKQLSQVFPQLTAKISDSDGEFLLIEAADLMPKWLIPEGIENRVFVYNGNLMIIPFLLGRYSLIHYQLSKPSLDQAIDLLRNFPEETRASRDQQKLIHNRINGIIKSFLAGTHKAYCYIPRPIATLLKRKPSLVSHAVETFYYRDPIDVKNCRNMEDFTNAERLRVDVRFTRVLYAQLVSQSFNPPKQMGIEAPDPEDKEFKRELLGMKLTCGFAMMAANLLPSTVDPSLNGWAYLEQFKRFRENVEKGNATAKISEPDDQLELISAVRKFLRYIVEDHIDASILKSLLVVELHRQKQMLPESEEAIRKIKKTLLERWNPGWQMSEEYREKTVGQVENGGDSSCEALKSDSKRADLADLDMGRVQDLSRFIDKESRPLERSKISDLQPEVVMGMEQEEDAAAAVSKVYKGGPYLVPIADLKERPEAVHPKATQVVQGELLLISAEDQESKTSNRRVRFGQHGQSQDQAAPMLVGCDRMTALDSIVPEKEEDKVKKGLGYIHLEKSTNSLLTHAIYKIQGSVSHVAARLADRGIDVTSDNVPIKPQTMEEG\" # @param {type:\"string\"}\n",
        "save_name = \"predicted_structure\" # @param {type:\"string\"}\n",
        "\n",
        "#@markdown Visualization settings\n",
        "color = \"lDDT\" #@param [\"chain\", \"lDDT\", \"rainbow\"]\n",
        "show_sidechains = False #@param {type:\"boolean\"}\n",
        "show_mainchains = False #@param {type:\"boolean\"}\n",
        "\n",
        "\n",
        "################################################################################\n",
        "############################### LOAD ESMFOLD ################################\n",
        "################################################################################\n",
        "try:\n",
        "  esmfold\n",
        "except Exception:\n",
        "  tokenizer = AutoTokenizer.from_pretrained(\"facebook/esmfold_v1\")\n",
        "  esmfold = EsmForProteinFolding.from_pretrained(\"facebook/esmfold_v1\")\n",
        "  esmfold.esm = esmfold.esm.half()\n",
        "  esmfold.trunk.set_chunk_size(64)\n",
        "\n",
        "  device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "  esmfold.to(device)\n",
        "\n",
        "################################################################################\n",
        "################################## PREDICT ###################################\n",
        "################################################################################\n",
        "tokenized_input = tokenizer(\n",
        "    [protein_sequence],\n",
        "    return_tensors=\"pt\",\n",
        "    add_special_tokens=False,\n",
        "    max_length=1024,\n",
        "    truncation=True,\n",
        "    )['input_ids']\n",
        "\n",
        "tokenized_input = tokenized_input.to(esmfold.device)\n",
        "with torch.no_grad():\n",
        "  output = esmfold(tokenized_input)\n",
        "\n",
        "################################################################################\n",
        "#################################### SAVE ####################################\n",
        "################################################################################\n",
        "save_path = f\"{root_dir}/SaprotHub/output/{save_name}.pdb\"\n",
        "pdb = convert_outputs_to_pdb(output)\n",
        "with open(save_path, \"w\") as f:\n",
        "  f.write(\"\".join(pdb))\n",
        "\n",
        "################################################################################\n",
        "################################# VISUALIZE ##################################\n",
        "################################################################################\n",
        "show_pdb(save_path, show_sidechains, show_mainchains, color).show()\n",
        "if color == \"lDDT\":\n",
        "  plot_plddt_legend().show()\n",
        "\n",
        "print(\"Predicted structure\")\n",
        "file_download(save_path)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "PecOgaEEZw4C"
      },
      "outputs": [],
      "source": [
        "#@title **3.3.4: Align proteins using TMalign**\n",
        "\n",
        "#@markdown You can find the **uploaded proteins** from /content/SaprotHub/structures (if you connect to your local server, then the path is /SaprotHub/structures).\n",
        "\n",
        "#@markdown You can find the **predicted proteins** from /content/SaprotHub/output (if you connect to your local server, then the path is /SaprotHub/output).\n",
        "\n",
        "#@markdown Right click the pdb file to copy the path and then paste it into the box:\n",
        "pdb_path_1 = \"/SaprotHub/structures/1qsf.pdb\" # @param {type:\"string\"}\n",
        "pdb_path_2 = \"/SaprotHub/output/predicted_structure.pdb\" # @param {type:\"string\"}\n",
        "\n",
        "pdb_path_1 = f\"{root_dir}{pdb_path_1}\"\n",
        "pdb_path_2 = f\"{root_dir}{pdb_path_2}\"\n",
        "\n",
        "assert os.path.exists(pdb_path_1) and os.path.exists(pdb_path_2), \"Input proteins do not exist!\"\n",
        "\n",
        "cmd = f\"{root_dir}/SaprotHub/bin/TMalign {pdb_path_1} {pdb_path_2}\"\n",
        "print(os.popen(cmd).read())"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "LIIoTQgJBI7o"
      },
      "source": [
        "# **4: (Optional) Data Preparation**"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "collapsed": true,
        "id": "BgsBSLcmBI7o"
      },
      "outputs": [],
      "source": [
        "# @title **4.1: Get Structure-Aware Sequence** <a name=\"get_sa\"></a>\n",
        "\n",
        "#@markdown AA Sequence, UniProt ID, PDB/CIF file -> SA Sequence\n",
        "\n",
        "################################################################################\n",
        "################################ input #########################################\n",
        "################################################################################\n",
        "\n",
        "data_type = \"Single PDB/CIF Structure\"  # @param [\"Single AA Sequence\", \"Single UniProt ID\", \"Single PDB/CIF Structure\", \"Multiple AA Sequences\", \"Multiple UniProt IDs\", \"Multiple PDB/CIF Structures\"]\n",
        "\n",
        "if data_type == data_type_list[7]:\n",
        "    # upload and unzip PDB files\n",
        "    print(Fore.BLUE + f\"Please upload your .zip file which contains {data_type} files\" + Style.RESET_ALL)\n",
        "    pdb_zip_path = upload_file(UPLOAD_FILE_HOME)\n",
        "    if pdb_zip_path.suffix != \".zip\":\n",
        "        logger.error(\"The data type does not match. Please click the run button again to upload a .zip file!\")\n",
        "        raise RuntimeError(\"The data type does not match.\")\n",
        "    print(Fore.BLUE + \"Successfully upload your .zip file!\" + Style.RESET_ALL)\n",
        "    print(\"=\"*100)\n",
        "\n",
        "    import zipfile\n",
        "    with zipfile.ZipFile(pdb_zip_path, 'r') as zip_ref:\n",
        "        file_names = zip_ref.namelist()\n",
        "        zip_ref.extractall(STRUCTURE_HOME)\n",
        "\n",
        "    uploaded_csv_path = UPLOAD_FILE_HOME / f\"{pdb_zip_path.stem}.csv\"\n",
        "    df = pd.DataFrame(file_names, columns=['sequence'])\n",
        "    df.to_csv(uploaded_csv_path, index=False)\n",
        "    raw_data = uploaded_csv_path\n",
        "\n",
        "else:\n",
        "    raw_data = input_raw_data_by_data_type(data_type)\n",
        "\n",
        "################################################################################\n",
        "############################### output #########################################\n",
        "################################################################################\n",
        "\n",
        "if data_type in [\"Single AA Sequence\", \"Single UniProt ID\", \"Single PDB/CIF Structure\"]:\n",
        "    def apply(button):\n",
        "        button.disabled = True\n",
        "        button.description = 'Clicked'\n",
        "        button.button_style = ''\n",
        "        sa_seq = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "\n",
        "        print(\"=\"*100)\n",
        "        print(f\"Amino Acid Sequence: {sa_seq[0::2]}\")\n",
        "        print(f\"Structure Sequence: {sa_seq[1::2]}\")\n",
        "        print(\"=\"*100)\n",
        "        print(Fore.BLUE + \"The Structure-Aware Sequence is here, double click to select and copy it:\" + Style.RESET_ALL)\n",
        "        print(sa_seq)\n",
        "\n",
        "    button_apply = ipywidgets.Button(\n",
        "        description='Apply',\n",
        "        disabled=False,\n",
        "        button_style='success',  # 'success', 'info', 'warning', 'danger' or ''\n",
        "        tooltip='Apply',\n",
        "        icon='check'  # (FontAwesome names without the `fa-` prefix)\n",
        "    )\n",
        "    button_apply.on_click(apply)\n",
        "    button_apply.layout.width = '500px'\n",
        "    display(button_apply)\n",
        "else:\n",
        "    csv_dataset = get_SA_sequence_by_data_type(data_type, raw_data)\n",
        "    print(Fore.BLUE + \"The Structure-Aware Sequences are saved in a .csv file here:\" + Style.RESET_ALL)\n",
        "    print(csv_dataset)\n",
        "    file_download(csv_dataset)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "IDkm_OeABI7o"
      },
      "outputs": [],
      "source": [
        "#@title **4.2: Convert `.fa/.fasta` file to `.csv` file in the data format of \"Multiple AA Sequences\"**\n",
        "\n",
        "\n",
        "#@markdown `.fa/.fasta` -> Multiple AA Sequences `.csv` <a name=\"fa2csv\"></a>\n",
        "\n",
        "from Bio import SeqIO\n",
        "import numpy as np\n",
        "\n",
        "aa_seq_dict = { 'sequence': [],\n",
        "                # \"label\": [],\n",
        "                # \"stage\":[]\n",
        "                }\n",
        "\n",
        "fa_file_path = upload_file(UPLOAD_FILE_HOME)\n",
        "assert Path(fa_file_path).name.split('.')[1] in ['fa', 'fasta'], \"Please upload a .fa or .fasta file.\"\n",
        "with fa_file_path.open(\"r\") as fa:\n",
        "  for record in tqdm(SeqIO.parse(fa, 'fasta'), leave=True):\n",
        "      aa_seq_dict['sequence'].append(str(record.seq))\n",
        "\n",
        "fa_df = pd.DataFrame(aa_seq_dict)\n",
        "print(fa_df[5:])\n",
        "\n",
        "csv_file_path = UPLOAD_FILE_HOME / f'{fa_file_path.stem}.csv'\n",
        "fa_df.to_csv(csv_file_path, index=None)\n",
        "# files.download(csv_file_path)\n",
        "file_download(csv_file_path)\n",
        "\n",
        "################################################################################\n",
        "############################ .fa 2 .csv and split ##############################\n",
        "################################################################################\n",
        "\n",
        "# automatically_split_dataset = False # @param {type:\"boolean\"}\n",
        "# split = ['train', 'valid', 'test']\n",
        "\n",
        "# aa_seq_dict = { 'sequence': [],\n",
        "#                 \"label\": [],\n",
        "#                 \"stage\":[]}\n",
        "\n",
        "\n",
        "\n",
        "# if automatically_split_dataset:\n",
        "\n",
        "#   fa_file_path = upload_file(UPLOAD_FILE_HOME)\n",
        "#   label = fa_file_path.stem\n",
        "\n",
        "#   with fa_file_path.open(\"r\") as fa:\n",
        "#       for record in tqdm(SeqIO.parse(fa, 'fasta'), leave=True):\n",
        "#           aa_seq_dict['sequence'].append(str(record.seq))\n",
        "#           aa_seq_dict[\"label\"].append(label)\n",
        "#   weights = [0.8, 0.1, 0.1]\n",
        "#   aa_seq_dict[\"stage\"] = np.random.choice(split, size=len(aa_seq_dict['sequence']), p=weights).tolist()\n",
        "\n",
        "# else:\n",
        "#   for i in range(3):\n",
        "#     print(Fore.BLUE+f\"Please upload a .fa file as your {split[i]} dataset\")\n",
        "#     fa_file_path = upload_file(UPLOAD_FILE_HOME)\n",
        "#     label = fa_file_path.stem\n",
        "\n",
        "#     with fa_file_path.open(\"r\") as fa:\n",
        "#         for record in tqdm(SeqIO.parse(fa, 'fasta')):\n",
        "#             aa_seq_dict['sequence'].append(str(record.seq))\n",
        "#             aa_seq_dict[\"label\"].append(label)\n",
        "#             aa_seq_dict[\"stage\"].append(split[i])\n",
        "\n",
        "#     print()\n",
        "#     print(\"=\"*100)\n",
        "\n",
        "# fa_df = pd.DataFrame(aa_seq_dict)\n",
        "# timestamp = datetime.now().strftime(\"%y%m%d%H%M%S\")\n",
        "# fa_df.to_csv(f'/content/SaprotHub/upload_files/{timestamp}.csv', index=None)\n",
        "# files.download(f'/content/SaprotHub/upload_files/{timestamp}.csv')\n",
        "# print(fa_df[5:])"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "cellView": "form",
        "id": "xtadHW9vBI7o"
      },
      "outputs": [],
      "source": [
        "#@title **4.3: Dataset Split**\n",
        "\n",
        "#@markdown Randomly split your .csv dataset <a name=\"split_dataset\"></a>\n",
        "\n",
        "#@markdown Please click the run button to upload your .csv dataset\n",
        "\n",
        "csv_dataset_path = upload_file(UPLOAD_FILE_HOME)\n",
        "dataset_df = pd.read_csv(csv_dataset_path)\n",
        "\n",
        "split = ['train', 'valid', 'test']\n",
        "split_ratio = [0.8, 0.1, 0.1]\n",
        "\n",
        "if ('stage' not in dataset_df.columns) or (dataset_df[\"stage\"].nunique()<3):\n",
        "  dataset_df[\"stage\"] = np.random.choice(split, size=len(dataset_df), p=split_ratio).tolist()\n",
        "\n",
        "dataset_df.to_csv(csv_dataset_path, index=None)\n"
      ]
    }
  ],
  "metadata": {
    "colab": {
      "provenance": [],
      "gpuType": "T4"
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    },
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "43bcba7abdfe423abdef9ec4925216a2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DropdownModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DropdownModel",
            "_options_labels": [
              "PDB",
              "AF2"
            ],
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "DropdownView",
            "description": "",
            "description_tooltip": null,
            "disabled": false,
            "index": 0,
            "layout": "IPY_MODEL_f469cbc8326c4d74882de4a1d860854b",
            "style": "IPY_MODEL_682d12c78c9f4b79a104d3cf9edaf7ec"
          }
        },
        "f469cbc8326c4d74882de4a1d860854b": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "682d12c78c9f4b79a104d3cf9edaf7ec": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "14ba4e672efd48b2812f1adaef897c4f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "TextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "TextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "TextView",
            "continuous_update": true,
            "description": "",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_7bbeedb8def74786b81633f1d6428f97",
            "placeholder": "Enter the name of chain here",
            "style": "IPY_MODEL_a98ba678e25b4fb186a589d416c238b3",
            "value": "A"
          }
        },
        "7bbeedb8def74786b81633f1d6428f97": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "a98ba678e25b4fb186a589d416c238b3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "703318884d354d5f8fb8fec1de6eccb8": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "a23903fc8c2d4ed88e77c1f3a41943f7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "558c855445594b8485b9fba5c1af3a84": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "TextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "TextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "TextView",
            "continuous_update": true,
            "description": "",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_703318884d354d5f8fb8fec1de6eccb8",
            "placeholder": "Enter Amino Acid Sequence here",
            "style": "IPY_MODEL_a23903fc8c2d4ed88e77c1f3a41943f7",
            "value": "ARIRHVQGDITEFQGDAIVNAANNYLKLGAGVAGAILRKGGPSIQEECDRIGKIRVGEAAVTGAGNLPVRYVIHAAVLGDEPASLETVRKATKSALEKAVELGLKTVAFTALGAWVGGLPAEAVLRVMLEEIKKAPDTLEVTGVHGTEKSAEAARRALLEHHHH"
          }
        },
        "8952766795894d0a98b19f7cc1097701": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "2fc9c110abd24ed1991bcf1d6bfd0ba5": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "a81da4401d1243ae9d342b19416439a4": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "TextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "TextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "TextView",
            "continuous_update": true,
            "description": "",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_8952766795894d0a98b19f7cc1097701",
            "placeholder": "Enter Structure Sequence here",
            "style": "IPY_MODEL_2fc9c110abd24ed1991bcf1d6bfd0ba5",
            "value": "daeaedadqqlqdqfaeeeaqaaqqrdqdddvsvvqcvqqhdvsvvvsvvvggddqlawdwdqshngnhgiyiyqhqhhphagdlvsllssllnslvvcvvvptaeyeyeqhcvpprvddsvssvvssvvnvvvhdnshyyyyydndpvvrvvvvvvvvvvvvd"
          }
        }
      }
    },
    "accelerator": "GPU"
  },
  "nbformat": 4,
  "nbformat_minor": 0
}